{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# This code is needed for using the project modules in the notebook\n",
    "\n",
    "# This path should point to the current project folder\n",
    "project_path = 'D:/Personal_Projects/Public/uh-trec-covid/uh-trec-covid/'\n",
    "raw_data_folder = 'D:/Personal_Projects/Public/uh-trec-covid/uh-trec-covid/dataset_sample/'\n",
    "pdf_path = raw_data_folder + 'document_parses/pdf_json/'\n",
    "\n",
    "import os\n",
    "import sys\n",
    "sys.path.insert(0, os.path.abspath(project_path))\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 81,
   "metadata": {},
   "outputs": [],
   "source": [
    "import json\n",
    "import os\n",
    "import io\n",
    "import pandas as pd\n",
    "import pickle\n",
    "\n",
    "\n",
    "class CovidDatasetManager:\n",
    "    \n",
    "    def __init__(self, metadata_file_path, data_folder_path):\n",
    "        self.metadata_file_path = metadata_file_path\n",
    "        self.data_folder_path = data_folder_path\n",
    "        self.metadata_dict = {}\n",
    "        self.paper_dict = {}\n",
    "    \n",
    "    def load_metadata_from_csv(self):\n",
    "        df = pd.read_csv(raw_data_folder + 'metadata.csv', low_memory=False, dtype=str)\n",
    "\n",
    "        for index, cord_uid, sha, source_x, title, doi, pmcid, pubmed_id, license, abstract, publish_time, \\\n",
    "            authors, journal, mag_id, who_covidence_id, arxiv_id, pdf_json_files, pmc_json_files, url, s2_id \\\n",
    "                in df.itertuples():\n",
    "        \n",
    "            # Check there are no repeated keys\n",
    "            if cord_uid not in self.metadata_dict:\n",
    "                # Check there is a pdf or pmc file corresponding to that key\n",
    "                if not (pd.isna(pdf_json_files) and pd.isna(pmc_json_files)):\n",
    "                    self.metadata_dict[cord_uid] = {'title': title, \n",
    "                                                    'abstract': abstract, \n",
    "                                                    'pdf_file': pdf_json_files,\n",
    "                                                    'pmc_file': pmc_json_files}\n",
    "                    \n",
    "    def save_metadata_as_pickle(self):\n",
    "        with open(raw_data_folder + 'metadata.pickle', 'wb') as file_handle:\n",
    "            pickle.dump(self.metadata_dict, file_handle, pickle.HIGHEST_PROTOCOL)\n",
    "            \n",
    "    def load_metadata_from_pickle(self):\n",
    "        with open(raw_data_folder + 'metadata.pickle', 'rb') as file_handle:\n",
    "            self.metadata_dict = pickle.load(file_handle)\n",
    "    \n",
    "    def _load_doc_from_json_(self, doc_file_path):\n",
    "        doc_json_file = io.open(file=self.data_folder_path + doc_file_path, mode='r', encoding='utf-8')\n",
    "        doc_json = json.load(doc_json_file)\n",
    "        \n",
    "        paper_title = doc_json['metadata']['title']\n",
    "        \n",
    "        paper_body_text = []\n",
    "        \n",
    "        for t in doc_json['body_text']:\n",
    "            paper_body_text.append(t['text'])\n",
    "        \n",
    "        doc_json_file.close()\n",
    "        \n",
    "        return {'title': paper_title, 'text': paper_body_text}\n",
    "            \n",
    "    def get_document_from_jsom(self, cord_uid):\n",
    "        metadata = self.metadata_dict[cord_uid]\n",
    "        \n",
    "        doc = {}\n",
    "        if not pd.isna(metadata['pmc_file']):\n",
    "            doc = self._load_doc_from_json_(metadata['pmc_file'])\n",
    "        else:\n",
    "            doc = self._load_doc_from_json_('pdf_file')\n",
    "            \n",
    "        doc['cord_uid'] = cord_uid\n",
    "        \n",
    "        return doc\n",
    "    \n",
    "    def create_papers_dict(self):\n",
    "        for cord_uid, metadata in self.metadata_dict.items():\n",
    "            doc = {}\n",
    "            pmc_file = metadata['pmc_file']\n",
    "            pdf_file = metadata['pdf_file']\n",
    "            if not pd.isna(pmc_file):\n",
    "                if os.path.isfile(self.data_folder_path + pmc_file):\n",
    "                    doc = self._load_doc_from_json_(pmc_file)\n",
    "            else:\n",
    "                if os.path.isfile(self.data_folder_path + pdf_file):\n",
    "                    doc = self._load_doc_from_json_(pdf_file)\n",
    "                    \n",
    "            # Check if a document was found in the folders\n",
    "            if doc:\n",
    "                doc['cord_uid'] = cord_uid\n",
    "                self.paper_dict[cord_uid] = doc\n",
    "\n",
    "    def save_docs_dict_as_pickle(self):\n",
    "        with open(raw_data_folder + 'docs_dict.pickle', 'wb') as file_handle:\n",
    "            pickle.dump(self.paper_dict, file_handle, pickle.HIGHEST_PROTOCOL)\n",
    "            \n",
    "    def load_docs_dict_from_pickle(self):\n",
    "        with open(raw_data_folder + 'docs_dict.pickle', 'rb') as file_handle:\n",
    "            self.paper_dict = pickle.load(file_handle)\n",
    "\n",
    "    def get_document_from_dict(self, cord_uid):\n",
    "        doc = self.paper_dict[cord_uid]\n",
    "        return doc\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 82,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'title': 'The gene of an archaeal α-l-fucosidase is expressed by translational frameshifting', 'abstract': 'The standard rules of genetic translational decoding are altered in specific genes by different events that are globally termed recoding. In Archaea recoding has been unequivocally determined so far only for termination codon readthrough events. We study here the mechanism of expression of a gene encoding for a α-l-fucosidase from the archaeon Sulfolobus solfataricus (fucA1), which is split in two open reading frames separated by a −1 frameshifting. The expression in Escherichia coli of the wild-type split gene led to the production by frameshifting of full-length polypeptides with an efficiency of 5%. Mutations in the regulatory site where the shift takes place demonstrate that the expression in vivo occurs in a programmed way. Further, we identify a full-length product of fucA1 in S.solfataricus extracts, which translate this gene in vitro by following programmed −1 frameshifting. This is the first experimental demonstration that this kind of recoding is present in Archaea.', 'pdf_file': 'document_parses/pdf_json/25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json', 'pmc_file': 'document_parses/pmc_json/PMC1616953.xml.json'}\n"
     ]
    }
   ],
   "source": [
    "\n",
    "cov_dm = CovidDatasetManager(raw_data_folder + 'metadata.csv', raw_data_folder)\n",
    "\n",
    "cov_dm.load_metadata_from_csv()\n",
    "\n",
    "print(cov_dm.metadata_dict['69gftii4'])\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 83,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "False\n"
     ]
    }
   ],
   "source": [
    "\n",
    "cov_dm.save_metadata_as_pickle()\n",
    "\n",
    "cov_dm2 = CovidDatasetManager(raw_data_folder + 'metadata.csv', raw_data_folder)\n",
    "\n",
    "cov_dm2.load_metadata_from_pickle()\n",
    "\n",
    "print(pd.isna(cov_dm2.metadata_dict['69gftii4']['pmc_file']))\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 84,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'title': 'The gene of an archaeal α-l-fucosidase is expressed by translational frameshifting',\n 'text': ['Translation is optimally accurate and the correspondence between the nucleotide and the protein sequences are often considered as an immutable dogma. However, the genetic code is not quite universal: in certain organelles and in a small number of organisms the meaning of different codons has been reassigned and all the mRNAs are decoded accordingly. More surprisingly, the standard rules of genetic decoding are altered in specific genes by different events that are globally termed recoding (1). In all cases, translational recoding occurs in competition with normal decoding, with a proportion of the ribosomes not obeying to the ‘universal’ rules. Translational recoding has been identified in both prokaryotes and eukaryotes. It has crucial roles in the regulation of gene expression and includes stop codon readthrough, ribosome hopping and ±1 programmed frameshifting [for reviews see (2–4)].',\n  'In stop codon readthrough a stop codon is decoded by a tRNA carrying an unusual amino acid rather than a translational release factor. Specific stimulatory elements downstream to the stop codon regulate this process (5). Hopping, in which the ribosome stops translation in a particular site of the mRNA and re-start few nucleotides downstream, is a rare event and it has been studied in detail only in the bacteriophage T4 (6). In programmed frameshifting, ribosomes are induced to shift to an alternative, overlapping reading frame 1 nt 3′-wards (+1 frameshifting) or 5′-wards (−1 frameshifting) of the mRNA. This process is regulated and its frequency varies in different genes. The ±1 programmed frameshifting has been studied extensively in viruses, retrotransposons and insertion elements for which many cases are documented (7–9). Instead, this phenomenon is by far less common in cellular genes. A single case of programmed +1 frameshifting is known in prokaryotes (10,11) while in eukaryotes, including humans, several genes regulated by this recoding event have been described previously [(4) and references therein]. Compared to +1 frameshifting, −1 frameshifting is less widespread with only two examples in prokaryotes (12–14) and few others in eukaryotes (15–17).',\n  'The programmed −1 frameshifting is triggered by several elements in the mRNA. The slippery sequence, showing the X-XXY-YYZ motif, in which X can be any base, Y is usually A or U, and Z is any base but G, has the function of favouring the tRNA misalignment and it is the site where the shift takes place (3,18). Frameshifting could be further stimulated by other elements flanking the slippery sequence: a codon for a low-abundance tRNA, a stop codon, a Shine–Dalgarno sequence and an mRNA secondary structure. It has been reported that these elements, alone or in combination, enhance frameshifting by pausing the translating ribosome on the slippery sequence (4,18).',\n  'Noticeably, known cases of recoding in Archaea [recently reviewed in (19)] are limited to termination codon readthrough events that regulate the incorporation of the 21st and 22nd amino acids selenocysteine and pyrrolysine, respectively (20–23).',\n  'No archaeal genes regulated by translational programmed frameshifting and ribosome hopping have been identified experimentally so far; therefore, if compared with the others domains of life, the study of translational recoding in Archaea is still at its dawn.',\n  'We showed that the α-l-fucosidase gene from the crenarchaeon Sulfolobus solfataricus is putatively expressed by programmed −1 frameshifting (24). This gene, named fucA1, is organized in the open reading frames (ORFs) SSO11867 and SSO3060 of 81 and 426 amino acids, respectively, which are separated by a −1 frameshifting in a 40 base overlap (Figure 1A). We have reported previously that the region of overlap between the two ORFs had the characteristic features of the genes expressed by programmed −1 frameshifting including a slippery heptanucleotide A-AAA-AAT (codons are shown in the zero frame) flanked by a putative stem–loop and the rare codons CAC (Figure 1A) resembling the prokaryotic stem–loops/hairpins and the Shine–Dalgarno-like sites (24). We showed that the frameshifting, obtained by mutating by site-directed mutagenesis the fucA1 gene exactly in the position predicted from the slippery site, produced a full-length gene, named fucA1A, encoding for a polypeptide of 495 amino acids (Figure 1B). This mutant gene expressed in Escherichia coli a fully functional α-l-fucosidase, named Ssα-fuc, which was thermophilic, thermostable and had an unusual nonameric structure (24,25). More recently, we determined the reaction mechanism and the function of the residues of the active site of the mutant enzyme (26,27).',\n  'The functionality of the product of the mutant gene fucA1A does not provide direct experimental evidence that programmed −1 frameshifting occurs in vivo and in S.solfataricus. To address these issues, we report here the study of the expression of the wild-type split gene fucA1 and of its mutants in the slippery sequence. We demonstrate here that fucA1 is expressed by programmed −1 frameshifting in both E.coli and S.solfataricus. This is the first experimental demonstration that this kind of recoding is present in the Archaea domain of life. The relevance of programmed −1 frameshifting in Archaea is also discussed.',\n  'S.solfataricus cells were grown, and cell extracts obtained, as described previously (24,28).',\n  'The expression in the E.coli strain BL21(RB791) of the wild-type gene fucA1 and of the mutant genes fucA1A [previously named FrameFuc in (24)], fucA1B, fucA1sm and fucA1tm as fusions of glutathione S-transferase (GST) and the purification of the recombinant proteins were performed as reported previously (23). The nomenclature used in this paper for the different α-fucosidase genes is listed in Table 1.',\n  \"For the western blot studies, equal amounts of E.coli cultures expressing the wild-type and mutant fucA1 genes, normalized for the OD600, were resuspended in SDS–PAGE loading buffer containing 0.03 M Tris–HCl buffer, pH 6.8, 3% SDS (w/v), 6.7% glycerol (w/v), 6.7% 2-mercaptoethanol (w/v) and 0.002% blue bromophenol (w/v). The samples were incubated at 100°C for 5 min (unless otherwise indicated) and were directly loaded on to the gel. Western blot analyses were performed by blotting SDS–PAGEs of the concentrations indicated on Hybond-P polyvinylidenfluorid filters (Amersham Biosciences, Uppsala, Sweden); polyclonal anti-Ssα-fuc antibodies from rabbit (PRIMM, Milan, Italy) and anti-GST antibodies (Amersham Biosciences) were diluted 1:5000 and 1:40 000, respectively. The filters were washed and incubated with the ImmunoPure anti-rabbit IgG antibody conjugated with the horseradish peroxidase (HRP) from Pierce Biotechnology (Rockford, IL, USA). Filters were developed with the ECL-plus Western Blotting Detection system (Amersham Biosciences) by following the manufacturer's indications. The molecular weight markers used in the western blot analyses were the ECL streptavidin–HRP conjugate (Amersham Biosciences).\",\n  'The protein concentration of the samples was measured with the method of Bradford (29) and the amounts of sample loaded on to the SDS–PAGEs are those indicated. The quantification of the bands identified by western blot was performed by using the program Quantity One 4.4.0 in a ChemiDoc EQ System (Bio-Rad, Hercules, CA, USA) with the volume analysis tool. The frameshifting efficiency was calculated as the ratio of the intensity of the bands of the frameshifted product/frameshifted product + termination product.',\n  'The mutants in the slippery sequence of the wild-type gene fucA1 were prepared by site-directed mutagenesis from the vector pGEX-11867/3060, described previously (24,27). The synthetic oligonucleotides used (PRIMM) were the following: FucA1sm-rev, 5′-TTTAGGTGATATTGGTGTTCTGGTCTATCT-3′; FucA1sm-fwd, 5′-GAACACCAATATCACCTAAAGAATTCGGCCCA-3′; FucA1tm-rev, 5′-AGGTGATATTGGTGTTCTGGTCTATCTGGC-3′; FucA1tm-fwd, 5′-CCAGAACACCAATATCACCTCAAGAACTCGGCCCAGT-3′, where the mismatched nucleotides in the mutagenic primers are underlined. Direct sequencing identified the plasmids containing the desired mutations and the mutant genes, named fucA1sm and fucA1tm, were completely re-sequenced.',\n  'The mutant Ssα-fucB was prepared by site-directed mutagenesis from the vector pGEX-11867/3060, by using the same site-directed mutagenesis kit described above. The synthetic oligonucleotides used were FucA1sm-rev (described above) and the following mutagenic oligonucleotide: Fuc-B, 5′-GAACACCAATATCACCTAAAGAAGTTCGGCCCAGT-3′, where the mismatched nucleotides are underlined. Direct sequencing identified the plasmid containing the desired mutations and the mutant gene, named fucA1B, was completely re-sequenced. The enzymatic characterization of Ssα-fucB was performed as described previously (24,27).',\n  'Samples of the proteins expressed in E.coli from the wild-type gene fucA1 and the mutants fucA1A and fucA1sm, purified as described, were fractionated on an SDS–PAGE. Protein bands were excised from the gel, washed in 50 mM ammonium bicarbonate, pH 8.0, in 50% acetonitrile, reduced with 10 mM DTT at 56°C for 45 min and alkylated with 55 mM iodoacetamide for 30 min at room temperature in the dark. The gel pieces were washed several times with the buffer, resuspended in 50 mM ammonium bicarbonate and incubated with 100 ng of trypsin for 2 h at 4°C and overnight at 37°C. The supernatant containing peptides was analysed by MALDIMS on an Applied Biosystem Voyager DE-PRO mass spectrometer using α-cyano-4-hydroxycynnamic acid as matrix. Mass calibration was performed by using the standard mixture provided by manufacturer.',\n  'Liquid chromatography online tandem mass spectrometry (LCMSMS) analyses were performed on a Q-TOF hybrid mass spectrometer (Micromass, Waters, Milford, MA, USA) coupled with a CapLC capillary chromatographic system (Waters). Peptide ions were selected in the collision cell and fragmented. Analysis of the daughter ion spectra led to the reconstruction of peptide sequences.',\n  'Genomic DNA from S.solfataricus P2 strain was prepared as described previously (24). A DNA fragment of 1538 nt containing the complete fucA1 gene, was prepared by PCR, by using the following synthetic oligonucleotides (Genenco, Florence, Italy): FucA1-fwd, 5′-CTGGAGGCGCGCTAATACGACTCACTATAGGTCAGTTAAATGTCACAAAATTCT-3′; FucA1-rev, 5′-GACTTGGCGCGCCTATCTATAATCTAGGATAACCCTTAT-3′, in which the sequence corresponding to the genome of S.solfataricus is underlined. In the FucA1-fwd primer, the sequence of the promoter of the T7 RNA polymerase is in boldface and the sequence of the BssHII site is shown in italics. The PCR amplification was performed as described previously (24) and the amplification products were cloned in the BssHII site of the plasmid pBluescript II KS+. The fucA1 gene was completely re-sequenced to check if undesired mutations were introduced by PCR and the recombinant vector obtained, named pBlu-FucA1, was used for translation in vitro experiments.',\n  'The plasmids expressing the mutant genes fucA1A, fucA1sm and fucA1tm for experiments of translation in vitro were prepared by substituting the KpnI–NcoI wild-type fragment, containing the slippery site, with those isolated from the mutants. To check that the resulting plasmids had the correct sequence, the mutant genes were completely re-sequenced.',\n  'The mRNAs encoding wild-type fucA1 and its various mutants were obtained by in vitro run-off transcription. About 2 μg of each plasmid was linearized with BssHII and incubated with 50 U of T7 RNA polymerase for 1 h 30 min at 37°C. The transcription mixtures were then treated with 10 U of DNAseI (RNAse free) for 30 min. The transcribed RNAs were recovered by extracting the samples twice with phenol (pH 4.7) and once with phenol/chloroform 1:1 followed by precipitation with ethanol. The mRNAs were resuspended in DEPC-treated H2O at the approximate concentration of 0.6 pmol/μl.',\n  'In vitro translation assays were performed essentially as described by Condò et al. (28). The samples (25 μl final volume) contained 5 μl of S.solfataricus cell extract, 10 mM KCl, 20 mM Tris–HCl, pH 7.0, 20 mM Mg acetate, 3 mM ATP, 1 mM GTP, 5 μg of bulk S.solfataricus tRNA, 2 μl of [35S]methionine (1200 Ci/mmol at 10 mCi/ml) and ∼10 pmol of each mRNA. The mixtures were incubated at 70°C for 45 min. After this time, the synthesized proteins were resolved by electrophoresis 12.5% acrylamide–SDS gels and revealed by autoradiography of the dried gels on an Instant Imager apparatus.',\n  'Cells of S.solfataricus, strain P2, were grown in minimal salts culture media supplemented with yeast extract (0.1%), casamino acids (0.1%), plus glucose (0.1%) (YGM) or sucrose (0.1%) (YSM). The extraction of total RNA was performed as reported previously (24). Total RNA was extensively digested with DNAse (Ambion, Austin, TX, USA) and the absence of DNA was assessed by the lack of PCR amplification with each sets of primers described below. The RT–PCR experiments were performed as reported previously (24) by using the primers described previously that allowed the amplification of a region of 833 nt (positions 1–833, in which the A of the first ATG codon is numbered as one) overlapping the ORFs SSO11867 and SSO3060 (24).',\n  'For real-time PCR experiments total cDNA was obtained using the kit Quantitect RT (Qiagen GmbH, Hilden, Germany) from 500 ng of the same preparation of RNA described above. cDNA was then amplified in a Bio-Rad LightCycler using the DyNAmo HS Syber Green qPCR Kit (Finnzymes Oy, Espoo, Finland). Synthetic oligonucleotides (PRIMM) used for the amplification of a region at the 3′ of the ORF SSO3060 were as follows: 5′-Real: 5′-TAAATGGCGAAGCGATTTTC-3′; 3′-Real: 5′-ATATGCCTTTGTCGCGGATA-3′ for the gene fucA1. 5′-GAATGGGGGTGATACTGTCG-3′ and 5′-TTTACAGCCGGGACTACAGG-3′ for the 16S rRNA gene.',\n  'For each amplification of the fucA1 gene was used ∼2500-fold more cDNA than that used for the amplification of the 16S rRNA. Controls with no template cDNA were always included. PCR conditions were 15 min at 95°C for initial denaturation, followed by 40 cycles of 10 s at 95°C, 25 s at 56°C and 35 s at 72°C, and a final step of 10 min at 72°C. Product purity was controlled by melting point analysis of setpoints with 0.5°C temperature increase from 72 to 95°C. PCR products were analysed on 2% agarose gels and visualized by ethidium bromide staining.',\n  'The expression values of fucA1 gene were normalized to the values determined for the 16S rRNA gene. Absolute expression levels were calculated as fucA1/16S ratio in YSM and YGM cells, respectively. Relative mRNA expression levels (YSM/YGM ratio) were calculated as (fucA1/16S ratio in YGM cells)/(fucA1/16S ratio in YSM). Each cDNA was used in triplicate for each amplification.',\n  'The wild-type fucA1 gene, expressed in E.coli as a GST-fused protein, produced trace amounts of α-fucosidase activity (2.3 × 10−2 units mg−1 after removal of GST), suggesting that a programmed −1 frameshifting may occur in E.coli (24). The enzyme was then purified by using the GST purification system and analysed by SDS–PAGE revealing a major protein band (Figure 2A). The sample and control bands were excised from the gel, digested in situ with trypsin and directly analysed by matrix-assisted laser desorption/ionization mass spectrometry (MALDIMS). As shown in Figure 2B and C, both spectra revealed the occurrence of an identical mass signal at m/z 1244.6 corresponding to a peptide (Peptide A) encompassing the overlapping region of the two ORFs. This result was confirmed by liquid chromatography online tandem mass spectrometry (LCMSMS) analysis of the peptide mixtures. The fragmentation spectra of the two signals showed the common sequence Asn-Phe-Gly-Pro-Val-Thr-Asp-Phe-Gly-Tyr-Lys in which the amino acid from the ORF SSO11867 is underlined. These results unequivocally demonstrate that the protein containing the Peptide A is produced in E.coli by a frameshifting event that occurred exactly within the slippery heptamer predicted from the analysis of the DNA sequence in the region of overlap between the ORFs SSO11867 and SSO3060 (Figure 1A).',\n  'Remarkably, the MALDIMS analysis of the products of the wild-type fucA1 gene revealed the presence of a second Peptide B at m/z 1258.6 that is absent in the spectra of the Ssα-fuc control protein (Figure 2B and C). The sequence of Peptide B obtained by LCMSMS (Figure 2D) was Lys-Phe-Gly-Pro-Val-Thr-Asp-Phe-Gly-Tyr-Lys. This sequence differs only by one amino acid from Peptide A demonstrating that the interrupted gene fucA1 expresses in E.coli two full-length proteins originated by different −1 frameshifting events. Polypeptide A results from a shift in a site A and it is identical to Ssα-fuc prepared by site-directed mutagenesis (24), suggesting that the expression occurred with the simultaneous P- and A-site slippage. Instead, polypeptide B, named Ssα-fucB, is generated by frameshifting in a second site B as the result of a single P-site slippage (Figure 2E).',\n  'To measure the global efficiency of frameshifting in the two sites of the wild-type gene fucA1 we analysed the total extracts of E.coli by western blot using anti-GST antibodies (Figure 2F). Two bands with marked different electrophoretic mobility were observed: the polypeptide of 78.7 ± 1.1 kDa migrated like GST-Ssα-fuc fusion and was identified as originated from frameshifting in either site A or B of fucA1. The protein of 38.1 ± 1.2 kDa, which is not expressed by the mutant gene fucA1A (not shown), had an electrophoretic mobility compatible with GST fused to the polypeptide encoded by the ORF SSO11867 solely (27 and 9.6 kDa, respectively). This polypeptide originated from the translational termination of the ribosome at the OCH codon of the fucA1 N-terminal ORF (Figure 1A). The calculated ratio of frameshifting to the termination products was 5%.',\n  'To test if the full-length α-fucosidase produced by the −1 frameshifting event in site B (Ssα-fucB), resulting from the single P-site slippage has different properties from Ssα-fuc, whose sequence arises from the simultaneous P- and A- site slippage, we prepared the enzyme by site-directed mutagenesis. The slippery sequence in fucA1 A-AAA-AAT was mutated in A-AAG-AAG-T where mutations are underlined. The new mutant gene was named fucA1B. The first G, producing the conservative mutation AAA→AAG, was made to disrupt the slippery sequence and hence reducing the shifting efficiency. The second G was inserted to produce the frameshifting that results in the amino acid sequence of Peptide B. Therefore, the sequence of the two full-length mutant genes fucA1A and fucA1B differs only in the region of the slippery sequence: A-AAG-AAT-TTC-GGC and A-AAG-AAG-TTC-GGC, respectively (the mutations are underlined, the nucleotides in boldface were originally in the −1 frame) (Table 1).',\n  'The recombinant Ssα-fucB was purified up to ∼95% (Materials and Methods). Gel filtration chromatography demonstrated that in native conditions Ssα-fucB had the same nonameric structure of Ssα-fuc with an identical molecular weight of 508 kDa (data not shown). In addition, Ssα-fucB had the same high substrate selectivity of Ssα-fuc. The two enzymes have high affinity for 4-nitrophenyl-α-l-fucoside (4NP-Fuc) substrate at 65°C; the KM is identical within the experimental error (0.0287 ± 0.005 mM) while the kcat of Ssα-fucB (137 ± 5.7 s−1) is ∼48% of that of Ssα-fuc (287 ± 11 s−1). In addition, 4-nitrophenyl-α-l-arabinoside, -rhamnoside, 4-nitrophenyl-α-d-glucoside, -xyloside, -galactoside and -mannoside were not substrates of Ssα-fucB as shown previously for Ssα-fuc (24). This suggests that the different amino acid sequence did not significantly affect the active site. Both enzymes showed an identical profile of specific activity versus temperature with an optimal temperature higher than 95°C (data not shown). The heat stability and the pH dependence of Ssα-fuc and Ssα-fucB are reported in Figure 3. At 80°C, the optimal growth temperature of S.solfataricus, the half-life of Ssα-fucB is 45 min, almost 4-fold lower than that of Ssα-fuc (Figure 3A). The two enzymes showed different behaviour at pH <6.0 at which Ssα-fucB is only barely active and stable (Figure 3B); however, the two enzymes showed similar values of specific activity at pHs above 6.0, which is close to the intracellular pH of S.solfataricus (30).',\n  'The experimental data reported above indicate that the predicted slippery heptanucleotide in the region of overlap between the ORFs SSO11867 and SSO3060 of the wild-type gene fucA1 could regulate in cis the frameshifting events observed in E.coli. To test this hypothesis, we mutated the sequence A-AAA-AAT into A-AAG-AAT and C-AAG-AAC (mutations are underlined) obtaining the fucA1 single mutant (fucA1sm) and triple mutant (fucA1tm) genes, respectively. It is worth noting that the mutations disrupt the slippery sequence, but they maintain the −1 frameshift between the two ORFs (Table 1).',\n  'Surprisingly, the expression of fucA1sm in E.coli produced a full-length polypeptide that, after purification by affinity chromatography and removal of the GST protein, showed the same electrophoretic migration of Ssα-fuc and Ssα-fucB (Figure 4A). This protein was then characterized by mass spectrometry analyses following in situ tryptic digestion. Interestingly, the MALDI spectra revealed the presence of a single peptide encompassing the overlapping region between the two ORFs with a mass value of 1259.7 Da (peptide C; Figure 4B). The sequence of peptide C, determined from the fragmentation spectra obtained by LCMSMS analysis, was Glu-Phe-Gly-Pro-Val-Thr-Asp-Phe-Gly-Tyr-Lys (Figure 4C). Remarkably, apart from the Glu residue, this sequence is identical to that of peptide B produced from fucA1, indicating that in the mutant gene fucA1sm only one of the two frameshifting events observed in the wild-type fucA1 gene had occurred. The presence of a Glu instead of Lys was not unexpected. The mutation A-AAA-AAT→A-AAG-AAT in fucA1sm was conservative in the zero frame of the ORF SSO11867 (AAA→AAG, both encoding Lys), but it produced the mutation AAA→GAA (Lys→Glu) in the −1 frame of the ORF SSO3060.',\n  'It is worth noting that the frameshifting efficiency of the gene fucA1sm, calculated by western blot as described above, was 2-folds higher (10%) if compared to fucA1 (5%) (Figure 4D). This indicates that the mutation cancelled the frameshifting site A and, in the same time, enhanced the frameshifting efficiency of site B.',\n  'In contrast, the triple mutant fucA1tm produced in E.coli only the low molecular weight band resulting from translational termination (Figure 4D). No full-length protein could be detected in western blots probed with either anti-GST (Figure 4D) or anti-Ssα-fuc antibodies (Figure 4E). These data show that the disruption of the heptameric slippery sequence completely abolished the frameshifting in E.coli confirming that this sequence has a direct role in controlling the frameshifting in vivo.',\n  'To test whether fucA1 is expressed in S.solfataricus we analysed the extracts of cells grown on yeast extract, sucrose and casaminoacids medium (YSM). Accurate assays showed that S.solfataricus extracts contained 3.4 × 10−4 units mg−1 of α-fucosidase activity. These very low amounts hampered the purification of the enzyme. The extracts of S.solfataricus cells grown on YSM revealed by western blot a band of a molecular mass >97 kDa and no signals were detected with the pre-immune serum confirming the specificity of the anti-Ssα-fuc antibodies (Figure 5A). The different molecular mass may result from post-translational modifications occurred in the archaeon or from the incomplete denaturation of a protein complex. In particular, the latter event is not unusual among enzymes from hyperthermophilic archaea (31,32). To test which hypotheses were appropriate, cellular extracts of S.solfataricus were analysed by western blot extending the incubation at 100°C to 2 h. Interestingly, this treatment shifted the high-molecular mass band to 67.6 ± 1.2 kDa (Figure 5B and C), which still differs from that of the recombinant Ssα-fuc, 58.9 ± 1.2 kDa, leaving the question on the origin of this difference unsolved. To try to shed some light we immunoprecipitated extracts of S.solfataricus with anti-Ssα-fuc antibodies and we analysed the major protein band by MALDIMS. Unfortunately, we could not observe any peptide compatible with the fucosidase because the heavy IgG chain co-migrated with the band of the expected molecular weight (data not shown).',\n  'To test if the scarce amounts of the α-fucosidase in S.solfataricus extracts was the result of reduced expression at transcriptional level, we performed a northern blot analysis of total RNA extracted from cells grown either on YSM or YGM media. We could not observe any signal by using probes matching the 3′ of the ORF SSO3060 (data not shown). These results suggest that fucA1 produced a rare transcript; therefore, we analysed the level of mRNA by RT–PCR and by real-time PCR. A band corresponding to the region of overlap between the ORFs SSO11867 and SSO3060 was observed in the RNA extracted from cells grown on YSM and YGM media, demonstrating that under these conditions the two ORFs were co-transcribed (Figure 6A).',\n  'The experiments of real-time PCR shown in Figure 6B demonstrated that rRNA16S was amplified after ∼17 cycles while the amplification of fucA1 mRNA was observed after 38 cycles, despite the fact that we used ∼2500-fold more cDNA for the amplification of fucA1. This indicates that the gene fucA1 is transcribed at very low level. No significant differences in the fucA1 mRNA level were observed in cells grown in YSM or YGM media. This is further confirmed by the analysis by western blot of the extracts of the same cells of S.solfataricus used to prepare the total RNAs, which revealed equal amounts of α-fucosidase in the two extracts (Figure 6C). Therefore, the low α-fucosidase activity observed under the conditions tested is the result of the poor transcription of the fucA1 gene.',\n  'To determine whether, and with what efficiency, the −1 frameshifting could be performed by S.solfataricus ribosomes, mRNAs obtained by in vitro transcription of the cloned wild-type fucA1 gene and the mutants thereof were used to program an in vitro translation system prepared as described by Condò et al. (28). To this aim, a promoter of T7 polymerase was inserted ahead of the gene of interest to obtain RNA transcripts endowed with the short 5′-untranslated region of 9 nt observed for the natural fucA1 mRNA (24). Autoradiography of an SDS–PAGE of the translation products (Figure 7) revealed that the wild-type fucA1 transcript produced a tiny but clear band whose molecular weight corresponded to that of the full-length Ssα-fuc obtained by site-directed mutagenesis (24); the latter was translated quite efficiently in the cell-free system in spite of being encoded by a quasi-leaderless mRNA. Judging from the relative intensity of the signals given by the translation products of the wild-type fucA1 and the full-length mutant fucA1A, the efficiency of the −1 frameshifting in the homologous system was ∼10%. No signals corresponding to the polypeptides expected from the separated ORFs SSO11867 and SSO3060 (9.6 and 46.5 kDa, respectively) were observed. However, it should be noted that the product of SSO11867, even if synthesized, is too small to be detected in the gel system employed for this experiment. The larger product of ORF SSO3060, on the other hand, is certainly absent. These data unequivocally demonstrate that the ribosomes of S.solfataricus can decode the split fucA1 gene by programmed −1 frameshifting with considerable efficiency producing a full-length polypeptide from the two ORFs SSO11867 and SSO3060.',\n  'Remarkably, under the same conditions at which fucA1 drives the expression of the full-length protein, we could not observe any product from the fucA1sm and fucA1tm constructs. These data demonstrate that the integrity of the heptanucleotide is essential for the expression of the fucA1 gene in S.solfataricus, thus further confirming that the gene is decoded by programmed −1 frameshifting in this organism. In addition, the lack of expression of fucA1sm by translation in vitro in S.solfataricus contrasts with the efficient expression of this mutant in E.coli, indicating that the two organisms recognize different sequences regulating the translational frameshifting.',\n  'The identification of genes whose expression is regulated by recoding events is often serendipitous. In the framework of our studies on glycosidases from hyperthermophiles, we identified in the genome of the archaeon S.solfataricus a split gene encoding a putative α-fucosidase, which could be expressed through programmed −1 frameshifting (24). We tackled this issue by studying the expression of fucA1 in S.solfataricus and in E.coli to overcome the problems connected to the scarcity of expression of the α-fucosidase gene and to the manipulation of hyperthermophiles. As already reported by others, in fact, it is a common strategy to study recoding events from different organisms in E.coli (23,33).',\n  'The expression in E.coli of the wild-type split gene fucA1 led to the production by frameshifting of two full-length polypeptides with an efficiency of 5%. This is a value higher than that observed in other genes expressed by translational frameshifting in a heterologous system such as the proteins gpG and gpGT (0.3–3.5%) (33).',\n  'The gene fucA1 is expressed in S.solfataricus at very low level under the conditions tested. In particular, the transcriptional analysis of the gene revealed that it is expressed at very low level in both YSM and YGM media. Similarly, no differences in the two media could be found by western blot probed with anti-Ssα-fuc antibodies, indicating that the low expression of the enzyme in S.solfataricus is the result of scarce transcription rather than suppressed translation.',\n  'Western blots allowed us to identify a specific band ∼8.7 kDa heavier than that of the recombinant Ssα-fuc and experiments of translation in vitro showed that the wild-type gene expresses a full-length polypeptide exhibiting the same molecular mass of the recombinant protein. This demonstrates that the translational machinery of S.solfataricus is fully competent to perform programmed frameshifting. It seems likely that the observed discrepancy in molecular mass might arise from post-translational modifications that cannot be produced by the translation in vitro. Further experiments are required to characterize the α-l-fucosidase identified in S.solfataricus.',\n  'MALDIMS and LCMSMS analyses of the products in E.coli of the wild-type split gene fucA1 demonstrated that two independent frameshifting events occurred in vivo in the proposed slippery site. In particular, the sequences obtained by LCMSMS demonstrate that peptide A results from a simultaneous backward slippage of both the P- and the A-site tRNAs (Figure 8A). Instead, the sequence of peptide B is the result of the re-positioning on the −1 frame of only the P-site tRNA; in fact, the next incorporated amino acid is specified by the codon in the new frame (Figure 8B). Therefore, the expression by −1 frameshifting of the wild-type gene fucA1 in E.coli follows the models proposed for ribosomal frameshifting (34). We confirmed the significance of the slippery heptanucleotide in promoting the programmed frameshifting in vivo by mutating the putative regulatory sequence. The triple mutant fucA1tm gave no full-length products; presumably, the mutations in both the P- and in the A-site of the slippery sequence dramatically reduced the efficiency of the −1 frameshifting as observed previously in metazoans (35). This result confirms that the intact slippery sequence in the wild-type gene fucA1 is absolutely necessary for its expression in E.coli. In contrast, surprisingly, the single mutant fucA1sm showed an even increased frequency of frameshifting (10%) if compared to the wild-type and produced only one polypeptide by shifting specifically in site B. We explained this result observing that the mutation in the P-site of the slippery sequence A-AAA-AAT→A-AAG-AAT created a novel slippery sequence A-AAG identical to that controlling the expression by programmed −1 frameshifting of a transposase gene in E.coli (36). Therefore, apparently, the single mutation inactivated the simultaneous P- and A-site tRNA re-positioning and, in the same time, fostered the shifting efficiency of the tRNA in the P-site. It is worth noting that, instead, in S.solfataricus, only the simultaneous slippage is effective (Figure 8B) and even the single mutation in the slippery sequence of fucA1sm completely annulled the expression of the gene. This indicates that this sequence is essential in the archaeon and that programmed frameshifting in S.solfataricus and E.coli exploits different mechanisms. Furthermore, since the only difference between the enzymes produced by the frameshifting sites A and B, Ssα-fuc and Ssα-fucB, respectively, is the stability at 80°C, which is the S.solfataricus physiological temperature, the functionality of Ssα-fucB in the archaeon appears questionable.',\n  'The reason why fucA1 is regulated by programmed −1 frameshifting is not known. However, the physiological significance of programmed frameshifting has been assigned to a minority of the cellular genes while for most of them it is still uncertain [see (4) and reference therein; (16)]. This mechanism of recoding is exploited to set the ratio of two polypeptides such as the τ and γ subunits of the DNA polymerase III holoenzyme in E.coli (12). Alternatively, programmed frameshifting balances the expression of a protein, as the bacterial translational release factor 2 and the eukaryotic ornithine decarboxylase antizyme [see (4) and (18) and references therein]. In the case of fucA1, the polypeptide encoded by the smaller ORF SSO11867 could never be detected by western blots analyses. In addition, the modelling of Ssα-fuc on the high-resolution crystal structure of the α-l-fucosidase from Thermotoga maritima (25,37) showed that the fucA1 N-terminal polypeptide is not an independent domain. Moreover, we have shown recently that SSO11867 includes essential catalytic residues (27), excluding the possibility that a functional α-fucosidase can be obtained from the ORF SSO3060 alone. Therefore, several lines of evidence allow us to exclude that programmed −1 frameshifting is used to set the ratio of two polypeptides of the α-fucosidase from S.solfataricus. More probably, this translational mechanism might be required to control the expression level of fucA1.',\n  'Noticeably, this is the only fucosidase gene expressed by programmed −1 frameshifting. Among carbohydrate active enzymes, the only example of expression through this recoding mechanism is that reported for a gene encoding for a α(1,2)-fucosyltransferase from Helicobacter pylori that is interrupted by a −1 frameshifting (38). In this case, the expression by programmed frameshifting would lead to a functional enzyme synthesizing components of the surface lipopolysaccharides to evade the human immune defensive system. It is hard to parallel this model to fucA1. Nevertheless, the monosaccharide fucose is involved in a variety of biological functions (39). Therefore, the α-l-fucosidase might play a role in the metabolism of fucosylated oligosaccharides; experiments are currently in progress to knockout the wild-type fucA1 gene and to insert constitutive functional mutants of this gene in S.solfataricus.',\n  'FucA1 is the only archaeal α-l-fucosidase gene identified so far; hence, it is probably the result of a horizontal gene transfer event in S.solfataricus. However, since there are no α-fucosidases genes regulated by programmed frameshifting in Bacteria and Eukarya, it is tempting to speculate that this sophisticated mechanism of translational regulation pre-existed in S.solfataricus and it was applied to the fucosidase gene for physiological reasons. The identification of other genes interrupted by −1 frameshifts in S.solfataricus would open the possibility that they are regulated by programmed −1 frameshifting. Recently, the computational analysis of prokaryotic genomes revealed that seven Archaea harbour interrupted coding sequences, but S.solfataricus is not included in this study (40). A computational analysis on several archaeal genomes revealed that 34 interrupted genes are present in the genome of S.solfataricus, 11 of these genes are composed by two ORFs separated by −1 frameshifting and could be expressed by recoding (B. Cobucci-Ponzano, M. Rossi and M. Moracci, manuscript in preparation).',\n  'We have experimentally shown here, for the first time, that programmed −1 frameshifting is present in the Archaea domain. This finding is the missing piece in the puzzle of the phylogenetic distribution of programmed frameshifting demonstrating that this mechanism is universally conserved.'],\n 'cord_uid': '69gftii4'}"
      ]
     },
     "execution_count": 84,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "doc = cov_dm.get_document_from_jsom('69gftii4')\n",
    "doc\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 85,
   "metadata": {},
   "outputs": [],
   "source": [
    "cov_dm.create_papers_dict()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 86,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'title': 'Antisense-induced ribosomal frameshifting', 'text': ['The standard triplet readout of the genetic code can be reprogrammed by signals in the mRNA to induce ribosomal frameshifting [reviewed in (1–3)]. Generally, the resulting trans-frame protein product is functional and may in some cases be expressed in equal amounts to the product of standard translation. This elaboration of the genetic code (4,5) demonstrates versatility in decoding.', 'Requirements for eukaryotic ribosomal frameshifting include a shift-prone sequence at the decoding site and often a downstream secondary structure in mRNA. The majority of −1 programmed frameshift sites consist of a heptanucleotide sequence X XXY YYZ [where X can be A, G, C or U; Y can be A or U; and Z can be any nucleotide (6)]. In this configuration, the P- and A-site tRNAs can re-pair with at least 2 out of 3 nt when shifted 1 nt towards the 5′ end of the mRNA. Similarly, for +1 frameshift sites, the identity of the codons in the P- and A-sites of the ribosome is critical for efficient frameshifting. One factor affecting +1 frameshift efficiency is the initial stability of the P-site tRNA–mRNA interaction in the 0 frame (7). High-efficiency frameshifting occurs when the P-site tRNA does not form standard codon–anticodon interactions (8). In some studies, a correlation between +1 frameshift efficiency and the final stability of the P-site tRNA–mRNA interaction in the +1 frame has been shown previously (9,10). However, in other systems there appears to be little correlation (11). In addition, competition between decoding of the 0 frame and +1 frame codons in the A-site may affect frameshifting efficiency (7). Slow to decode 0 frame codons such as stop codons or those decoded by low abundance tRNAs favor frameshifting, as do +1 frame codons with high levels of corresponding cognate tRNAs (12–16).', 'High levels of frameshifting are often achieved by the stimulatory action of a cis-acting element located downstream of the shift site. A wide variety of structures, most commonly H-type pseudoknots (17), have been identified which stimulate −1 frameshifting in eukaryotes [for reviews see (18,19)]. Mutagenic and structural data for several of the frameshift stimulators have demonstrated that each pseudoknot has key structural features required for frameshift stimulation (20–28). However, unifying structural feature essential for frameshifting has not yet been identified. This observation combined with recent reports that simple antisense oligonucleotides can functionally mimic cis-acting 3′ stimulators of −1 frameshifting (29,30) demonstrates that many different structures can stimulate frameshifting. Although it should be noted that not all structures of equal thermodynamic stability can stimulate frameshifting (Discussion).', 'RNA pseudoknots have also been shown to stimulate programmed +1 frameshifting in many eukaryotic antizyme genes (31,32). Antizyme is a negative regulator of cellular polyamine levels through its ability to target ornithine decarboxylase (the rate-limiting enzyme in polyamine biosynthesis) for degradation (33–35), inhibits polyamine import (36,37) and stimulates export (38). Antizyme expression is induced by high-intracellular polyamine levels, and decreased with lowered levels. The polyamine sensor is a programmed +1 frameshift event that is required for antizyme synthesis. At low polyamine levels, termination at the end of open reading frame 1 (ORF1) is efficient, whereas at high levels of polyamines, a substantial proportion of ribosomes shift to the +1 reading frame and then resume standard decoding to synthesize the full-length and active antizyme protein. Frameshifting at the mammalian antizyme mRNA shift site, UCC UGA, is stimulated by two cis-acting signals (39,40). One of these, the 5′ element, encompasses ∼50 bases upstream of shift site and is important for the polyamine effect (39–41). The other cis-acting element is a pseudoknot located 3′ of the shift site. The mammalian antizyme pseudoknot and a structurally distinct counterpart in a subset of invertebrate antizyme mRNAs (31) are the only pseudoknots known to act as stimulators for +1 frameshifting in eukaryotes.', 'Although it is unknown if pseudoknots stimulate −1 frameshifting and +1 frameshifting by different mechanisms, one notable difference is found in positioning of the downstream structure relative to the shift site. Naturally occurring pseudoknots or stem–loop stimulators of −1 frameshifting typically begin ∼6–9 nt downstream of the A-site codon of the shift site (18), whereas +1 frameshift pseudoknots are located closer with only a 2–3 nt separation from the A-site codon (31). Mutagenic studies have revealed that altering the size of the spacer affects frameshifting and, in general, reduces efficiency (27,31,42–44).', 'Here we have tested the ability of antisense oligonucleotides, annealed downstream of the shift-prone site, UCC UGA, to induce shifting of the ribosome to the +1 reading frame. The directionality of frameshifting (either into the +1 or −1 reading frame) is shown to be dependent upon the position of the duplex region relative to the shift site, and the efficiency of frameshifting is responsive to polyamine levels and enhanced by the inclusion of stimulatory sequences found upstream of the human antizyme +1 programmed frameshift site.', 'Complementary oligonucleotides, to construct the sequences described in this paper, were synthesized at the University of Utah DNA/Peptide Core Facility such that when annealed they would have appropriate ends to ligate into the SalI/BamHI sites of the dual luciferase vector, p2luc (45). Dual luciferase constructs were prepared and their sequence was verified as described previously (46).', 'Insert sequences with shift site in boldface is given as follows:', 'P2lucAZ1wt: TCGACGGTCTCCCTCCACTGCTGTAGTAACCCGGGTCCGGGGCCTCGGTGGTGCTCCTGATGCCCCTCACCCACCCCTGAAGA TCCCAGGTGGGCGAGGGAATAGTCAGAGGGATCACAACGGATC;', 'P2lucAZ10sp: TCGACGGTCTCCCTCCACTGCTGTAGTAACCCGGGTCCGGGGCCTCGGTGGTGCTCCTGACCCTCACCCACCCCTGAA GATCCCAGGTGGGCGAGGGAATAGTCAGAGGGATCACAACGGATC;', 'P2lucAZ1hp: TCGACGGTCTCCCTCCACTGCTGTAGTAACCCGGGTCCGGGGCCTCGGTGGTGCTCCTGATGCCCCTCACCCACCCCTGAAGA TCCCAGGTGGGCGAGGGAATGGATC;', 'P2lucAZ1PKdel: TCGACGGTCTCCCTCCACTGCTGTAGTAACCCGGGTCCGGGGCCTCGGTGGTGCTCCTGATGCCCCTGGATC;', 'P2lucAZ1PKm1: TCGACGGTCTCCCTCCACTGCTGTAGTAACCCGGGTCCGGGGCCTCGGTGGTGCTCCTGATGCCCCTCACCCACCGGGATCACA AGGATC;', 'P2lucAZ1sl: TCGACGGTCTCCCTCCACTGCTGTAGTAACCCGGGTCCGGGGCCTCGGTGGTGCTCCTGATGCCCCTCACCCACCCGGATC;', 'P2lucAZ1FS: TCGACGTGCTCCTGATGCCCCTGGATC;', 'P2lucAZ1FSUGG: TCGACGTGCTCCTGGTGCCCCTGGATC.', '2′-O-Methyl antisense oligonucleotides were synthesized by Integrated DNA Technologies (Coralville, IA).', 'AZ1A: AGUUGAAGGAUCCAGGGGCA; AZ1B: GGAAGUUGAAGGAUCCAGGG; AZ1C: CAGGGAAGUUGAAGGAUCCA; PKm1: GAUCCCGGUGGGUGAGGG; PKm2: GAUCCCAGGUGGGCGAGGG; SL1: GGUGGGUGAGGG; and SL2: GGAUCCGGGUGGGUGAGGG.', 'The dual luciferase constructs (0.1 μg) described above were added directly to TNT coupled reticulocyte lysate reactions (Promega) with 35S-labeled methionine in a volume of 10 μl. Reactions were incubated at 30°C for 1 h. Radiolabeled proteins were separated by SDS–PAGE and the gels were fixed with 7.5% acetic acid and methanol for 20 min. After drying under vacuum, the gels were visualized using a Storm 860 PhosphorImager (Molecular Dynamics) and radioactive bands quantified using ImageQuant software. Percent frameshifting was calculated as the percentage of full-length (frameshift) product relative to the termination product and the full-length product combined. The value of each product was corrected for the number of methionine codons present in the coding sequence. The reported values are the average and standard deviations obtained from at least three independent measurements. Tables showing percent frameshifting and standard deviations can be found in Supplementary Data.', 'Plasmid p2lucAZ1PKdel was co-transfected into CV-1 cells with varying concentrations of AZ1B 2′-O-Methyl antisense oligonucleotides under the following conditions. CV-1 cells (1.5 × 104) in 50 μl of DMEM + 5% fetal bovine serum were added to wells (1/2 area 96-well tissue culture treated plates) containing 25 ng of DNA, varying amounts of AZ1B antisense oligonucleotides and 0.4 μl Lipofectamine 2000 (Invitrogen) in 25 μl of Optimem. Cells were incubated at 37°C (5% CO2) for 20 h. Media were then removed from the cells and the transfected cells were lysed in 12.5 μl lysis buffer and luciferase activity determined by measuring light emission following injection of 25 μl of luminescence reagent (Promega). Percent frameshifting was calculated by comparing firefly/Renilla luciferase ratios of experimental constructs with those of control constructs: (firefly experimental RLUs/Renilla experimental RLUs)/(firefly control RLUs/Renilla control RLUs) × 100.', 'Initially, three dual luciferase reporter vectors were generated containing the human antizyme 1 frameshift cassette (p2luc-AZ1wt) with the 5′ and 3′ stimulators of frameshifting, with the pseudoknot deleted (p2luc-AZ1PKdel), or replaced with a stem–loop (p2luc-AZ1hp) (Figure 1). Each constructs was then subjected to coupled transcription and translation reactions in the presence of increasing amounts of spermidine, and the 35S-labeled products separated by SDS–PAGE. Frameshifting efficiency was measured by comparing the amount of full-length frameshift product (+1 FS) to the product of termination (Term) at the shift site stop codon (Figure 2 and Supplementary Table 1). Maximum levels of frameshifting (AZ1wt 5.6%, AZ1PKdel 2.1% and AZ1hp 1.5%) were observed in the presence of 0.4 mM spermidine. Low-level frameshifting, 0.1% or less, was observed in the absence of exogenous spermidine.', '2′-O-Methyl antisense oligonucleotides were designed to anneal downstream of the UCC UGA shift site of RNA produced from p2luc-AZ1PKdel (AZ1PKdel) such that the 3′ ends were located 0 (AZ1A), 3 (AZ1B) or 6 (AZ1C) nt downstream of the UGA codon of the shift site (Figure 1B). Frameshift efficiency was measured following transcription/translation reactions of p2luc-AZ1PKdel in the presence of 2 μM of each antisense oligonucleotide and increasing amounts of spermidine (Figure 3A–C and Supplementary Table 2). Maximal levels of frameshifting were found to occur when 2–4 μM of antisense oligonucleotide was added to the transcription/translation reactions (Supplementary Table 3). In the presence of 0.4 mM exogenous spermidine, highly efficient shifting of ribosomes into the +1 reading frame (higher than that observed in the wild-type antizyme frameshift cassette) was observed with the addition of AZ1A (26.1%), AZ1B (51.8%) and AZ1C (31.8%) (Supplementary Table 2). The most efficient frameshifting is observed with the antisense oligonucleotide AZ1B which anneals such that spacing between the shift site and the beginning of the duplex region is the same as that observed between the shift site and the beginning of stem 1 of the natural antizyme 3′ pseudoknot structure (i.e. each has a 3 nt spacer). To verify that the antisense oligonucleotide was activating ribosomal frameshifting and not transcription slippage, RNA was transcribed from p2luc-AZ1PKdel in the absence of oligonucleotide and added to reticulocyte lysate translations in the presence of increasing amounts of 2′-O-Methyl AZ1B oligonucleotide. Frameshifting levels were increased to the same level as that observed in coupled transcription and translation reactions demonstrating that the oligonucleotide acts to induce frameshifting during translation (Supplementary Figure).', 'Surprisingly, the addition of AZ1A (0 spacer) also induced high-level frameshifting into the −1 reading frame in a manner which was modestly inhibited by the addition of spermidine (19% in the absence and 10% in the presence of 0.4 mM exogenous spermidine) (Figure 3A and Supplementary Table 2). No −1 frameshift product was observed when the wild-type antizyme cassette was examined in the absence of antisense oligonucleotide addition (Figure 2; AZwt). As the AZ1A antisense oligonucleotide was designed to anneal directly adjacent to the UGA codon of the shift site, it was of interest to determine whether the wild-type antizyme pseudoknot could induce −1 frameshifting when located in the equivalent position. To address this, a new construct p2luc-AZ1-0sp (Figure 1A) was made by deleting the 3 nt spacer between the pseudoknot and the shift site of p2luc-AZ1wt. In this case, the wild-type pseudoknot is directly 3′ adjacent to the shift site. The products of in vitro transcription and translation were separated by SDS–PAGE. No −1 frameshift product was observed and levels of the +1 frameshift product were significantly reduced to ∼3% (Figure 3D and Supplementary Table 2).', 'AZ1A, AZ1B and AZ1C were designed to complement RNA sequences encoded by the originating vector. To determine if duplexes formed between the antisense oligonucleotide and 3′ adjacent antizyme sequences would result in more efficient frameshift stimulation, reporter vectors were designed to contain a portion of the antizyme 3′ stimulator. Construct p2luc-AZ1PKm1 contains sequences from the 5′ half of the axis formed by the stacking of stem 1 and stem 2 of the pseudoknot (Figure 1A). Two complementary 2′-O-Methyl antisense oligonucleotides were designed. First, PKm1 has perfect complimentarity to the region starting 3 nt and ending 21 nt downstream of the UGA shift site codon. Second, PKm2 is the same except that a mispaired C and bulged A were located at positions 9 and 16, respectively. These two alterations were included to more closely mimic the natural pseudoknot which also contains a mispaired C and bulged A at equivalent positions along the extended stem formed by the stacking of pseudoknot stems 1 and 2 (Figure 1; compare p2luc-AZ1wt with the duplex formed between p2luc-PKm1 and antisense oligonucleotide PKm2). PKm1 and PKm2 induced 30 and 22% frameshifting, respectively, when added to coupled transcription and translation reactions of p2luc-AZ1PKm1 in the presence spermidine (Figure 4A and B, and Supplementary Table 4). Neither PKm1 nor Pkm2 induced frameshifting to the same levels seen with AZ1B, suggesting that the sequence content of the duplex region can affect the efficiency of frameshift stimulation and that native antizyme sequences are not required.', 'A second construct, p2luc-AZ1sl, was designed to contain only the 5′ half of stem 1 of the antizyme pseudoknot downstream from the UCC UGA shift site (Figure 1A). 2′-O-Methyl antisense oligonucleotides were designed to anneal between 3 and 15 nt (SL1) or 3 and 22 nt (SL2) downstream from the UGA codon of the shift site. Frameshift efficiency induced by these two antisense oligonucleotides, 8 and 22% respectively, was somewhat lower than that observed with PKm1 and PKm2 (Figure 4C and D and Supplementary Table 4). In these cases frameshift efficiency was higher for the longer antisense oligonucleotide (SL2), suggesting that frameshift efficiency most probably correlates with stability of the duplex. As was seen with AZ1A, AZ1B and AZ1C, frameshifting efficiency stimulated by antisense oligonucleotides PKm1, PKm2, SL1 and SL2 was also strongly correlated with the concentration of exogenously added spermidine (Supplementary Table 5).', 'The importance of the antizyme 5′ sequence context to antisense oligonucleotide induced ribosome frameshifting was examined by testing the frameshift site, UCC UGA, without the 5′ and 3′ stimulatory antizyme sequences. To this end, the 5′ antizyme stimulatory sequences were deleted from p2luc-AZ1PKdel to make p2luc-AZ1FS. Each of the antisense oligonucleotides AZ1A, AZ1B or AZ1C was added to coupled transcription and translation reactions with p2luc-AZ1FS in the presence or absence of spermidine. Frameshift efficiency was measured at 11, 8 and 4%, in the presence of spermidine and 3, 0.4 and 0.2% in its absence for AZ1A, AZ1B and AZ1C, respectively (Figure 5A and B).', 'To determine whether the stop codon of the shift site is essential for frameshifting, the UGA codon of p2luc-AZ1FS was altered to UGG such that the shift site was UCC UGG (p2luc-AZ1-UGG). Frameshift efficiency was significant, but reduced, compared to the shift site UCC UGA, and shows little stimulation by the addition of spermidine; AZ1A, AZ1B and AZ1C induced 3, 1 and 1.4% frameshifting in the presence of spermidine, and 1.9, 0.8 and 1.7% frameshifting in its absence, respectively (Figure 5C and D).', 'The ability of antisense oligonucleotides to induce frameshifting in cultured mammalian cells was examined by co-transfection of CV-1 cells with p2lucAZ1PKdel and increasing amounts of 2′-O-Methyl antisense oligonucleotides AZ1B as described in Materials and Methods. In the absence of antisense oligonucleotide frameshifting levels were determined to be 1.1%, whereas a graded increase in frameshift levels was observed upon the addition of AZ1B (Figure 6). Maximal frameshifting levels were 13% in the presence of 2 μM AZ1B in the transfection media.', 'Several models attempting to explain pseudoknot stimulation of programmed −1 frameshifting have been proposed [for reviews see (18,19)]. Most models invoke a pausing mechanism whereby the ribosome is paused over the shift site such that time is allowed for the tRNAs to reposition in the new reading frame. This explanation is clearly too simplistic as stem–loops and pseudoknots of similar thermodynamic stability that cause ribosome pausing are not necessarily effective frameshift stimulators (47–49). In addition, variations of the IBV pseudoknot have demonstrated a lack of correlation between the extent of pausing and the efficiency of frameshifting (47). A recent publication by Brierley and co-workers (50) presents structural data demonstrating that the IBV frameshift stimulating pseudoknot blocks the mRNA entrance tunnel and leads to a structural deformation of the P-site tRNA. The resulting movement of the tRNA displaces the anticodon loop towards the 3′ end of the mRNA. A model is presented in which this movement results in disruption of the codon–anticodon interactions, thus allowing for tRNA slippage relative to the mRNA. Similar tRNA movements were not observed with non-frameshift stimulating stem–loop structures. This model provides a feasible mechanistic explanation for the ability of some downstream structures to induce frameshifting.', 'The ability of antisense oligonucleotides to induce high-level −1 frameshifting (29,30) demonstrates that elaborate tertiary structures are not required, and that a duplex formed by complementary antisense oligonucleotides (with a variety of chemistries, including RNA, 2′-O-Methyl, morpholino) is sufficient to induce high-level frameshifting. Here we demonstrate for the first time that trans-acting antisense oligonucleotides may stimulate ribosome shifting to the +1 reading frame at surprisingly high levels, levels which are greater than those achieved by natural 3′ cis-acting mRNA pseudoknot structures in programmed +1 frameshifting.', 'Structural studies indicating that the mRNA begins to enter the ribosome 7–9 nt downstream from the A-site codon is of direct relevance to this study (50,51). Our results indicate that maximal frameshifting is induced when the antisense–mRNA duplex begins 3 nt downstream of the UGA of the shift site, in agreement with the distance found between the UGA of the shift site and the beginning of stem 1 of the pseudoknot stimulator found in antizyme genes. Given this distance, the implication is that the stimulatory secondary structure would be encountered by the ribosome when the UCC codon enters the A-site of the ribosome. Perhaps as suggested by the structural studies of the IBV-1 frameshift inducing pseudoknot, the codon–anticodon interactions between the UCC codon and Ser-tRNASer are disrupted during translocation to the P-site. Given the importance of the UGA codon during frameshifting at the UCC UGA shift site, subsequent events following translocation of the UCC codon to the P-site and UGA to the A-site must influence frameshifting efficiency. This latter event most probably involves competition between termination and +1 frame decoding when the UGA codon is in the A-site. Various discussions have been presented for the importance of A-site and P-site events during ribosomal frameshifting (7,52) and clearly, further investigations of this topic are warranted.', 'The observation presented here that the antisense oligonucleotide, AZ1A, which anneals directly adjacent to the UGA stop codon can induce ribosome frameshifts to either the +1 or −1 reading frame is surprising. In light of the above discussion of spacing for naturally occurring cis-acting frameshift stimulators, it is possible that frameshifting may occur at codons upstream of the known UCC UGA shift site. However, visual examination of upstream codons does not reveal an obvious −1 or +1 frameshift site.', 'The ability of spermidine to stimulate antisense oligonucleotide induced ribosome frameshifting to the +1 reading frame at the UCC UGA shift site in the absence of the natural 3′ stimulator demonstrates that this cis-acting element is not required for polyamine responsiveness. Similarly, spermidine stimulation was observed in the absence of the 5′ element but virtually eliminated by altering the UGA codon of the shift site to UGG. These observations are in agreement with previous studies examining the importance of cis-acting elements for polyamine induced frameshifting during expression of antizyme genes (39–41).', 'Finally, the ability to direct ribosomes to the +1 reading frame in living cells (Figure 6) suggests a potential therapeutic application for antisense oligonucleotides. Directed frameshifting to the +1 reading frame near a disease causing −1 frameshift mutation would cause some ribosomes to resume decoding in the wild-type ORF, thus restoring partial production of full-length protein from mutant alleles. The importance of the stop codon for efficient frameshifting suggests that the stop codon following the frameshift mutation presents a promising target for antisense induced phenotypic suppression, and that modulation of intracellular polyamine levels, although not essential, may increase the effectiveness of this approach. Further experiments are required to determine the therapeutic potential of this approach in vivo including the generality and efficiency of frameshift induction at non-programmed frameshift sites.', 'Supplementary Data are available at NAR Online.'], 'cord_uid': 'xgwbl8em'}\n{'title': 'The gene of an archaeal α-l-fucosidase is expressed by translational frameshifting', 'text': ['Translation is optimally accurate and the correspondence between the nucleotide and the protein sequences are often considered as an immutable dogma. However, the genetic code is not quite universal: in certain organelles and in a small number of organisms the meaning of different codons has been reassigned and all the mRNAs are decoded accordingly. More surprisingly, the standard rules of genetic decoding are altered in specific genes by different events that are globally termed recoding (1). In all cases, translational recoding occurs in competition with normal decoding, with a proportion of the ribosomes not obeying to the ‘universal’ rules. Translational recoding has been identified in both prokaryotes and eukaryotes. It has crucial roles in the regulation of gene expression and includes stop codon readthrough, ribosome hopping and ±1 programmed frameshifting [for reviews see (2–4)].', 'In stop codon readthrough a stop codon is decoded by a tRNA carrying an unusual amino acid rather than a translational release factor. Specific stimulatory elements downstream to the stop codon regulate this process (5). Hopping, in which the ribosome stops translation in a particular site of the mRNA and re-start few nucleotides downstream, is a rare event and it has been studied in detail only in the bacteriophage T4 (6). In programmed frameshifting, ribosomes are induced to shift to an alternative, overlapping reading frame 1 nt 3′-wards (+1 frameshifting) or 5′-wards (−1 frameshifting) of the mRNA. This process is regulated and its frequency varies in different genes. The ±1 programmed frameshifting has been studied extensively in viruses, retrotransposons and insertion elements for which many cases are documented (7–9). Instead, this phenomenon is by far less common in cellular genes. A single case of programmed +1 frameshifting is known in prokaryotes (10,11) while in eukaryotes, including humans, several genes regulated by this recoding event have been described previously [(4) and references therein]. Compared to +1 frameshifting, −1 frameshifting is less widespread with only two examples in prokaryotes (12–14) and few others in eukaryotes (15–17).', 'The programmed −1 frameshifting is triggered by several elements in the mRNA. The slippery sequence, showing the X-XXY-YYZ motif, in which X can be any base, Y is usually A or U, and Z is any base but G, has the function of favouring the tRNA misalignment and it is the site where the shift takes place (3,18). Frameshifting could be further stimulated by other elements flanking the slippery sequence: a codon for a low-abundance tRNA, a stop codon, a Shine–Dalgarno sequence and an mRNA secondary structure. It has been reported that these elements, alone or in combination, enhance frameshifting by pausing the translating ribosome on the slippery sequence (4,18).', 'Noticeably, known cases of recoding in Archaea [recently reviewed in (19)] are limited to termination codon readthrough events that regulate the incorporation of the 21st and 22nd amino acids selenocysteine and pyrrolysine, respectively (20–23).', 'No archaeal genes regulated by translational programmed frameshifting and ribosome hopping have been identified experimentally so far; therefore, if compared with the others domains of life, the study of translational recoding in Archaea is still at its dawn.', 'We showed that the α-l-fucosidase gene from the crenarchaeon Sulfolobus solfataricus is putatively expressed by programmed −1 frameshifting (24). This gene, named fucA1, is organized in the open reading frames (ORFs) SSO11867 and SSO3060 of 81 and 426 amino acids, respectively, which are separated by a −1 frameshifting in a 40 base overlap (Figure 1A). We have reported previously that the region of overlap between the two ORFs had the characteristic features of the genes expressed by programmed −1 frameshifting including a slippery heptanucleotide A-AAA-AAT (codons are shown in the zero frame) flanked by a putative stem–loop and the rare codons CAC (Figure 1A) resembling the prokaryotic stem–loops/hairpins and the Shine–Dalgarno-like sites (24). We showed that the frameshifting, obtained by mutating by site-directed mutagenesis the fucA1 gene exactly in the position predicted from the slippery site, produced a full-length gene, named fucA1A, encoding for a polypeptide of 495 amino acids (Figure 1B). This mutant gene expressed in Escherichia coli a fully functional α-l-fucosidase, named Ssα-fuc, which was thermophilic, thermostable and had an unusual nonameric structure (24,25). More recently, we determined the reaction mechanism and the function of the residues of the active site of the mutant enzyme (26,27).', 'The functionality of the product of the mutant gene fucA1A does not provide direct experimental evidence that programmed −1 frameshifting occurs in vivo and in S.solfataricus. To address these issues, we report here the study of the expression of the wild-type split gene fucA1 and of its mutants in the slippery sequence. We demonstrate here that fucA1 is expressed by programmed −1 frameshifting in both E.coli and S.solfataricus. This is the first experimental demonstration that this kind of recoding is present in the Archaea domain of life. The relevance of programmed −1 frameshifting in Archaea is also discussed.', 'S.solfataricus cells were grown, and cell extracts obtained, as described previously (24,28).', 'The expression in the E.coli strain BL21(RB791) of the wild-type gene fucA1 and of the mutant genes fucA1A [previously named FrameFuc in (24)], fucA1B, fucA1sm and fucA1tm as fusions of glutathione S-transferase (GST) and the purification of the recombinant proteins were performed as reported previously (23). The nomenclature used in this paper for the different α-fucosidase genes is listed in Table 1.', \"For the western blot studies, equal amounts of E.coli cultures expressing the wild-type and mutant fucA1 genes, normalized for the OD600, were resuspended in SDS–PAGE loading buffer containing 0.03 M Tris–HCl buffer, pH 6.8, 3% SDS (w/v), 6.7% glycerol (w/v), 6.7% 2-mercaptoethanol (w/v) and 0.002% blue bromophenol (w/v). The samples were incubated at 100°C for 5 min (unless otherwise indicated) and were directly loaded on to the gel. Western blot analyses were performed by blotting SDS–PAGEs of the concentrations indicated on Hybond-P polyvinylidenfluorid filters (Amersham Biosciences, Uppsala, Sweden); polyclonal anti-Ssα-fuc antibodies from rabbit (PRIMM, Milan, Italy) and anti-GST antibodies (Amersham Biosciences) were diluted 1:5000 and 1:40 000, respectively. The filters were washed and incubated with the ImmunoPure anti-rabbit IgG antibody conjugated with the horseradish peroxidase (HRP) from Pierce Biotechnology (Rockford, IL, USA). Filters were developed with the ECL-plus Western Blotting Detection system (Amersham Biosciences) by following the manufacturer's indications. The molecular weight markers used in the western blot analyses were the ECL streptavidin–HRP conjugate (Amersham Biosciences).\", 'The protein concentration of the samples was measured with the method of Bradford (29) and the amounts of sample loaded on to the SDS–PAGEs are those indicated. The quantification of the bands identified by western blot was performed by using the program Quantity One 4.4.0 in a ChemiDoc EQ System (Bio-Rad, Hercules, CA, USA) with the volume analysis tool. The frameshifting efficiency was calculated as the ratio of the intensity of the bands of the frameshifted product/frameshifted product + termination product.', 'The mutants in the slippery sequence of the wild-type gene fucA1 were prepared by site-directed mutagenesis from the vector pGEX-11867/3060, described previously (24,27). The synthetic oligonucleotides used (PRIMM) were the following: FucA1sm-rev, 5′-TTTAGGTGATATTGGTGTTCTGGTCTATCT-3′; FucA1sm-fwd, 5′-GAACACCAATATCACCTAAAGAATTCGGCCCA-3′; FucA1tm-rev, 5′-AGGTGATATTGGTGTTCTGGTCTATCTGGC-3′; FucA1tm-fwd, 5′-CCAGAACACCAATATCACCTCAAGAACTCGGCCCAGT-3′, where the mismatched nucleotides in the mutagenic primers are underlined. Direct sequencing identified the plasmids containing the desired mutations and the mutant genes, named fucA1sm and fucA1tm, were completely re-sequenced.', 'The mutant Ssα-fucB was prepared by site-directed mutagenesis from the vector pGEX-11867/3060, by using the same site-directed mutagenesis kit described above. The synthetic oligonucleotides used were FucA1sm-rev (described above) and the following mutagenic oligonucleotide: Fuc-B, 5′-GAACACCAATATCACCTAAAGAAGTTCGGCCCAGT-3′, where the mismatched nucleotides are underlined. Direct sequencing identified the plasmid containing the desired mutations and the mutant gene, named fucA1B, was completely re-sequenced. The enzymatic characterization of Ssα-fucB was performed as described previously (24,27).', 'Samples of the proteins expressed in E.coli from the wild-type gene fucA1 and the mutants fucA1A and fucA1sm, purified as described, were fractionated on an SDS–PAGE. Protein bands were excised from the gel, washed in 50 mM ammonium bicarbonate, pH 8.0, in 50% acetonitrile, reduced with 10 mM DTT at 56°C for 45 min and alkylated with 55 mM iodoacetamide for 30 min at room temperature in the dark. The gel pieces were washed several times with the buffer, resuspended in 50 mM ammonium bicarbonate and incubated with 100 ng of trypsin for 2 h at 4°C and overnight at 37°C. The supernatant containing peptides was analysed by MALDIMS on an Applied Biosystem Voyager DE-PRO mass spectrometer using α-cyano-4-hydroxycynnamic acid as matrix. Mass calibration was performed by using the standard mixture provided by manufacturer.', 'Liquid chromatography online tandem mass spectrometry (LCMSMS) analyses were performed on a Q-TOF hybrid mass spectrometer (Micromass, Waters, Milford, MA, USA) coupled with a CapLC capillary chromatographic system (Waters). Peptide ions were selected in the collision cell and fragmented. Analysis of the daughter ion spectra led to the reconstruction of peptide sequences.', 'Genomic DNA from S.solfataricus P2 strain was prepared as described previously (24). A DNA fragment of 1538 nt containing the complete fucA1 gene, was prepared by PCR, by using the following synthetic oligonucleotides (Genenco, Florence, Italy): FucA1-fwd, 5′-CTGGAGGCGCGCTAATACGACTCACTATAGGTCAGTTAAATGTCACAAAATTCT-3′; FucA1-rev, 5′-GACTTGGCGCGCCTATCTATAATCTAGGATAACCCTTAT-3′, in which the sequence corresponding to the genome of S.solfataricus is underlined. In the FucA1-fwd primer, the sequence of the promoter of the T7 RNA polymerase is in boldface and the sequence of the BssHII site is shown in italics. The PCR amplification was performed as described previously (24) and the amplification products were cloned in the BssHII site of the plasmid pBluescript II KS+. The fucA1 gene was completely re-sequenced to check if undesired mutations were introduced by PCR and the recombinant vector obtained, named pBlu-FucA1, was used for translation in vitro experiments.', 'The plasmids expressing the mutant genes fucA1A, fucA1sm and fucA1tm for experiments of translation in vitro were prepared by substituting the KpnI–NcoI wild-type fragment, containing the slippery site, with those isolated from the mutants. To check that the resulting plasmids had the correct sequence, the mutant genes were completely re-sequenced.', 'The mRNAs encoding wild-type fucA1 and its various mutants were obtained by in vitro run-off transcription. About 2 μg of each plasmid was linearized with BssHII and incubated with 50 U of T7 RNA polymerase for 1 h 30 min at 37°C. The transcription mixtures were then treated with 10 U of DNAseI (RNAse free) for 30 min. The transcribed RNAs were recovered by extracting the samples twice with phenol (pH 4.7) and once with phenol/chloroform 1:1 followed by precipitation with ethanol. The mRNAs were resuspended in DEPC-treated H2O at the approximate concentration of 0.6 pmol/μl.', 'In vitro translation assays were performed essentially as described by Condò et al. (28). The samples (25 μl final volume) contained 5 μl of S.solfataricus cell extract, 10 mM KCl, 20 mM Tris–HCl, pH 7.0, 20 mM Mg acetate, 3 mM ATP, 1 mM GTP, 5 μg of bulk S.solfataricus tRNA, 2 μl of [35S]methionine (1200 Ci/mmol at 10 mCi/ml) and ∼10 pmol of each mRNA. The mixtures were incubated at 70°C for 45 min. After this time, the synthesized proteins were resolved by electrophoresis 12.5% acrylamide–SDS gels and revealed by autoradiography of the dried gels on an Instant Imager apparatus.', 'Cells of S.solfataricus, strain P2, were grown in minimal salts culture media supplemented with yeast extract (0.1%), casamino acids (0.1%), plus glucose (0.1%) (YGM) or sucrose (0.1%) (YSM). The extraction of total RNA was performed as reported previously (24). Total RNA was extensively digested with DNAse (Ambion, Austin, TX, USA) and the absence of DNA was assessed by the lack of PCR amplification with each sets of primers described below. The RT–PCR experiments were performed as reported previously (24) by using the primers described previously that allowed the amplification of a region of 833 nt (positions 1–833, in which the A of the first ATG codon is numbered as one) overlapping the ORFs SSO11867 and SSO3060 (24).', 'For real-time PCR experiments total cDNA was obtained using the kit Quantitect RT (Qiagen GmbH, Hilden, Germany) from 500 ng of the same preparation of RNA described above. cDNA was then amplified in a Bio-Rad LightCycler using the DyNAmo HS Syber Green qPCR Kit (Finnzymes Oy, Espoo, Finland). Synthetic oligonucleotides (PRIMM) used for the amplification of a region at the 3′ of the ORF SSO3060 were as follows: 5′-Real: 5′-TAAATGGCGAAGCGATTTTC-3′; 3′-Real: 5′-ATATGCCTTTGTCGCGGATA-3′ for the gene fucA1. 5′-GAATGGGGGTGATACTGTCG-3′ and 5′-TTTACAGCCGGGACTACAGG-3′ for the 16S rRNA gene.', 'For each amplification of the fucA1 gene was used ∼2500-fold more cDNA than that used for the amplification of the 16S rRNA. Controls with no template cDNA were always included. PCR conditions were 15 min at 95°C for initial denaturation, followed by 40 cycles of 10 s at 95°C, 25 s at 56°C and 35 s at 72°C, and a final step of 10 min at 72°C. Product purity was controlled by melting point analysis of setpoints with 0.5°C temperature increase from 72 to 95°C. PCR products were analysed on 2% agarose gels and visualized by ethidium bromide staining.', 'The expression values of fucA1 gene were normalized to the values determined for the 16S rRNA gene. Absolute expression levels were calculated as fucA1/16S ratio in YSM and YGM cells, respectively. Relative mRNA expression levels (YSM/YGM ratio) were calculated as (fucA1/16S ratio in YGM cells)/(fucA1/16S ratio in YSM). Each cDNA was used in triplicate for each amplification.', 'The wild-type fucA1 gene, expressed in E.coli as a GST-fused protein, produced trace amounts of α-fucosidase activity (2.3 × 10−2 units mg−1 after removal of GST), suggesting that a programmed −1 frameshifting may occur in E.coli (24). The enzyme was then purified by using the GST purification system and analysed by SDS–PAGE revealing a major protein band (Figure 2A). The sample and control bands were excised from the gel, digested in situ with trypsin and directly analysed by matrix-assisted laser desorption/ionization mass spectrometry (MALDIMS). As shown in Figure 2B and C, both spectra revealed the occurrence of an identical mass signal at m/z 1244.6 corresponding to a peptide (Peptide A) encompassing the overlapping region of the two ORFs. This result was confirmed by liquid chromatography online tandem mass spectrometry (LCMSMS) analysis of the peptide mixtures. The fragmentation spectra of the two signals showed the common sequence Asn-Phe-Gly-Pro-Val-Thr-Asp-Phe-Gly-Tyr-Lys in which the amino acid from the ORF SSO11867 is underlined. These results unequivocally demonstrate that the protein containing the Peptide A is produced in E.coli by a frameshifting event that occurred exactly within the slippery heptamer predicted from the analysis of the DNA sequence in the region of overlap between the ORFs SSO11867 and SSO3060 (Figure 1A).', 'Remarkably, the MALDIMS analysis of the products of the wild-type fucA1 gene revealed the presence of a second Peptide B at m/z 1258.6 that is absent in the spectra of the Ssα-fuc control protein (Figure 2B and C). The sequence of Peptide B obtained by LCMSMS (Figure 2D) was Lys-Phe-Gly-Pro-Val-Thr-Asp-Phe-Gly-Tyr-Lys. This sequence differs only by one amino acid from Peptide A demonstrating that the interrupted gene fucA1 expresses in E.coli two full-length proteins originated by different −1 frameshifting events. Polypeptide A results from a shift in a site A and it is identical to Ssα-fuc prepared by site-directed mutagenesis (24), suggesting that the expression occurred with the simultaneous P- and A-site slippage. Instead, polypeptide B, named Ssα-fucB, is generated by frameshifting in a second site B as the result of a single P-site slippage (Figure 2E).', 'To measure the global efficiency of frameshifting in the two sites of the wild-type gene fucA1 we analysed the total extracts of E.coli by western blot using anti-GST antibodies (Figure 2F). Two bands with marked different electrophoretic mobility were observed: the polypeptide of 78.7 ± 1.1 kDa migrated like GST-Ssα-fuc fusion and was identified as originated from frameshifting in either site A or B of fucA1. The protein of 38.1 ± 1.2 kDa, which is not expressed by the mutant gene fucA1A (not shown), had an electrophoretic mobility compatible with GST fused to the polypeptide encoded by the ORF SSO11867 solely (27 and 9.6 kDa, respectively). This polypeptide originated from the translational termination of the ribosome at the OCH codon of the fucA1 N-terminal ORF (Figure 1A). The calculated ratio of frameshifting to the termination products was 5%.', 'To test if the full-length α-fucosidase produced by the −1 frameshifting event in site B (Ssα-fucB), resulting from the single P-site slippage has different properties from Ssα-fuc, whose sequence arises from the simultaneous P- and A- site slippage, we prepared the enzyme by site-directed mutagenesis. The slippery sequence in fucA1 A-AAA-AAT was mutated in A-AAG-AAG-T where mutations are underlined. The new mutant gene was named fucA1B. The first G, producing the conservative mutation AAA→AAG, was made to disrupt the slippery sequence and hence reducing the shifting efficiency. The second G was inserted to produce the frameshifting that results in the amino acid sequence of Peptide B. Therefore, the sequence of the two full-length mutant genes fucA1A and fucA1B differs only in the region of the slippery sequence: A-AAG-AAT-TTC-GGC and A-AAG-AAG-TTC-GGC, respectively (the mutations are underlined, the nucleotides in boldface were originally in the −1 frame) (Table 1).', 'The recombinant Ssα-fucB was purified up to ∼95% (Materials and Methods). Gel filtration chromatography demonstrated that in native conditions Ssα-fucB had the same nonameric structure of Ssα-fuc with an identical molecular weight of 508 kDa (data not shown). In addition, Ssα-fucB had the same high substrate selectivity of Ssα-fuc. The two enzymes have high affinity for 4-nitrophenyl-α-l-fucoside (4NP-Fuc) substrate at 65°C; the KM is identical within the experimental error (0.0287 ± 0.005 mM) while the kcat of Ssα-fucB (137 ± 5.7 s−1) is ∼48% of that of Ssα-fuc (287 ± 11 s−1). In addition, 4-nitrophenyl-α-l-arabinoside, -rhamnoside, 4-nitrophenyl-α-d-glucoside, -xyloside, -galactoside and -mannoside were not substrates of Ssα-fucB as shown previously for Ssα-fuc (24). This suggests that the different amino acid sequence did not significantly affect the active site. Both enzymes showed an identical profile of specific activity versus temperature with an optimal temperature higher than 95°C (data not shown). The heat stability and the pH dependence of Ssα-fuc and Ssα-fucB are reported in Figure 3. At 80°C, the optimal growth temperature of S.solfataricus, the half-life of Ssα-fucB is 45 min, almost 4-fold lower than that of Ssα-fuc (Figure 3A). The two enzymes showed different behaviour at pH <6.0 at which Ssα-fucB is only barely active and stable (Figure 3B); however, the two enzymes showed similar values of specific activity at pHs above 6.0, which is close to the intracellular pH of S.solfataricus (30).', 'The experimental data reported above indicate that the predicted slippery heptanucleotide in the region of overlap between the ORFs SSO11867 and SSO3060 of the wild-type gene fucA1 could regulate in cis the frameshifting events observed in E.coli. To test this hypothesis, we mutated the sequence A-AAA-AAT into A-AAG-AAT and C-AAG-AAC (mutations are underlined) obtaining the fucA1 single mutant (fucA1sm) and triple mutant (fucA1tm) genes, respectively. It is worth noting that the mutations disrupt the slippery sequence, but they maintain the −1 frameshift between the two ORFs (Table 1).', 'Surprisingly, the expression of fucA1sm in E.coli produced a full-length polypeptide that, after purification by affinity chromatography and removal of the GST protein, showed the same electrophoretic migration of Ssα-fuc and Ssα-fucB (Figure 4A). This protein was then characterized by mass spectrometry analyses following in situ tryptic digestion. Interestingly, the MALDI spectra revealed the presence of a single peptide encompassing the overlapping region between the two ORFs with a mass value of 1259.7 Da (peptide C; Figure 4B). The sequence of peptide C, determined from the fragmentation spectra obtained by LCMSMS analysis, was Glu-Phe-Gly-Pro-Val-Thr-Asp-Phe-Gly-Tyr-Lys (Figure 4C). Remarkably, apart from the Glu residue, this sequence is identical to that of peptide B produced from fucA1, indicating that in the mutant gene fucA1sm only one of the two frameshifting events observed in the wild-type fucA1 gene had occurred. The presence of a Glu instead of Lys was not unexpected. The mutation A-AAA-AAT→A-AAG-AAT in fucA1sm was conservative in the zero frame of the ORF SSO11867 (AAA→AAG, both encoding Lys), but it produced the mutation AAA→GAA (Lys→Glu) in the −1 frame of the ORF SSO3060.', 'It is worth noting that the frameshifting efficiency of the gene fucA1sm, calculated by western blot as described above, was 2-folds higher (10%) if compared to fucA1 (5%) (Figure 4D). This indicates that the mutation cancelled the frameshifting site A and, in the same time, enhanced the frameshifting efficiency of site B.', 'In contrast, the triple mutant fucA1tm produced in E.coli only the low molecular weight band resulting from translational termination (Figure 4D). No full-length protein could be detected in western blots probed with either anti-GST (Figure 4D) or anti-Ssα-fuc antibodies (Figure 4E). These data show that the disruption of the heptameric slippery sequence completely abolished the frameshifting in E.coli confirming that this sequence has a direct role in controlling the frameshifting in vivo.', 'To test whether fucA1 is expressed in S.solfataricus we analysed the extracts of cells grown on yeast extract, sucrose and casaminoacids medium (YSM). Accurate assays showed that S.solfataricus extracts contained 3.4 × 10−4 units mg−1 of α-fucosidase activity. These very low amounts hampered the purification of the enzyme. The extracts of S.solfataricus cells grown on YSM revealed by western blot a band of a molecular mass >97 kDa and no signals were detected with the pre-immune serum confirming the specificity of the anti-Ssα-fuc antibodies (Figure 5A). The different molecular mass may result from post-translational modifications occurred in the archaeon or from the incomplete denaturation of a protein complex. In particular, the latter event is not unusual among enzymes from hyperthermophilic archaea (31,32). To test which hypotheses were appropriate, cellular extracts of S.solfataricus were analysed by western blot extending the incubation at 100°C to 2 h. Interestingly, this treatment shifted the high-molecular mass band to 67.6 ± 1.2 kDa (Figure 5B and C), which still differs from that of the recombinant Ssα-fuc, 58.9 ± 1.2 kDa, leaving the question on the origin of this difference unsolved. To try to shed some light we immunoprecipitated extracts of S.solfataricus with anti-Ssα-fuc antibodies and we analysed the major protein band by MALDIMS. Unfortunately, we could not observe any peptide compatible with the fucosidase because the heavy IgG chain co-migrated with the band of the expected molecular weight (data not shown).', 'To test if the scarce amounts of the α-fucosidase in S.solfataricus extracts was the result of reduced expression at transcriptional level, we performed a northern blot analysis of total RNA extracted from cells grown either on YSM or YGM media. We could not observe any signal by using probes matching the 3′ of the ORF SSO3060 (data not shown). These results suggest that fucA1 produced a rare transcript; therefore, we analysed the level of mRNA by RT–PCR and by real-time PCR. A band corresponding to the region of overlap between the ORFs SSO11867 and SSO3060 was observed in the RNA extracted from cells grown on YSM and YGM media, demonstrating that under these conditions the two ORFs were co-transcribed (Figure 6A).', 'The experiments of real-time PCR shown in Figure 6B demonstrated that rRNA16S was amplified after ∼17 cycles while the amplification of fucA1 mRNA was observed after 38 cycles, despite the fact that we used ∼2500-fold more cDNA for the amplification of fucA1. This indicates that the gene fucA1 is transcribed at very low level. No significant differences in the fucA1 mRNA level were observed in cells grown in YSM or YGM media. This is further confirmed by the analysis by western blot of the extracts of the same cells of S.solfataricus used to prepare the total RNAs, which revealed equal amounts of α-fucosidase in the two extracts (Figure 6C). Therefore, the low α-fucosidase activity observed under the conditions tested is the result of the poor transcription of the fucA1 gene.', 'To determine whether, and with what efficiency, the −1 frameshifting could be performed by S.solfataricus ribosomes, mRNAs obtained by in vitro transcription of the cloned wild-type fucA1 gene and the mutants thereof were used to program an in vitro translation system prepared as described by Condò et al. (28). To this aim, a promoter of T7 polymerase was inserted ahead of the gene of interest to obtain RNA transcripts endowed with the short 5′-untranslated region of 9 nt observed for the natural fucA1 mRNA (24). Autoradiography of an SDS–PAGE of the translation products (Figure 7) revealed that the wild-type fucA1 transcript produced a tiny but clear band whose molecular weight corresponded to that of the full-length Ssα-fuc obtained by site-directed mutagenesis (24); the latter was translated quite efficiently in the cell-free system in spite of being encoded by a quasi-leaderless mRNA. Judging from the relative intensity of the signals given by the translation products of the wild-type fucA1 and the full-length mutant fucA1A, the efficiency of the −1 frameshifting in the homologous system was ∼10%. No signals corresponding to the polypeptides expected from the separated ORFs SSO11867 and SSO3060 (9.6 and 46.5 kDa, respectively) were observed. However, it should be noted that the product of SSO11867, even if synthesized, is too small to be detected in the gel system employed for this experiment. The larger product of ORF SSO3060, on the other hand, is certainly absent. These data unequivocally demonstrate that the ribosomes of S.solfataricus can decode the split fucA1 gene by programmed −1 frameshifting with considerable efficiency producing a full-length polypeptide from the two ORFs SSO11867 and SSO3060.', 'Remarkably, under the same conditions at which fucA1 drives the expression of the full-length protein, we could not observe any product from the fucA1sm and fucA1tm constructs. These data demonstrate that the integrity of the heptanucleotide is essential for the expression of the fucA1 gene in S.solfataricus, thus further confirming that the gene is decoded by programmed −1 frameshifting in this organism. In addition, the lack of expression of fucA1sm by translation in vitro in S.solfataricus contrasts with the efficient expression of this mutant in E.coli, indicating that the two organisms recognize different sequences regulating the translational frameshifting.', 'The identification of genes whose expression is regulated by recoding events is often serendipitous. In the framework of our studies on glycosidases from hyperthermophiles, we identified in the genome of the archaeon S.solfataricus a split gene encoding a putative α-fucosidase, which could be expressed through programmed −1 frameshifting (24). We tackled this issue by studying the expression of fucA1 in S.solfataricus and in E.coli to overcome the problems connected to the scarcity of expression of the α-fucosidase gene and to the manipulation of hyperthermophiles. As already reported by others, in fact, it is a common strategy to study recoding events from different organisms in E.coli (23,33).', 'The expression in E.coli of the wild-type split gene fucA1 led to the production by frameshifting of two full-length polypeptides with an efficiency of 5%. This is a value higher than that observed in other genes expressed by translational frameshifting in a heterologous system such as the proteins gpG and gpGT (0.3–3.5%) (33).', 'The gene fucA1 is expressed in S.solfataricus at very low level under the conditions tested. In particular, the transcriptional analysis of the gene revealed that it is expressed at very low level in both YSM and YGM media. Similarly, no differences in the two media could be found by western blot probed with anti-Ssα-fuc antibodies, indicating that the low expression of the enzyme in S.solfataricus is the result of scarce transcription rather than suppressed translation.', 'Western blots allowed us to identify a specific band ∼8.7 kDa heavier than that of the recombinant Ssα-fuc and experiments of translation in vitro showed that the wild-type gene expresses a full-length polypeptide exhibiting the same molecular mass of the recombinant protein. This demonstrates that the translational machinery of S.solfataricus is fully competent to perform programmed frameshifting. It seems likely that the observed discrepancy in molecular mass might arise from post-translational modifications that cannot be produced by the translation in vitro. Further experiments are required to characterize the α-l-fucosidase identified in S.solfataricus.', 'MALDIMS and LCMSMS analyses of the products in E.coli of the wild-type split gene fucA1 demonstrated that two independent frameshifting events occurred in vivo in the proposed slippery site. In particular, the sequences obtained by LCMSMS demonstrate that peptide A results from a simultaneous backward slippage of both the P- and the A-site tRNAs (Figure 8A). Instead, the sequence of peptide B is the result of the re-positioning on the −1 frame of only the P-site tRNA; in fact, the next incorporated amino acid is specified by the codon in the new frame (Figure 8B). Therefore, the expression by −1 frameshifting of the wild-type gene fucA1 in E.coli follows the models proposed for ribosomal frameshifting (34). We confirmed the significance of the slippery heptanucleotide in promoting the programmed frameshifting in vivo by mutating the putative regulatory sequence. The triple mutant fucA1tm gave no full-length products; presumably, the mutations in both the P- and in the A-site of the slippery sequence dramatically reduced the efficiency of the −1 frameshifting as observed previously in metazoans (35). This result confirms that the intact slippery sequence in the wild-type gene fucA1 is absolutely necessary for its expression in E.coli. In contrast, surprisingly, the single mutant fucA1sm showed an even increased frequency of frameshifting (10%) if compared to the wild-type and produced only one polypeptide by shifting specifically in site B. We explained this result observing that the mutation in the P-site of the slippery sequence A-AAA-AAT→A-AAG-AAT created a novel slippery sequence A-AAG identical to that controlling the expression by programmed −1 frameshifting of a transposase gene in E.coli (36). Therefore, apparently, the single mutation inactivated the simultaneous P- and A-site tRNA re-positioning and, in the same time, fostered the shifting efficiency of the tRNA in the P-site. It is worth noting that, instead, in S.solfataricus, only the simultaneous slippage is effective (Figure 8B) and even the single mutation in the slippery sequence of fucA1sm completely annulled the expression of the gene. This indicates that this sequence is essential in the archaeon and that programmed frameshifting in S.solfataricus and E.coli exploits different mechanisms. Furthermore, since the only difference between the enzymes produced by the frameshifting sites A and B, Ssα-fuc and Ssα-fucB, respectively, is the stability at 80°C, which is the S.solfataricus physiological temperature, the functionality of Ssα-fucB in the archaeon appears questionable.', 'The reason why fucA1 is regulated by programmed −1 frameshifting is not known. However, the physiological significance of programmed frameshifting has been assigned to a minority of the cellular genes while for most of them it is still uncertain [see (4) and reference therein; (16)]. This mechanism of recoding is exploited to set the ratio of two polypeptides such as the τ and γ subunits of the DNA polymerase III holoenzyme in E.coli (12). Alternatively, programmed frameshifting balances the expression of a protein, as the bacterial translational release factor 2 and the eukaryotic ornithine decarboxylase antizyme [see (4) and (18) and references therein]. In the case of fucA1, the polypeptide encoded by the smaller ORF SSO11867 could never be detected by western blots analyses. In addition, the modelling of Ssα-fuc on the high-resolution crystal structure of the α-l-fucosidase from Thermotoga maritima (25,37) showed that the fucA1 N-terminal polypeptide is not an independent domain. Moreover, we have shown recently that SSO11867 includes essential catalytic residues (27), excluding the possibility that a functional α-fucosidase can be obtained from the ORF SSO3060 alone. Therefore, several lines of evidence allow us to exclude that programmed −1 frameshifting is used to set the ratio of two polypeptides of the α-fucosidase from S.solfataricus. More probably, this translational mechanism might be required to control the expression level of fucA1.', 'Noticeably, this is the only fucosidase gene expressed by programmed −1 frameshifting. Among carbohydrate active enzymes, the only example of expression through this recoding mechanism is that reported for a gene encoding for a α(1,2)-fucosyltransferase from Helicobacter pylori that is interrupted by a −1 frameshifting (38). In this case, the expression by programmed frameshifting would lead to a functional enzyme synthesizing components of the surface lipopolysaccharides to evade the human immune defensive system. It is hard to parallel this model to fucA1. Nevertheless, the monosaccharide fucose is involved in a variety of biological functions (39). Therefore, the α-l-fucosidase might play a role in the metabolism of fucosylated oligosaccharides; experiments are currently in progress to knockout the wild-type fucA1 gene and to insert constitutive functional mutants of this gene in S.solfataricus.', 'FucA1 is the only archaeal α-l-fucosidase gene identified so far; hence, it is probably the result of a horizontal gene transfer event in S.solfataricus. However, since there are no α-fucosidases genes regulated by programmed frameshifting in Bacteria and Eukarya, it is tempting to speculate that this sophisticated mechanism of translational regulation pre-existed in S.solfataricus and it was applied to the fucosidase gene for physiological reasons. The identification of other genes interrupted by −1 frameshifts in S.solfataricus would open the possibility that they are regulated by programmed −1 frameshifting. Recently, the computational analysis of prokaryotic genomes revealed that seven Archaea harbour interrupted coding sequences, but S.solfataricus is not included in this study (40). A computational analysis on several archaeal genomes revealed that 34 interrupted genes are present in the genome of S.solfataricus, 11 of these genes are composed by two ORFs separated by −1 frameshifting and could be expressed by recoding (B. Cobucci-Ponzano, M. Rossi and M. Moracci, manuscript in preparation).', 'We have experimentally shown here, for the first time, that programmed −1 frameshifting is present in the Archaea domain. This finding is the missing piece in the puzzle of the phylogenetic distribution of programmed frameshifting demonstrating that this mechanism is universally conserved.'], 'cord_uid': '69gftii4'}\n{'title': 'Role of RNA helicases in HIV-1 replication', 'text': ['Helicases are enzymes that separate in an energy-dependent manner stretches of duplexed DNA and/or RNA into single-stranded components. Currently, based on characteristic motifs and the sequence comparisons, three superfamilies (SF1 through 3) and two smaller families (F4, F5) of helicases have been identified (1). Superfamily 1 and 2 contain helicases which share seven or more recognized signature amino acid motifs while SF3 and F4 and F5 helicases are characterized generally by three conserved motifs (2); the F4 and F5 proteins are largely bacterial and bacteriophage proteins. Currently, it should be cautioned that many ‘helicases’ are not bona fide helicases, but may only function as RNA translocases, perhaps to fulfill functions in the remodeling of ribonucleoprotein complexes (RNP).', 'DEAD-box and the related DEAH, DExH and DExD (3) helicases are the most numerous members of SF2 and are ubiquitously present in eukaryotic genomes. These helicases share eight conserved motifs and are commonly refered to as the DExH/D family of helicases. Humans, Arabidopsis and Saccharomyces have ∼38, 55 and 25 such entities, respectively (4). Differing from DNA helicases and DExH proteins, DEAD helicases are poor in unwinding long nucleic acid duplexes and are best suited for separating short RNA hybrids. DEAD-box proteins bind with high-affinity RNA–protein complexes while exhibiting little RNA sequence preference. This suggests that the specificity determinants for DEAD helicases may be through the recognition of protein factors. In this regard, a better understanding of the roles for DEAD proteins depends on the clear characterization of their respective interacting proteins.', 'Although the precise substrate for most helicases awaits definition, DEAD helicases are generally thought to participate pleiotropically in many aspects of RNA metabolism including transcription, mRNA splicing, mRNA export, translation, RNA stability and mitochondrial gene expression (5–8). Some examples of helicases and their attributed functions include the following. UAP56, Brr2, Prp16, Prp22 and Prp43 play roles in RNA-splicing (4,9), while Dbp5 (10,11) and DDX3 (12) chaperone RNAs from the nucleus into the cytoplasm. eIF4a and Ded1 serve for translation of mRNAs while Rh1B, Ski2, Dob1, Dhh1 helicases contribute to mRNA stability (4). Other DEAD helicases act in ribosome biogenesis through regulation of small nucleolar RNAs and ribosomal RNAs (rRNAs) interactions (13,14). Finally, Neurospora and Trypanosoma DEAD proteins contribute to mitochondrial gene expression (15,16); a Cryptococcus DEAD helicase is required for cryptococcosis pathogenesis (17), and the dipteran Chironomus tentans uses a hrp84 DEAD helicase to regulate mRNA transport from the nucleus into the cytoplasm onto polyribosomes (18).', 'Given that helicases significantly contribute to normal cellular metabolism, are they similarly essential to viruses? The operational answer appears to be a qualified ‘yes’. Indeed, when DEAD/DEAH-box helicase motif (InterPro IPR001410) was used to search the EMBL-EBI database, 1561 matches to individual viral sequence entries were found (), suggesting that many viruses have evolved to encode directly helicase or helicase-like proteins. The strongest biological evidence which supports the importance of a helicase in the virus life cycle comes from those viruses with an RNA genome. Hence, all positive-strand RNA viruses encode one or more helicase/helicase-like open reading frame (ORF) which, aside from the RNA-dependant RNA polymerase, is the most highly conserved viral sequence. Although less ubiquitous, helicases are also found in other types of viruses (see some examples listed in Table 1). Direct mutagenesis studies have confirmed that a helicase function is biologically required for the replication of many viruses including vaccinia virus (19), poliovirus (20), alphaviruses (21), brome mosaic virus (22), nidoviruses (23,24) and flaviviruses (25–27).', 'In 1981, the first cases of acquired immunodeficiency syndrome (AIDS) were described in American homosexual men. Thereafter, within three short years, French and American scientists confirmed that the human immunodeficiency virus (HIV) is the causative agent for AIDS. In the ensuing 20 years, >20 million individuals have died from AIDS; and currently, in 2006, ∼50 million people worldwide are infected by HIV-1 with 3 million incremental AIDS deaths and 4–5 million new infections occurring annually. The magnitude of this burden casts urgency to medical research on HIV/AIDS.', \"HIV-1 is a retrovirus of the lentivirus genus with an RNA genome of ∼9 kilobases which encodes nine polypeptides. The major HIV-1 structural proteins are encoded by three genes, gag (group-specific antigen), pol (polymerase) and env (envelope), while the accessory proteins, Vif, Vpu, Vpr and Nef, and the regulatory proteins, Tat and Rev, are the primary translation products of multiply-spliced mRNA. HIV-1 infects CD4+ human T-cells and macrophages and integrates as a provirus into the host cell's DNA. Gene expression of HIV-1 is governed transcriptionally by a viral protein, Tat (28,29), via its binding to a nascent viral TAR RNA (30), and post-transcriptionally by a second viral protein Rev (31,32) through its association with the viral RRE RNA. Both Tat and Rev interact with several host cell proteins in their transcriptional and post-transcriptional functions (33). HIV-1 does not encode for any RNA helicase; however, findings suggest that host cell RNA helicases may be involved in the reverse transcription of HIV-1 RNA, in HIV-1 mRNA transcription and in the nucleus-to-cytoplasm transport of viral mRNA.\", 'A recent unexpected finding revealed the possibility that an RNA helicase may potentially contribute roles in HIV-1 particle assembly and reverse transcription (34). Using proteomic analyses, Roy et al. (34) reported that the DEAH protein RNA helicase A (RHA) was found associated with HIV-1 Gag and packaged into HIV-1 virions in an RNA-dependent manner. When RHA was knocked down in cells, HIV-1 particles which were produced from these cells were significantly less infectious. This appears to be compatible with two possible explanations. First, it is conceivable that RHA participates in the formation of infectious virus particles either by shaping Gag–RNA interaction during viral particle assembly or by budding. Failure of RHA to properly restructure viral RNP could explain the observed reduced infectivity. Second, Roy et al. (34) reported evidence that HIV-1 particles that do not contain RHA showed reduced virion-endogenous reverse transcriptase activity. In this respect, it may be that RHA assists HIV-1 reverse transcriptase to more efficiently copy RNA by unwinding RNA secondary structure or by promoting the interaction of viral RNA with the nucleocapsid protein in order to assemble a better reverse transcription complex.', \"Separate from reverse transcription, the unwinding of highly structured RNAs might also be reasoned to be important for transcription (35). However, direct evidence for an RNA helicase role has been somewhat elusive. There are several examples which seemingly support an activity for RNA helicase in transcription. First, in vaccinia virus, it has been postulated that the NPH-II helicase assists transcription by strand-separating duplexed RNA structures to prevent R-loop formation behind the elongating RNA polymerase (36). Second, RHA has been invoked to provide a factor-recruitment role, bridging at the promoter the CREB-binding protein (37) and RNA polymerase II (37). Third, the p68 DEAD-box helicase was shown recently to be a novel transcriptional co-activator for p53's transcriptional function (38). Interestingly, in the latter two instances, neither the ATPase nor the helicase activity of RHA and p68 is apparently required for their attributed transcriptional roles.\", 'For HIV-1, two recent studies provide clues that RNA helicases may also serve co-factor function for transcription from the viral long terminal repeat (LTR). Fujii et al. (39) observed that RHA conserves in its N-terminus two double-stranded RNA-binding (dsRBD) domains characterized previously for the TAR RNA-binding protein, TRBP (40,41). These investigators found in both reporter and virus replication assays that RHA activated, in a TAR RNA-binding dependant manner, HIV-1 LTR-directed transcription (39). Next, Cocude et al. (42) separately described that the expression of a DExH RNA helicase, RH116, was significantly induced after infection of HeLa-CD4 cells by HIV-1. RH116 was found localized in the nucleus of HIV-1 infected cells and was observed to augment the transcription of unspliced HIV-1 transcripts. While both studies offer tantalizing evidence for roles by two different RNA helicases in HIV-1 transcription, critical details on how the helicases function by binding to TAR RNA and how this binding might cooperate with the activities of the viral transcriptional activator, Tat, are lacking. Pending more direct experimental tests, it remains unclear whether RHA and RH116 provide direct roles in transcription or indirectly influence the milieu of polymerase II initiation/elongation at the LTR.', 'Downstream from transcription, the fate of HIV-1 encoded RNA is regulated at the step of export of unspliced/partially spliced moieties from the nucleus into the cytoplasm. Unspliced and partially spliced viral RNAs code for genomic RNAs that are packaged into progeny virions and structural proteins. Hence, the egress of these RNAs from the nucleus into the cytoplasm is critical to the life cycle of the virus. Exit of HIV RNAs from the nucleus is a significant issue because unspliced/partially spliced cellular mRNAs are routinely retained in and not permitted export from the nucleus (43–46). A large body of work has suggested an elegant solution to this conundrum. Thus, it was established that the HIV-1 encoded Rev protein binds a highly secondary structured element (Rev responsive element; RRE) present in all unspliced and partially spliced HIV transcripts (47–57); and this binding specifically distinguishes, for purposes of nuclear export, viral transcripts from cellular RNAs.', \"New evidence now suggests that RNA helicases are also co-factors for Rev-directed export of HIV-1 mRNAs (58). In its role of transporting unspliced and incompletely spliced viral RNAs from the nucleus, Rev directly interacts with nuclear export receptor CRM1 (59,60), and CRM1 is required for Rev-mediated export of HIV RNAs (59,61,62). A recent report provides data that an RNA helicase, DDX3, is an additional player in the Rev–CRM1–RRE complex (12). Thus, it was shown that DDX3 over-expression enhanced Rev-dependent, but not other export, pathway; and that DDX3 is a nucleo-cytoplasmic shuttling protein which binds CRM1 and Rev. Moreover, DDX3's necessity for Rev/RRE/CRM1 function was demonstrated by knock-down of cell endogenous DDX3. Finally, because DDX3 locates to nuclear pore complexes (NPC), Yedavalli et al. (12) further proposed that this human helicase, like the analogous yeast Dbp5p (11), may function with Rev/CRM1 to remodel and ‘thread’ large unspliced HIV-1 RNAs through the nuclear pore, facilitating their final release to the cytoplasmic side of the NPC.\", 'The above DDX3 results are consistent with two additional papers which described similar findings for a related RNA helicase, DDX1. Thus, Pomerantz and co-workers (63) showed that DDX1 binds directly to the N-terminus of Rev and to the RRE-RNA motif and participates in the export of unspliced HIV-1 RNA from the nucleus to the cytoplasm. Additionally, they illustrated that reduced expression of DDX1 in astrocytes explains the previously observed tissue restricted function of HIV-1 Rev (64). Fully spliced viral mRNAs encoding for viral Tat, Rev and Nef, proteins have been shown previously to exit the nucleus using the cellular mRNA export pathway. Export of these mRNA may require the RNA helicase Dbp5 (65,66). As yet, the involvement of Dbp5 in export of spliced HIV-1 viral RNA has not been fully clarified.', 'The story of HIV-1 and RNA helicases is, however, likely to be more complex than and unlikely to conclude simply with RHA, RH116, DDX1 and DDX3 (Figure 1). HIV-1 RNAs are extensively regulated through splicing. Splicing is a multiple-step process requiring the recognition of splice sites by spliceosomes. It is generally believed that remodeling of RNA–RNA and RNA–protein interactions within the spliceosome is catalyzed by a family of DEAD/DExH box RNA helicases. To date, seven mammalian proteins that are RNA helicases have been implicated in mRNA splicing (67,68). Whether there is specific preference by subclasses of RNA helicases for viral mRNA splicing remains to be clarified. Moreover, how cellular RNA helicases might contribute to the translation of viral mRNAs also require further investigation.', \"Recently both Van't Wout et al. (69) and Krishnan and Zeichner (70) have provided evidence that the expression of several cellular RNA helicases including DDX24, DDX21, DDX18, DDX11 and DDX9 is modulated during HIV-1 infection; however, the precise cellular role and significance of these helicases for HIV-1 pathogenesis have not be characterized. Interestingly, Krishnan and Zeichner reported microarray data which examined the transition of HIV-1 infection from latency to productive replication, and found that several cellular RNA helicases were upregulated (71). For future understanding of functions, it will be important to design experiments which can segregate helicases which serve direct, although perhaps overlapping and redundant, roles on HIV-1 from those that might participate indirectly in the viral life cycle. Nevertheless, the convergence of evidence would support that several discrete cellular RNA helicases contribute importantly to the efficient execution of several steps in the HIV-1 replicative cycle.\", \"Given that the HIV-1/AIDS disease burden has reached pandemic global proportions, new antiviral strategies that target molecularly delineated mechanisms used by this virus are urgently needed (72). Is there a possibility that host cell helicases can be therapeutic targets for anti-HIV-1 chemotherapy? Implicit within this question is the concept that one could attack a host cell protein in order to treat an infecting pathogen. Although targeting a cellular protein involved in a viral pathway risks obvious cytotoxicity, this approach avoids the inherent problem posed by rapid HIV-1 mutation to all currently utilized chemotherapeutics targeted to virus-encoded proteins. We note that inhibition of cell-encoded enzymes in medical therapy is not an unprecedented strategy. Suppression of angiotensin-converting enzyme (ACE) is widely used to treat hypertension, congestive heart failure, myocardial infarction, endothelial dysfunction and renal disease (73). Elsewhere, aromatase inhibitors have been used to treat hormone-dependent breast cancer (74), and inhibitors of cellular secretory proteases are contemplated for Alzheimer's disease (75). We recently inhibited the cellular polyprotein convertase, furin, at minimal toxicity to the cell in order to block HIV-1 replication (76). Thus, a priori exclusion of cellular helicase as an antiviral target is not warranted.\", 'Guarded optimism that small molecule helicase inhibitors can be developed against viruses arises from encouraging progress in non-retroviral systems. Unlike HIV-1, human herpesviruses physically encode helicases. The herpes simplex virus UL5 and UL9 genes are helicases in superfamily 1 and 2, respectively (77). HSV UL5 together with UL8 and UL52 form a heterotrimeric helicase–primase complex responsible for unwinding duplex viral DNA at replication forks. Two recent studies provide proof-of-concept that the HSV helicase–primase can be targeted at low host cell toxicity by two new classes of drugs, amino-thiazolyphenyl-molecules (78) and thiazole amide derivatives (79). In addition, other studies suggest that the NS3 protein, a RNA helicase encoded by Hepatitis C virus and related West Nile virus and Japanese Encephalitis virus can be targeted to inhibit viral replication (80–82). This conceptual break through in drug development is important because it indicates that target discrimination between different helicases by small molecule inhibitors is possible. Of relevance to HIV-1, a synthetic immunomodulator Murabutide was shown recently to suppress HIV-1 replication in macrophages and T cells. Murabutide was shown to inhibit the activity of RNA helicase RH116, blocking its positive transcriptional activity for HIV-1 gene expression (42).', 'If one looks beyond the signature motifs conserved amongst helicases, then it becomes clear that the different proteins are widely divergent in their coding sequences. In principle, this suggests that individual helicases can be abrogated with specificity in a knowledge-directed manner. In theory, a helicase can be attacked by (i) inhibition of NTPase activity through direct competition for NTP binding, (ii) inhibition of substrate binding through direct competition at active site, (iii) allosteric mechanisms to affect NTP-binding/NTP hydrolysis and/or polynucleotide binding, and (iv) inhibition of unwinding activity by steric hinderance of helicase translocation along the polynucleotide substrate (83,84). Because the NTP-binding and substrate-binding pockets may be sufficiently similar between various helicases, specificity of inhibition through these sites will likely be extremely difficult, although perhaps not impossible. On the other hand, the tremendous variations in sequence and sizes of helicases, in their oligomerization states, in their discrete domains responsible for protein–protein interactions and/or for targeting to specific nucleic acids (85), and in their differential localizations within cells (86) offer interventional possibilities outside of the NTP- or polynucleotide-binding sites.', 'We are in the preliminary stages of screening ring-expanded nucleoside analogs found previously to be successful NTPase/helicase inhibitors of West Nile virus, Hepatitis C virus and Japanese encephalitis virus (81,82). We have observed that a few of these candidate inhibitors have substantial anti-HIV-1 activity at doses that do not incur cytotoxicity to cells treated in tissue culture for 1 week. Further studies are needed before concluding that these compounds exert specific inhibition of DDX3, one of the other cellular helicases, or some other target altogether.', \"There is another area where a cellular helicase activity and HIV-1 are likely to intersect. An emerging research focus is the role of small interfering RNAs (87) and microRNAs (miRNA) as innate cell defenses against viruses including HIV-1 (87–90). In human cells, the precursor for miRNA (pre-miRNA) is processed by DICER (Figure 2A) which is a ribonuclease with a bona fide RNA helicase domain (91,92). A surprising recent finding revealed that the human TRBP, which has been shown to be a potent binder of the HIV-1 TAR RNA RNA (40,41), is an indispensable dsRNA-binding partner of DICER which allows the latter to associate with pre-miRNA (91,92). Without TRBP, DICER's miRNA processing activity is lost. Thus an intriguing scenario can potentially unfold. Accordingly, whereas the ribonuclease–helicase protein DICER requires TRBP to process duplex-structured miRNAs in order that the cell can use such matured miRNAs for antiviral defense, it could be speculated that HIV-1 has evolved to restrict this defense by the ability to transcribe viral TAR RNA to squelch TRBP away from DICER (Y. Bennasser, M. L. Yeung and K. T. Jeang, manuscript submitted) (Figure 2B). If this thinking is correct, then HIV-1 has developed mechanisms not only to co-opt the active functions of a virus-propitious cellular helicase (i.e. DDX3) but also to inactivate the role of a second virus-pernicious helicase (i.e. DICER) for purposes of selfish gain.\", 'In a separate perspective, virus infection can trigger through double-stranded viral RNAs an innate antiviral immune response. Thus viral dsRNAs can be recognized by cellular proteins [pattern-recognition receptors (PRRs)] which initiate antiviral responses by inducing the production of a variety of cytokines including type I interferons (IFN-α and IFN-β) and initiating additional inflammatory and adaptive immune responses. Recently, DExD/H RNA helicases such as RIG-1 (retinoic acid inducible gene-1) (93) and Mda5 (melanoma differentiation-associated gene 5) (94) have been identified as suppressors of viral replication by binding to virus associated dsRNA and activating type I interferon-dependent antiviral immunity. Over-expression of RIG-1 and Mda5 was found to enhance dsRNA induced type I interferon antiviral response. Currently, it remains speculative whether helicases like RIG-1 and Mda5 may recognize HIV-1 dsRNA and trigger an innate immune response. Intriguingly, several reports exist in the literature that HIV-1 infection does induce activation of type 1 interferons (95,96).', \"In conclusion, by studying helicase proteins one can gain insights into normal cellular metabolic processes, abnormal inherited human diseases (e.g. Bloom syndrome, Werner syndrome, Cockayne's syndrome and xeroderma pigmentosum; all diseases with mutations in cellular helicases), and remarkably also the biology of viruses.\"], 'cord_uid': 'vefs1h6o'}\n{'title': 'Amino acid residues critical for RNA-binding in the N-terminal domain of the nucleocapsid protein are essential determinants for the infectivity of coronavirus in cultured cells', 'text': ['RNA viruses, including coronaviruses, encode a group of structural phosphoproteins with basic patches on their surface. As a pivotal structural component of the virion, this group of proteins plays essential roles in packaging the RNA genome to form a ribonucleoprotein (RNP) complex resulting from assembly of the viral RNA and multiple copies of the (nucleo)capsid protein. A detailed dissection and characterization of the intrinsic RNA-binding properties of these proteins are thus essential for understanding several important processes in the life cycle of RNA viruses, including assembly of the nucleocapsid, the specific encapsidation of viral RNA and morphogenesis of virions. In this study, amino acid residues in the N-terminal domain of the nucleocapsid (N) protein of a coronavirus that are critical for its RNA-binding activity are identified and their roles in viral infectivity are analyzed, using site-directed mutagenesis based on our previous structural studies.', 'Coronaviruses are enveloped viruses containing the largest known single-stranded, positive-sense RNA genome of ∼30 kb (1). During the coronavirus life cycle, the N protein is synthesized in large amounts and is thought to play an important role by specifically packaging the viral genome into a filamentous nucleocapsid of ∼10 to 15 nm in diameter and several 100 nm in length, a macromolecular structure that is visible by using electron microscopy (2). In addition to its structural role, the N protein also participates in viral RNA transcription, replication and in modulating the metabolism of host cells (3–13). Using X-ray crystallography techniques, we recently determined a 3D structure of the N protein from coronavirus infectious bronchitis virus (IBV), a prototype coronavirus, and showed that the basic building block for nucleocapsid assembly was a dimer of the N protein (14). The 409 amino acids long IBV N protein is composed of two globular domains that are resistant to proteolysis. Amino acids 29 to 160 form the N-terminal RNA binding domain and residues 218 to 329 form its C-terminal dimerization domain (5,14). Even though the structure of the N-terminal domain was determined in the absence of a nucleic acid ligand, its overall shape and the distribution of electrostatic charges suggest a plausible model for RNA binding. The outer surface of the N-terminal domain is enriched in aromatic and basic residues, a common feature for RNA-binding proteins (15). The shape of this domain is reminiscent of a hand having basic fingers, a hydrophobic palm and an acidic ‘wrist’. The positively charged ‘fingers-like’ β-hairpin extension could neutralize the phosphate groups emanating from RNA, whilst the base moieties could make contact with exposed aromatic residues from the hydrophobic palm (14).', 'Based on this structural hypothesis, we chose to mutate a number of evolutionarily conserved residues in the N-terminal domain of the IBV N protein in order to assess their roles in viral genomic RNA binding and in viral replication. We used purified recombinant wild type and mutant proteins containing the N-terminal domain expressed in Escherichia coli for RNA binding assays. These studies led to the identification of a number of amino acid residues essential for the RNA-binding activity of the domain. Subsequently, we introduced mutations that either severely or mildly impair the RNA binding activities of the N-terminal domain into an infectious cDNA clone system derived from the genomic RNA of IBV to assess their effects on viral replication and infectivity. We identified several residues exposed to the solvent whose substitutions to alanine yield a decrease of RNA binding and a concomitant reduction in virus replication. Interestingly, residues Arg-76 and Tyr-94, located at the base of the positively charged flexible hairpin loop and on the hydrophobic platform, respectively, were critical for RNA binding and viral infectivity.', 'His-tagged wild type and mutant N-terminal domains of IBV N protein were expressed in E.coli BL-21 by induction with 1 mM isopropyl-β-d-thiogalactopyranoside (IPTG). Cells were lysed by sonication and purified by metal affinity chromatography with Protino-Ni 150 kit (Macherey Nagel).', 'PCR fragments covering the IBV genome from 27 100 to 27 608 nt were cloned into a plasmid in either forward or reverse orientation under the control of a T7 promoter. The Dig-labeled sense (+) and anti-sense (−) RNA probes were made in vitro using the DIG RNA labeling kit according to the instructions of the manufacturer (Roche).', \"Five micrograms of purified proteins were resolved on an SDS-15% polyacrylamide gel and transferred onto nitrocellulose membranes (Hybond C-Extra, Amersham Biosciences) using a semi-dry transfer apparatus. Membranes were washed for 10 min with the probe buffer [1× Denhardt's Reagent, 1 mM EDTA, 10 mM Tris–HCl (pH 7.5) and 50 mM NaCl], blocked for 1 h with 25 µg/ml yeast tRNA (Ambion) and subsequently incubated with 10 µg of DIG-labeled RNA probe in the same probe buffer for 1 h. Membranes were washed three times for 15 min each with the probe buffer, before proceeding to detection with CDP-Star (Roche) according to the manufacturer's instructions.\", 'Construction of an infectious IBV clone was carried out essentially as described (16,17). Briefly, five fragments spanning the entire IBV genome were obtained by RT–PCR from Vero cells infected with the Vero cell-adapted IBV p65. The PCR products were purified from agarose gels and cloned into pCR-XL-TOPO (Invitrogen) or pGEM-T Easy (Promega) vectors. Subsequently, fragment A was removed from pCR-XL-TOPO by digestion with NheI and EcoRI, and subcloned into pKT0 vector. Plasmids were digested with either BsmBI (fragment A) or BsaI (fragments B, C, D and E). The digested plasmids were separated on 0.8% agarose gels containing crystal violet. Bands corresponding to each of the fragments were cut from the gels and purified with QIAquick gel extraction kit (QIAGEN Inc.). Fragments A and B, and fragments C, D and E were first ligated with T4 DNA ligase at 4°C overnight. The two reaction mixtures were then mixed and further ligated at 4°C overnight. The final ligation products were extracted with phenol/chloroform/isoamyl alcohol (25:24:1), precipitated with ethanol and detected by electrophoresis on 0.4% agarose gels.', 'Full-length transcripts were generated in vitro using the mMessage mMachine T7 kit (Ambion, Austin, TX). The N transcripts were generated by using a linearized pKT0-IBVN containing IBV N gene and the 3′-untranslated region (3′-UTR) as templates. The in vitro synthesized full-length and N transcripts were treated with DNase I and purified with phenol/chloroform. Vero cells were grown to 90% confluence, trypsinized, washed twice with cold phosphate-buffered saline (PBS), and resuspended in PBS. RNA transcripts were added to 400 µl of Vero cell suspension in an electroporation cuvette, and electroporated with one pulse at 450 V, 50 µF with a Bio-Rad Gene Pulser II electroporator. The transfected Vero cells were cultured overnight in 1% FBS-containing MEM in a 60 mm dish or a 6-well plate and further cultured in MEM without FBS.', 'Mutations were introduced into the corresponding fragments by using QuickChange site-directed mutagenesis kit (Stratagene), and confirmed by sequencing of the whole fragments.', 'Confluent monolayers of Vero cells on 6-well plates were infected with wild-type and mutant viruses at a multiplicity of ∼1 PFU/cell. After 1 h of incubation at 37°C, cells were washed twice with PBS and cultured in 3 ml of MEM containing 0.5% carboxymethy cellulose for 3 days. The cells were fixed and stained with 0.1% toluidine.', 'Vero cells were infected with wild-type and recombinant IBV, and harvested at different times post-infection. Viral stocks were prepared by freezing/thawing of the cells three times. The plaque-forming units per ml of each sample were determined by infecting Vero cells on 6-well plates in duplicate with 10-fold serial dilution of each viral stock.', \"Vero cells were infected with wild type and Q74A mutant virus at a multiplicity of ∼1 PFU/cell. Total RNA was extracted from cells infected with wild type or mutant viruses. Ten micrograms of RNA were added to a mixture of 1× MOPS, 37% formaldehyde and formamide and incubated at 65°C for 20 min before subjected to gel electrophoresis. The segregated RNA bands were transferred onto a Hybond N+ membrane (Amersham Biosciences) via capillary action overnight and fixed by ultraviolet (UV) crosslinking (Stratalinker). Hybridization of Dig-labeled DNA probes was carried out at 50°C in hybridization oven overnight. Membranes were washed three times for 15 min each with the probe buffer, before proceeding to detection with CDP-Star (Roche) according to the manufacturer's instructions.\", \"Vero cells were infected with wild type and Q74A mutant virus at a multiplicity of ∼1 PFU/cell. Total proteins extracted from Vero cells were lysed with 2× SDS loading buffer in the presence of 200 mM DTT plus 10 mM of iodoacetamide and subjected to SDS–PAGE. Proteins were transferred to PVDF membrane (Stratagene) and blocked overnight at 4°C in blocking buffer (5% fat free milk powder in PBST buffer). The membrane was incubated with 1:2000 diluted primary antibodies in blocking buffer for 2 h at room temperature. After washing three times with PBST, the membrane was incubated with 1:2000 diluted anti-mouse or anti-rabbit IgG antibodies conjugated with horseradish peroxidase (HRP) (DAKO) in blocking buffer for 1 h at room temperature. After washing three times with PBST, the polypeptides were detected with a chemiluminescence detection kit (ECL, Amersham Biosciences) according to the manufacturer's instructions.\", 'PCR products covering the IBV sequence from 25 957–26 352 nt were amplified by using the forward primer 5′-CGGGCATATGTCTTCTGGAAATGCATCTTGG-3′ and the reverse primer 5′-CGGGATCCTTACAGGGGAATGAAGTCCCAAC-3′. The PCR fragments were digested with NdeI and BamHI, and ligated into NdeI- and BamHI-digested pET-16b (Novagen). Each mutation was introduced by two-round PCR and the mutation introduced was confirmed by automated nucleotide sequencing.', 'The N-terminal RNA-binding domain of N protein from both IBV and SARS-CoV contains a flexible and positively charged hairpin loop that extends much beyond the protein core and that could grasp a nucleic acid substrate by neutralizing its phosphate groups (Figure 1) (14,18). Positively charged lysine and arginine residues in the N-terminal domain of SARS-CoV N protein were proposed to bind a 32 nt stem–loop structure located at the 3′ end of the SARS-CoV RNA genome (18). Indeed, the structure of this domain possesses some features reminiscent of RNA-binding proteins sharing the RNP fold (19–22). These RNA-binding proteins, including the U1A spliceosomal protein (23) and the coat protein from MS2 bacteriophage (24), bind the RNA ligand with residues emanating from the surface of a four-stranded anti-parallel β sheet. Presumably, strands β2 and β3 as well as the flexible β-hairpin from the IBV nucleocapsid protein could fulfill a similar role by interacting with phosphate groups from an RNA ligand (Figure 1). The β3 strand is the longest in the N-terminal domain of IBV N protein and several evolutionarily conserved residues project from it (Figure 1). Nucleotide bases in single-stranded RNA have a tendency to stack either on adjacent bases or with aromatic side chains of the protein (20).', 'The above structural model was partially supported by the demonstration that the N-terminal domain of IBV N protein could interact with RNA fragments corresponding to the 3′ end of the viral genome (14). To more vigorously test this model, systematic mutagenesis of residues essential for the RNA-binding activity was carried out. We targeted the exposed hydrophobic residues Tyr-92 and Tyr-94 (strand β3) that could form stacking interactions with the nucleotide bases as well as several basic or polar residues emanating from the β-hairpin (Figure 2a). As summarized in Figure 2a, 10 amino acids were mutated to alanine. These residues are either exposed to the solvent (Gln-74, Arg-76, Lys-78, Lys-81, Tyr-92 and Tyr-94) or have been conserved during evolution (Tyr-70 and Arg-73) and are thus likely to be involved in RNA binding or to play an undefined functional role. In addition, mutation of Asp-111 to an alanine was included as an additional control for the RNA-binding assay (Figure 2a).', 'The wild type and mutant constructs, spanning 29 to 160 amino acids with a hexa-histidine tag at their N-termini, were cloned into a bacterial expression vector, expressed in E.coli and purified to near homogeneity. Approximately equal amounts of the purified proteins were separated on SDS–PAGE (Figure 2b, upper panels). After transfer to a Hybond C extra membrane, the RNA-binding activities of wild type and mutant proteins were analyzed by northwestern blot with two probes corresponding to the positive- and negative-sense 3′-UTR of IBV, respectively. A moderate decrease in binding to both the positive and negative sense RNA probes was consistently observed with most mutant proteins, and representative gels are shown in Figure 2b (Figure 2b, middle and lower panels). Among all mutants, mutation of Tyr-94 to an alanine residue showed the most significant reduction in binding of the domain to both positive and negative probes. The RNA binding activities of this mutant to positive and negative probes were reduced to 27 and 25%, respectively, of the wild type were observed (Figure 2b, lane 9). Relatively less reduction in the RNA-binding activity was observed for the R76A, K78A, K81A and Y92A mutant proteins which retain between 50–74% of the wild-type binding activity (Figure 2b, lanes 5–8). With the exception of the Q74A and D111A mutants, which displayed a slight increase in the RNA binding activity to both the positive- and negative-sense RNAs, (Figure 2b, lanes 4 and 10), all other mutants showed a certain degree of reduction in their binding activity to either RNA probes (Figure 2b). As a control, a circular dichroism analysis of the two corresponding R76A and Y94A single mutants was performed to check whether the mutant proteins were properly folded. The spectra obtained for these two mutants were similar to the wild-type protein, ruling out any drastic conformational change introduced by the mutations (data not shown).', 'As none of the single mutations introduced could totally abrogate the RNA-binding activity of the IBV N-terminal domain, two mutant constructs carrying double mutations were made. As can be seen in Figure 2b (lanes 13 and 14), mutations of both Arg-73 and Lys-78 to alanine residues (R73/K78A) reduced the RNA-binding activities to the positive- and negative-sense RNA probes between 44 and 10% of wild-type, respectively. Mutations of both Arg-76 and Tyr-94 (R76/Y94A) to alanine reduced the RNA-binding to the positive- and negative-sense RNA probes to only 7 and 19% of wild-type, respectively (Figure 2b, lanes 15 and 16). Taken together, these results are consistent with the presence of an extended RNA binding site covering a large part of the accessible surface of the N-terminal domain of the coronavirus N protein. They are also in agreement with the assumption that removal of a single positive charge at the protein surface by mutation of a lysine or arginine residue to an alanine is insufficient to totally disrupt the formation of the protein–RNA complex.', 'The R76A and Y94A mutations were first introduced into an infectious clone of IBV, and in vitro transcribed full-length RNA molecules derived from wild type and mutant constructs were generated by in vitro transcription using the T7 RNA polymerase in the presence of a cap analog. Electroporation of wild-type RNA transcripts together with an RNA fragment covering the N protein region into Vero cells showed the formation of massive CPE at 2 days post-electroporation (Figure 3, panel rIBV+N). In contrast, no CPE formation was observed in cells transfected with transcripts derived from either mutant up to 5 days post-electroporation, suggesting that no infectious virus could be rescued from the two mutant transcripts (Figure 3, panels R76A+N and Y94A+N). These results indicate that R76A and Y94A mutations, which severely reduce the RNA-binding activity of the N-terminal domain of the N protein, may abolish the infectivity of the in vitro synthesized full-length IBV transcripts.', 'Introduction of Q74A and Y92A mutations into the full-length infectious clone of IBV was then carried out. Typical CPEs were observed in cells transfected with Q74A mutant transcripts at 3 days post-electroporation and recombinant viruses were recovered (Figure 3, panel Q74A). In cells electroporated with Y92A transcripts, a typical CPE was observed at 2 days post-electroporation [Figure 3, panel Y92A(1)]. Incubation of the transfected cells for one more day saw the appearance of more CPEs [Figure 3, panel Y92A(2)]. However, prolonged incubation of the transfected cells up to 6 days saw no spread of the infection from the two initiated infected loci to neighboring cells. Re-infection of fresh Vero cells with either medium collected from these cells or total cell lysates prepared by freezing and thawing of the transfected cells showed no detection of the CPE formation, indicating that the minute amount of viruses rescued from the initially transfected cells could not maintain infectivity in subsequent passage.', 'The R76A and Y94A mutant N constructs were then used to test if they could facilitate the rescue of the in vitro synthesized full-length IBV transcripts in trans. As mentioned, efficient recovery of infectious virus from the in vitro transcribed full-length IBV RNA requires co-transfection of these in vitro transcripts together with the in vitro synthesized RNA covering the N gene. Electroporation of wild-type full-length IBV transcripts together with RNAs synthesized in vitro from wild-type and the two mutant N constructs, respectively, showed efficient recovery of the infectious virus with CPE observed on almost the whole monolayer at 3 days post-electroporation (Figure 3, panels rIBV+N-R76A and rIBV+N-Y94A). In contrast, cells transfected with the full-length IBV transcripts only (without co-electroporation with the N transcripts) showed the formation of a single CPE on the whole monolayer at 3 days post-electroporation (Figure 3, panel rIBV-N).', 'As no infectious virus was recovered from cells transfected with R76A and Y94A mutant transcripts, total RNA was extracted from cells electroporated with wild-type and mutant full-length transcripts and RT–PCR amplification of subgenomic mRNAs was carried out to check whether a low level of RNA replication and subgenomic mRNA transcription occurred in cells transfected with the mutant transcripts. The forward primer used in this reaction corresponds to the leader sequence from 26–46 nt in the genomic RNA and the downstream primers covers IBV sequences from 24 784 to 24 803 nt. If transcription of subgenomic mRNAs did occur, a 415 bp PCR product corresponding to the 5′-terminal region of the subgenomic mRNA4 and a 1010 bp fragment corresponding to the 5′-terminal region of the subgenomic mRNA3 would be expected. As shown in Figure 4a, a dominant 415 bp band and a weak 1010 bp band were observed in cells electroporated with wild-type full-length transcripts at 2 days post-electroporation (lane 2). Sequencing of the PCR fragments confirmed that they represent the correct sequences of the corresponding regions of the subgenomic mRNAs 3 and 4, respectively. The same PCR products were not detected in cells electroporated with the R76A and Y94A mutant transcripts, respectively, at 24, 48 and 72 h post-electroporation (Figure 4a, lanes 3–8). In cells electroporated with Q74A mutants at 3 days post-electroporation, RT–PCR amplification of the subgenomic mRNA 3 and 4 showed the detection of the 415 and 1010 bp bands (Figure 4a, lane 9). The same PCR products were not evident in cells electroporated with the Y92A mutant transcripts (Figure 4a, lane 10).\\n', 'RT–PCR amplification of the negative strand RNA was performed to check if RNA replication occurred in these transfected cells. The primer pair was chosen so that the IBV sequence from 14 931 to 15 600 nt would be amplified by the RT–PCR. If replication of viral RNA occurred, a 670 bp PCR fragment would be expected. As shown in Figure 4b, RT–PCR fragments amplified from both positive (lanes 4, 7 and 9) and negative (lanes 3, 6 and 10) strand RNA templates were obtained from cells transfected with the mutant transcripts. The amount of the negative strand RNA was approximately half of that of the positive strand RNA (Figure 4b). As positive control, the same RT–PCR fragment for the negative strand RNA was observed in cells transfected with wild-type transcripts (Figure 4b, lanes 8 and 11). In the negative controls, total RNA extracted from cells transfected with Y94A (Figure 4b, lane 2) and R76A (Figure 4b, lane 5) was analyzed by PCR directly using the primer set for the negative strand RNA. No corresponding PCR fragment was detected from these samples, demonstrating that the detection of the negative strand RNA in cells transfected with wild-type and mutant constructs by RT–PCR is due to the replication of viral RNA. These results confirm that transcription of the negative strand RNA has taken place in cells transfected with the mutant transcripts.', 'The transcription of negative strand RNA in cells electroporated with wild-type, R76A, Y92A and Y94A transcripts was then quantitatively determined and compared by real time RT–PCR at 24 and 48 h post-electroporation, respectively. Compared to cells transfected with wild-type transcripts, the relative amounts of negative strand RNA in cells transfected with R76A, Y92A and Y94A were 48, 46 and 50%, respectively, at 24 h post-electroporation (Figure 4c, lanes 1–4). These figures were dropped to 41, 45 and 23%, respectively, at 48 h post-electroporation (Figure 4c, lanes 5–6). Taken together, these results demonstrate that transcription of the negative strand RNA has taken place in cells transfected with the mutant transcripts. As the level of negative strand RNA was not increased in cells transfected with the mutant transcripts over a time-course experiment, it suggests that transcription of these RNA species may occur only in the initially transfected cells using the in vitro transcripts as templates.', 'The growth properties of the Q74A mutant virus on Vero cells were tested by analysis of plaque sizes and growth curves of passage 3 mutant virus. Compared to cells infected with wild-type recombinant virus (rIBV), slightly smaller sized plaques were observed in cells infected with the Q74A mutant virus (Figure 5a). Analysis of the growth curves of wild-type and the mutant virus demonstrated that the mutant virus grew slightly more slowly than the wild type recombinant virus. When 10-fold more mutant virus was used, 2- to 10-fold more mutant virus was produced at 8–24 h post-infection (Figure 5a). The titers of the mutant virus dropped more rapidly than wild-type virus at 36 and 48 h post-infection (Figure 5a).\\n', 'Further characterization of the mutant virus was subsequently carried out by analysis of the genomic and subgenomic RNA, and the expression of viral structural proteins S and N. Northern blot analysis of cells infected with the 5th passage of rIBV and Q74A mutant viruses showed that similar amounts of genomic and subgenomic RNAs were detected (Figure 5b). Quantitative comparison of the subgenomic RNA synthesis by densitometry analysis of the bands shown in Figure 5b demonstrated that in cells infected with Q74A mutant virus (p5), the relative amounts of mRNA 2, 3, 4, 5 and 6 are ∼98, 101, 121, 104, 106 and 101%, respectively, of those in cells infected with rIBV. However, Western blot analysis of cells infected with rIBV and Q74A mutant virus showed that more S and N proteins were detected in cells infected with wild-type virus (Figure 5c, lanes 1–14).', 'To confirm if the Q74A mutation is genetically stable during passage of the mutant virus in cells, viral RNA was isolated from cells infected with passage 2 and 5 virus and amplified by RT–PCR. Sequencing analysis showed that the two mutated nucleotides (CA-GC) remained stable (Figure 5d). Furthermore, no compensatory mutation was found in other region of the N protein, confirming that Q74A mutant virus is stable in cell culture system.', 'The intrinsic RNA-binding activity of the nucleocapsid protein of RNA viruses is essential for packaging the RNA genome into a RNP complex. In previous structural studies of the N protein from two coronaviruses, the N-terminal domain was shown to be resistant to proteolysis and to fold independently (14,15). It serves as a functional unit critical for the RNA-binding activity of the protein (14). Based on these structural results, the importance of individual residues in RNA-binding was studied. Here we showed that Tyr-94 in strand β3 binds to the RNA ligand. The positively charged Arg-76 is located in the immediate vicinity of this aromatic residue at the base of the extended flexible hairpin loop, within a cluster of positively charged residues. Thus, the interaction between Arg-76 and Tyr-94 with RNA appears to play a crucial role during virus assembly. Since no single mutation could totally disrupt nucleic acid binding, other aromatic or basic residues located close to these two residues at the surface of the N-terminal domain are likely to contribute to RNA binding by creating a broad surface that makes contact with the genomic RNA. Using a simple docking experiment, the RNA binding site seems capable of accommodating up to 8–9 nt bases per N-terminal domain monomer. The Arg-76 and Tyr-94 residues identified in this paper thus provide more precise mapping of the location of the RNA-binding site on the IBV N protein.', 'It is interesting to note that the main effect of these mutations on viral life cycles was on subgenomic RNA transcription. However, the replication of negative strand RNA is nearly normal in cells transfected with these mutant constructs. Is it possible that these mutations will directly impair an undefined function of the N protein that is essential for subgenomic RNA transcription? Two possibilities were considered. First, the coronavirus N protein is directly involved in subgenomic RNA replication. The MHV N protein was shown to be involved in coronaviral RNA transcription (4), but the underlying mechanism was unrevealed. Alternatively, the observed phenotypic changes in subgenomic RNA replication caused by these mutations may be due to a generally reduced viral replication rate and infectivity. In fact, several other mutations that affect viral replication cycles were found to have defects in subgenomic RNA transcription (25–27). In this study, we have repeatedly observed that detection of subgenomic RNA replication in cells infected with low multiplicity of infectivity of IBV by RT–PCR is technically challenging. Positive results can be obtained only if transcription of subgenomic RNA reaches a certainly high level. When in vitro transcripts containing Y92A mutation were introduced into cells, one or two CPEs were always observable in a monolayer, suggesting the presence of low level viral RNA replication. However, detection of subgenomic RNA was consistently unsuccessful in cells electroporated with this mutant transcript.', 'Coronavirus N protein is a multi-functional protein. The protein was shown to be able to facilitate rescue of infectious virus from several coronaviruses by co-electroporation of the in vitro synthesized N transcripts with the in vitro synthesized full-length transcripts (17,28–30). The mechanism for this trans-effect is currently unknown. In this study, the R76A and Y94A mutant N constructs could promote rescue of the infectivity of the in vitro synthesized IBV RNA as efficiently as did the wild type N transcripts, suggesting that the reduced RNA-binding activity in the N-terminal domain does not affect this function of the N protein. It also suggests that RNA-binding per se is not a major factor that contributes to this important function of the N protein.', 'Are the reduced replication rate and loss of the infectivity of the mutant transcripts caused by the effects of these mutations on the functionality of the N protein or on the integrity of the genomic RNA itself? The nice correlation between the reduced RNA-binding activity of the N-terminal domain from the mutant proteins and the loss of infectivity of the corresponding mutant transcripts would support that the failure to rescue the viral infectivity is due to the specific mutation that impairs the functionality of the N protein. In addition, in the cases of Q74A and R76A mutants, the two mutations are physically adjacent in the genome, but yet totally different phenotypes were observed from the two mutants. It would support that the failure to recover infectious viruses from the mutant transcripts is due to the fact that these mutations reduce/impair the RNA-binding activity of the N protein. If this were the case, however, supplement of wild type N protein in trans would rescue the infectivity of the mutant transcripts. Two experiments were conducted to test this possibility, but none of them were successful. First, co-transfection of the mutant full-length transcripts with wild-type N transcripts into cells failed to recover infectious virus from the mutant transcripts, suggesting that wild-type N RNA cannot rescue the infectivity of the mutant transcripts in trans. The second experiment was to introduce the mutant full-length transcripts into cells stably expressing a six-His-tagged IBV N protein. Once again, it failed to rescue the infectivity of the mutant transcripts. At present, it is unclear if this was due to the low-level expression of N protein in these cells or because the N protein expressed in these cells contains the six-His tag at its C-terminus.', 'Residues Arg-76 and Tyr-94 in the IBV N protein are well conserved across the whole family of coronaviruses. Structurally, these two residues may correspond to the Arg-94 and Tyr-112 residues in the SARS-CoV N protein (see Figure 1a). It is possible that introduction of similar alanine mutations into the SARS-CoV genome may yield a similar phenotype observed here for IBV. The fact that substitutions of Gln-74, Arg-76, Tyr-72 and Tyr-94 in the IBV genome by using an infectious clone of IBV showed nice correlation between the RNA-binding activity and viral infectivity would suggest potential strategies for intervention of viral replication. A compound which could specifically bind to this region of the N protein might interfere with RNA binding and thus inhibit virus replication. In addition, this study points to a number of residues on the surface of the N protein that contribute less to the RNA-binding activity and viral infectivity for the design of attenuated coronavirus strains as potential vaccine candidates.'], 'cord_uid': 'at9jg710'}\n{'title': 'Automated identification of multiple micro-organisms from resequencing DNA microarrays', 'text': ['For both surveillance and diagnostic applications, fine-scale pathogen identification and near-neighbor discrimination is important; therefore, an assay that monitors at this very specific level is desirable for many types of samples such as clinical and environmental (1–3). To successfully use any method based on DNA or RNA detection, these assays must be coupled with large databases of nucleic acid sequence information for assay design to ensure that the desired information is provided and for the interpretation of raw data. Several well-established techniques use PCR to amplify individual target pieces of sequenced genomes to provide detection of organisms (4). These methods can roughly be divided into approaches that target individual short sequence lengths or probes (<40 bp) and methods that examine longer probes. The advantage of using short probes is that when the uniqueness of the probe has been assured and unique primers are also selected, this method gives good specificity. This approach is capable of providing fine-scale identification of several genetically close organisms by selecting a sufficient number of probes. However, this can rapidly lead to a very large number of total probes being required to detect all organisms of interest. In addition these selected probes, which in the initial selection process were determined to be unique, are often later found to be less specific as more organisms are sequenced or are less specific under conditions that differ from the original conditions. This is particularly a problem for organisms belonging to a family with a high mutation rate and also for pathogens that have relatively few neighboring pathogens sequenced. In addition, PCR approaches focused on short unique probes are not capable of detecting the presence of new significant mutations nor can they easily resolve base sequence details. Approaches that use longer individual probes avoid many of these issues at the cost of being less specific. This issue means most of these approaches are not suitable for providing the information desired, providing impetus to this work.', 'High-density resequencing microarrays produce variable length segments, 102–105 bp, of direct sequence information. This target sequence falls in the longer target regime of PCR approaches but rather than being hybridized to a longer less-specific probe on the microarray, many shorter specific probes are placed on the microarray to allow more detailed determinations from the entire PCR amplicon. This also means that the specificity of the primers used can be relaxed. They have been successfully used to detect single nucleotide polymorphism (SNP) and genetic variants from viral, bacterial and eukaryotic genomes (5–12). Their use for SNP detection has clearly established their ability to provide reliable quality sequence information. In most cases, the microarrays were designed to study a limited number of genetically similar target pathogens and for many cases, the detection methods relied only on recognizing hybridization patterns for identification (6,9,10,13,14). Taking advantage of the sequential base resolution capability of resequencing microarrays that is required for SNP detection, resequencing has recently been successfully adapted recently using a different approach for organism identification of multiple bacterial and viral pathogens while allowing for fine detailed discrimination of closely related organisms and tracking mutations within the targeted pathogen (15–16). The new methodology differed from earlier work by using the resolved bases as the query of a similarity search of DNA databases to identify the most likely species and variants that match the base calls from the hybridization observed. The system was capable of testing for 26 pathogens simultaneously and could detect the presence of multiple pathogens. A software program, resequencing pathogen identifier (REPI), was used to simplify data analysis by performing similarity searches of a genetic database using basic local alignment search tool (BLAST) (17). The REPI program used BLAST default settings and would only return sequences that might represent the hybridization if the expect value, a quantity calculated by the BLAST program that indicates the likelihood that the sequence match found would have occurred by random chance in the database, was <10−9. This screened out all cases that had insufficient signal; however, the final determination of what pathogen(s) was detected and to what degree discrimination was possible required manual examination of the returned results. This method successfully allowed fine discrimination of various adenoviruses and strain identifications of Flu A and B samples in agreement with conventional sampling results (15,16). Two important advantages of this approach were that the information was always recovered at the most detailed level possible and that it was capable of still recognizing organisms with recent mutations. This approach also maintained specificity well, as it was not dependent on the uniqueness of a few individual short probes.', 'Although this analysis method has utility, there are several shortcomings: it is time consuming, not optimized to maximize sensitivity, has complicated results, is suitable only for an expert, and contains redundant or duplicate information. The process was time consuming because only the initial screening was handled automatically while the remaining steps required manual interpretation before the detection analysis was complete. Because a simple criterion (expect value cutoff of 10−9) and non-optimized BLAST parameters were used to consider a pathogen detected, the REPI algorithm provided a list of candidate organisms but did not make a final simple conclusion or relate the results of one prototype sequence to another. Instead a manual process was used to make the final determination, but because the REPI program provided all similar results and the use of public nucleic acid databases containing redundant entries, a large amount of data was presented to a user that was not useful. In addition, with a manual process it was not possible to establish that the algorithm developed was generally applicable for any organism where nucleic acid base resolved sequence information has been provided.', 'In this paper, we describe a new software expert system, Computer-Implemented Biological Sequence Identifier system 2.0 (CIBSI 2.0), that successfully uses resolved base sequence information from custom designed Affymetrix resequencing microarrays to provide a simple list of organisms that are detected. This algorithm addresses the most important shortcoming of previous methods by incorporating new features to completely automate pathogen identification. We have demonstrated the effectiveness of this algorithm for identification via several examples. The single program is capable of making correct decisions for all 26 pathogens contained on the Respiratory Pathogen Microarray v.1 (RPM v.1), whether detected alone or in combinations, with improved sensitivity. Although the program is currently applied to resequencing microarrays, the methodologies developed remain generally applicable. Only the first portion of the algorithm handles issues specific to microarrays while the remainder deals with sequences that are suitable for use as a query by the BLAST algorithm. In developing the general identification algorithm, we have identified and resolved issues specific to resequencing microarrays that complicate their use. Because the entire decision process for what is detected has been automated, it is straightforward to test whether the rules used to make identifications are rigorous and applicable to any pathogen. With this efficient program, resequencing based assays can provide a competitive method to test simultaneously for many possible pathogens, providing output that can be interpreted by a non-expert.', 'The details of the RPM v.1 design and the experimental methods have been discussed in previous work (15,16,18) (Lin et al., submitted for publication). Briefly, the RPM v.1 chip design includes 57 tiled regions allowing resequencing of 29.7 kb of sequences from 27 respiratory pathogens and biothreat agents. These were selected based upon clinical relevance for the population of immediate interest (United States military recruit in training) (19–21). Partial sequences from the genes containing diagnostic regions were tiled for the detection of these pathogens. The experimental microarray data used in the present analysis were obtained using a variety of purified nucleic acid templates and clinical samples culture (throat swabs and nasal washes) using random and multiplexed RT–PCR amplification schemes (for more detail description of amplification methods see Supplementary Data). Resequencing microarrays provide base call resolution by comparing the intensities between a set of four 25mer probes that differ from each other at the same position (13th base). An amplicon or target sequence is represented by numerous overlapping probe sets. GCOS™ software v1.3 (Affymetrix Inc., Santa Clara, CA) was used to align and scan hybridized microarrays to determine the intensity of each probe in every probe set. Base calls were made based on the intensity data of each probe set using GDAS v3.0.2.8 software (Affymetrix Inc.) which used an implementation of the ABACUS algorithm (5). The sequences were represented in FASTA format for later analysis steps.', 'In this paper, target pathogens are the organisms the assay was specifically designed to detect. The sets of probes that represent reference sequence selected from target pathogen genomes are referred to as a Prototype Sequence or ‘ProSeq’ for brevity. The set of resolved bases that result from hybridization of genomic material to a ProSeq is referred to as the hybridized sequence or ‘HybSeq’. The HybSeq is split into possible subsequences or ‘SubSeqs’.', 'The CIBSI 2.0 program implemented in Perl described in this study handled a hierarchy of three tasks (Figure 1): (I) ProSeq identification; (II) ProSeq grouping; and (III) pathogen determination. The most developed and important portion of the algorithm deals with ProSeq identification Task(I) and is handled in three important subtasks: initial filtering of individual HybSeqs into SubSeqs suitable for sequence similarity comparisons (Figure 2), database querying of individual SubSeqs (Figure 3) and taxonomic comparison of BLAST returns for each SubSeq (Figure 4). The NCBI BLAST and taxonomy databases were used for the queries and images were obtained on February 7, 2006. For the ProSeq grouping Task(II), ProSeqs were compared to determine if they supported the same identified organism. In the pathogen determination Task(III), detected organisms were compared to the list of target pathogens the assay was designed for in order to determine if any were positively detected or were possibly related close genetic near neighbors. The level of discrimination that a particular sample supported was automatically determined.', 'An initial filtering algorithm, REPI, was developed previously (16) and the general concepts with revisions were incorporated into the current (automated detection) algorithm used in the CIBSI 2.0 program. Filtering and subsequence selection were used to remove potential biasing caused by reference sequence choice and by other sources (i.e. primers). When PCR amplification was used, microarrays were hybridized in the presence of only primers to determine locations where they resulted in hybridization. Any portions of the ProSeqs that hybridized with the primers were masked as N calls so that the HybSeq did not contain biased information. Normally the primers are designed to be outside the ProSeq region to minimize the interference caused by primers, and so minimize the bases to be masked. There is still the chance that some bases require masking because with the large number of primers used in the multiplex, short stretches of a ProSeq not corresponding to primer locations may still hybridize with the primers. Such regions could be removed from the reference sequences and so not appear on the microarray. However, determining such locations are a difficult and time-consuming task that for most cases is not worth the effort. The first subtask of ProSeq identification Task(I) is noted in Figure 1 and shown schematically in detail in Figure 2. This subtask uses a procedure to examine a HybSeq to find the longest possible subsequence of base calls (SubSeq) that can be submitted as a query to BLAST. It produces a group of SubSeq that contain all portions of a HybSeq that have a chance of producing a limited list of returns from a BLAST query. When a HybSeq has two regions separated by a long stretch of continuous N calls, the relational positioning of the two regions cannot be trusted and so must be sent as separate queries. In addition for shorter subsequences, the number of base calls that must be made is dependent on the length. It was also recognized that for very long sequences a longer WORD size in BLAST may be used. A detailed description of the criteria and process used for each step is contained in Supplementary Data. Upon completion, the algorithm returned to the Task(I) loop and performed the BLAST subtask.', 'The database query subtask performed a batch similarity search of a database using SubSeq as the queries. The BLAST program used was the NCBI Blastall –p blastn version 2.12 with a defined set of parameters. The masking of low complex regions was performed for the seeding phase to speed up the query; however, low complexity repeats were included in the actual scoring. The entire nucleotide database from NCBI acquired on February 7, 2006 was used as the reference database. (Note that earlier images of the database were used during development but all experiments were rerun with the algorithm as described with the image of the database obtained on this date.) The default gap penalty and nucleotide match score were used. The nucleotide mismatch penalty, –q, parameter was set to −1 rather than the default. The results of any BLAST query with an expect value <0.0001 were returned in tabular format from the blastall program. The information about each return (bit score, expect value, mismatches, length of match) was placed in the Return{hash key}{info} hash using the SubSeq identity as the hash key for further analysis.', 'The next subtask of ProSeq identification Task(I) carried out was the determination of SubSeq() states and is shown in Figure 3. The BLAST algorithm gives a ranking score which can be reported as accounting for the size of the database (expect value) or not (bit score). The full taxonomic classifications of every return for a SubSeq were retrieved from the NCBI taxonomy database obtained on February 7, 2006. Using the scores and taxonomy relationships it was possible to find a reduced number of returns that had the best match with the HybSeq. These results were summarized by identifying the taxonomic class to which all the returns belonged to, ‘identified organism’, and a parameter that indicated how they are related to each other, ‘organism uniqueness’. A detailed description of the steps is contained in Supplementary Data.', 'After each SubSeq was examined, the algorithm moved to the next subtask, which was to determine the identified organism of the ProSeq from the SubSeq (Figure 4). The subsequences from the same ProSeq were only allowed to support a single ‘identified organism’ determination. The procedure shown in Figure 4 demonstrates the decision method used to arrive at this determination (detailed description in Supplementary Data). After the subtask covered in Figure 4 was completed, the ProSeq identification Task(I) loop continued until all ProSeqs were examined. A list of ProSeqs that had detected organisms was built up in the Result1 array.', 'After the ProSeq identification Task(I) was completed, ProSeq grouping Task(II) (Figure 1) was used to examine the identified organism values listed in Result1 and grouped them together if they identified the same taxonomic class. Each entry in Result1 was examined and a new entry was created in Result2 if the identified organism did not appear in this list. The entries of Result2 represented the distinct individual organisms identified, but might still contain redundant information. When the ProSeqs were designed to detect the same organism and they all hybridized well, this grouping led to a reduction in redundant information being reported. But, when one ProSeq did not hybridize as well for a variety of possible reasons, multiple entries would appear in Result2 that actually represent hybridization from the same pathogen. This is because there is an alternative cause for the ProSeq hybridizing in this manner. This hybridization could be caused by two different but closely related organisms both being present in a sample and hybridizing to the microarray. Because we have not yet developed methods to distinguish these cases, no further reduction of the list of organisms is made for in ProSeq grouping Task(II) in cases where the level of identification varied on different ProSeq targeted for the same organism.', 'Although it was difficult to relate results from separate ProSeqs to each other, it was desired to have a simple final detection decision be made in pathogen determination Task(III). The first task was specifically implemented so that information about what a ProSeq was intended to detect was not considered and the second task only minimal consideration of this was taken into account. This allowed these initial tasks to be capable of recognizing not only just positive and negative identifications of target pathogens but also cases that were indeterminate. In the final task, the algorithm considered whether the identified organisms belonged to the list of organisms the ProSeqs were designed to detect. The task would group organisms from ProSeq grouping Task(II) together that belonged to or were child classes of the taxonomic class of a target organism. The taxonomic class reported was the common taxonomic group of all the organisms. When all the ProSeqs for a pathogen hybridized well, a fine level discrimination was reported. But if one or more ProSeqs hybridized less well, the reported positive target pathogen was only identified at the level of the less detailed level. This is conservative because methods have not yet been developed to clearly discriminate mixtures of very closely related organisms causing different ProSeqs to hybridize from variable hybridization of a single organism on several ProSeqs. The results of all three tasks were reported and a more experienced user can view ProSeq grouping Task(II) results to clarify some cases. Note that organisms identified in ProSeq grouping Task(II) that only belonged to target pathogens were reported as positives. Clear negative ProSeqs were not mentioned in the output. ProSeqs that were indeterminate or that detected close genetic organisms were never reported as positives. These organisms were instead reported as being detected.', \"Purified Chlamydia pneumoniae nucleic acid samples with 10–1000 genome copies (via method in Lin et al., submitted for publication) were chosen to illustrate how pathogen detection and identification were done when multiple ProSeqs were targeted for the same pathogen. RPM v.1 has three highly conserved ProSeqs selected from the genes encoding for the major outer membrane proteins VD2 and VD4, and the DNA-directed RNA polymerase (rpoB) gene. The HybSeqs from the different samples differed only in the number of unique base calls as shown in Table 1. The percentage of the ProSeq called varied from 80 to 100% except for one case at a concentration of 10 that had only 11% of the rpoB ProSeq producing unique base calls. Because the samples at this concentration are not reproducibly generating the same percentage of base calls, this is probably the detection limit of this ProSeq of the assay. Table 1 listed the determinations made for the SubSeq and at the end of each task for the various samples. The ProSeq from the different cases produced the same number of SubSeqs. These SubSeqs from different samples reported different bit scores for the same top ranked returns from BLAST. In fact VD2 and VD4 produced exactly the same results. The NCBI taxonomy database classified the returns into four distinct groups, which represented the C.pneumoniae taxonomic group and three child strain groups. AE001652, AE002167, AE017159 and BA000008 appeared in the returns of all the ProSeqs for each sample, since they represented database entries of completely sequenced genomes. One rpoB SubSeq produced for its organism uniqueness, SeqUniqu. All other SubSeqs were TaxAmbig as multiple returns from different taxonomic classes were returned. Since the VD2 and VD4 ProSeq each have a single SubSeq, Task(I) assigned the ProSeq the state of the SubSeq. For the rpoB ProSeq, the bit score of one SubSeq was large enough that the algorithm assigned that SubSeq's identification to the ProSeq. Task II of the algorithm grouped all three ProSeqs together since they all had the same identified organism and TaxAmbig was assigned. The result of Task(III) was positive for target pathogen C.pneumoniae and this decision was straightforward as all the ProSeqs agreed with each other and belonged to the same target pathogen taxonomic class. Although the rpoB ProSeq was SeqUniqu, this was not the final conclusion for Task(II) as the ProSeq that was SeqUniqu was not the child taxonomic group and other ProSeq were TaxAmbig. The three recognized strains scored the same, which indicated that the sequence selected for the ProSeqs was very conserved and would not allow discrimination between the strains.\", 'Influenza and Human Adenovirus (HAdV) were the only pathogens that had ProSeq selected that would permit detailed strain level discrimination as discussed in previous work (15,16). This previous work using manual analysis found that the microarray results were in excellent agreement with the conventional sequencing results for clinical samples. A few of the results of running the CIBSI 2.0 program using the updated NCBI database on the raw microarray results are presented in Table 2 (the results for all samples used in the previous work are presented in Supplementary Table A). The identified organisms were not identical to the original findings due to the difference in database used and because all ProSeqs were considered rather than only the Flu A and B hemagglutinin. In fact, the conventional sequencing results that were submitted to NCBI from that work were found for every sample to be among the returns with the best score for the hemagglutinin ProSeq (Supplementary Table B). It should be noted that the previous work based its analysis upon only the results of the hemagglutinin ProSeq. For 8 of 13 Influenza A and 3 of 12 Influenza B cases, the results of ProSeq identification Task(I) and ProSeq grouping Task(II) found that the conventional sequencing was the single best return for the hemagglutinin ProSeq. Owing to the large number of isolate sequences in the database for the hemagglutinin gene it was not surprising that in some cases a single unique entry was not found. In each of the remaining five Influenza A samples, the other sequences returned differed by <0.2% from the conventional sequence. The fewer samples with unique isolate identifications for Influenza B were due to an older reference sequence used for the ProSeq, which allowed less hybridization to occur (18). This also meant that when multiple sequences were returned for a sample they represented greater genetic variation, up to 2%. As a result of the current method of making pathogen determination Task(III) level identification, the final organism reported was less specific (H3N2 or Flu B) for every sample than what was reported as possible in ProSeq grouping Task(II). For HAdV samples, the algorithm also reproduced the finer scale discriminations that had been made previously by manual methods (data not shown).', 'The next example of detection for the Mycoplasma pneumoniae pathogen demonstrated a case where there was only a single ProSeq for the target pathogen. A total of 48 test samples were performed using multiplex PCR (via method in Lin et al., submitted for publication) where for 46 of the samples M.pneumoniae organism was spiked into nasal wash with several other pathogens from 100 to 100 000 colony forming units per ml, the remaining 2 samples were purified with nucleic acid from culture stock at a concentration of 1000 genome copies per reaction volume. This ProSeq was also not optimal for fine discrimination because it was selected from a highly conserved region (345 bp) of the cytadhesin P1 gene. In every case taxonomic database entries for M.pneumoniae or its one recognized distinct strain tied for MaxScore (Supplementary Table 3). To better understand these returns, the database sequences were examined and subdivided into three groups of sequences, A, B and C, based on how well they matched the reference sequence used to make the ProSeq. The placement of the database entries into the three groups was determined from a CLUSTAL alignment of the sequences of this gene. This alignment confirmed that the database entries differed significantly more from each other in regions not represented by the ProSeq and contained sufficient variability that would have allowed finer discrimination. Members of Group A exactly matched the ProSeq and could not be distinguished between on the microarray. Similarly, members of group B matched the ProSeq except at the 199th position where the base called was C rather than T. Group C sequences contained a few database entries that were more variable and might be distinguished from other entries within the ProSeq. For the 48 experimental tests of M.pneumoniae, as much as 80% of the ProSeq hybridized for 19 samples, yet only 5 of these samples had an unambiguous base call at the 199th position. When it was unambiguous, it always matched group B sequences. In the cases where an N base call was made at the 199th location, both groups A and B sequences were returned with the same score. Regardless of this, the target pathogen positively identified was M.pneumoniae for every sample tested.', 'These examples showed how decisions were made independent of whether single or multiple ProSeqs were dedicated to a target pathogen. They also illustrated that the level of discrimination possible was strongly determined by the quality of the selected ProSeq. It is possible that for some pathogens fine level discrimination is not required and the currently tested selections on RPM v.1 would provide satisfactory information. The CIBSI 2.0 algorithm demonstrated its capability to automatically report the maximum level of discrimination that could be supported by the HybSeq information.', 'To demonstrate how the algorithm handled closely related genetic species, a sample of a non-targeted pathogen was considered using multiplex PCR (via method in Lin et al., submitted for publication). For Variola major virus, one of the biothreat pathogens on the RPMv.1, the validation runs demonstrated that Variola major virus purified DNA templates of plasmids were always positively identified when detected (Table 3). Table 4 shows the results when purified Vaccinia genomic DNA was spiked into nasal washes and processed at various concentrations using multiplex PCR. The array has two ProSeqs from hemagglutinin (VMVHA, ∼500 bp) and cytokine response modifier B (VMVcrmB, ∼300 bp) genes for Variola major virus detection. The percentage of the ProSeq that hybridizes is sufficient that if hybridization patterns were only considered one might assume that this tile is identifying the presence of its target. This would indicate that reference sequence selected was not the best choice. However, when our algorithm was applied none of the samples is in fact identified as Variola major or minor virus. Vaccinia was always one of the Orthopoxvirus species listed with the highest scores for VMVcrmB ProSeq, but in only seven cases was it uniquely identified as the probable species detected. In only one sample at the lowest concentration and fraction of VMVcrmB hybridizing, did this ProSeq even identify Variola major and minor virus as one among the Orthopoxvirus species that could be the cause of the hybridization. The lower limit of detection for the amplification method used was between this concentration and the one above it for Variola major itself. The VMVHA ProSeq exhibited much lower sensitivity and made identifications of Orthopoxvirus species in only two experiments and Variola major virus was listed as one of the tied best scoring returns. In both cases, VMVcrmB ProSeq specifically identified Vaccinia virus as the best match. The percentage of the hybridized ProSeq correlated with concentration of the sample.', 'This example demonstrated the importance of the filtering portion of the algorithm by considering the HybSeqs of the ProSeqs for the H1N1 neuraminidase (NA1) and matrix genes from human Influenza A/Puerto Rico/8/34 (H1N1) strain. Filtering was necessary because sending the HybSeq of a ProSeq to BLAST in a single query can bias the scores against strains that have insertions or deletions relative to the ProSeq, especially when using BLAST parameters that maximized the use of base calls. The sliding window test was the portion of the algorithm that controlled filtering. If filtering were turned off, the entire HybSeq would be used in a single subsequence for two influenza ProSeqs that showed significant hybridization. A/Weiss/43 (H1N1) strain was identified as the most likely strain from the HybSeq of the NA1 ProSeq while the HybSeq of the matrix ProSeq correctly identified A/Puerto Rico/8/34. To better understand the source of biasing, CLUSTAL alignment of the NA1 gene of the two strains and the reference sequence used to make the ProSeq are shown in Figure 5. The two strains showed 95% identity (67 mismatches in 1362 aligned bases); however, there was a stretch of 45 bases inserted in both A/Weiss/43 and the NA1 ProSeq compared to A/Puerto Rico/8/34. With the default filtering on, the NA1 ProSeq was split into five SubSeqs as the algorithm encountered large stretches of no calls. In ProSeq identification Task(I), the algorithm determined that three of the SubSeqs, which were shorter, had an identified organism of H1N1 as several isolates including A/Puerto Rico/8/34 tied with the best score, while the two other SubSeqs had an identified organism of only the A/Puerto Rico/8/34 strain as being the closest match. The organism identified by the NA1 ProSeq was A/Puerto Rico/8/34 because one of the SubSeqs had a much higher score. This ProSeq supported the same strain identification made in the matrix ProSeq. The organism identified was A/Puerto Rico/8/34 because two ProSeqs detected only that organism. The correct target pathogen was detected with filtering while without filtering the target pathogen level of identification would have been Influenza A (H1N1 subtype) because two organisms were detected, A/Puerto Rico/8/34 and A/Weiss/43. Splitting the HybSeq into SubSeqs to remove biasing may reduce the level of identification as occurred for the three of the five SubSeqs in this case. The previous example for Vaccinia was another instance where the wrong species (Camel Pox or Callithrix jacchus) identification would occur if the filtering was not used. Clinical samples in Table 2 showed that HybSeqs split into multiple SubSeqs were capable of very specific identification.', 'To illustrate the other filtering performed, when multiplex strategies rather than generic were used for amplification Figure 5 also contains the raw and mask filtered results of this region for the A/Puerto Rico/8/34 sample. It was necessary to perform additional filtering to remove potential biasing caused by the specific primers as described in the methods. In the case shown in Figure 5, a sequence of 18 bases present in the raw result are made N after filtering since they are in a location that interacts with the primers. If these base calls were included in the subsequences constructed, even though the HybSeq would still be split into the same number of SubSeq, the query for the ProSeq would favor an incorrect strain.', 'The algorithm we have developed successfully provided pathogen identification to the maximum level of detail possible (species or strain) depending on the quality of each ProSeq. This identification capability requires minimal input on the identity of the pathogens, making non-expert use feasible. The crucial feature incorporated that allowed complete automation was the use of taxonomic databases, which classify organisms into ordered groups and provide relationships between organism entries, allowing removal of redundancies, comparison of different related prototype sequences and simplification of data presentation. This allows databases, i.e. NCBI, that are redundant and subject to minimal curation but which constantly receive updated and new sequence information to be used with great success. Although we have demonstrated this using only the NCBI databases, other databases or custom made ones could have easily been used, which might improve performance. The algorithm is capable of providing accurate identifications at all analysis levels for pathogens that are less variable or are represented by highly conserved ProSeqs. For more variable or rapidly mutating pathogens, e.g. Influenza A virus, ProSeq identification Task(I) and ProSeq grouping Task(II) still provided accurate detailed identifications, but the pathogen determination Task(III) was unable to report fine scale discrimination. The comparison of the conventionally sequenced Influenza virus gene sequences illustrated that the algorithm is capable of automatically adjusting for updates in databases. The algorithm demonstrated its capability to properly distinguish hybridizations on a ProSeq caused by the specified pathogen from those caused by genetically close (near neighbor) strains and did not make incorrect identifications, eliminating one potential cause of false positives. Filtering the raw hybridization results served to reduce the computation time, accounted for potential primer interference and more importantly reduced potential biasing. This simple integrated algorithm provided sufficient and accurate identification, so that immediate use of the RPM v.1 or similar resequencing arrays and assay is possible. Although not discussed in this paper, the algorithm has successfully detected the presence of simulated multi-infections (Lin et al., submitted for publication). The algorithm as currently developed will detect mixtures when the organisms have sufficient variation; however, detection of a mixture of an organism and its mutation strain in a sample is uncertain in its present phase. In principle it may be possible to detect such mixtures as the resequencing microarray can detect and sequence diploid organisms.', 'Besides demonstrating the success of the CIBSI 2.0 program, the work involved in developing the algorithm allowed insight into the importance of proper ProSeq selection. The RPM v.1 was the first resequencing array designed specifically for multiple pathogen detection using database similarity searching and served as a prototype for this application. We have demonstrated that a single ProSeq with as few as 100 bp, when designed correctly, can be sufficient to unambiguously identify an organism. However, it is clearly indicated that several longer ProSeqs provide better confirmation and more detailed information of a pathogen. Although the algorithm provides accuracy equivalent to manual analysis for determinations of individual ProSeqs, the current algorithm is only partially successful in integrating information from multiple ProSeqs. The emphasis of the design to this point has been on capabilities that are generally applicable to any pathogen. We are incorporating these insights in our newer more comprehensive resequencing array designs. Improving on level of detail reported in pathogen determination Task(III) will require more information about an individual pathogen and may have to be developed for each specific pathogen or class of pathogens. This information is also required for the algorithm to identify which differences between a sample and database entries represent significant mutations. Future work will involve improving the use of the current taxonomic database or potentially developing a new relational database that is specific to our needs and then incorporating more specific information of target pathogens. The hierarchal design of the data analysis makes it easy to incorporate analysis that build upon the analysis already performed.', 'We have met with some success in the current version but want to have increased automated discrimination. We have a well-defined path to completing this aim. The use of properly designed resequencing microarrays and this automated detection algorithm provides a way forward to developing assays that can test for multiple organisms simultaneously while providing fine strain level discrimination giving access to information about detailed strain recognition, antibiotic resistance markers and pathogenicity. This is a capability that other approaches cannot currently provide. In addition, since the design of the original 30 kb RPM microarray, the possible sequence content of the current array has increased 10-fold to 300 kb and further increases in array density are still attainable. This, coupled with our identification algorithms, will allow the analysis of partial sequence information from even more organisms for applications such as differential diagnostics for illnesses with multiple potential causes (i.e. febrile respiratory illness), tracking of emergent pathogens, distinction of biological threats from harmless near genetic neighbors in surveillance applications and for tracking the impact of co-infections or super infections. The concept of categorizing and reporting different degrees of identification depending on the quality of samples and set of target sequences is not limited to resequencing microarrays but is more generally applicable to any platform that is capable of returning sequence level calls that can be used to query a reference DNA database. As the trend for assays that test for multiple pathogens increases, automated analysis tools, such as this one, become more crucial for rapid identification in simple formats useful to the non-expert on a day to day basis. The remaining hurdle to using resequencing microarrays as a routine assay method is now clearly the sample processing methods. Further automating these steps is an important area of future research and development.', 'The program can be obtained free of charge for research purposes by contacting the authors.', 'Supplementary Data are available at NAR Online.'], 'cord_uid': 'cxzlmfst'}\n{'title': 'A DNA biochip for on-the-spot multiplexed pathogen identification', 'text': [\"Decentralized medical testing plays a vital role in today's health care system. The blood glucose meter, which was the first commercial handheld device for medical diagnostics and developed three decades ago, is by far one of the most successful examples in point-of-care testing (POCT). In the years to come, driven by the ever-growing threats from emerging infectious diseases (e.g. avian flu and severe acute respiratory syndrome), the development of small-size instruments for on-the-spot pathogen detection is expected to be an important segment of the POCT market. This trend has already commenced with a few companies having launched self-test products for hepatitis and human immunodeficiency virus detection based on antibody–antigen interactions (e.g. OraSure Technologies and ACON Laboratories). These systems give a visual readout indicating the presence or absence of the target virus in ∼15 min. One of the main shortcomings of these immunological techniques is their limited sensitivity. To address this issue, there have been significant efforts to develop nucleic acid (NA)-based analyzers (1–3). The miniaturization of NA analytical platforms has many advantages over the conventional bench-top counterparts. These include low sample/reagent consumption (volume of micro- down to picoliter) as well as short assay time (minutes rather than days). Most importantly, they permit the integration of a number of functions including sample preparation, target amplification, and product detection, thus enabling a fully automated operation that can be used by untrained individuals.\\nTo date, several integrated NA-based analytical systems have been commercialized [ (GeneXpert from Cepheid),  (Direct Tube Sampling Systems from Gen-Probe),  (RAZOR from Idaho Technology) and  (Liat Analyzer from IQuum)]. Despite their wide use in clinical/central laboratories, their application for on-site pathogen monitoring on a routine basis is still limited due to the large footprint and high instrument cost (mainly the complex optics). A promising alternative would be the inherently simple and lost-cost electrochemical method. Over the past decade, despite a great deal of work having been carried out on electrochemical sequence-specific NA sensing (4–6), little work has been undertaken on the integration of these with upstream functionalities. In 2004, Liu et al. (7) successfully demonstrated a fully integrated biochip for cell isolation and lysis, target amplification, as well as electrochemical amplicon detection. One relevant feature of their approach is the incorporation of on-chip mixers, valves, and pumps in a self-contained device. However, the design and fabrication of the chip involves many complicated steps, which limits its practical application.\", 'In a previous study, our group demonstrated a proof-of-concept experiment that both DNA amplification by the PCR and sequence-specific electrochemical amplicon detection could be done in a single microchamber (8), in contrast to the commonly used device which involved multi-chambers with complex microfluidic control elements (9). This microdevice had an 8 μL reaction chamber etched in a silicon substrate with a thin-film heater and temperature sensor patterned on top for rapid thermal cycling. An oligonucleotide capture probe-modified detection electrode was placed on a glass substrate used to seal the microchamber. To develop this prototype device into practical use, sample preparation functionality as well as the ability to perform multiplexed analysis would need to be addressed.', 'In this work, we present a complete DNA-based assay in a single silicon–glass microchamber for multiple pathogen detection. A model system of E.coli and B.subtilis was used. The assay involves the following steps: (i) sample preparation using thermal cell lysis and magnetic particle-based target genome isolation; (ii) target DNA amplification by the PCR; (iii) hybridization of the amplicons to their complementary oligonucleotide capture probes immobilized onto individual detection electrode surfaces and (iv) electrochemical transduction of the recognition event via gold nanoparticles with signal amplification using electrocatalytic silver deposition (10). An issue of importance much needed to be addressed was the compatibility of all the materials and processing steps. In particular, the chemistry used for the site-specific probe immobilization together with that of the magnetic particles used for genome isolation should be PCR-compatible.', 'Oligonucleotides and PCR reagents were obtained from Invitrogen (Carlsbad, CA, USA), unless otherwise stated. Other chemicals were purchased from Sigma–Aldrich (St. Louis, MO, USA). Electrochemical measurements were performed using a VMP multichannel potentiostat (Princeton Applied Research, Oak Ridge, TN, USA) controlled by EC-Lab software (version 6.70, Bio-Logic Science Instruments, Claix, France). The thermal control system for the PCR consisted of a data acquisition card (PCI-MIO-16E-1, National Instruments, Austin, TX, USA) along with a signal conditioning board (SC-2042-RTD, National Instruments) connected to the temperature sensors. A digital feedback proportional-integral-derivative (PID) control algorithm was implemented in LabVIEW software (National Instruments) to control voltage supply to the heater from a power source (HP6629A, Hewlett-Packard, Rockville, MD, USA).', \"The silicon chip (thickness of 400 μm) had two fluid injection holes (top side, diameter of 500 μm, depth of 100 μm) and a chamber (bottom side, length and width of 5 mm, depth of 325 μm) etched by the inductively coupled plasma/deep reactive ion etching (ICP/DRIE) process, see Figure 1A (left panels). Thin-film platinum (100 nm) was patterned on top of the silicon substrate as heater and temperature sensors (Figure 1A, upper left). The glass chip had platinum pseudo-reference and counter electrodes (thickness of 100 nm) as well as four working electrodes made of indium tin oxide (ITO) (thickness of 100 nm), see right panel of Figure 1A. Ultra-violet curing optical cement (Type UV-69, Summers Optical, Hatfield, PA, USA) was used to bond the silicon and glass chips, the curing procedure was in accordance with the manufacturer's instruction. Pipet tips were glued to the fluid injection holes with epoxy (Figure 1B).\", 'An oligonucleotide capture probe specific for E.coli amplicon [pyrrole-5′-ACAACACGTTTAGCCTGACC-3′ (pyrrole-EC), Apibio, France] was first electrochemically polymerized onto two of the four ITO working electrodes. A mixture of 60 mM pyrrole, 20 μM pyrrole-EC, and 0.1 M LiClO4 was introduced into the microchamber, followed by a cyclic voltammetric scan of the two electrodes between −0.5 and +0.65 V at a scan rate of 50 mV/s for three times. The microchamber was then washed with deionized water and dried with nitrogen gas. The same procedure was repeated for the other two ITO working electrodes, except an oligonucleotide capture probe specific for B.subtilis amplicon [pyrrole-5′-CCTACGGGAGGCAGCAG-3′ (pyrrole-BS), Apibio, France] was used.', 'Ten microliters of avidin-coated magnetic particles (3.0 μm, VMS-30-10, Spherotech, Libertyville, IL, USA) were washed with an equal volume of saline/sodium citrate buffer (SSC, 150 mM NaCl/15 mM sodium citrate, pH 7.0). After centrifugation and pipetting, the supernatant was removed and the magnetic particles were incubated with 10 μL of 10 nM biotinylated genome capture probe overnight at room temperature. Oligonucleotide sequence of the genome capture probe for E.coli was 5′-biotin-GACAAGAAAATCTCCAACATCC-3′ while that for B.subtilis was 5′-biotin-CCAGTTTCCAATGACCCTCCCC-3′. The capture probe functionalized magnetic particles were finally washed with 10 μL of the SSC buffer, resuspended in 10 μL of the SSC buffer, and stored at 4°C.', \"The sample containing E.coli or B.subtilis or both (1 μL), which was cultured in Luria–Bertaini broth overnight at 37°C, was mixed with 1 μL each of the biotinylated genome capture probe (1 nM) for E.coli and B.subtilis along with 7 μL of the SSC buffer. The mixture was injected into the reaction chamber and sealed with Bostik's Blu-Tack. Then, the silicon–glass device was placed in a plexiglass holder with contact pins for electrical connections to the heater and temperature sensors (Figure 1C). The chamber was maintained at 90°C for 5 min to lyse the cell and at the same time denature the genomic DNAs. After that, the temperature was cooled to 50°C and held for 10 min to allow specific hybridization between the denatured genomic DNAs and capture probes. Subsequently, 1 μL of the functionalized magnetic particles were added into the chamber and incubated for 10 min to capture the specific genomes onto the magnetic particles. Finally, SSC buffer was used to remove any unwanted materials with an external magnet to keep the particles within the microchamber.\", 'The PCR master mix consisted of 1× buffer (20 mM Tris−HCl, pH 8.4, 50 mM KCl), 2 mM MgCl2, 0.2 mM dNTPs, 200 nM of biotinylated E.coli forward primer (5′-biotin-GACAAGAAAATCTCCAACATCC-3′), 2 nM of E.coli reverse primer (5′-ACAACACGTTTAGCCTGACC-3′), 2 nM of B.subtilis forward primer (5′-CCTACGGGAGGCAGCAG-3′), 200 nM of B.subtilis biotinylated reverse primer (5′-biotin-CCAGTTTCCAATGACCCTCCCC-3′), 0.5 μg/μL BSA, and 0.4 U/μL Taq polymerase. The mixture was pipetted into the microchamber and subjected to the following thermal cycling profile: initial denaturation at 94°C for 5 min; 40 cycles at 94°C for 30 s, at 60°C for 30 s, 72°C for 30 s; and final extension at 72°C for 5 min.', 'After the asymmetric PCR, the solution was allowed to stand at room temperature for 1 h. Unhybridized amplicons were washed away with the SSC buffer. Gold nanoparticle label was bound to the hybridized amplicons by exposing the electrode to a streptavidin−gold nanoparticle (5 nm) solution (the stock was diluted 10 times with 0.05 M 4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid/0.2 M NaCl) for 30 min at room temperature. The unbound gold nanoparticles were removed by flushing the microchamber with phosphate-buffered nitrate solution (0.3 M NaNO3/10 mM sodium phosphate, pH 7.0). Electrocatalytic silver deposition onto the hybrid-bound gold nanoparticles was then achieved by applying a potential of −0.18 V in a silver nitrate solution (1 mM AgNO3/1 M KNO3) for 20 s. Finally, the amount of deposited silver was determined by measuring the oxidative silver dissolution response with an applied anodic current of 10 μA in the same silver nitrate solution, and the time to reach a potential of +0.65 V was taken as the signal.', 'The single microchamber design poses particular challenges to the electrochemical platform used for the sequence-specific PCR amplicons detection. Addressability and compatibility are two important considerations regarding immobilization chemistry for the oligonucleotide detection capture probes. For a multiplexed assay, it is necessary to individually modify the detection platform so that each individual electrode in an electrode array has a specific capture probe. When using either high temperature or ultra-violet glue to seal the microchamber, it is recommended that immobilization should be carried out after the silicon−glass bonding process so as to prevent damage to the capture probes. In doing so, the more common chemical attachment (spotting) method cannot be used because all the active electrode surfaces are embedded within the same microchamber and they would receive identical modifications. One simple way to achieve site-specific probe immobilization onto individual electrode surfaces can be achieved by electrochemical copolymerization of pyrrole and pyrrole−oligonucleotide (11). Figure 2 illustrates the strategy to immobilize different capture probes onto each individual electrode. A solution of pyrrole and oligonucleotide 1 bearing a pyrrole group is introduced into the microchamber. When a cyclic voltammetric scan is applied to electrode 1, with other electrodes disconnected or grounded, oligonucleotide 1 is selectively deposited on this particular electrode. Then, the microchamber is washed with water to ensure there is no pyrrole−oligonucleotide 1 monomer is left. This procedure is repeated for the other electrodes with different pyrrole−oligonucleotide polymerization solutions. In our model system with two target analytes and four working electrodes, the capture probes specific to E.coli and B.subtilis are immobilized in duplicate. Before proceeding to the complete analytical protocol, the ability of these immobilized capture probes to recognize their complementary targets should be tested. Figure 3 shows the fluorescence images of the four functionalized electrodes (A and D: B.subtilis probe; B and C: E.coli probe) exposed to a sample containing a fluorescently-labeled sequence complementary to the E.coli probe. It is clear that electrodes B and C exhibit much higher fluorescence intensity than electrodes A and D, indicating the highly specific probe immobilization as well as hybridization offered by the electrochemical pyrrole-based attachment chemistry. Another criterion for the selection of immobilization method is the compatibility with other processes, in particular the PCR. Due to the fact that the detection electrodes are within the reaction chamber, the linkage between the immobilized capture probe and electrode surface must be strong enough to survive through the thermal cycling process (especially the high denaturation temperature). Moreover, the detector surface should interact only with the specific amplicon but not with other components employed in the assay protocol.', 'The assay procedure used in this work is schematically represented in Figure 4. It involves three main steps: sample preparation, target DNA amplification, and product detection, all performed within the same microchamber. Intact cells are first broken down by applying a high temperature (90°C, controlled by the on-chip heater and temperature sensor) to free the genomic DNA. To remove all the interfering substances (e.g. cell debris and protein) that may affect the subsequent DNA amplification process, magnetic particles are used to isolate the specific genomes. Biotinylated genome capture probes for the two model species are mixed with the intact cells before injecting into the microchamber. When the temperature is lowered to 50°C after the thermal lysis step, these probes hybridize to their complementary target genomes. These probe−genome hybrids are then isolated by the addition of the avidin-coated magnetic particles, followed by thorough washing. It is worth noting that the magnetic particles are pretreated with a small amount of the genome capture probes to minimize nonspecific adsorption of the interfering substances and other genomic DNAs. Subsequently, with the genomes captured on the magnetic particles serving as the template, asymmetric PCR is conducted to generate single-stranded rich target amplicons. After the amplification step, these amplicons hybridize to their corresponding detection electrodes. Next, the hybridized amplicons are labeled with gold nanoparticles via biotin–avidin interaction. Finally, silver metal is electrocatalytically deposited onto the gold nanoparticles and the amount is determined by the electrochemical oxidative dissolution technique (10,12). The detailed procedure for the sample preparation, target DNA amplification, and product detection steps is given in the Materials and Methods section.', 'The ability of this single microchamber to detect specific cell type using the above protocol is demonstrated by running a series of experiments with E.coli cells of different concentrations, taking the signal from the B.subtilis detection capture probe-modified electrode as the background. Figure 5 gives a semi-log plot of the sample to background ratio against the number of cells in the sample. A linear relationship is obtained in the concentration range investigated (102–105 cells/sample). This result confirms the successful isolation of the genome with the magnetic particle-based approach, compatibility of the magnetic particle with the PCR, thermal stability of the immobilized detection capture probe through the temperature cycling process, as well as negligible nonspecific adsorption on the electrode surface.', 'Another attractive feature of this microdevice is multiplexing. By constructing an electrode array, it is possible to identify several species in a single run. The results for the detection of the two model species are presented in Figure 6. If the sample contains E.coli cells only, there is a significant increase in the analytical signal (silver metal stripping time) for the E.coli detection capture probe-functionalized electrode while that for the B.subtilis remains the same as the background signal. When the sample contains B.subtilis cells only, opposite results from the two different electrodes are obtained. Another case is the inclusion of both cell types; not surprisingly, both electrodes have much higher signals than the background.', 'We have demonstrated the utilization of a silicon−glass-based microchamber for DNA-based detection of E.coli and B.subtilis cells. Thermal cell lysis, magnetic particle-based target genome isolation, DNA amplification, and electrochemical sequence-specific amplicons detection have been successfully implemented in this microdevice platform. The portable electrochemical instrumentation as well as a simple microchip design is conducive to the realization of on-site pathogen detection. The selective immobilization of capture probes using the pyrrole-based electropolymerization process provides good thermal stability and PCR process compatibility, which are crucial for the multiplexed analysis. Future work will be directed towards interfacing the microchip with macro-world so as to achieve a fully automatic device that can be used by untrained individuals.'], 'cord_uid': '0sg0hv9w'}\n{'title': 'Molecular dynamics simulations of human tRNAUUULys,3: the role of modified bases in mRNA recognition', 'text': ['Nonstandard bases are most prevalently found in tRNA molecules and may involve a variety of modifications from simple methylation to addition of a peptide moiety (1,2). Some tRNA molecules require posttranscriptionally modified nucleic acid bases for proper translation (3,4), recognition by proteins (5,6) and initiation of human immunodeficiency virus reverse transcription (7). Specifically, ribosomal binding (8,9), proper translocation from the aminoacyl site (A) to peptidyl site (P site) (10,11) and translational speed (12) may be dependent upon nonstandard bases, while the lack of modified bases may result in frameshifting errors (13). Most tRNAs contain at least one modified base (14) and modifications often occur at positions 34 and 37 (4). Modified bases are generally thought to increase stability of tRNA structure by providing additional hydrogen-bonding contacts, better base–base stacking interactions (4,15–19) and metal ion binding sites (20–22). Specifically, base modifications at the 34th and 37th positions of the tRNA are often important for facilitating cognate codon recognition (4). Still, the role specific modifications play in determining RNA structure, stability and function are not well understood.', 'Functional groups in modified nucleosides can dramatically affect the ability of tRNA anticodons to recognize their cognate mRNA codon and therefore participate in accurate translation of the genetic code (15,16). The third human tRNA coding for lysine (tRNALys,3) contains three posttranscriptionally modified bases in the anticodon stem–loop (ASL, Figure 1). Biochemical assays have established that 5-methoxycarbonylmethyl-2-thiouridine at position 34 (mcm3s2U34) and 2-methylthio-N6-threonylcarbamoyl-adenosine at position 37 (ms2t6A37) are required to mimic wild-type human tRNALys,3 binding to programmed ribosomes (9). Pseudouridine (ψ) presence at position 39 is not critical to anticodon–codon recognition but has been shown to contribute to helical stability (17). Escherichia coli tRNALys,, analogous to its human counterpart, contains a 5-methylaminomethyluridine-2-thiouridine at position 34 (mnm5s2U34) and N6-threonylcarbamoyl-adenosine at position 37 (t6A37) that are both important for ribosomal A site binding (8,9,23), translocation from the A site (11) and discrimination between AAA and AAG codons (24,25).', 'Several structures of human tRNALys,3 and E.coli tRNALys, variants with variously modified bases in a variety of contexts have been determined by both NMR spectroscopy (18,26–29) and X-ray crystallography (19,30). The expensive and often complex synthesis of nonstandard bases further complicates incorporation of all fully modified bases. Structural studies indicate that preservation of a U-turn motif in E.coli anticodon loop is important for recognition of cognate mRNA (28). Systematic, detailed NMR studies of fully modified E.coli and partially modified human tRNALys,3 indicate flexibility in the U36 position (29). In the context of the 30S ribosomal subunit, an anticodon loop containing t6A37 has been found to recognize the cognate mRNA codon (19) in a canonical Fuller–Hodgson tRNA form (31).', 'The contributions of each modified base to free tRNA and ribosomally bound tRNA structures remains elusive (4,16,32). Molecular dynamics (MD) studies considering specific contributions of modified bases mcm5s2U34, ms2t6A37, ψ39 to the structure and function of the human ASLLys,3 are reported. Specifically, the role of modified bases in preserving canonical tRNA anticodon structure is examined.', 'Fully refined NMR structures are currently not available for fully modified tRNALys,3; therefore all simulations were begun from X-ray crystal structures. MD simulations of the tRNAUUULys,3 anticodon stem–loop with bases 27–43 (ASLLys,3 wild type) derived from the fully modified tRNALys,3 X-ray crystal structure solved to 3.3 Å resolution (PDB code 1FIR) (30) were performed with different combinations of mcm5s2U34, ms2t6A37, ψ39 and unmodified bases (Table 1). All variations of ASLLys,3 were derived from the Bénas et al. (30) structure. While this study was being conducted, the crystal structure of E.coli ASLLys,, which contains t6A37, with the 30S ribosomal subunit and mRNA was solved to 3.0 Å resolution (PDB code 1XMQ) (19). In E.coli ASLLys,, the ureido ring of t6A37 stacks with A38 stabilizing anticodon–codon interactions. To ensure adequate sampling of relevant structures, an additional system, wild-type 2, was derived by grafting the E.coli ASLLys, anticodon loop containing an ureido ring in t6A onto the Bénas ASLLys,3 stem and applying the appropriate full-human ASLLys,3 modifications.', 'All MD simulations were performed according to the same protocol. Trajectories were propagated with the Cornell et al. force field (33), as implemented in the SANDER module of the program AMBER 6.0 (34). ASLs were solvated by a 10 Å rectangular box of TIP3P water (35) and Na+ ions were added to neutralize the system. Simulations were performed under periodic boundary conditions, employing the particle mesh Ewald (36) formalism to account for long range electrostatics.', 'MD trajectories were propagated with a 2.0 fs time step, employing the SHAKE algorithm (37) on all hydrogen atoms, with a nonbonded cutoff of 9.0 Å. The pairlist was updated every 25 steps. Constant pressure (1 atm) and temperature (300 K) were maintained according to the Berendsen coupling algorithm (38). The equilibration protocol of Cheatham was followed (39) similar to previous simulations of nucleic acids (40–45). Simulations were allowed to equilibrate under production-run conditions for at least 2.0 ns before collection of statistics over 4.0 ns.', 'Parameters for nonstandard bases were derived to be consistent with the Cornell et al. force field (33) and specific parameters are available in Supplementary Data. Atom types, force constants and equilibrium values were assigned by analogy to the Cornell et al. nucleic acid, threonine side chain (t6) or methionine (ms2) force-field parameters (33), or the General Amber Force Field (46) for sulfur-related parameters. Partial atomic charges were derived from Hartree–Fock (47)/6-31G* (48–50) electrostatic potential charges fitted with a two-step restrained electrostatic potential procedure (51). Electronic structure calculations were performed with Gaussian 03M, revision B.04 (52).', 'Quantum mechanical calculations employing the modified Perdew-Wang 1-parameter model for kinetics (MPW1K) (53,54) and the 6-31+G (d,p) basis set (55,56) were used to determine the rotational barrier around the C10-N11 amide bond involved in ureido ring formation.', 'Trajectories were analyzed using the CARNAL and PTRAJ modules of AMBER 6.0 (34). All statistics were collected on 1 ps snapshots over 4.0 ns of data. Helical parameters were measured using the 3DNA program (57). Trajectories and structures were visualized using VMD (58) and UCSF chimera (59). Criteria for hydrogen bonding was a 3.5 Å distance between heavy atoms and a heavy atom-H-heavy atom angle of 180° ± 60°. Solvent accessible surface area (SASA) was calculated as a Connolly surface (60), as implemented in the MM-PBSA module of AMBER 8.0 (61). When determining if bases were parallel, the angle between normal vectors to the plane of the respective bases was used. Planes were defined using three points; N1, N3 and C5 for pyrimidines and N3, C6 and C8 for purines and the normal vectors were defined to point in the 3′ direction.', 'Trajectory stability was based upon steady fluctuations around constant values for root mean square deviation (r.m.s.d.) values from the average (Figure 2) and starting structures (Supplementary Data) as well as monitoring of helical parameters of the double-stranded region such as local twist, x-displacement from the helical axis and base pair inclination (Supplementary Data).', 'Overall, the helical portion of all ASL structures exhibited classic A-form geometry (62) with x-displacement values ranging from −4.5 ± 0.6 to −5.3 ± 0.9 Å and base pair rise parameters of 2.5 ± 0.3 to 2.8 ± 0.3 Å. The majority of structural analysis occurred in the loop region of the ASLs.', 'The wild-type simulation was unusually stable with an average r.m.s.d. from the starting structure of only 1.8 ± 0.3 Å. To ensure that the wild-type simulation was indeed naturally stable and that periodic effects or lack of sampling time did not influence the results, data for two additional simulations were collected (N. E. McCrate, unpublished data). In the first simulation, the water box surrounding the RNA was increased to 15 Å beyond the solute. In the other simulation, with 10 Å of water added beyond the solute, data were collected for 40 ns. Both additional simulations retained the structure observed in the reported wild-type simulation.', 'All simulations contained a hydrogen bond between C32 O2 or N3 and the A38 exocyclic amine to varying extents of the simulation but averaging 74% occupancy. Simulations containing no modified bases or only ψ39, exhibited an additional hydrogen bond between the exocyclic amine of A37 and the O2 of U33 56 and 62% of the simulations time, respectively. Addition of either mcm5s2U34 and/or ms2t6A37 decreased the A37–U33 interaction to <4% of the simulation time.', 'The side chain of mcm5s2U34 did not form hydrogen bonds often with other parts of the ASL. The only detectable hydrogen bond observed occurred between the mcm5 carbonyl oxygen and the hydroxyl of ms2t6A37 in the wild-type simulation and the simulation containing both mcm5s2U34 and ms2t6A37 for <20% of the simulation time.', 'In the models presented herein there seems to be no correlation between presence of a U33 NH3 hydrogen bond to U36 O2P and any one conformation. Simulations containing mcm5s2U34 seem to lose a U33 NH3 hydrogen bond with U36 O2P (<3%) while simulations lacking a modification at position 34 close to form this hydrogen bond >90% of the simulation time. The exceptions to this trend are both wild-type simulations, which contain this hydrogen bond 100 and 91% for wild-type and wild-type 2 simulations, respectively.', 'In the fully modified ASL structure including mcm5s2U34, ms2t6A37 and ψ39 (wild-type ASLLys,3), the structure retained a stair-stepped conformation, in which the uridines (Figure 3) are offset and nearly parallel to each other, observed previously in the X-ray crystal structures of free human tRNALys,3 (30) and when E.coli t6A ASLLys, pairs with its cognate mRNA codon in the context of the ribosome (19).', \"To characterize the stair-stepped conformation, the relative positions of the anticodon bases to each other were determined. To analyze the positions of the anticodon bases relative to each other, the distances between the center of mass (COM) of anticodon bases' aromatic rings were measured. The distance vectors were transformed into a relative coordinate system utilizing a rotation matrix and decomposed into x (dx), y (dy) and z (dz) components, to describe the relative positions of the bases. By analogy, the dx, dy and dz components describe the height width and length of ‘the stair’. From visual inspection, the wild-type simulation was determined to exhibit the prototypical stair-stepped conformation. To determine whether the geometry of the anticodon for each model matched that of the wild-type, dx, dy and dz between each set of rings (U34–U35 and U35–U36) were classified as ‘stair stepped’ if the values for both pairs of rings fell within 4 SDs of the average values calculated for the wild-type model and the angle between the normal vectors of the planes of the bases was 0 ± 45° (63) (Table 1).\", 'To aid in data analysis, snapshots of each simulation were assessed for presence of a stair-stepped conformation. Based upon the criteria outlined above, the percent occupancies of stair-stepped interactions for positions 34–36 were determined (Table 2). Simulations that contained an ms2t6 modification at position 37 exhibited the stair-stepped anticodon configuration 75–100% of the simulation time, while those ASLs lacking base modifications at position 37 were not stair-stepped for most, if not all, of the simulation. In the wild-type 2 model, in which the anticodon appears to be stair-stepped, the U36 base is positioned closer to the N9 of ms2t6A37 than is observed in the other models. As a result of this difference in position, the wild-type 2 simulation is classified as having a less stair-stepped codon (46%) than the other models because it is stair-stepped in a slightly different conformation.', 'When ms2t6A37 is substituted with A37, U36 rotates around the glycosidic bond and is exposed to solvent. This was observed in all simulations lacking ms2t6 modifications at position 37. Quantification of U36 solvent exposure through the SASA of the U36 base indicates that the presence of the ms2t6 modification at position 37 reduces the SASA of U36 (Table 2). Relatively smaller U36 SASA values correspond to the stair-stepped conformation, while larger SASA values correspond to U36 unstacking. In addition, mcm5s2U at position 34 appears to slightly lower the SASA of U36 as all models lacking mcm5s2 modifications have slightly higher U36 SASA than their modified counterparts (10–30 Å2, Table 1).', 'From visual inspection of the wild-type simulations, it seemed that bases 36–38 were interacting, perhaps by base–base stacking interactions. All simulations were examined to determine the extent of base–base stacking in the anticodon loop. A stacking interaction was defined to occur when the distance measured from the COM between two aromatic rings was <4 Å and the angle between the normal vectors of planes formed by the bases was ±20°, as determined appropriate by previous studies (63,64).', 'Based on the stacking criteria outlined above, the percent occupancy of stacking interactions for positions 36–38 was determined (Table 2). In all simulations containing ms2t6A37, for a portion of the simulation (3–34%), U36 was in close proximity to the A37 5-membered ring. The stacking data indicate that the presence of ms2t6A37 may slightly enhance base–base stacking between U36 and position 37.', 'Modification of A37 encourages the base to remain intercalated between A38 and U36. In general, A37, modified or unmodified, hydrogen bonds with functional groups in the anticodon loop. Simulations containing ms2t6A37 tended to interact across the loop with base and sugar atoms in residues 32–35 for relatively short lifetimes. For instance, in the wild-type trajectory, the hydroxyl group from t6 was hydrogen bonded for most of the simulation time to some group in the tRNA. It preferred to interact with sugar oxygens in residue 33 but also hydrogen bonded with base oxygens in U34 and U35 carbonyls. When ms2t6 modifications were removed, the A37 disrupted the U36–U35 interaction and formed new h-bonds through N6 with bases in closer proximity, such as residues U35, U36 and A38. In the absence of ms2t6, the ASL rearranges to provide new hydrogen-bonding partners for the truncated N6 position of A37.', 'In the context of the ribosome, E.coli t6A37 contains an ureido ring which stacks with the 6-membered ring of A38 (19). The crystal structure of t6A alone contains an ureido ring (65). In the fully refined tRNALys3 X-ray structure (PDB code 1FIR), ms2t6A37 seems to adopt a conformation without an ureido ring although initial attempts were to try to have a clearer electron density showing an ureido ring-like conformation [P. Bénas, personal communication and figures in Ref. (30)]. Quantum mechanical calculations on N9-methylated ms2t6A at the MPW1K/6-31+G (d,p) level of theory indicate that the barrier to rotation around the C10-N11 bond to form an ureido ring is ∼9.0 kcal/mol at room temperature.', 'When ψ39 is present, a water bridge forms between A38 O2P, ψ39 NH3 and ψ39 O1P. Although the full water bridge is only present for ∼30% of the production run (detailed tables in Supplementary Data), the component hydrogen bonds between ψ39 NH3-water-A38 O2P and ψ39 NH3-water-ψ39 O1P are present for larger percentages of the production run, typically 50–80%. The waters involved in the water bridge are short lived with the bulk of the bridging interaction being attributable to water molecules having lifetimes at or below 40 ps and nearly all residence lifetimes being attributable to waters with lifetimes ≤120 ps.', 'MD simulations with the Cornell et al. force field (33) and TIP3P (35) water have been shown to identify long-lived, structural water molecules in other systems (41,44). The water molecule that coordinates with ψ39 is present most of the time but is not long lived. Pseudouridine has been shown to stabilize tRNA structure through a water-mediated hydrogen-bonding network (67). The crystal structure of yeast tRNAPhe, which contains a ψ55, has a water molecule bridging the N1-H position to the neighboring phosphate oxygens of T54 and ψ55 (68). Additionally, a long-lived water molecule coordinated to ψ has been found in tRNAAsp through MDs simulations (41). This water-mediated hydrogen bond with ψ is believed to restrict ψ motion and stabilize A–ψ base pairs (41,69,70), which are most prevalently found at positions 31–39 (14).', 'Overall the wild-type simulation retained the stair-stepped conformation with dx, dy and dz values all similar to that found in the X-ray crystal structure of human tRNALys,3 (30) and E.coli t6A-ASL bound in the 30S ribosome (19). Modifications at position 37 stabilize the anticodon conformation. From this data it appears that the presence of ms2t6A37 encourages retention of a highly ordered stair-stepped conformation.', 'Base modifications at position 37 seem to stabilize U36 participation in the stair-stepped conformation. U36 is the first base of the anticodon and is required for proper reading of the cognate codon base. The ms2t6A37 base is required for proper positioning of U36 to recognize its cognate codon base. Uridine with its hydrophilic carbonyls and N3, in general, is predisposed to an extra-helical position resulting in an increase in solvent exposure as noted in a systematic study of RNAs containing a single-base bulge (71). The highly ordered stair-stepped conformation of the anticodon bases seems to be important to codon recognition as originally hypothesized (31) and observed in crystal structures of tRNAs alone (72), interacting with other tRNAs through crystal packing effects (30,73–75), and in the context of the ribosome (19,76).', 'Base–base stacking data indicate that modifications on A37 may position the base to better interact with U36. However, this stacking interaction is not long lived in simulations. In order to have favorable base–base stacking interactions, the interaction of two hydrophobic bases must be enthalpically favorable (77,78). In this case, the negative electron densities of the aromatic bases may repel each other and electrostatic interactions, in which positive portions of the base above are interacting with the negative electron density of the base below, may predominate. This favorable electrostatic interaction has been observed in simple systems, such as benzene and water. In these benzene–water pairs, the partially positive hydrogens of the water molecules are interacting with the negative π-cloud of the benzene ring (79). It may be more favorable for a canonical anticodon to be stair-stepped because the positive portions of bases are aligned to interact with the negative π-cloud of the neighboring, lower base. When A37 is modified, it can partake in hydrogen bonds across the anticodon loop, effectively holding it up above U36. When modifications are removed, the base no longer has enthalpically favorable hydrogen-bonding interactions unless U36 rotates into solution, leaving unmodified A37 to favorably interact with U35 through electrostatic interactions and hydrogen bond with closer bases through its truncated 6-amino group.', \"The bulky ms2t6 modification containing hydrogen bond donors and acceptors may act sterically to restrict A37 movement while discouraging the displacement of U36 through weak stacking interactions and stabilizing the position of A37 through hydrogen bond contacts across the loop. In the wild-type 2 simulation, where the bulk of the t6A substituent's hydrogen bond donors and acceptors are solvent exposed, U36 exhibits markedly increased conformational freedom. This behavior suggests a delicate balance of forces, of which cross-strand hydrogen bonding is a portion, preventing the displacement of U36 (Figure 4).\", 'NMR studies indicate that the SCH3 of ms2t6A weakens the ureido hydrogen-bonding interaction as compared to t6A (29). Taken together with quantum calculations indicating facile interconversion at room temperature between an ureido ring conformation and a non-ureido ring structure, the conformation around C10-N11 may change based upon the context of the ribosome. In the E.coli 30S ribosomal subunit, primarily responsible for decoding (80), A1493, A1492 and G530 have been found to recognize the first two base pairs of the anticodon–codon complex in the minor groove (76), triggering a closure of the small subunit around the tRNA (81). In this system, the ureido ring allows greater stacking interactions with A38 (19), possibly giving U36 more flexibility to adjust to the codon. The prokaryotic and eukaryotic ribosomes contain conserved regions of rRNA around the decoding site (82) and mRNA crosslinks to A1493 and A1492 corresponding residues in human 18S subunits (83). However, human tRNALys,3 may rotate the t6 moiety around the C10-N11 bond allowing the t6 side chain to hydrogen bond across the loop, thereby stabilizing U36 for correct positioning in the ribosome in a canonical fashion. Accuracy in tRNA–mRNA recognition is both dependent upon a kinetic proofreading mechanism and the stability of the cognate codon–anticodon complex (84). The role of modified bases in stablizing codon–anticodon complexes as opposed to destabilizing the loop conformation for an induced-fit in the ribosome seems to be base and system dependent (4).', 'Supplementary Data are available at NAR Online.'], 'cord_uid': 'br2p09pg'}\n{'title': 'A novel endonuclease IV post-PCR genotyping system', 'text': ['Nucleic acid assays utilizing cleavage enzymes to generate a fluorescent signal from dual-dye-labeled oligonucleotide probes have been extensively used in diagnostics (1–3). In these assays, the ability of fluorescent dyes to transfer energy absorbed from light to nearby molecules forms the basis of homogeneous nucleic acid based assays (1,4,5). In the unhybridized state the fluorophore and quencher are within close proximity, which allows the quencher to absorb energy from the fluorophore to affect quenching. The use of fluorogenic probes in a 5′-nuclease polymerase assay, where the probe is enzymatically cleaved to release the fluorophore (2), is well known. The signal amplification reaction utilizes invasive cleavage with structure-specific 5′-nucleases (6). The method requires annealing of two oligonucleotides, called the upstream oligonucleotide and the probe, to a target sequence, which results in the formation of a unique substrate for the 5′-nuclease. In the DNAzyme-PCR strategy the primer contains a target-specific sequence and harbors the antisense sequence of a 10–23 DNAzyme. During amplification, amplicons are produced containing active sense copies of DNAzymes that cleave a reporter substrate included in the reaction mixture (7). A target nucleic acid sequence can be amplified exponentially in vitro under isothermal conditions by using three enzymatic activities essential to retroviral replication: reverse transcriptase, RNase H (RNA cleavage) and a DNA-dependent RNA polymerase (8). Cycling probe technology represents a simple method for the detection of DNA target sequences, utilizing a chimeric DNA–RNA–DNA probe which is cleaved by RNase H when hybridized with its complementary target (9).', 'Apurinic/apyrimidinic (AP) sites in DNA arise via spontaneous or mutagen-induced hydrolysis of the N-glycosylic bond, or through the repair activity of DNA glycosylases (10). If left unrepaired, AP sites are potentially lethal or mutagenic (11). To cope with the deleterious consequences of AP sites, organisms possess AP endonucleases that initiate the repair of these DNA lesions (10). Class II AP endonucleases initiate repair by catalyzing the hydrolysis of the 5′-phosphodiester of an abasic site to generate a 3′-OH group and a 5′-abasic phosphate residue (10). The substrate specificity of human apurinic endonuclease on synthetic substrate analogs has been reported previously (12).', 'Here we describe a novel endonuclease IV (Endo IV) assay substrate (Figure 1C) that mimics the abasic lesions that normally occur in double-stranded DNA (Figure 1A and B). The first lesion (Figure 1A) is a typical abasic (apurinic or apyrimidinic site generated by spontaneous or enzymatic loss of a nucleic acid base. The second lesion (Figure 1B) is an atypical abasic site appearing as a result of inherent instability of the 3′-phosphodiester bond in abasic deoxyribose in lesion 1 or its cleavage by a Class I AP endonuclease. As shown by the arrow, AP endonucleases cleave the phosphodiester linkage at the abasic sites. In the novel Endo IV assay a short probe with a fluorophore linked through a phosphate at the 3′ end and a short enhancer oligonucleotide generate an artificial, lesion 2 type Endo IV substrate. This arrangement allows the specific and efficient cleavage of the phosphodiester bond of the probe by Endo IV and release of fluorescent dye (Figure 1C). The addition of a quencher at the 5′ end of the probe allows quenching of the fluorescence in the uncleaved probe. The specific cleavage of the phosphate bond and the generation of fluorescence is the basis of the new assay.', 'In this study we used endonuclease IV from Escherichia coli (Endo IV). The enzyme functions as a Class II AP endonuclease and as a 3′ terminal phosphodiesterase (13,14). Endo IV is enzymatically similar to exonuclease III, the major AP endonuclease of E.coli, but it differs from the latter enzyme mainly in lacking the associated exonuclease activity (14). It was also found that E.coli Endo IV is fairly heat stable to ∼70°C (15,16).', 'Full characterization for all the compounds described below can be found in the Supplementary Data.', 'A total of 102 unrelated Centre Etude Polymorphism Humaine DNA samples were obtained from the Coriell Institute of Medical Research () after specifying that the DNA samples should be used for research purpose only. The list of templates used is available at .', 'For the synthesis of the fluorophore-modified controlled pore glass(CPG) supports, please see the Supplementary Data.', 'Fluorogenic probes were synthesized on fluorophore-linker-based CPG supports (Scheme 1) using standard 3′-DNA-phosphoramidites. An Eclipse Dark Quencher was introduced at the 5′ end using the corresponding phosphoramidite (). Probes were purified by reverse-phase high-performance liquid chromatography (HPLC), dried and re-dissolved in 1× TE buffer. A nearest-neighbor model was applied to calculate extinction coefficient (ɛ260) of oligonucleotides (17). A260 measurements were made in PBS (pH 7.2) at ambient temperature and assumed to be a random coil DNA structure in solution. For each Eclipse Quencher, fluorophore 1 (FL1), fluorophore 2 (FL2) or fluorophore 3 (FL3) substitution an ɛ260 correction of +6600, +18 500, or +25 100 or 28 600 M−1 cm−1 was used, respectively.', '3′-Hydroxyprolinol modified probe precursor was synthesized using a modified CPG support (18), and 3′-DNA-phosphoramidites and Eclipse Quencher phosphoramidite on a 0.2 μmol scale. The probe was purified by reverse-phase HPLC using a gradient of acetonitrile in 0.1 M triethylammonium bicarbonate buffer (pH 8–9) followed by drying in a SpeedVac evaporator. The probe precursor was re-dissolved in 50 μl of dry DMSO and treated with 10 μmol of pentafluorophenyl N-FMOC-6-aminohexanoate (19) for 10 h. The FMOC-aminohexanoyl modified probe was isolated by reverse-phase HPLC and deprotected by treatment with concentrated ammonium hydroxide for 1 h at room temperature. The deprotected oligonucleotide was further purified by reverse-phase HPLC using the triethylammonium bicarbonate buffer as described above, dried and re-dissolved in 50 μl of dry DMSO. To this solution, 10 μmol compound 8c (Scheme 2) and 1 μl triethylamine were added. After being kept at room temperature for 10 h the reaction was combined with 1 ml of 2% solution of NaClO4 in acetone. The precipitated crude conjugate was further purified by reverse-phase HPLC, dried and re-constituted in 1× TE buffer. An additional molar absorbance (ɛ260) of 28 500 M−1 cm−1 was used in the probe concentration calculation to correct for the presence of the dye.', 'The synthesis of special CPG supports for the preparation of required oligonucleotide probes are shown in Schemes 1–3.', 'Methyl 3-(3-chloro-2,4-dihydroxyphenyl) propanoate (4a) (Scheme 1) was synthesized starting from 3-chloro-2,4-dimethoxybenzaldehyde (1) (20), which was converted to substituted cinnamic acid 2 by Knoevenagel condensation. Intermediate 2 was hydrogenated in the presence of Pd/C to yield substituted phenyl propionic acid 3, which was first demethylated with HBr/HOAc and then converted to the methyl ester 4a.', 'Synthesis of the pentafluorophenyl ester dye intermediates 11 required to synthesize the dye-modified CPG supports 12 is shown in Scheme 2. Compounds 4a and b were reacted with substituted phthalic anhydrides in the presence of AlCl3 to give the benzophenones 5, which were reacted with the resorcinol analogs 6 (a–c) in trifluoroacetic acid/methanesulfonic acid to yield the carboxyethyl substituted dyes 7. Treatment of compounds 7 with trifluoroacetic anhydride generated lactones 8, which were first reacted with the DMT-protected hydroxyprolinol 9 (21) and then with trimethylacetic anhydride to afford the protected dyes 10. Reaction of intermediates 10 first with succinic anhydride and then with pentafluorophenyl trifluoroacetate, yielded the PFP esters 11.', 'The reaction of the PFP ester 11 with the long chain aminoalkyl CPG (LCAA-CPG) yielded the desired CPG supports 12 as shown in Scheme 1.', 'Endo IV cleavage activity was measured in real-time as shown in Figure 2. The model system in Figure 3A was used to determine optimum E.coli Endo IV (Trevigen, Gaithersburg, MD) concentration, magnesium concentration and pH; the latter two were determined to be at 20 mM Tris buffer, pH 8.6. Results are shown in the Supplementary Data.', 'PCR was conducted in 384-well plates on MJ Research PTC-200 Peltier Thermal Cycler (Waltman, MA). Fifty cycles of three-step PCR profile (95°C for 5 s, 58°C for 30 s, 72°C for 30 s) were run after an initial 2 min denaturation step at 95°C. Each 5 μl PCR contained 2.5 μl JumpStart Taq ReadyMix (Sigma, MI) and 5–10 ng lyophilized human genomic DNA from Coriell Biorepository (Camdem, NJ). Primer concentrations were 0.1 and 1.0 μM for forward and reverse primers, respectively. A Biomek® 2000 Laboratory Automation Station (Beckman Coulter, USA) was used to set up PCRs.', 'The Endo IV Master Mixture was prepared by mixing appropriate volumes of 5× Endo IV reaction buffer [100 mM Tris–HCl (pH 8.5), 25 mM MgCl2, 5× E.coli Endo IV enzyme solution, 50% glycerol, 50 mM Tris–HCl (pH 7.5), 0.1 M NaCl, 1 mM DTT, 0.1% Triton X-100, 0.02 U E.coli Endo IV enzyme and 20× of Endo IV Probes/Enhancer Master Mixture], containing each probe and enhancer at 20 μM concentration.', 'The endo IV reaction was prepared by adding 5 μl Endo IV Master Mixture to 5 μl PCR mixture. Reaction was incubated at 50°C for 50–60 min. Fluorescent detection was performed on ABI 7700 or 7900 in the two channels typically used to detect FAM (for FL1) and VIC (for FL2, FL3 and FL4) utilizing the ABI software for Allelic Discrimination. Alternatively, the fluorescence could also be detected in a LightCycler® LC24 Real-Time PCR System. All assays unless otherwise indicated were performed at least in triplicate.', \"Thermodynamic parameters of duplex formation were derived by the van't Hoff analysis methods. The shape of each melting curve was fitted to the two-state model with linear base lines (22) using a nonlinear least-square program (23). Unless otherwise stated, Tm of DNA duplexes were measured at a concentration of 5 × 10−7 M in 1× PCR buffer containing 10 mM Tris–HCl (pH 8.5), 5 mM MgCl2 and 40 mM NaCl, as described earlier (23).\", 'The MGB Eclipse® Design Software 3.0 () was used to design the primers for PCR. It was also used to calculate the Tm of the enhancer and probe.', 'An example of the new Endo IV-based assay is shown in Figure 2. A probe and an enhancer are hybridized to a complementary synthetic target with a one base gap between them.', 'The signal is substantially diminished in the absence of the enhancer. A rigid hydroxyprolinol linker (Figure 3B) between the oligonucleotide and fluorophore FL1, critical for sensitivity and specificity, was used in this experiment. The results in Figure 2 illustrate the importance of all components used in the assay. The cleavage of the fluorophore is highly target dependent and its rate is amplified in the presence of the enhancer oligonucleotide. Very low non-target-dependent cleavage is an important feature of the Endo IV assay.', 'Effects of pH and cations on E.coli Endo IV activity was determined using the probe, enhancer and target (Figure 3A). It was shown that Endo IV cleavage of the probe has a relatively flat pH optimum in the range between pH 8.5 and 9.5. For compatibility with post-PCR Endo IV amplification the assay buffer with pH 8.6 was chosen, similar to that of the PCR master mixture. Inclusion of the divalent magnesium cation in the buffer ranging from 0 to 20 mM, resulted in more than 2-fold increase in cleavage rate, the optimum range was ∼5–8 mM. A concentration of 5 mM was chosen for the Endo IV assay. It was determined that the monovalent cations, such as Li, Na, K and Rb, inhibit Endo IV cleavage activity for the concentrations studied in the range from 0 to 100 mM. An inhibition of ∼30% was observed at 20 mM monovalent cation concentration, which increased to ∼60–70% at 100 mM concentration. The optimum probe concentration was determined to be ∼600 nM, please see Supplementary Figures 1–3.', 'Figure 3A shows a model Endo IV assay that requires a target complementary to an enhancer and a probe labeled at the 5′ end with an Eclipse Dark Quencher and a fluorescent dye at the 3′ end. As shown, the probe (calculated Tm = 46.7°C) and enhancer (calculated Tm = 50.2°C) are separated by one base to mimic a natural abasic site. Four probes with different fluorescent dyes and linkers (Figure 3B) were evaluated for their performance in the new assay. Fluorophores FL1, FL2 and FL3 of the probes I, II and III (with similar quantum yields) contained a hydroxyprolinol linker between the 3′-phosphate of the probe sequence and the core of the dyes. Fluorophore FL4 of probe IV was analogous to FL3 of probe III, with an additional aminocaproic spacer introduced between the hydroxylprolinol moiety and the dye; the purpose of this spacer was to distance the fluorophore from the cleavage site.', 'The probe specificity was evaluated by comparing the cleavage rates in the presence and absence of target. Target-dependent (specific) and target-independent (non-specific) cleavage rates of the probe were measured as the percent target cleaved per minute. The results are summarized in Table 1. Probes I–III demonstrated similar specific cleavage. A 2- to 3-fold decrease in the rate of non-specific cleavage for probes II and III versus probe I was observed. These low, non-specific cleavage rates translate into high-specific to non-specific cleavage ratios. It appears that chloro- or chloro/methyl substitutions in the fluorophores of probes II and III, respectively, are beneficial for the reduction of non-specific cleavage. One possible explanation for this reduction may be an effect of increased hydrophobicity of FL2 and FL3 on enzyme activity. In the case of probe IV, with the extended hydroxyprolinol linker, the rate of specific cleavage almost doubled. However, the corresponding increase in the rate of non-specific cleavage was ∼20 times higher than that observed in probe III, substantially compromising the ratio of specific to non-specific cleavage rates. Once again, this result implies the existence of inhibiting interactions between Endo IV enzymatic activity and fluorophore which can be adjusted by moving the fluorophore away from the active center.', 'The rigid hydroxyprolinol linker with low non-specific cleavage rates utilized in probes I–III was identified as the linker of choice in the Endo IV assays.', 'The use of a flexible, straight chain linker (C6) instead of hydroxyprolinol-based linker increased the rates of non-specific cleavage to even higher degree than the extended hydroxyprolinol linker and reduced the specific to non-specific ratios to an unacceptable level (data not shown).', 'It appears that both the structure of the sterically constrained hydroxyprolinol linker and proximity of the bulky dye moiety to the cleavage site moderate cleavability of the 3′-terminal phosphodiester bond and practically eliminate non-template-dependent cleavage.', 'Figure 4A shows the target, probe and the enhancer aligned with the target to indicate different gaps. The Tm of the enhancer is typically chosen to be at least 5°C higher than that of the probe. The cleavage rates in the presence of enhancer are shown with 1–5 base gaps (Figure 4B). As expected, based on the natural abasic substrate requirement shown in Figure 1B, a one base gap gave the highest cleavage rate. The probe in the absence of enhancer showed a cleavage rate nearly equivalent to that of a 4 base gap in the presence of enhancer. As shown in Figure 4C, the maximum cleavage rate is achieved at a concentration of enhancer greater than ∼80 nM. In practice, the concentration of the enhancer is typically at least equivalent or larger than that of the probe. Some cleavage is observed with no enhancer present.', 'Figure 5 shows how the cleavage rate of probes with different Tms changes with temperature. Probe lengths ranging from a 6mer to a 14mer were investigated and are shown in Figure 5 with calculated Tms of between 14 and 60°C. All the probes showed a bell-shaped relationship for cleavage rate with temperature. As expected, the cleavage rate for all probes increased with increased temperature. A relatively sharp cleavage rate optimum was observed for the probes a, b and c with calculated Tms of 60, 48 and 42, respectively. It appears that the best probe cleavage occurs close to the calculated Tm or slightly above it. Performing the assay at a temperature slightly higher than the Tm (Figure 5) showed not only optimum activity but also appears to allow probe cleavage cycling, an important feature of this assay.', 'The ability of the Endo IV assay to detect different target concentrations is shown in Figure 6. It was observed that in the case of 125 nM target concentration, the probe-based fluorescence plateaued in ∼1 h, while in the presence of 25 nM target a fluorescence plateau was reached in ∼3 h. The limit of detection of this assay is 0.04 nM, clearly distinguishable from the stable background (Figure 6). This stable background, a characteristic feature of the assay, is maintained over a period of 15 h. This property is also observed in the assay with other nucleic acid targets.', 'The ability of the Endo IV assay to discriminate mismatches at different positions of a 14mer probe is illustrated in Figure 7. Mismatches were introduced in the probe one at a time, from positions 1 to 8. The Endo IV assay shows excellent specificity for the different mismatches from base 1 to base 6 with large match/mismatch signal ratios. As shown in Figure 7, the mismatch signal is often about the same as that of the NTC or slightly higher. The exceptions are the difficult 4-G/T- and 8-T/G-mismatches (24) where match/mismatch ratios of ∼16 and 7 were observed, respectively. These ratios are quite adequate to differentiate matched and mismatched sequences. It appears (data not shown) that the discrimination in positions 1 and 2 is largely determined by substrate requirements while discrimination in other positions is determined by thermodynamic considerations. Experiments with a large number of assays have also indicated that, with 10mer to 12mer probes, satisfactory discrimination is observed with the mismatch position in all except the last three bases at the 5′ end.', 'The exquisite specificity of the Endo IV assay appears to be the result of at least two factors. The first is the strict intrinsic substrate specificity requirement of the Endo IV enzyme which appears to require the rigid prolinol linker (Figure 3B, n = 0) used in the probes I–III. Second, the use of short probes (10mer to 12mer) gives the Endo IV a distinct advantage over most other assays where probes are typically much longer (1,2,6,25).', 'The application of the Endo IV cleavage as a post-PCR detection system is shown in Figures 8 and 9. Following an asymmetric PCR using primers to produce predominantly single-stranded amplicon, two allele-specific probes, an enhancer and Endo IV enzyme are added to the amplification mixture and incubated isothermally for about an hour. The cleaved fluorescence from the two allele-specific probes, each labeled with a different fluorescent dye, is then plotted in a scatter diagram as shown in Figures 8 and 9 for the genotyping analysis of two single nucleotide polymorphisms (SNPs) which are of immediate importance to in cancer (). The scatter diagrams of two polymorphisms, namely agouti signal molecular epidemiology studies protein (ASIP-01) and adenomatous polyposis coli (APC-03) () are shown in Figures 8 and 9, respectively. In both cases, a probe specific to the wild-type allele is labeled with FL1 and a probe specific for the mutant allele labeled with FL3 is used.', 'The probe sets, designed to analyze ASIP-01 and APC-03 polymorphisms, were used to assay 102 unrelated human DNA samples obtained from the Coriell Institute. In the case of the ASIP-01 target the mismatch was situated two bases from the 3′ end of the probe, while in the case of APC-03, the mismatch was six bases from the 3′ end of the probe.', 'From our experience, the spacing of the allele clusters in Figures 8 and 9 are comparable or better than those scatter plots observed in other methods (2,26,27). The exquisite specificity achieved with the Endo IV enzyme, combined with the shorter probes used and the high signal accumulation, gives this assay an advantage over other SNP assays. In comparison with other signal accumulating assays (2), the Endo IV assay has similar levels of high signal accumulation, but uses shorter probes with higher specificity than those assays. In the case of hybridization-based assays (5,25), the Endo IV assay has not only a signal accumulation advantage, but also uses shorter probes with a specificity advantage. In addition, the shorter probes allow much more flexibility in assay design compared to other methods with longer probes.', 'We disclosed a novel nucleic acid signal detection system based on the cleavage of a phosphodiester linkage in a dual-dye-labeled oligonucleotide probe by the Endo IV enzyme from E.coli. The parameters influencing the activity of the Endo IV enzyme were optimized. This optimized Endo IV detection assay requires, in addition to an oligonucleotide probe substrate, a fluorophore attached through a relatively rigid linker to the oligonucleotide and a helper oligonucleotide. The rigid linker conveys the exquisite specificity of Endo IV cleavage of the phosphodiester linkage in the probe and shows high-specific/non-specific rate ratios. The synthesis of fluorophore analogs coupled to a rigid linker and solid support, used in the synthesis of labeled oligonucleotides, was also disclosed. Excellent mismatch discrimination is observed from positions 1 to 6 in a 12mer probe. The ability to put the mismatch in any one of positions 1–6, allows for flexibility in probe design for genotyping assays where sequence and secondary structural constraints are considered. This method allows the use of probes shorter than those typically used in other methods (28) contributing to the improved mismatch discrimination.', 'The combination of the Endo IV assay with a PCR as a post-amplification detection system has been demonstrated. Post-PCR Endo IV genotyping shows excellent spacing of scatter plot endpoint signals of no template controls, wild-type, mutant and heterozygous samples allowing clear allele differentiation. In contrast, other endpoint genotyping methods may suffer from overlapping endpoint signals due to PCR efficiency issues (25,29). This Endo IV post-PCR genotyping has been successfully used in an industry setting for >5000 genotype assays.', 'Supplementary Data are available at NAR online.'], 'cord_uid': 'kt9pmfdr'}\n{'title': 'PATRIC: The VBI PathoSystems Resource Integration Center', 'text': [\"Bioterrorism became an important national security issue (1) following the deliberate release of anthrax spores into the US postal system in October 2001 (2). Meanwhile, emerging and reemerging infectious diseases (3) have had profound effects on public health in many parts of the world. Recognizing the pathogens responsible for these diseases as threats to homeland security, the National Institute of Allergy and Infectious Diseases (NIAID) of the US National Institutes of Health has embarked upon a series of initiatives aimed at developing a comprehensive understanding of the organisms identified as NIAID category A, B and C priority pathogens (for a complete list, see ). The Virginia Bioinformatics Institute's PathoSystems Resource Integration Center (PATRIC) is one of eight Bioinformatics Resource Centers (BRCs) established to study the NIAID priority pathogens and develop these information resources for the research community. While database resources for bacterial ((4) and those cited in (5)) and viral (6,7) genomics have been available for number of years, this project seeks to integrate genomics with comparative genomics and pathway analysis and ultimately proteomics, transcriptomics, immune epitope mapping, host-response and other downstream technologies. The goal is to help researchers and clinicians better detect and respond to biothreat agents (and infectious diseases in general) by facilitating the development of diagnostics, vaccines and therapeutics. This requires access to comprehensive information on the molecular biology, physiology and pathogenicity of these organisms.\", 'PATRIC is responsible for the eight organism categories listed in Table 1. The three genera of proteobacteria are all intracellular pathogens that are known or potential biowarfare agents. In the 1950s, Brucella suis was the first infectious agent developed for use as a biowarfare agent by the United States. Brucellosis, caused by Brucella sp., is an important agricultural disease infecting cattle, sheep, goats and swine as well as humans. It is highly contagious and readily dispersed as an aerosol (8). Coxiella burnetii, the causative agent of Q fever, is a highly infectious agent of relatively low lethality. Its interest as a biowarfare agent stems from its high infectivity, stability to heat and desiccation and potential for aerosol dispersal. The genus Rickettsia contains the organisms responsible for numerous types of typhus and arthropod-borne spotted fevers (9,10). Rickettsia prowazekii was developed as a bioweapon by the USSR in the 1930s and was used by the Japanese in Manchuria during World War II (11).', 'The five categories of viruses studied by PATRIC are all positive-strand ssRNA viruses, with the exception of Lyssaviruses, which have negative-strand ssRNA genomes. While there are no reports of any of these viruses being weaponized, they represent the causative agents for a number of emerging and reemerging diseases including Severe Acute Respiratory Syndrome (SARS), rabies and transmissible gastroenteritis. Recombinant vaccines for these viruses are either still in development or unavailable in areas where these infections are endemic or epidemic, compounding the public health risk.', 'The pace of research on these organisms has increased significantly since the turn of the millennium, with outbreaks, such as that of SARS in 2003 (12,13), spawning a flurry of scientific activity. The widespread use of automated DNA sequencing, microarray gene expression analysis and other high-throughput laboratory technologies has increased the volume of data produced, but not necessarily its accessibility. Currently, significant genomics and bioinformatics expertise is required to extract, process and interpret this wealth of data.', 'To address these problems, PATRIC has created an interdisciplinary team of bioinformaticians, software engineers, computational biologists and organism experts to build a publicly accessible resource aimed at providing high quality, analyzed and curated data to the infectious disease community working on these pathogens. To date, we have achieved the following objectives:\\ncollection and organization of existing genomic data for the eight pathosystems under a single, unified frameworkgenome annotation and curation following standardized proceduresvisualization of raw data from analytical programs, as well as curated datacreation of orthologous gene groups within each organism category allowing comparative analysis of gene contentprediction and visualization of bacterial metabolic pathways to complement functional analysis of proteinsintegration of online literature reviews from PathInfo (14) for selected organisms.', 'Longer-term goals include integration of data from gene expression and proteomics experiments (including host-response), predicted protein and RNA secondary and tertiary structures, and well-cataloged literature compilations. Ultimately, we hope our website will become an essential tool for researchers working on these pathogens and provide networking opportunities within the pathogen research communities.', 'PATRIC is implemented on Oracle 9i RDBMS using the Genomics Unified Schema (GUS) version 3.5, developed at the Computational Biology and Informatics Laboratory at the University of Pennsylvania (see ). GUS is used to store all sequence data and associated annotation with the exception of metabolic pathway data, which is stored in a separate schema developed for use with the Pathway Tools software (15) (see section 4.3). To minimize modifications to GUS and ensure compatibility with future releases, the Curation/Annotation Schema was developed to support PATRIC-specific curation activities.', 'The database is populated with all known full-length or nearly full-length genomic sequences for the eight organism categories listed in Table 1. Automated scripts query GenBank (16) daily to identify new or updated records. The corresponding sequences, annotation and associated literature are retrieved from NCBI and loaded following curatorial review to remove redundancies and assign unique names to each genome. RefSeq (17) records are used when available to take advantage of their more thorough and consistent annotation. Draft genome sequences from Joint Genome Institute (JGI)/Los Alamos National Labs (LANL) and the NIAID-funded Microbial Sequencing Centers will also be part of the PATRIC dataset. In addition to genome sequences and primary annotation from the original GenBank or RefSeq entry, the database stores the results of all automated and manual analyses described in the following section.', 'Due to the large number of closely related genomes in each organism category, we have adopted an annotation strategy in which automated methods are applied to all genomes while detailed manual curation is applied to a limited number of reference genomes. The species B.suis 1330, C.burnetii RSA 493 and R.prowazekii str. Madrid E were chosen as reference genomes for their respective categories. Each viral category has (or will have) multiple reference genomes, representing phylogenetically diverse strains.', 'Automated nucleic acid and protein sequence annotation is accomplished using a Java-based genome annotation pipeline (unpublished), which reads an XML script containing the names and parameters of the analytical applications. The bacterial pipeline executes the gene prediction programs Glimmer (20) and GeneMark (21,22) followed by start site correction programs RBSfinder (23) and TICO (24). BLASTX (25) searches the non-redundant protein database, complementing the ab initio gene prediction methods. RNA genes are identified by tRNAscan-SE (26) and BLASTN searching against a ribosomal RNA database (27,28). The annotation protocol containing the full list of applications and parameters is available online at  under ‘standard operating procedures’.', 'Results of the genome analysis pipeline are merged with original GenBank or RefSeq features for automated interpretation. A decision tree is used to classify genes into categories based on the level of agreement between the various prediction methods. Genes that are unambiguously predicted by multiple methods are automatically ‘finalized’, creating new ‘gene’, ‘CDS’ and/or ‘[t/r]RNA’ features. The remaining genes are marked for manual curation. For viral genomes, an abbreviated pipeline is executed that emphasizes sequence alignment for gene identification and employs GeneMarkHMM optimized for mammalian (host) genomes.', 'After curatorial review, finalized protein-coding (CDS) features are translated and subjected to another pipeline executing InterProScan and structure prediction methods such as MEMSAT 2 (29). Currently, each protein is associated with GO terms (30), TIGRroles, Enzyme Commission numbers (31) based on Pfam (32) and TIGRfam alignments (for a description of TIGRfam and TIGRroles, see: ). The protocol for automated proteome annotation is also available online. Manually curated protein sequences will be available in early 2007.', 'Once protein sequences are inferred from each genome in an organism category, putative ortholog groups are generated using BLASTP for all pairwise genome combinations and applying the conventional bidirectional-best-hit (BBH) criterion (33). While putative ortholog groups within the bacterial categories are generally well defined, many viral proteins cannot be readily clustered using the stringent BBH criterion. This is an active area of curation. Using the ortholog groups as a starting point, a reference protein list is created for each bacterial category consisting of the proteins of the reference genome (each representing one ortholog group) plus a representative protein from each ortholog group identified in the associated genomes. A gene occurring in only a single genome constitutes a ‘group’ of one and would be included in the reference list. The reference protein lists will be manually curated and include, whenever possible, detailed functional descriptions, gene symbols, GO terms and EC numbers. Thus, every protein in the database will either be manually curated or be linked to an ortholog group member that has been manually curated.', 'The ortholog groups are further processed to create multiple sequence alignments (MSAs) using MUSCLE (34) with default parameters. Phylogenetic estimations using the neighbor-joining method (35) were created based on trimmed alignments using PHYLIP (36). Trees were validated by bootstrapping (37) using a minimum of 100 replicates.', \"To help users understand the function of the bacterial proteins in context, we have adopted the Pathway Tools system (15) to derive pathways from genome annotation and to fill potential gaps in annotation known as pathway holes. The system takes a list of protein names, descriptions and EC numbers as input. Proteins with EC numbers can be assigned roles directly; the roles of other proteins are suggested by lexicographic analysis of descriptive information and/or analysis of gene order from homologous regions of related genomes and confirmed or rejected by the curation staff. The output is a database with integrated web server that allows users to browse and query the organism's metabolic pathways. This system has been integrated with the PATRIC web site, allowing users to access pathway information for all bacterial reference genomes. The current analysis was based on preexisting RefSeq or GenBank annotations; later releases will incorporate data curated in house, unifying the genomic and pathway versions of the data. The analysis of pathways can facilitate the identification of metabolic choke points, critical enzymes that could be targeted by drugs that may have valuable antimicrobial properties. Pathway analysis can also yield clues to pathogenesis by comparing virulent and avirulent strains and examining the roles of genes not present in both strains.\", 'The PATRIC website is hosted on a Sun Microsystems v20z server running SuSE Linux using the Apache web server. Applications are written in PHP and Perl, accessing data from an Oracle 9i server hosted on a Sun Microsystems E15000 running Sun OS.', \"The conceptual organization of the website is described in Figure 1. The website's home page contains news, a navigation bar and the list of PATRIC organisms. Users can select their organism of interest from the list to access the corresponding organism category page. This page contains a table of genomes currently in our database with links to the three principal representations of individual genomes: the genome summary, genome browser and gene table. These pages allow users to view a summary of genome sequencing information and to identify specific genes and link to their corresponding gene, protein and pathway information pages. The gene information page displays the output of sequence analysis software run by the annotation pipeline, as well as curated data. Similarly, the protein information page displays InterProScan and TIGRfam alignments and associated information such as GO terms and EC numbers. For bacterial genomes, the pathway information page illustrates the protein's position in the organism's metabolic network and links to a wealth of information provided by PathwayTools.\", 'The organism category page also contains links to a pathogen summary, ortholog group table and a phylogenetic tree based on 16S rRNAs for bacteria or a selected protein family for viruses. For bacterial genomes, detailed pathosystem information is available, provided by the VBI PathInfo documents (14). The ortholog group table shows the presence or absence of reference gene list proteins for each organism in the organism category and provides links to an MSA and tree viewer and the Base-By-Base MSA editor (38) for every ortholog group. Base-By-Base allows users to add sequences to the MSA, recalculate it using Clustal (39), T-Coffee (40) or MUSCLE and generate the corresponding tree using neighbor-joining or a number of clustering algorithms.', 'The PATRIC website also supports analytical and query tools. A database search page allows user-supplied sequences to be BLASTed against reference and curated sequences from PATRIC organisms. The page also supports MUMMER (41) comparisons between genomes in the database or with a user-supplied sequence. A query tool is available throughout the site by which users can retrieve genes by name, ID, description, as well as GO and EC identifiers and descriptions. Questions, comments and suggestions concerning the website and its contents may be submitted via the ‘feedback’ page, accessible from the menu bar.', 'The PATRIC database is hosted at the Virginia Bioinformatics Institute at Virginia Tech and can be accessed via web browser at . Sequences and annotation in GFF3 format (see ) can be downloaded by following the ‘downloads’ link on the main menu bar. GFF3 files are also available through BRC-Central at: .', \"This paper presents the first detailed description of the PATRIC website. Future development will advance on several fronts. Genome and proteome curation will continue, complemented by improved tools for query, analysis and visualization. For viruses, we will transition to the more widely accepted ICTV taxonomy (42). The website's user interface is being enhanced to integrate organism-, tool/task- and data-centric approaches to data access, allowing users more efficient and effective access to PATRIC resources. This will be followed up by prioritized curation targeted at potential drug and vaccine targets, virulence factors and genes with differential representation or polymorphisms associated with clinically significant phenotypes. Leveraging another NIAID-funded VBI project, the Administrative Resource for Biodefense Proteomics Research (), we plan to integrate expression profiling and proteomics data from pathogen and host to better understand the pathosystem's biology and help the community identify targets for counter-measures. The integration of these disparate data types into a single, easy-to-use system is a goal that we anticipate will enable pathogen researchers to make full use of available data to develop diagnostics, vaccines and therapeutics.\"], 'cord_uid': '3ulketgy'}\n{'title': \"MMDB: annotating protein sequences with Entrez's 3D-structure database\", 'text': [\"The molecular modeling database (MMDB) is Entrez's ‘Structure’ database (1). Querying MMDB with text terms, e.g. one may identify structures of interest based on a protein name. Links between databases provide other search mechanisms. A query of Entrez PubMed database, e.g. will identify articles citing a particular protein name. Links from this set of articles to ‘Structure’ may identify structures not found by direct query, since PubMed abstracts contain additional descriptive terms. Currently, MMDB and its visualization services handle ∼25 000 user queries per day.\", 'Experimental three-dimensional (3D) structure data are obtained from the Protein Data Bank (PDB) (2). Author-annotated features provided by PDB are recorded in MMDB. The agreement between atomic coordinate and sequence data is verified, and sequence data are obtained from PDB coordinate records, if necessary, to resolve ambiguities(3). Data are mapped into a computer friendly format and transferred between applications using Abstract Syntax Notation 1 (ASN.1). This validation and encoding supports the interoperable display of sequence, structure and alignment. Uniformly defined secondary-structure and 3D-domain features are added to support structure neighbor calculations. MMDB currently contains ∼39 000 structure entries, corresponding to ∼90 000 chains and 170 000 3D domains.', \"The MMDB web server generates structure summary pages, which provide a concise description of an MMDB entry's content and the available annotation (4). Sequences derived from MMDB are entered into Entrez's protein or nucleic acid sequence database, preserving links to the corresponding 3D structures. Links to PubMed are generated by matching citations. Links to Entrez's organism taxonomy database are generated by semi-automatic processing of ‘source records' and other descriptive text provided by PDB. Ligands and other small molecules are identified and added to the PubChem resource, accessible at , also preserving reciprocal links to 3D structure. Sequence neighbors are identified by BLAST (5), and links to the Conserved Domain Database (CDD) (6) by the RPS-BLAST algorithm (5). Structure neighbors are identified by VAST (7). The 3D structure viewer supported by Entrez, Cn3D (8), provides molecular-graphics visualization.\", 'In the Entrez database system, protein sequences are neighbored to each other by comparing each newly entered sequence to all other database entries. These database scans are run with the BLAST (5) engine, which identifies sequence neighbors with significant similarity, and the resulting sequence identifiers and taxonomy indices are stored, so that Entrez can provide ‘Related Sequences’ links for all protein records in the collection. The ‘Related Structure’ service is built on top of this system. Sequence neighbors directly linked to MMDB are identified and alignments are re-computed by employing the ‘BlastTwoSequences’ tool (9) to restore alignment footprints. The ‘Related Structure’ web interface provides direct access to this information. Initially this service had been restricted to sequences from microbial genomes (10), but it has now been expanded to cover all proteins in Entrez and is updated daily to provide a comprehensive 3D-structure annotation service. Identification of structure-linked neighbors and the visualization of sequence-structure alignment is also possible using Entrez and the Cn3D alignment viewer/editor, but ‘Related Structures’ provides a convenient new summary and ‘one click’ shortcuts to 3D visualization. These 3D views may be used to identify conserved residues and map site-specific features derived from the 3D structure. Currently ∼48% of non-identical protein sequences in Entrez have been linked to at least one related structure, employing a conservative threshold for alignment length (50 aligned residues or more) and similarity (30% or more identical residues in the aligned footprint); see Figure 1 for details.', \"A search with the term ‘Angiotensin converting enzyme’ in Entrez's protein database retrieves >400 hits. One may configure the Entrez browser to filter search results by various criteria, and one pre-configured filter selects those protein sequences with ‘Related Structures’ (configuration of Entrez can be achieved by following links to ‘My NCBI’, or by clicking on the ‘toolbox’ icon shown at the top of Entrez document summaries.). In this example, the ‘Related Structures’ filter shows that >240 of the identified sequence records have links to related structures.\", 'One such protein sequence is the ACE protein from Rattus norvegicus (accession no. ‘NP_036676’). On the ‘Links’ menu for this record, ‘Related structures’ generates a request to the Related Structure service (). The resulting page indicates with a horizontal bar, the sequence region annotated by each related structure (Figure 2). The display also supports sorting by a variety of alignment parameters such as score or length and selection of sequence-dissimilar ‘non redundant’ subsets. A ‘Table’ option switches to a text view, listing descriptions of each structure as well as alignment scores.', \"Using the table view with this example, one may notice that several related structures are complexes of the same protein with different drugs/inhibitors, e.g. structures with PDB codes 1O86 (11), 1UZF (12) and 1UZE (12). Clicking on the graphical alignment footprint of 1O86, a human ACE enzyme in complex with lisinopril, one can see a text representation of the corresponding BLAST alignment, and a Cn3D view of the alignment can be launched by clicking on ‘Get 3D Structure data’ (Figure 3). One may see that the query protein is highly similar in sequence to the human ACE enzyme, as identical residue pairs are colored red by default. The sequence identity across the aligned region is 82%, and it appears that the core of the structure is mostly formed by residues conserved between the two aligned rows, while non-conserved residues are mainly located on the structure's surface.\", \"One may further identify the catalytic center by identifying residues that contact the catalytic Zinc ion. Those sites can then be mapped from the structure to aligned regions in the sequence window using Cn3D's highlighting functionality. One may also examine the sequence-structure alignments with related structures 1UZE and 1UZF, human ACE binding to enalaprilat and captopril, respectively, drugs with chemical structures similar to that of lisinopril. This allows one to identify conserved interactions between the ACE enzyme and this series of antihypertensive drugs. Similarly, by examining the related structure 2AJF (13), one may be able to identify residues critical for cross-species infection by studying the protein–protein interactions between the receptor binding domain from SARS Coronavirus Spike and human versus rat angiotensin-converting enzyme 2.\", \"The ‘Related Structure’ service is also integrated with NCBI's protein BLAST service. A ‘Related Structures’ link is provided when one or more similar proteins with known 3D structures have been identified by BLAST. The NCBI single-nucleotide polymorphism resource (SNP) also links to the ‘Related Structure’ service, which in this context provides a mapping of both synonymous and non-synonymous coding SNPs onto experimentally determined 3D structures. ‘Related Structure’ may be expanded further in the future, to provide visualization for other NCBI resources and to support additional filtering and selection among related structures, e.g. to highlight those annotated with conserved domain footprints by the CDD resource or those linked to small molecules in the PubChem database.\"], 'cord_uid': '6qpsxmgi'}\n{'title': 'VIRUS ON T H E CEREBRAL ACTIVITY OF P L E U R O P N E U M O N I A -L I K E ORGANISMS I N M I C E', 'text': ['Shortly after the isolation of MHV(C) from Balb C mice, in 1955, it was found that Swiss weanlings were equally susceptible and because of their availability they were subsequently used in activity tests (1) . Princeton mice were consistently resistant. Intraperitoneal injection in Swiss mice was generally followed by focal or diffuse necrosis of the liver but was not attended by nervous manifestations. On intracerebral injection the virus produced significant changes in the brain, with some destruction of nerve cells. Lymphocytic infiltration was commonly observed in brain sections but polymorphonuclear leukocytes were absent. Lesions produced in the liver, following carriage of the virus from the brain, were generally focal in character. The death rate was variable. In one series of passages, 12 of 25 mice died between the 3rd and the 5th day after injection. Survival after the 3rd day was irregularly attended by paralysis of the hind legs. The most common sign was a rigid backward extension of one or both limbs. The affected leg was functionless and was dragged by the mouse in moving. This type of paralysis was observed in 9 of the 25 mice. At autopsy on the 5th or 6th day, 5 of them were erect and active while 4 were prone and unable to move. A tendency to roll sideways, if lightly prodded, was observed in one of the latter mice. All of the cranially injected animals showed weight losses of 1 to 3 gin.', 'Unusually acute signs of illness appeared in Swiss mice during the first cranial passage series. By the 3rd transfer, leukocytes were demonstrable in direct films 179 and later in sections from the brain. A cerebral suspension prepared from the mice of the 5th transfer yielded a pure growth of PPLO. The passage series was continued with a Coors filtrate of the contaminated suspension and subsequently maintained in the absence of PPLO. In 1938, Sabin (2) described a nervous disorder of mice that appeared during the cerebral passage of toxoplasma. The characteristic sign was a rolling motion on the long axis of the body. It was also observed after intraabdominal and intrathoracic injection. The incidence was 20 to 40 per cent. Two kinds of brain lesions were found in sections. The first one was indicated by destruction of the periventricular tissue and extensive leukocytic infiltration. It was present in all mice. The second lesion, consisting of almost complete necrosis and lysis of the posterior pole of the cerebellum, occurred only in mice with rolling signs.', 'Sabin (3) subsequently found that a pleuropneumonia-like organism, later termed type A (5) , was the etiologic factor. The disease was fully reproduced by intracerebral injection of the agent in pure culture. If grown in glucose-serum broth the organism produced a true exotoxin which resulted in the same nervous signs and cerebellar lesion on intravenous injection. Findlay et al. (4) , also in 1938, noted that mice injected intracerebrally, in series, with a neurotropic strain of yellow fever virus developed nervous signs within 2 days, instead of 4 to 5 days. Rolling with a sideways motion was observed in some of the mice. An agent that was not cultivable in ordinary media was separated from the brain but was subsequently lost on passage.', 'A similar condition was later observed during brain passage of the S strain of lymphocytic choriomeningitis virus. Rolling occurred in about 10 per cent of the injected mice. Seventy-five per cent died in 2 to 7 days. Some of the survivors developed hydrocephalus between the 8th and the 21st day.', 'Brain sections showed neurolysis and an intense infiltration of leukocytes. The lateral ventricles, meninges, and choroid plexus were invaded. In hydrocephalus the leukocytes were largely replaced by lymphocytes and mononuclears.', 'PPLO designated L5 were invariably isolated from the brains of mice with rolling signs. This organism by itself was incapable of reproducing the disease but mixed with a neurotropic virus or agar, or after rapid intracerebral passage, its virulence was raised and the rolling signs resulted. Several other types of PPLO (L,, Ls, and L4) were inactive in the presence of virus.', \"Cross-protection tests indicated that Sabin's type A organism was closely related to Ls but his was considerably more virulent.\", 'The first experiments on the behavior of PPLO in the brain were made in normal mice with 2 strains of the catarrhal type, designated S and J. Swiss and Princeton weanlings were injected intracerebrally with pure cultures of the organisms in the absence of any other microbial agent. Subsequent passages were made with brain suspensions.', 'The mice used in all of the following experiments were Swiss and Princeton weanlings, 10 to 12 gin. in weight, from colonies maintained at The Rockefeller Institute. They were injected in groups of 4 or 6, using mice of either sex but the same sex in any one group. The intracerebrai injections were made with approximately 0.02 ml. of inoculum, in mice lightly anesthetized with ether. The brain suspensions were prepared in saline from minced tissue, using a TenBroeck grinder, and were about 10 per cent in concentration. Freshly prepared suspensions were generally held for a short period, until sedimentation had occurred, and the supernatant used. In some instances penicillin was added in the amount of 1000 units per ml.', 'The PPLO strain designated S was the one originally isolated from the brains of Swiss mice during the cranial passage of MHV(C). The results of nasal and intraperitoneal injection in Princeton weanlings had indicated that it was a catarrhal type. The intracerebral passage in Swiss mice was begun with the 2nd transfer of the organism grown for 48 hours in 20 per cent horse serum--heart infusion bouillon (pH 8).', \"Most of the injected mice were killed on the 7th day and their brains removed aseptically. At least one brain from each passage group was placed in Zenker's solution, for histologic study, and sections cut from 3 levels including the cerebellum. They were stained with eosinmethylene blue. A small amount of tissue was aspirated with a capillary pipette from 2 or more of the remaining brains, the pipette being inserted between the hemispheres. Gramstained films of the tissue were prepared for microscopic examination. Cultures were also made on 20 per cent horse serum-nutrient agar plates containing 2500 units of penicillin. Finally, a suspension was prepared from the pooled brains for passaging. In some of the later tests cultures were also made from the suspensions. Several of the mice in this experiment were held under observation for 14 days and killed without a complete examination. Five subsequent passages were made, in each case with a brain suspension from the preceding transfer. Essentially the same methods were used in all of the following tests.\", 'The PPLO strain designated J was originally isolated during an outbreak of infectious catarrh in a small colony of Japanese mice. It has been maintained for many years in this laboratory by nasal passage or contact in Princeton mice. Intracerebral passage was begun in Princeton weanlings with a freshly isolated 72 hour old culture, the 3rd transfer on isolation from middle ear exudate. Four subsequent cerebral passages were made in Princeton mice and one in Swiss mice.', 'Twenty-eight of the 32 Swiss mice used in the 6 cerebral passages of the S strain of PPLO were killed on the 7th day after injection, and 4 on the 14th day. All of the animals gained weight during the observation period and were normal in appearance. At autopsy they showed no consistent pathologic findings. Leukocytes were not observed in either direct films or sections and there was no indication of encephalitis or meningitis. Thirteen of 16 brain cultures gave a pure growth of PPLO and 3 were negative. The amount of growth was regularly scanty, 8 of the plates having 10 colonies or less.', 'Similar results were obtained with the cerebral passage of the J strain in Princeton mice. The 25 animals used in the 5 transfers were normal in appearance during the observation period of 7 days. At autopsy, leukocytes were not observed in either brain films or sections. Cultures from the suspensions yielded only a scanty growth of PPLO colonies. In the single experiment with Swiss mice the injected animals were held for 4 weeks. They were likewise normal during life and at autopsy. The terminal culture from the brain suspension yielded 8 PPLO colonies.', 'The Reaction with Catarrhal Strains of PPLO.--This series was begun with the S strain of PPLO. Eight successive cranial transfers were made in Swiss weanlings, starting with a brain suspension which contained both the organisms and the virus. Prior to use this suspension had been maintained in a frozen state at --20°C. for 21 weeks.', 'Five mice were used in each of the 8 brain passages. Mter injection they were held under observation until one or more deaths had occurred. At this time, usually the 3rd or 4th day, the survivors were killed with ether and autopsied. Direct brain films and cultures were made from at least one mouse in each passage. Six brains were sectioned for later study.', 'Twenty of the 40 mice in this series of passages died, 14 on the 3rd day and 6 on the 4th. The 20 survivors were acutely ill when killed for autopsy. They regularly showed a marked loss in weight, up to 4 grn. per mouse. Roughening of the hair was a variable finding. Many of the mice were inactive and some were prone. Convulsive movements were irregularly observed and a tendency to roll was seen in 3. At autopsy the livers commonly showed a normal red color but were regularly spotted with small white loci. These areas were often few in number, 10 or less, and in some instances were visible only on low power magnification. The Gram-stained films regularly showed leukocytes in considerable numbers. In some films extra-and intracellular groupings of PPLO were clearly demonstrable. Areas of inflammation with innumerable leukocytes were observed in the brain sections. A pure growth of PPLO colonies, varying from several thousand to innumerable, was obtained on the plate cultures from 7 of the passages. The growth from the brains of the first passage, which had been made with a previously frozen suspension, was relatively meagre. A culture made from the pooled livers of the 2 survivors in the second passage was bacteriologically sterile.', 'Supplementary experiments in this group were carried out with 3 additional PPLO strains of the catarrhal type. One strain (J) was recovered from the middle ear exudate of experimentally infected mice and 2 strains (RZ and RH) from naturally infected albino rats. All 3 strains were of recent isolation. Two cerebral passages were made in Swiss weanlings with each of the strains. In preparing the inoculum for the first passage a brain suspension containing only MHV(C) was mixed with an equal volume of a fluid culture of PPLO. The second passage was made with a brain suspension which contained both agents.', 'With the RZ strain of PPLO the outward signs of illness were delayed until the 5th day. In other respects the outcome of these tests essentially duplicated the results with the S strain. Signs of acute illness were regularly observed in the injected mice. Thirteen of the 30 animals used in the combined tests died. A tendency to roll was observed in only 2 of the 17 survivors. At autopsy a sparse distribution of small surface foci was observed in all of the livers. Leukocytes were regularly present in brain films and numerous colonies of PPLO were obtained on plate cultures. Brain sections from 2 mice in each group showed extensive leukocytic infiltration.', \"In the experiment with the J strain of PPLO, cultures were also made from the heart's blood, liver, and spleen. All of these suspensions were bacteriologically sterile. Aside from the liver, the abdominal and thoracic organs of these mice, and indeed of all the mice in the entire series of experiments, were normal in the gross.\", 'The effect of long continued cultivation on the cerebral activity of PPLO was determined with a culture of the J strain that had been transferred 100 times in serum-bouillon at intervals of 2 to 3 days. In the presence of MHV(C) its pathogenicity was fully as great as that of the recently isolated strains.', 'One titration experiment was carried out with brains removed on the 3rd day from Swiss mice injected intracerebrally with MHV(C) and the J strain of PPLO. The highest dilution of the suspension that gave a leukocytic response with positive cultures was 10 -4 . The brains from mice which received the 10 -5 dilution showed no lesions and were bacteriologically sterile.', 'The Reaction with Conjunctival Strains of PPLO.--The observations on the activity of MHV(C) and PPLO in the brain were continued with 2 strains of the conjunctival type, designated P and B. These organisms are much less pathogenic for mice than those of the catarrhal type and are commonly inactive on nasal or intraperitoneal injection in susceptible weanlings. Under natural conditions they are associated with a low grade inflammation of the conjunctivae (6).', 'The P strain was obtained from Princeton mice infected by direct contact. A passage series with this strain was begun in 1950 and has been maintained to date. A culture of recent isolation from conjunctival washings (the 3rd subculture) and one that had been recultured after 79 transfers and then stored for 8 months at -20°C. were used. The B strain was obtained from a naturally infected mouse, one of a group sent to us for examination from the Brookhaven National Laboratory. The 2nd culture transfer was employed.', 'In the presence of active MHV(C) 2 cerebral passages were made in Swiss weanlings with each of the 2 P strain cultures and 10 passages with the B strain. A single passage was also made with the latter culture in the absence of virus.', 'On combination with MHV(C) the activity of the P strain in the brain was essentially the same as that of the catarrhal types. The subculture of the transplant that had been transferred 79 times and then stored in a frozen state was fully as active as the recently isolated one. The first experiments with the B strain showed evidence of an accelerated lethal outcome which was substantiated by subsequent passages. Outward signs of illness were also more con-spicuous with this strain. In other respects its behavior resembled that of the P strain. In the absence of virus the organisms survived in the brain for 2 weeks but produced no demonstrable reaction.', 'Twelve of the 20 mice used in the 2 experiments with the P strain died on the 3rd to the 5th day after injection. Prior to death they showed external signs of illness. One animal tended to roll. Eight mice were killed and autopsied on the 3rd or 4th day. Their livers were spotted with surface areas of focal necrosis. Leukocytes were regularly present in the brains and numerous colonies of PPLO were obtained on culture.', 'Deaths in the passage series with the B strain began a day earlier, 15 of the 50 mice dying on the 2nd day after injection. Signs of acute illness were observed in many of the survivors at this time. Fourteen additional deaths occurred between the 3rd and the 5th day. Leukocytes were invariably seen in films or sections from the 21 mice that were killed for autopsy. All of the brain cultures yielded many colonies of PPLO. Two of these mice were hydrocephalic, one showed circling movements, and 5 displayed a tendency to roll.', 'Earlier observations on the behavior of MHV(C) in different strains of mice had indicated that the virus alone was commonly inactive on intraperitoneal or intracerebral injection in Princeton weanlings (1) . As an extension of the present findings on the behavior of MHV(C) and PPLO in the brains of Swiss mice a group of experiments was carried out in Princeton mice to determine whether or not either agent was influenced by the presence of the other.', 'In a preliminary experiment 5 Princeton weanlings were injected intracerebrally with a brain suspension containing only the virus. These mice showed no outward signs of illness and when killed on the 7th day were strictly normal. Leukocytes were not demonstrable in brain films or sections and cultures were bacteriologically sterile.', 'Ten mice were then injected intracerebrally with a mixture of the virus and the S strain of catarrhal PPLO. A similar number received the virus and the J strain of the organisms. The 20 mice, in the 2 groups, were unaffected by the inoculum. When killed on the 7th day the livers were free from focal lesions and the brains showed no indication of leukocytic infiltration. On culture, brain suspensions prepared from each of the 2 groups yielded a sparse growth of PPLO colonies.', 'The highly virulent Princeton type of mouse hepatitis virus, MHV(Pr), is pathogenic for Princeton weanlings on intracerebral injection (1). Cerebral multiplication of the virus is not attended, however, by outward signs of nervous disorder and at autopsy there is little evidence of cellular injury in the brain. Carriage of the virus from the brain to the liver commonly occurs and is followed by the customary necrosis. The hepatic reaction is generally fatal.', 'Swiss weanlings are naturally resistant to MHV(Pr). The virus fails to multiply in the brain and is usually not recoverable by passage to susceptible Princeton mice. At autopsy the brains and livers are regularly lesion-free.', 'Two additional groups of experiments were carried out to determine the cerebral response of Princeton and Swiss weanlings to MHV(Pr) in combination with PPLO.', 'Five cerebral passages were made in a total of 25 Princeton mice with a mixture of MHV(Pr) and the J strain of PPLO. Fifteen deaths were recorded as follows: 1 on the 2nd day, 10 on the 3rd, 3 on the 4th, and 1 on the 5th. Ten mice were killed and autopsied, most of them on the 3rd day. Prior to death they appeared acutely ill but were free from outward signs of nervous disorder. The livers of these mice showed focal or semidiffuse necrosis. Leukocytes were observed in brain films and massive areas of leukocytic infiltration in sections. Innumerable colonies of PPLO were obtained on culture.', 'Twenty Swiss weanlings were injected intracerebrally with a mixture of MHV(Pr) and PPLO, 10 receiving the J strain and 10 the S strain. There were no deaths. All of the mice gained weight normally and showed no outward signs of illness when killed on the 7th or the 10th day. At autopsy the livers were normal. Leukocytes were not observed in either films or sections from the brains. Eight of 10 brain cultures yielded a few discrete colonies of PPLO and 2 were without growth.', 'A group of experiments was carried out in Princeton and Swiss weaniings to determine the effect of agar on the growth and activity of PPLO in the brain. Findlay et al. (4) had reported that cultures of PPLO which failed to produce rolling signs on intracerebral injection did so in some mice on the addition of agar.', 'The J strain of PPLO was used in all of these tests. Heart infusion agar (pH 8) was liquefied, cooled to 40°C., and diluted 1 to 5 with saline, The undiluted medium contained 1.5 per cent agar and was used as routine in the isolation of PPLO. The diluted semisolid agar was mixed with an equal volume of young PPLO cultures in 20 per cent horse serum-bouillon. Twenty-four weanlings of each mouse strain were injected intracerebrally with this inoculum. One series of 3 cerebral passages was made in Princeton mice, starting with a brain suspension from 2 animals injected with the PPLO-agar mixture. Finally, as a control on the activity of the medium, mice of both strains were injected with a sterile mixture of agar and horse serum-bouillon.', 'Six of the 48 mice used in the first group of experiments died on the 7th to the 19th day after injection. The survivors were killed and autopsied at in-tervals of 1 and 3 weeks. All of the mice, including those that died, gained weight normally and were free from any outward signs of nervous disorder. Thirteen of the 21 Princeton mice that were killed and 6 of the Swiss showed leukocytes in either films or sections of the brain and a good growth of PPLO on culture. In the absence of an inflammatory reaction the growth was meagre.', 'As Findlay et al. (4) had previously noted in their experiments, a significant number of the mice were frankly hydrocephalic. Twelve Princeton weanlings and 10 Swiss showed free fluid in the cranial cavity, accompanied in some instances by a marked elevation of the skull and circumscribed sunken areas in the anterior portion of one or both hemispheres. The brain usually appeared paler than normal.', 'All of the 18 mice in the passage series survived and were normal in appearance through the 14th day when they were killed. At autopsy leukocytes were not observed in brain films or sections and only a meagre growth of PPLO was obtained on culture. In the first passage, however, 3 of the mice were hydrocephalic. Their skulls were not elevated but the cranial cavities contained free fluid and areas of collapse were present in one or both hemispheres.', 'Two of the 6 Princeton mice injected with the sterile mixture of agar and horse serum-bouillon died. The 10 survivors in both groups showed no outward signs of nervous disorder and were killed on the 21st day. The skulls of 2 Princeton mice were elevated and free fluid was present in the cranial cavities. Four of the Swiss mice were also hydrocephalic, though their skulls were not elevated. Films and sections from the brains of these mice showed no leukocytes and cultures were bacteriologically sterile.', 'Brain sections from the mice injected with the fluid cultures of PPLO alone were essentially normal. A few lymphocytes were occasionally observed but leukocytes were regularly absent. In some sections the ventricles were somewhat dilated.', 'Sections of the cerebrum from the mice injected with a mixture of virus with neurotropic activity and PPLO invariably showed an inflammatory reaction with a preponderance of leukocytes. The most common lesion was a roughly semicircular band composed largely of leukocytes, a few lymphocytes, much necrotic debris, red blood cells, and fibrin (Figs. 1 and 2). It partially enclosed the ventricular spaces and the intermeshing horns of the bilateral ganglia. Large mononuclear cells packed with fat vacuoles (gitter cells) and droplets of free fat were generally observed at its margin. The nerve cells of the adjacent ganglia often showed necrosis with deep blue pyknotic nuclei and pink staining debris. In a few sections the reaction was limited to one or more sharply demarcated abscesses. Dilation of the ventricles sometimes occurred but was not a characteristic finding. Leukocytie involvement of the meninges was observed but was rarely conspicuous. Leukocytes also invaded the cerebellum but much less frequently. A few sections showed degeneration of nerve cells particularly those in the granular layer (Fig. 5) . It was not consistent, however~ in respect to the appearance of rolling signs prior to autopsy.', 'Brain sections from the mice injected with the agar-PPLO mixture showed 2 types of cellular response. In the presence of actively growing PPLO there was a marked inflammatory reaction which was commonly circumscribed (Fig. 4) . Leukocytes predominated but lymphocytes and large mononuclear cells, with and without fat droplets, were also present. There was much necrotic debris. In some mice there was definite evidence of encapsulation by connective tissue. Necrosis of nerve cells in the nearby ganglia was generally not observed. Portions of the meninges, particularly in the cerebellum, were distended by an infiltration of leukocytes and lymphocytes. Some of the sections also showed widely dilated ventricles.', 'In the absence of actively growing PPLO there was commonly a lymphocytic infiltration together with hydrocephalus. With no or low magnification the ground substance of the brain was separated symmetrically by unstained open spaces (Fig. 3) . Microscopically the central and lateral ventricles were greatly dilated and often extended beyond the limits of the ependymal cell layer. In some instances this layer had broken away from the margin of the cavity. Rupture of the distended ventricles, possibly at autopsy, with release of the fluid may account for the areas of collapse observed in the intact brain. Lymphocytes and large mononuclear cells were usually present within and around the margin of the ventricles and also in the meninges. Leukocytes were absent. The nerve ceils of the ganglia were intact and appeared normal. Sections from some of the mice injected with the sterile mixture of agar and horse serum-bouillon showed a similar distension of the ventricles but little or no cellular reaction.', 'In naturally infected mice the sites most often frequented by PPLO are the nasal passages and the middle ears. Localization in the lungs, conjunctivae, and joints also occurs but is more sporadic. The present findings together with the earlier ones of Sabin (2, 3) and of Findlay et al. (4) indicate that PPLO may occasionally invade the central nervous system.', 'It is not surprising that PPLO have been recovered in a few instances from the brains of supposedly normal mice. The first isolate that we obtained was an organism of the catarrhal type which is widely distributed in nature. This type is enzootic in the Swiss colony, from which our mice were obtained, but is not present in the Princeton colony. In diseased mice the organisms occasionally migrate inwardly from the tympanic cavity to the labyrinth. A small local abscess that impinges on the brain may be produced and provide access to the meninges. Carriage via the circulating blood is also a possibility and would account for the transport of the conjunctival type which ordinarily does not invade the middle ear.', \"The normal mouse brain evidently fails to provide optimal conditions for the growth of most PPLO. Sabin's type A organism (5) which produces an extotoxin is an outstanding exception. On cranial injection the organisms that we studied survived in the brain for a period of weeks. The level of growth was barely sufficient, however, to maintain them by serial passage and too low to provoke a counter-reaction. Growth was markedly increased by the simultaneous injection of mouse hepatitis virus and was attended by a vigorous leukocytic response. There was no indication of specificity in relation to the type of PPLO, as earlier reported by Findlay et al. (4) . Multiplication of the virus in the brain was a prerequisite but encephalitis was not essential. AI-though direct proof is lacking, it is tempting to attribute the growth enhancement to fatty substances liberated in the brain by the virus. Edward (7) and Smith, Lecce, and Lynn (8) have presented evidence that lipoids, probably lipoproteins, are generally needed for the growth of PPLO in vitro.\", 'Rolling occurred so infrequently in mice injected with the mixture of MHV and PPLO that it could hardly be called a characteristic sign. It was also observed in an occasional mouse injected with MHV(C) alone. This virus was productive of considerable injury to the brain but rarely affected the nerve cells of the cerebral ganglia. In the absence of the virus or agar PPLO showed little or no evidence of neurotropism. The intracerebral injection of both microbic agents commonly resulted, however, in extensive destruction of the nerve cells in the anterior horns of the cerebrum. This outcome was indicative of an interaction between the 2 agents and suggestive of synergism.', 'Our findings in regard to the effect of agar on the cerebral behavior of PPLO in mice agreed with the earlier observations of Findlay et al. (4) as to its enhancing properties but differed from theirs in detail. In our experiments none of the mice injected with the agar-PPLO mixture showed any outward signs of nervous disorder. In Swiss mice there was a marked difference between agar and MHV(C) in respect to the number of individuals with an increased growth of PPLO and a leukocytic response at autopsy. The actual rates were about 30 per cent with agar and practically 100 per cent with the virus. Princeton mice were more responsive, the rate with agar being 60 per cent. The present observations suggest that hydrocephalus which frequently appeared in the injected mice was in part a response to the agar as the same condition also followed the injection of a sterile agar inoculum. SUMMARY Pleuropneumonia-like organisms (PPLO) of the catarrhal type were isolated from the brain of a Swiss mouse during the cranial passage of mouse hepatitis virus-MHV(C). Cranial injection of the PPLO alone in Swiss and Princeton weanlings was attended by a meagre growth of the organisms in the brain, with no pathologic change.', 'The growth of both catarrhal and conjunctival strains of PPLO in the brains of Swiss mice was greatly enhanced by the simultaneous injection of MHV(C). Rolling was not a characteristic sign prior to autopsy. Brain sections regularly showed a vigorous leukocytic response, commonly accompanied by the destruction of nerve cells in the anterior horns of the cerebrum. Injected in Princeton mice together with the virus, the organisms barely survived and were inactive. MHV(Pr) enhanced the growth and pathogenicity of PPLO in the brains of Princeton mice but failed to do so in Swiss.', 'The behavior of PPLO in the brain was likewise affected by the presence of agar, as earlier observed by Findlay et al. In comparison with the effect of MHV, the enhancement was reduced in rate in both strains of mice and was not accompanied by outward signs of nervous disorder. Hydrocephalus which often followed injection of the PPLO-agar mixture was also produced by agarbouillon alone.', 'All of the following cross-sections of the brain were stained with eosin-rnethyl ene blue. They were photographed by Mr. J. A. Carlile. '], 'cord_uid': 'vhfticne'}\n"
     ]
    }
   ],
   "source": [
    "for k, v in cov_dm.paper_dict.items():\n",
    "    print(v)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 91,
   "metadata": {},
   "outputs": [],
   "source": [
    "cov_dm.save_docs_dict_as_pickle()\n",
    "\n",
    "cov_dm.load_docs_dict_from_pickle()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 92,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'title': 'Antisense-induced ribosomal frameshifting', 'text': ['The standard triplet readout of the genetic code can be reprogrammed by signals in the mRNA to induce ribosomal frameshifting [reviewed in (1–3)]. Generally, the resulting trans-frame protein product is functional and may in some cases be expressed in equal amounts to the product of standard translation. This elaboration of the genetic code (4,5) demonstrates versatility in decoding.', 'Requirements for eukaryotic ribosomal frameshifting include a shift-prone sequence at the decoding site and often a downstream secondary structure in mRNA. The majority of −1 programmed frameshift sites consist of a heptanucleotide sequence X XXY YYZ [where X can be A, G, C or U; Y can be A or U; and Z can be any nucleotide (6)]. In this configuration, the P- and A-site tRNAs can re-pair with at least 2 out of 3 nt when shifted 1 nt towards the 5′ end of the mRNA. Similarly, for +1 frameshift sites, the identity of the codons in the P- and A-sites of the ribosome is critical for efficient frameshifting. One factor affecting +1 frameshift efficiency is the initial stability of the P-site tRNA–mRNA interaction in the 0 frame (7). High-efficiency frameshifting occurs when the P-site tRNA does not form standard codon–anticodon interactions (8). In some studies, a correlation between +1 frameshift efficiency and the final stability of the P-site tRNA–mRNA interaction in the +1 frame has been shown previously (9,10). However, in other systems there appears to be little correlation (11). In addition, competition between decoding of the 0 frame and +1 frame codons in the A-site may affect frameshifting efficiency (7). Slow to decode 0 frame codons such as stop codons or those decoded by low abundance tRNAs favor frameshifting, as do +1 frame codons with high levels of corresponding cognate tRNAs (12–16).', 'High levels of frameshifting are often achieved by the stimulatory action of a cis-acting element located downstream of the shift site. A wide variety of structures, most commonly H-type pseudoknots (17), have been identified which stimulate −1 frameshifting in eukaryotes [for reviews see (18,19)]. Mutagenic and structural data for several of the frameshift stimulators have demonstrated that each pseudoknot has key structural features required for frameshift stimulation (20–28). However, unifying structural feature essential for frameshifting has not yet been identified. This observation combined with recent reports that simple antisense oligonucleotides can functionally mimic cis-acting 3′ stimulators of −1 frameshifting (29,30) demonstrates that many different structures can stimulate frameshifting. Although it should be noted that not all structures of equal thermodynamic stability can stimulate frameshifting (Discussion).', 'RNA pseudoknots have also been shown to stimulate programmed +1 frameshifting in many eukaryotic antizyme genes (31,32). Antizyme is a negative regulator of cellular polyamine levels through its ability to target ornithine decarboxylase (the rate-limiting enzyme in polyamine biosynthesis) for degradation (33–35), inhibits polyamine import (36,37) and stimulates export (38). Antizyme expression is induced by high-intracellular polyamine levels, and decreased with lowered levels. The polyamine sensor is a programmed +1 frameshift event that is required for antizyme synthesis. At low polyamine levels, termination at the end of open reading frame 1 (ORF1) is efficient, whereas at high levels of polyamines, a substantial proportion of ribosomes shift to the +1 reading frame and then resume standard decoding to synthesize the full-length and active antizyme protein. Frameshifting at the mammalian antizyme mRNA shift site, UCC UGA, is stimulated by two cis-acting signals (39,40). One of these, the 5′ element, encompasses ∼50 bases upstream of shift site and is important for the polyamine effect (39–41). The other cis-acting element is a pseudoknot located 3′ of the shift site. The mammalian antizyme pseudoknot and a structurally distinct counterpart in a subset of invertebrate antizyme mRNAs (31) are the only pseudoknots known to act as stimulators for +1 frameshifting in eukaryotes.', 'Although it is unknown if pseudoknots stimulate −1 frameshifting and +1 frameshifting by different mechanisms, one notable difference is found in positioning of the downstream structure relative to the shift site. Naturally occurring pseudoknots or stem–loop stimulators of −1 frameshifting typically begin ∼6–9 nt downstream of the A-site codon of the shift site (18), whereas +1 frameshift pseudoknots are located closer with only a 2–3 nt separation from the A-site codon (31). Mutagenic studies have revealed that altering the size of the spacer affects frameshifting and, in general, reduces efficiency (27,31,42–44).', 'Here we have tested the ability of antisense oligonucleotides, annealed downstream of the shift-prone site, UCC UGA, to induce shifting of the ribosome to the +1 reading frame. The directionality of frameshifting (either into the +1 or −1 reading frame) is shown to be dependent upon the position of the duplex region relative to the shift site, and the efficiency of frameshifting is responsive to polyamine levels and enhanced by the inclusion of stimulatory sequences found upstream of the human antizyme +1 programmed frameshift site.', 'Complementary oligonucleotides, to construct the sequences described in this paper, were synthesized at the University of Utah DNA/Peptide Core Facility such that when annealed they would have appropriate ends to ligate into the SalI/BamHI sites of the dual luciferase vector, p2luc (45). Dual luciferase constructs were prepared and their sequence was verified as described previously (46).', 'Insert sequences with shift site in boldface is given as follows:', 'P2lucAZ1wt: TCGACGGTCTCCCTCCACTGCTGTAGTAACCCGGGTCCGGGGCCTCGGTGGTGCTCCTGATGCCCCTCACCCACCCCTGAAGA TCCCAGGTGGGCGAGGGAATAGTCAGAGGGATCACAACGGATC;', 'P2lucAZ10sp: TCGACGGTCTCCCTCCACTGCTGTAGTAACCCGGGTCCGGGGCCTCGGTGGTGCTCCTGACCCTCACCCACCCCTGAA GATCCCAGGTGGGCGAGGGAATAGTCAGAGGGATCACAACGGATC;', 'P2lucAZ1hp: TCGACGGTCTCCCTCCACTGCTGTAGTAACCCGGGTCCGGGGCCTCGGTGGTGCTCCTGATGCCCCTCACCCACCCCTGAAGA TCCCAGGTGGGCGAGGGAATGGATC;', 'P2lucAZ1PKdel: TCGACGGTCTCCCTCCACTGCTGTAGTAACCCGGGTCCGGGGCCTCGGTGGTGCTCCTGATGCCCCTGGATC;', 'P2lucAZ1PKm1: TCGACGGTCTCCCTCCACTGCTGTAGTAACCCGGGTCCGGGGCCTCGGTGGTGCTCCTGATGCCCCTCACCCACCGGGATCACA AGGATC;', 'P2lucAZ1sl: TCGACGGTCTCCCTCCACTGCTGTAGTAACCCGGGTCCGGGGCCTCGGTGGTGCTCCTGATGCCCCTCACCCACCCGGATC;', 'P2lucAZ1FS: TCGACGTGCTCCTGATGCCCCTGGATC;', 'P2lucAZ1FSUGG: TCGACGTGCTCCTGGTGCCCCTGGATC.', '2′-O-Methyl antisense oligonucleotides were synthesized by Integrated DNA Technologies (Coralville, IA).', 'AZ1A: AGUUGAAGGAUCCAGGGGCA; AZ1B: GGAAGUUGAAGGAUCCAGGG; AZ1C: CAGGGAAGUUGAAGGAUCCA; PKm1: GAUCCCGGUGGGUGAGGG; PKm2: GAUCCCAGGUGGGCGAGGG; SL1: GGUGGGUGAGGG; and SL2: GGAUCCGGGUGGGUGAGGG.', 'The dual luciferase constructs (0.1 μg) described above were added directly to TNT coupled reticulocyte lysate reactions (Promega) with 35S-labeled methionine in a volume of 10 μl. Reactions were incubated at 30°C for 1 h. Radiolabeled proteins were separated by SDS–PAGE and the gels were fixed with 7.5% acetic acid and methanol for 20 min. After drying under vacuum, the gels were visualized using a Storm 860 PhosphorImager (Molecular Dynamics) and radioactive bands quantified using ImageQuant software. Percent frameshifting was calculated as the percentage of full-length (frameshift) product relative to the termination product and the full-length product combined. The value of each product was corrected for the number of methionine codons present in the coding sequence. The reported values are the average and standard deviations obtained from at least three independent measurements. Tables showing percent frameshifting and standard deviations can be found in Supplementary Data.', 'Plasmid p2lucAZ1PKdel was co-transfected into CV-1 cells with varying concentrations of AZ1B 2′-O-Methyl antisense oligonucleotides under the following conditions. CV-1 cells (1.5 × 104) in 50 μl of DMEM + 5% fetal bovine serum were added to wells (1/2 area 96-well tissue culture treated plates) containing 25 ng of DNA, varying amounts of AZ1B antisense oligonucleotides and 0.4 μl Lipofectamine 2000 (Invitrogen) in 25 μl of Optimem. Cells were incubated at 37°C (5% CO2) for 20 h. Media were then removed from the cells and the transfected cells were lysed in 12.5 μl lysis buffer and luciferase activity determined by measuring light emission following injection of 25 μl of luminescence reagent (Promega). Percent frameshifting was calculated by comparing firefly/Renilla luciferase ratios of experimental constructs with those of control constructs: (firefly experimental RLUs/Renilla experimental RLUs)/(firefly control RLUs/Renilla control RLUs) × 100.', 'Initially, three dual luciferase reporter vectors were generated containing the human antizyme 1 frameshift cassette (p2luc-AZ1wt) with the 5′ and 3′ stimulators of frameshifting, with the pseudoknot deleted (p2luc-AZ1PKdel), or replaced with a stem–loop (p2luc-AZ1hp) (Figure 1). Each constructs was then subjected to coupled transcription and translation reactions in the presence of increasing amounts of spermidine, and the 35S-labeled products separated by SDS–PAGE. Frameshifting efficiency was measured by comparing the amount of full-length frameshift product (+1 FS) to the product of termination (Term) at the shift site stop codon (Figure 2 and Supplementary Table 1). Maximum levels of frameshifting (AZ1wt 5.6%, AZ1PKdel 2.1% and AZ1hp 1.5%) were observed in the presence of 0.4 mM spermidine. Low-level frameshifting, 0.1% or less, was observed in the absence of exogenous spermidine.', '2′-O-Methyl antisense oligonucleotides were designed to anneal downstream of the UCC UGA shift site of RNA produced from p2luc-AZ1PKdel (AZ1PKdel) such that the 3′ ends were located 0 (AZ1A), 3 (AZ1B) or 6 (AZ1C) nt downstream of the UGA codon of the shift site (Figure 1B). Frameshift efficiency was measured following transcription/translation reactions of p2luc-AZ1PKdel in the presence of 2 μM of each antisense oligonucleotide and increasing amounts of spermidine (Figure 3A–C and Supplementary Table 2). Maximal levels of frameshifting were found to occur when 2–4 μM of antisense oligonucleotide was added to the transcription/translation reactions (Supplementary Table 3). In the presence of 0.4 mM exogenous spermidine, highly efficient shifting of ribosomes into the +1 reading frame (higher than that observed in the wild-type antizyme frameshift cassette) was observed with the addition of AZ1A (26.1%), AZ1B (51.8%) and AZ1C (31.8%) (Supplementary Table 2). The most efficient frameshifting is observed with the antisense oligonucleotide AZ1B which anneals such that spacing between the shift site and the beginning of the duplex region is the same as that observed between the shift site and the beginning of stem 1 of the natural antizyme 3′ pseudoknot structure (i.e. each has a 3 nt spacer). To verify that the antisense oligonucleotide was activating ribosomal frameshifting and not transcription slippage, RNA was transcribed from p2luc-AZ1PKdel in the absence of oligonucleotide and added to reticulocyte lysate translations in the presence of increasing amounts of 2′-O-Methyl AZ1B oligonucleotide. Frameshifting levels were increased to the same level as that observed in coupled transcription and translation reactions demonstrating that the oligonucleotide acts to induce frameshifting during translation (Supplementary Figure).', 'Surprisingly, the addition of AZ1A (0 spacer) also induced high-level frameshifting into the −1 reading frame in a manner which was modestly inhibited by the addition of spermidine (19% in the absence and 10% in the presence of 0.4 mM exogenous spermidine) (Figure 3A and Supplementary Table 2). No −1 frameshift product was observed when the wild-type antizyme cassette was examined in the absence of antisense oligonucleotide addition (Figure 2; AZwt). As the AZ1A antisense oligonucleotide was designed to anneal directly adjacent to the UGA codon of the shift site, it was of interest to determine whether the wild-type antizyme pseudoknot could induce −1 frameshifting when located in the equivalent position. To address this, a new construct p2luc-AZ1-0sp (Figure 1A) was made by deleting the 3 nt spacer between the pseudoknot and the shift site of p2luc-AZ1wt. In this case, the wild-type pseudoknot is directly 3′ adjacent to the shift site. The products of in vitro transcription and translation were separated by SDS–PAGE. No −1 frameshift product was observed and levels of the +1 frameshift product were significantly reduced to ∼3% (Figure 3D and Supplementary Table 2).', 'AZ1A, AZ1B and AZ1C were designed to complement RNA sequences encoded by the originating vector. To determine if duplexes formed between the antisense oligonucleotide and 3′ adjacent antizyme sequences would result in more efficient frameshift stimulation, reporter vectors were designed to contain a portion of the antizyme 3′ stimulator. Construct p2luc-AZ1PKm1 contains sequences from the 5′ half of the axis formed by the stacking of stem 1 and stem 2 of the pseudoknot (Figure 1A). Two complementary 2′-O-Methyl antisense oligonucleotides were designed. First, PKm1 has perfect complimentarity to the region starting 3 nt and ending 21 nt downstream of the UGA shift site codon. Second, PKm2 is the same except that a mispaired C and bulged A were located at positions 9 and 16, respectively. These two alterations were included to more closely mimic the natural pseudoknot which also contains a mispaired C and bulged A at equivalent positions along the extended stem formed by the stacking of pseudoknot stems 1 and 2 (Figure 1; compare p2luc-AZ1wt with the duplex formed between p2luc-PKm1 and antisense oligonucleotide PKm2). PKm1 and PKm2 induced 30 and 22% frameshifting, respectively, when added to coupled transcription and translation reactions of p2luc-AZ1PKm1 in the presence spermidine (Figure 4A and B, and Supplementary Table 4). Neither PKm1 nor Pkm2 induced frameshifting to the same levels seen with AZ1B, suggesting that the sequence content of the duplex region can affect the efficiency of frameshift stimulation and that native antizyme sequences are not required.', 'A second construct, p2luc-AZ1sl, was designed to contain only the 5′ half of stem 1 of the antizyme pseudoknot downstream from the UCC UGA shift site (Figure 1A). 2′-O-Methyl antisense oligonucleotides were designed to anneal between 3 and 15 nt (SL1) or 3 and 22 nt (SL2) downstream from the UGA codon of the shift site. Frameshift efficiency induced by these two antisense oligonucleotides, 8 and 22% respectively, was somewhat lower than that observed with PKm1 and PKm2 (Figure 4C and D and Supplementary Table 4). In these cases frameshift efficiency was higher for the longer antisense oligonucleotide (SL2), suggesting that frameshift efficiency most probably correlates with stability of the duplex. As was seen with AZ1A, AZ1B and AZ1C, frameshifting efficiency stimulated by antisense oligonucleotides PKm1, PKm2, SL1 and SL2 was also strongly correlated with the concentration of exogenously added spermidine (Supplementary Table 5).', 'The importance of the antizyme 5′ sequence context to antisense oligonucleotide induced ribosome frameshifting was examined by testing the frameshift site, UCC UGA, without the 5′ and 3′ stimulatory antizyme sequences. To this end, the 5′ antizyme stimulatory sequences were deleted from p2luc-AZ1PKdel to make p2luc-AZ1FS. Each of the antisense oligonucleotides AZ1A, AZ1B or AZ1C was added to coupled transcription and translation reactions with p2luc-AZ1FS in the presence or absence of spermidine. Frameshift efficiency was measured at 11, 8 and 4%, in the presence of spermidine and 3, 0.4 and 0.2% in its absence for AZ1A, AZ1B and AZ1C, respectively (Figure 5A and B).', 'To determine whether the stop codon of the shift site is essential for frameshifting, the UGA codon of p2luc-AZ1FS was altered to UGG such that the shift site was UCC UGG (p2luc-AZ1-UGG). Frameshift efficiency was significant, but reduced, compared to the shift site UCC UGA, and shows little stimulation by the addition of spermidine; AZ1A, AZ1B and AZ1C induced 3, 1 and 1.4% frameshifting in the presence of spermidine, and 1.9, 0.8 and 1.7% frameshifting in its absence, respectively (Figure 5C and D).', 'The ability of antisense oligonucleotides to induce frameshifting in cultured mammalian cells was examined by co-transfection of CV-1 cells with p2lucAZ1PKdel and increasing amounts of 2′-O-Methyl antisense oligonucleotides AZ1B as described in Materials and Methods. In the absence of antisense oligonucleotide frameshifting levels were determined to be 1.1%, whereas a graded increase in frameshift levels was observed upon the addition of AZ1B (Figure 6). Maximal frameshifting levels were 13% in the presence of 2 μM AZ1B in the transfection media.', 'Several models attempting to explain pseudoknot stimulation of programmed −1 frameshifting have been proposed [for reviews see (18,19)]. Most models invoke a pausing mechanism whereby the ribosome is paused over the shift site such that time is allowed for the tRNAs to reposition in the new reading frame. This explanation is clearly too simplistic as stem–loops and pseudoknots of similar thermodynamic stability that cause ribosome pausing are not necessarily effective frameshift stimulators (47–49). In addition, variations of the IBV pseudoknot have demonstrated a lack of correlation between the extent of pausing and the efficiency of frameshifting (47). A recent publication by Brierley and co-workers (50) presents structural data demonstrating that the IBV frameshift stimulating pseudoknot blocks the mRNA entrance tunnel and leads to a structural deformation of the P-site tRNA. The resulting movement of the tRNA displaces the anticodon loop towards the 3′ end of the mRNA. A model is presented in which this movement results in disruption of the codon–anticodon interactions, thus allowing for tRNA slippage relative to the mRNA. Similar tRNA movements were not observed with non-frameshift stimulating stem–loop structures. This model provides a feasible mechanistic explanation for the ability of some downstream structures to induce frameshifting.', 'The ability of antisense oligonucleotides to induce high-level −1 frameshifting (29,30) demonstrates that elaborate tertiary structures are not required, and that a duplex formed by complementary antisense oligonucleotides (with a variety of chemistries, including RNA, 2′-O-Methyl, morpholino) is sufficient to induce high-level frameshifting. Here we demonstrate for the first time that trans-acting antisense oligonucleotides may stimulate ribosome shifting to the +1 reading frame at surprisingly high levels, levels which are greater than those achieved by natural 3′ cis-acting mRNA pseudoknot structures in programmed +1 frameshifting.', 'Structural studies indicating that the mRNA begins to enter the ribosome 7–9 nt downstream from the A-site codon is of direct relevance to this study (50,51). Our results indicate that maximal frameshifting is induced when the antisense–mRNA duplex begins 3 nt downstream of the UGA of the shift site, in agreement with the distance found between the UGA of the shift site and the beginning of stem 1 of the pseudoknot stimulator found in antizyme genes. Given this distance, the implication is that the stimulatory secondary structure would be encountered by the ribosome when the UCC codon enters the A-site of the ribosome. Perhaps as suggested by the structural studies of the IBV-1 frameshift inducing pseudoknot, the codon–anticodon interactions between the UCC codon and Ser-tRNASer are disrupted during translocation to the P-site. Given the importance of the UGA codon during frameshifting at the UCC UGA shift site, subsequent events following translocation of the UCC codon to the P-site and UGA to the A-site must influence frameshifting efficiency. This latter event most probably involves competition between termination and +1 frame decoding when the UGA codon is in the A-site. Various discussions have been presented for the importance of A-site and P-site events during ribosomal frameshifting (7,52) and clearly, further investigations of this topic are warranted.', 'The observation presented here that the antisense oligonucleotide, AZ1A, which anneals directly adjacent to the UGA stop codon can induce ribosome frameshifts to either the +1 or −1 reading frame is surprising. In light of the above discussion of spacing for naturally occurring cis-acting frameshift stimulators, it is possible that frameshifting may occur at codons upstream of the known UCC UGA shift site. However, visual examination of upstream codons does not reveal an obvious −1 or +1 frameshift site.', 'The ability of spermidine to stimulate antisense oligonucleotide induced ribosome frameshifting to the +1 reading frame at the UCC UGA shift site in the absence of the natural 3′ stimulator demonstrates that this cis-acting element is not required for polyamine responsiveness. Similarly, spermidine stimulation was observed in the absence of the 5′ element but virtually eliminated by altering the UGA codon of the shift site to UGG. These observations are in agreement with previous studies examining the importance of cis-acting elements for polyamine induced frameshifting during expression of antizyme genes (39–41).', 'Finally, the ability to direct ribosomes to the +1 reading frame in living cells (Figure 6) suggests a potential therapeutic application for antisense oligonucleotides. Directed frameshifting to the +1 reading frame near a disease causing −1 frameshift mutation would cause some ribosomes to resume decoding in the wild-type ORF, thus restoring partial production of full-length protein from mutant alleles. The importance of the stop codon for efficient frameshifting suggests that the stop codon following the frameshift mutation presents a promising target for antisense induced phenotypic suppression, and that modulation of intracellular polyamine levels, although not essential, may increase the effectiveness of this approach. Further experiments are required to determine the therapeutic potential of this approach in vivo including the generality and efficiency of frameshift induction at non-programmed frameshift sites.', 'Supplementary Data are available at NAR Online.'], 'cord_uid': 'xgwbl8em'}\n{'title': 'The gene of an archaeal α-l-fucosidase is expressed by translational frameshifting', 'text': ['Translation is optimally accurate and the correspondence between the nucleotide and the protein sequences are often considered as an immutable dogma. However, the genetic code is not quite universal: in certain organelles and in a small number of organisms the meaning of different codons has been reassigned and all the mRNAs are decoded accordingly. More surprisingly, the standard rules of genetic decoding are altered in specific genes by different events that are globally termed recoding (1). In all cases, translational recoding occurs in competition with normal decoding, with a proportion of the ribosomes not obeying to the ‘universal’ rules. Translational recoding has been identified in both prokaryotes and eukaryotes. It has crucial roles in the regulation of gene expression and includes stop codon readthrough, ribosome hopping and ±1 programmed frameshifting [for reviews see (2–4)].', 'In stop codon readthrough a stop codon is decoded by a tRNA carrying an unusual amino acid rather than a translational release factor. Specific stimulatory elements downstream to the stop codon regulate this process (5). Hopping, in which the ribosome stops translation in a particular site of the mRNA and re-start few nucleotides downstream, is a rare event and it has been studied in detail only in the bacteriophage T4 (6). In programmed frameshifting, ribosomes are induced to shift to an alternative, overlapping reading frame 1 nt 3′-wards (+1 frameshifting) or 5′-wards (−1 frameshifting) of the mRNA. This process is regulated and its frequency varies in different genes. The ±1 programmed frameshifting has been studied extensively in viruses, retrotransposons and insertion elements for which many cases are documented (7–9). Instead, this phenomenon is by far less common in cellular genes. A single case of programmed +1 frameshifting is known in prokaryotes (10,11) while in eukaryotes, including humans, several genes regulated by this recoding event have been described previously [(4) and references therein]. Compared to +1 frameshifting, −1 frameshifting is less widespread with only two examples in prokaryotes (12–14) and few others in eukaryotes (15–17).', 'The programmed −1 frameshifting is triggered by several elements in the mRNA. The slippery sequence, showing the X-XXY-YYZ motif, in which X can be any base, Y is usually A or U, and Z is any base but G, has the function of favouring the tRNA misalignment and it is the site where the shift takes place (3,18). Frameshifting could be further stimulated by other elements flanking the slippery sequence: a codon for a low-abundance tRNA, a stop codon, a Shine–Dalgarno sequence and an mRNA secondary structure. It has been reported that these elements, alone or in combination, enhance frameshifting by pausing the translating ribosome on the slippery sequence (4,18).', 'Noticeably, known cases of recoding in Archaea [recently reviewed in (19)] are limited to termination codon readthrough events that regulate the incorporation of the 21st and 22nd amino acids selenocysteine and pyrrolysine, respectively (20–23).', 'No archaeal genes regulated by translational programmed frameshifting and ribosome hopping have been identified experimentally so far; therefore, if compared with the others domains of life, the study of translational recoding in Archaea is still at its dawn.', 'We showed that the α-l-fucosidase gene from the crenarchaeon Sulfolobus solfataricus is putatively expressed by programmed −1 frameshifting (24). This gene, named fucA1, is organized in the open reading frames (ORFs) SSO11867 and SSO3060 of 81 and 426 amino acids, respectively, which are separated by a −1 frameshifting in a 40 base overlap (Figure 1A). We have reported previously that the region of overlap between the two ORFs had the characteristic features of the genes expressed by programmed −1 frameshifting including a slippery heptanucleotide A-AAA-AAT (codons are shown in the zero frame) flanked by a putative stem–loop and the rare codons CAC (Figure 1A) resembling the prokaryotic stem–loops/hairpins and the Shine–Dalgarno-like sites (24). We showed that the frameshifting, obtained by mutating by site-directed mutagenesis the fucA1 gene exactly in the position predicted from the slippery site, produced a full-length gene, named fucA1A, encoding for a polypeptide of 495 amino acids (Figure 1B). This mutant gene expressed in Escherichia coli a fully functional α-l-fucosidase, named Ssα-fuc, which was thermophilic, thermostable and had an unusual nonameric structure (24,25). More recently, we determined the reaction mechanism and the function of the residues of the active site of the mutant enzyme (26,27).', 'The functionality of the product of the mutant gene fucA1A does not provide direct experimental evidence that programmed −1 frameshifting occurs in vivo and in S.solfataricus. To address these issues, we report here the study of the expression of the wild-type split gene fucA1 and of its mutants in the slippery sequence. We demonstrate here that fucA1 is expressed by programmed −1 frameshifting in both E.coli and S.solfataricus. This is the first experimental demonstration that this kind of recoding is present in the Archaea domain of life. The relevance of programmed −1 frameshifting in Archaea is also discussed.', 'S.solfataricus cells were grown, and cell extracts obtained, as described previously (24,28).', 'The expression in the E.coli strain BL21(RB791) of the wild-type gene fucA1 and of the mutant genes fucA1A [previously named FrameFuc in (24)], fucA1B, fucA1sm and fucA1tm as fusions of glutathione S-transferase (GST) and the purification of the recombinant proteins were performed as reported previously (23). The nomenclature used in this paper for the different α-fucosidase genes is listed in Table 1.', \"For the western blot studies, equal amounts of E.coli cultures expressing the wild-type and mutant fucA1 genes, normalized for the OD600, were resuspended in SDS–PAGE loading buffer containing 0.03 M Tris–HCl buffer, pH 6.8, 3% SDS (w/v), 6.7% glycerol (w/v), 6.7% 2-mercaptoethanol (w/v) and 0.002% blue bromophenol (w/v). The samples were incubated at 100°C for 5 min (unless otherwise indicated) and were directly loaded on to the gel. Western blot analyses were performed by blotting SDS–PAGEs of the concentrations indicated on Hybond-P polyvinylidenfluorid filters (Amersham Biosciences, Uppsala, Sweden); polyclonal anti-Ssα-fuc antibodies from rabbit (PRIMM, Milan, Italy) and anti-GST antibodies (Amersham Biosciences) were diluted 1:5000 and 1:40 000, respectively. The filters were washed and incubated with the ImmunoPure anti-rabbit IgG antibody conjugated with the horseradish peroxidase (HRP) from Pierce Biotechnology (Rockford, IL, USA). Filters were developed with the ECL-plus Western Blotting Detection system (Amersham Biosciences) by following the manufacturer's indications. The molecular weight markers used in the western blot analyses were the ECL streptavidin–HRP conjugate (Amersham Biosciences).\", 'The protein concentration of the samples was measured with the method of Bradford (29) and the amounts of sample loaded on to the SDS–PAGEs are those indicated. The quantification of the bands identified by western blot was performed by using the program Quantity One 4.4.0 in a ChemiDoc EQ System (Bio-Rad, Hercules, CA, USA) with the volume analysis tool. The frameshifting efficiency was calculated as the ratio of the intensity of the bands of the frameshifted product/frameshifted product + termination product.', 'The mutants in the slippery sequence of the wild-type gene fucA1 were prepared by site-directed mutagenesis from the vector pGEX-11867/3060, described previously (24,27). The synthetic oligonucleotides used (PRIMM) were the following: FucA1sm-rev, 5′-TTTAGGTGATATTGGTGTTCTGGTCTATCT-3′; FucA1sm-fwd, 5′-GAACACCAATATCACCTAAAGAATTCGGCCCA-3′; FucA1tm-rev, 5′-AGGTGATATTGGTGTTCTGGTCTATCTGGC-3′; FucA1tm-fwd, 5′-CCAGAACACCAATATCACCTCAAGAACTCGGCCCAGT-3′, where the mismatched nucleotides in the mutagenic primers are underlined. Direct sequencing identified the plasmids containing the desired mutations and the mutant genes, named fucA1sm and fucA1tm, were completely re-sequenced.', 'The mutant Ssα-fucB was prepared by site-directed mutagenesis from the vector pGEX-11867/3060, by using the same site-directed mutagenesis kit described above. The synthetic oligonucleotides used were FucA1sm-rev (described above) and the following mutagenic oligonucleotide: Fuc-B, 5′-GAACACCAATATCACCTAAAGAAGTTCGGCCCAGT-3′, where the mismatched nucleotides are underlined. Direct sequencing identified the plasmid containing the desired mutations and the mutant gene, named fucA1B, was completely re-sequenced. The enzymatic characterization of Ssα-fucB was performed as described previously (24,27).', 'Samples of the proteins expressed in E.coli from the wild-type gene fucA1 and the mutants fucA1A and fucA1sm, purified as described, were fractionated on an SDS–PAGE. Protein bands were excised from the gel, washed in 50 mM ammonium bicarbonate, pH 8.0, in 50% acetonitrile, reduced with 10 mM DTT at 56°C for 45 min and alkylated with 55 mM iodoacetamide for 30 min at room temperature in the dark. The gel pieces were washed several times with the buffer, resuspended in 50 mM ammonium bicarbonate and incubated with 100 ng of trypsin for 2 h at 4°C and overnight at 37°C. The supernatant containing peptides was analysed by MALDIMS on an Applied Biosystem Voyager DE-PRO mass spectrometer using α-cyano-4-hydroxycynnamic acid as matrix. Mass calibration was performed by using the standard mixture provided by manufacturer.', 'Liquid chromatography online tandem mass spectrometry (LCMSMS) analyses were performed on a Q-TOF hybrid mass spectrometer (Micromass, Waters, Milford, MA, USA) coupled with a CapLC capillary chromatographic system (Waters). Peptide ions were selected in the collision cell and fragmented. Analysis of the daughter ion spectra led to the reconstruction of peptide sequences.', 'Genomic DNA from S.solfataricus P2 strain was prepared as described previously (24). A DNA fragment of 1538 nt containing the complete fucA1 gene, was prepared by PCR, by using the following synthetic oligonucleotides (Genenco, Florence, Italy): FucA1-fwd, 5′-CTGGAGGCGCGCTAATACGACTCACTATAGGTCAGTTAAATGTCACAAAATTCT-3′; FucA1-rev, 5′-GACTTGGCGCGCCTATCTATAATCTAGGATAACCCTTAT-3′, in which the sequence corresponding to the genome of S.solfataricus is underlined. In the FucA1-fwd primer, the sequence of the promoter of the T7 RNA polymerase is in boldface and the sequence of the BssHII site is shown in italics. The PCR amplification was performed as described previously (24) and the amplification products were cloned in the BssHII site of the plasmid pBluescript II KS+. The fucA1 gene was completely re-sequenced to check if undesired mutations were introduced by PCR and the recombinant vector obtained, named pBlu-FucA1, was used for translation in vitro experiments.', 'The plasmids expressing the mutant genes fucA1A, fucA1sm and fucA1tm for experiments of translation in vitro were prepared by substituting the KpnI–NcoI wild-type fragment, containing the slippery site, with those isolated from the mutants. To check that the resulting plasmids had the correct sequence, the mutant genes were completely re-sequenced.', 'The mRNAs encoding wild-type fucA1 and its various mutants were obtained by in vitro run-off transcription. About 2 μg of each plasmid was linearized with BssHII and incubated with 50 U of T7 RNA polymerase for 1 h 30 min at 37°C. The transcription mixtures were then treated with 10 U of DNAseI (RNAse free) for 30 min. The transcribed RNAs were recovered by extracting the samples twice with phenol (pH 4.7) and once with phenol/chloroform 1:1 followed by precipitation with ethanol. The mRNAs were resuspended in DEPC-treated H2O at the approximate concentration of 0.6 pmol/μl.', 'In vitro translation assays were performed essentially as described by Condò et al. (28). The samples (25 μl final volume) contained 5 μl of S.solfataricus cell extract, 10 mM KCl, 20 mM Tris–HCl, pH 7.0, 20 mM Mg acetate, 3 mM ATP, 1 mM GTP, 5 μg of bulk S.solfataricus tRNA, 2 μl of [35S]methionine (1200 Ci/mmol at 10 mCi/ml) and ∼10 pmol of each mRNA. The mixtures were incubated at 70°C for 45 min. After this time, the synthesized proteins were resolved by electrophoresis 12.5% acrylamide–SDS gels and revealed by autoradiography of the dried gels on an Instant Imager apparatus.', 'Cells of S.solfataricus, strain P2, were grown in minimal salts culture media supplemented with yeast extract (0.1%), casamino acids (0.1%), plus glucose (0.1%) (YGM) or sucrose (0.1%) (YSM). The extraction of total RNA was performed as reported previously (24). Total RNA was extensively digested with DNAse (Ambion, Austin, TX, USA) and the absence of DNA was assessed by the lack of PCR amplification with each sets of primers described below. The RT–PCR experiments were performed as reported previously (24) by using the primers described previously that allowed the amplification of a region of 833 nt (positions 1–833, in which the A of the first ATG codon is numbered as one) overlapping the ORFs SSO11867 and SSO3060 (24).', 'For real-time PCR experiments total cDNA was obtained using the kit Quantitect RT (Qiagen GmbH, Hilden, Germany) from 500 ng of the same preparation of RNA described above. cDNA was then amplified in a Bio-Rad LightCycler using the DyNAmo HS Syber Green qPCR Kit (Finnzymes Oy, Espoo, Finland). Synthetic oligonucleotides (PRIMM) used for the amplification of a region at the 3′ of the ORF SSO3060 were as follows: 5′-Real: 5′-TAAATGGCGAAGCGATTTTC-3′; 3′-Real: 5′-ATATGCCTTTGTCGCGGATA-3′ for the gene fucA1. 5′-GAATGGGGGTGATACTGTCG-3′ and 5′-TTTACAGCCGGGACTACAGG-3′ for the 16S rRNA gene.', 'For each amplification of the fucA1 gene was used ∼2500-fold more cDNA than that used for the amplification of the 16S rRNA. Controls with no template cDNA were always included. PCR conditions were 15 min at 95°C for initial denaturation, followed by 40 cycles of 10 s at 95°C, 25 s at 56°C and 35 s at 72°C, and a final step of 10 min at 72°C. Product purity was controlled by melting point analysis of setpoints with 0.5°C temperature increase from 72 to 95°C. PCR products were analysed on 2% agarose gels and visualized by ethidium bromide staining.', 'The expression values of fucA1 gene were normalized to the values determined for the 16S rRNA gene. Absolute expression levels were calculated as fucA1/16S ratio in YSM and YGM cells, respectively. Relative mRNA expression levels (YSM/YGM ratio) were calculated as (fucA1/16S ratio in YGM cells)/(fucA1/16S ratio in YSM). Each cDNA was used in triplicate for each amplification.', 'The wild-type fucA1 gene, expressed in E.coli as a GST-fused protein, produced trace amounts of α-fucosidase activity (2.3 × 10−2 units mg−1 after removal of GST), suggesting that a programmed −1 frameshifting may occur in E.coli (24). The enzyme was then purified by using the GST purification system and analysed by SDS–PAGE revealing a major protein band (Figure 2A). The sample and control bands were excised from the gel, digested in situ with trypsin and directly analysed by matrix-assisted laser desorption/ionization mass spectrometry (MALDIMS). As shown in Figure 2B and C, both spectra revealed the occurrence of an identical mass signal at m/z 1244.6 corresponding to a peptide (Peptide A) encompassing the overlapping region of the two ORFs. This result was confirmed by liquid chromatography online tandem mass spectrometry (LCMSMS) analysis of the peptide mixtures. The fragmentation spectra of the two signals showed the common sequence Asn-Phe-Gly-Pro-Val-Thr-Asp-Phe-Gly-Tyr-Lys in which the amino acid from the ORF SSO11867 is underlined. These results unequivocally demonstrate that the protein containing the Peptide A is produced in E.coli by a frameshifting event that occurred exactly within the slippery heptamer predicted from the analysis of the DNA sequence in the region of overlap between the ORFs SSO11867 and SSO3060 (Figure 1A).', 'Remarkably, the MALDIMS analysis of the products of the wild-type fucA1 gene revealed the presence of a second Peptide B at m/z 1258.6 that is absent in the spectra of the Ssα-fuc control protein (Figure 2B and C). The sequence of Peptide B obtained by LCMSMS (Figure 2D) was Lys-Phe-Gly-Pro-Val-Thr-Asp-Phe-Gly-Tyr-Lys. This sequence differs only by one amino acid from Peptide A demonstrating that the interrupted gene fucA1 expresses in E.coli two full-length proteins originated by different −1 frameshifting events. Polypeptide A results from a shift in a site A and it is identical to Ssα-fuc prepared by site-directed mutagenesis (24), suggesting that the expression occurred with the simultaneous P- and A-site slippage. Instead, polypeptide B, named Ssα-fucB, is generated by frameshifting in a second site B as the result of a single P-site slippage (Figure 2E).', 'To measure the global efficiency of frameshifting in the two sites of the wild-type gene fucA1 we analysed the total extracts of E.coli by western blot using anti-GST antibodies (Figure 2F). Two bands with marked different electrophoretic mobility were observed: the polypeptide of 78.7 ± 1.1 kDa migrated like GST-Ssα-fuc fusion and was identified as originated from frameshifting in either site A or B of fucA1. The protein of 38.1 ± 1.2 kDa, which is not expressed by the mutant gene fucA1A (not shown), had an electrophoretic mobility compatible with GST fused to the polypeptide encoded by the ORF SSO11867 solely (27 and 9.6 kDa, respectively). This polypeptide originated from the translational termination of the ribosome at the OCH codon of the fucA1 N-terminal ORF (Figure 1A). The calculated ratio of frameshifting to the termination products was 5%.', 'To test if the full-length α-fucosidase produced by the −1 frameshifting event in site B (Ssα-fucB), resulting from the single P-site slippage has different properties from Ssα-fuc, whose sequence arises from the simultaneous P- and A- site slippage, we prepared the enzyme by site-directed mutagenesis. The slippery sequence in fucA1 A-AAA-AAT was mutated in A-AAG-AAG-T where mutations are underlined. The new mutant gene was named fucA1B. The first G, producing the conservative mutation AAA→AAG, was made to disrupt the slippery sequence and hence reducing the shifting efficiency. The second G was inserted to produce the frameshifting that results in the amino acid sequence of Peptide B. Therefore, the sequence of the two full-length mutant genes fucA1A and fucA1B differs only in the region of the slippery sequence: A-AAG-AAT-TTC-GGC and A-AAG-AAG-TTC-GGC, respectively (the mutations are underlined, the nucleotides in boldface were originally in the −1 frame) (Table 1).', 'The recombinant Ssα-fucB was purified up to ∼95% (Materials and Methods). Gel filtration chromatography demonstrated that in native conditions Ssα-fucB had the same nonameric structure of Ssα-fuc with an identical molecular weight of 508 kDa (data not shown). In addition, Ssα-fucB had the same high substrate selectivity of Ssα-fuc. The two enzymes have high affinity for 4-nitrophenyl-α-l-fucoside (4NP-Fuc) substrate at 65°C; the KM is identical within the experimental error (0.0287 ± 0.005 mM) while the kcat of Ssα-fucB (137 ± 5.7 s−1) is ∼48% of that of Ssα-fuc (287 ± 11 s−1). In addition, 4-nitrophenyl-α-l-arabinoside, -rhamnoside, 4-nitrophenyl-α-d-glucoside, -xyloside, -galactoside and -mannoside were not substrates of Ssα-fucB as shown previously for Ssα-fuc (24). This suggests that the different amino acid sequence did not significantly affect the active site. Both enzymes showed an identical profile of specific activity versus temperature with an optimal temperature higher than 95°C (data not shown). The heat stability and the pH dependence of Ssα-fuc and Ssα-fucB are reported in Figure 3. At 80°C, the optimal growth temperature of S.solfataricus, the half-life of Ssα-fucB is 45 min, almost 4-fold lower than that of Ssα-fuc (Figure 3A). The two enzymes showed different behaviour at pH <6.0 at which Ssα-fucB is only barely active and stable (Figure 3B); however, the two enzymes showed similar values of specific activity at pHs above 6.0, which is close to the intracellular pH of S.solfataricus (30).', 'The experimental data reported above indicate that the predicted slippery heptanucleotide in the region of overlap between the ORFs SSO11867 and SSO3060 of the wild-type gene fucA1 could regulate in cis the frameshifting events observed in E.coli. To test this hypothesis, we mutated the sequence A-AAA-AAT into A-AAG-AAT and C-AAG-AAC (mutations are underlined) obtaining the fucA1 single mutant (fucA1sm) and triple mutant (fucA1tm) genes, respectively. It is worth noting that the mutations disrupt the slippery sequence, but they maintain the −1 frameshift between the two ORFs (Table 1).', 'Surprisingly, the expression of fucA1sm in E.coli produced a full-length polypeptide that, after purification by affinity chromatography and removal of the GST protein, showed the same electrophoretic migration of Ssα-fuc and Ssα-fucB (Figure 4A). This protein was then characterized by mass spectrometry analyses following in situ tryptic digestion. Interestingly, the MALDI spectra revealed the presence of a single peptide encompassing the overlapping region between the two ORFs with a mass value of 1259.7 Da (peptide C; Figure 4B). The sequence of peptide C, determined from the fragmentation spectra obtained by LCMSMS analysis, was Glu-Phe-Gly-Pro-Val-Thr-Asp-Phe-Gly-Tyr-Lys (Figure 4C). Remarkably, apart from the Glu residue, this sequence is identical to that of peptide B produced from fucA1, indicating that in the mutant gene fucA1sm only one of the two frameshifting events observed in the wild-type fucA1 gene had occurred. The presence of a Glu instead of Lys was not unexpected. The mutation A-AAA-AAT→A-AAG-AAT in fucA1sm was conservative in the zero frame of the ORF SSO11867 (AAA→AAG, both encoding Lys), but it produced the mutation AAA→GAA (Lys→Glu) in the −1 frame of the ORF SSO3060.', 'It is worth noting that the frameshifting efficiency of the gene fucA1sm, calculated by western blot as described above, was 2-folds higher (10%) if compared to fucA1 (5%) (Figure 4D). This indicates that the mutation cancelled the frameshifting site A and, in the same time, enhanced the frameshifting efficiency of site B.', 'In contrast, the triple mutant fucA1tm produced in E.coli only the low molecular weight band resulting from translational termination (Figure 4D). No full-length protein could be detected in western blots probed with either anti-GST (Figure 4D) or anti-Ssα-fuc antibodies (Figure 4E). These data show that the disruption of the heptameric slippery sequence completely abolished the frameshifting in E.coli confirming that this sequence has a direct role in controlling the frameshifting in vivo.', 'To test whether fucA1 is expressed in S.solfataricus we analysed the extracts of cells grown on yeast extract, sucrose and casaminoacids medium (YSM). Accurate assays showed that S.solfataricus extracts contained 3.4 × 10−4 units mg−1 of α-fucosidase activity. These very low amounts hampered the purification of the enzyme. The extracts of S.solfataricus cells grown on YSM revealed by western blot a band of a molecular mass >97 kDa and no signals were detected with the pre-immune serum confirming the specificity of the anti-Ssα-fuc antibodies (Figure 5A). The different molecular mass may result from post-translational modifications occurred in the archaeon or from the incomplete denaturation of a protein complex. In particular, the latter event is not unusual among enzymes from hyperthermophilic archaea (31,32). To test which hypotheses were appropriate, cellular extracts of S.solfataricus were analysed by western blot extending the incubation at 100°C to 2 h. Interestingly, this treatment shifted the high-molecular mass band to 67.6 ± 1.2 kDa (Figure 5B and C), which still differs from that of the recombinant Ssα-fuc, 58.9 ± 1.2 kDa, leaving the question on the origin of this difference unsolved. To try to shed some light we immunoprecipitated extracts of S.solfataricus with anti-Ssα-fuc antibodies and we analysed the major protein band by MALDIMS. Unfortunately, we could not observe any peptide compatible with the fucosidase because the heavy IgG chain co-migrated with the band of the expected molecular weight (data not shown).', 'To test if the scarce amounts of the α-fucosidase in S.solfataricus extracts was the result of reduced expression at transcriptional level, we performed a northern blot analysis of total RNA extracted from cells grown either on YSM or YGM media. We could not observe any signal by using probes matching the 3′ of the ORF SSO3060 (data not shown). These results suggest that fucA1 produced a rare transcript; therefore, we analysed the level of mRNA by RT–PCR and by real-time PCR. A band corresponding to the region of overlap between the ORFs SSO11867 and SSO3060 was observed in the RNA extracted from cells grown on YSM and YGM media, demonstrating that under these conditions the two ORFs were co-transcribed (Figure 6A).', 'The experiments of real-time PCR shown in Figure 6B demonstrated that rRNA16S was amplified after ∼17 cycles while the amplification of fucA1 mRNA was observed after 38 cycles, despite the fact that we used ∼2500-fold more cDNA for the amplification of fucA1. This indicates that the gene fucA1 is transcribed at very low level. No significant differences in the fucA1 mRNA level were observed in cells grown in YSM or YGM media. This is further confirmed by the analysis by western blot of the extracts of the same cells of S.solfataricus used to prepare the total RNAs, which revealed equal amounts of α-fucosidase in the two extracts (Figure 6C). Therefore, the low α-fucosidase activity observed under the conditions tested is the result of the poor transcription of the fucA1 gene.', 'To determine whether, and with what efficiency, the −1 frameshifting could be performed by S.solfataricus ribosomes, mRNAs obtained by in vitro transcription of the cloned wild-type fucA1 gene and the mutants thereof were used to program an in vitro translation system prepared as described by Condò et al. (28). To this aim, a promoter of T7 polymerase was inserted ahead of the gene of interest to obtain RNA transcripts endowed with the short 5′-untranslated region of 9 nt observed for the natural fucA1 mRNA (24). Autoradiography of an SDS–PAGE of the translation products (Figure 7) revealed that the wild-type fucA1 transcript produced a tiny but clear band whose molecular weight corresponded to that of the full-length Ssα-fuc obtained by site-directed mutagenesis (24); the latter was translated quite efficiently in the cell-free system in spite of being encoded by a quasi-leaderless mRNA. Judging from the relative intensity of the signals given by the translation products of the wild-type fucA1 and the full-length mutant fucA1A, the efficiency of the −1 frameshifting in the homologous system was ∼10%. No signals corresponding to the polypeptides expected from the separated ORFs SSO11867 and SSO3060 (9.6 and 46.5 kDa, respectively) were observed. However, it should be noted that the product of SSO11867, even if synthesized, is too small to be detected in the gel system employed for this experiment. The larger product of ORF SSO3060, on the other hand, is certainly absent. These data unequivocally demonstrate that the ribosomes of S.solfataricus can decode the split fucA1 gene by programmed −1 frameshifting with considerable efficiency producing a full-length polypeptide from the two ORFs SSO11867 and SSO3060.', 'Remarkably, under the same conditions at which fucA1 drives the expression of the full-length protein, we could not observe any product from the fucA1sm and fucA1tm constructs. These data demonstrate that the integrity of the heptanucleotide is essential for the expression of the fucA1 gene in S.solfataricus, thus further confirming that the gene is decoded by programmed −1 frameshifting in this organism. In addition, the lack of expression of fucA1sm by translation in vitro in S.solfataricus contrasts with the efficient expression of this mutant in E.coli, indicating that the two organisms recognize different sequences regulating the translational frameshifting.', 'The identification of genes whose expression is regulated by recoding events is often serendipitous. In the framework of our studies on glycosidases from hyperthermophiles, we identified in the genome of the archaeon S.solfataricus a split gene encoding a putative α-fucosidase, which could be expressed through programmed −1 frameshifting (24). We tackled this issue by studying the expression of fucA1 in S.solfataricus and in E.coli to overcome the problems connected to the scarcity of expression of the α-fucosidase gene and to the manipulation of hyperthermophiles. As already reported by others, in fact, it is a common strategy to study recoding events from different organisms in E.coli (23,33).', 'The expression in E.coli of the wild-type split gene fucA1 led to the production by frameshifting of two full-length polypeptides with an efficiency of 5%. This is a value higher than that observed in other genes expressed by translational frameshifting in a heterologous system such as the proteins gpG and gpGT (0.3–3.5%) (33).', 'The gene fucA1 is expressed in S.solfataricus at very low level under the conditions tested. In particular, the transcriptional analysis of the gene revealed that it is expressed at very low level in both YSM and YGM media. Similarly, no differences in the two media could be found by western blot probed with anti-Ssα-fuc antibodies, indicating that the low expression of the enzyme in S.solfataricus is the result of scarce transcription rather than suppressed translation.', 'Western blots allowed us to identify a specific band ∼8.7 kDa heavier than that of the recombinant Ssα-fuc and experiments of translation in vitro showed that the wild-type gene expresses a full-length polypeptide exhibiting the same molecular mass of the recombinant protein. This demonstrates that the translational machinery of S.solfataricus is fully competent to perform programmed frameshifting. It seems likely that the observed discrepancy in molecular mass might arise from post-translational modifications that cannot be produced by the translation in vitro. Further experiments are required to characterize the α-l-fucosidase identified in S.solfataricus.', 'MALDIMS and LCMSMS analyses of the products in E.coli of the wild-type split gene fucA1 demonstrated that two independent frameshifting events occurred in vivo in the proposed slippery site. In particular, the sequences obtained by LCMSMS demonstrate that peptide A results from a simultaneous backward slippage of both the P- and the A-site tRNAs (Figure 8A). Instead, the sequence of peptide B is the result of the re-positioning on the −1 frame of only the P-site tRNA; in fact, the next incorporated amino acid is specified by the codon in the new frame (Figure 8B). Therefore, the expression by −1 frameshifting of the wild-type gene fucA1 in E.coli follows the models proposed for ribosomal frameshifting (34). We confirmed the significance of the slippery heptanucleotide in promoting the programmed frameshifting in vivo by mutating the putative regulatory sequence. The triple mutant fucA1tm gave no full-length products; presumably, the mutations in both the P- and in the A-site of the slippery sequence dramatically reduced the efficiency of the −1 frameshifting as observed previously in metazoans (35). This result confirms that the intact slippery sequence in the wild-type gene fucA1 is absolutely necessary for its expression in E.coli. In contrast, surprisingly, the single mutant fucA1sm showed an even increased frequency of frameshifting (10%) if compared to the wild-type and produced only one polypeptide by shifting specifically in site B. We explained this result observing that the mutation in the P-site of the slippery sequence A-AAA-AAT→A-AAG-AAT created a novel slippery sequence A-AAG identical to that controlling the expression by programmed −1 frameshifting of a transposase gene in E.coli (36). Therefore, apparently, the single mutation inactivated the simultaneous P- and A-site tRNA re-positioning and, in the same time, fostered the shifting efficiency of the tRNA in the P-site. It is worth noting that, instead, in S.solfataricus, only the simultaneous slippage is effective (Figure 8B) and even the single mutation in the slippery sequence of fucA1sm completely annulled the expression of the gene. This indicates that this sequence is essential in the archaeon and that programmed frameshifting in S.solfataricus and E.coli exploits different mechanisms. Furthermore, since the only difference between the enzymes produced by the frameshifting sites A and B, Ssα-fuc and Ssα-fucB, respectively, is the stability at 80°C, which is the S.solfataricus physiological temperature, the functionality of Ssα-fucB in the archaeon appears questionable.', 'The reason why fucA1 is regulated by programmed −1 frameshifting is not known. However, the physiological significance of programmed frameshifting has been assigned to a minority of the cellular genes while for most of them it is still uncertain [see (4) and reference therein; (16)]. This mechanism of recoding is exploited to set the ratio of two polypeptides such as the τ and γ subunits of the DNA polymerase III holoenzyme in E.coli (12). Alternatively, programmed frameshifting balances the expression of a protein, as the bacterial translational release factor 2 and the eukaryotic ornithine decarboxylase antizyme [see (4) and (18) and references therein]. In the case of fucA1, the polypeptide encoded by the smaller ORF SSO11867 could never be detected by western blots analyses. In addition, the modelling of Ssα-fuc on the high-resolution crystal structure of the α-l-fucosidase from Thermotoga maritima (25,37) showed that the fucA1 N-terminal polypeptide is not an independent domain. Moreover, we have shown recently that SSO11867 includes essential catalytic residues (27), excluding the possibility that a functional α-fucosidase can be obtained from the ORF SSO3060 alone. Therefore, several lines of evidence allow us to exclude that programmed −1 frameshifting is used to set the ratio of two polypeptides of the α-fucosidase from S.solfataricus. More probably, this translational mechanism might be required to control the expression level of fucA1.', 'Noticeably, this is the only fucosidase gene expressed by programmed −1 frameshifting. Among carbohydrate active enzymes, the only example of expression through this recoding mechanism is that reported for a gene encoding for a α(1,2)-fucosyltransferase from Helicobacter pylori that is interrupted by a −1 frameshifting (38). In this case, the expression by programmed frameshifting would lead to a functional enzyme synthesizing components of the surface lipopolysaccharides to evade the human immune defensive system. It is hard to parallel this model to fucA1. Nevertheless, the monosaccharide fucose is involved in a variety of biological functions (39). Therefore, the α-l-fucosidase might play a role in the metabolism of fucosylated oligosaccharides; experiments are currently in progress to knockout the wild-type fucA1 gene and to insert constitutive functional mutants of this gene in S.solfataricus.', 'FucA1 is the only archaeal α-l-fucosidase gene identified so far; hence, it is probably the result of a horizontal gene transfer event in S.solfataricus. However, since there are no α-fucosidases genes regulated by programmed frameshifting in Bacteria and Eukarya, it is tempting to speculate that this sophisticated mechanism of translational regulation pre-existed in S.solfataricus and it was applied to the fucosidase gene for physiological reasons. The identification of other genes interrupted by −1 frameshifts in S.solfataricus would open the possibility that they are regulated by programmed −1 frameshifting. Recently, the computational analysis of prokaryotic genomes revealed that seven Archaea harbour interrupted coding sequences, but S.solfataricus is not included in this study (40). A computational analysis on several archaeal genomes revealed that 34 interrupted genes are present in the genome of S.solfataricus, 11 of these genes are composed by two ORFs separated by −1 frameshifting and could be expressed by recoding (B. Cobucci-Ponzano, M. Rossi and M. Moracci, manuscript in preparation).', 'We have experimentally shown here, for the first time, that programmed −1 frameshifting is present in the Archaea domain. This finding is the missing piece in the puzzle of the phylogenetic distribution of programmed frameshifting demonstrating that this mechanism is universally conserved.'], 'cord_uid': '69gftii4'}\n{'title': 'Role of RNA helicases in HIV-1 replication', 'text': ['Helicases are enzymes that separate in an energy-dependent manner stretches of duplexed DNA and/or RNA into single-stranded components. Currently, based on characteristic motifs and the sequence comparisons, three superfamilies (SF1 through 3) and two smaller families (F4, F5) of helicases have been identified (1). Superfamily 1 and 2 contain helicases which share seven or more recognized signature amino acid motifs while SF3 and F4 and F5 helicases are characterized generally by three conserved motifs (2); the F4 and F5 proteins are largely bacterial and bacteriophage proteins. Currently, it should be cautioned that many ‘helicases’ are not bona fide helicases, but may only function as RNA translocases, perhaps to fulfill functions in the remodeling of ribonucleoprotein complexes (RNP).', 'DEAD-box and the related DEAH, DExH and DExD (3) helicases are the most numerous members of SF2 and are ubiquitously present in eukaryotic genomes. These helicases share eight conserved motifs and are commonly refered to as the DExH/D family of helicases. Humans, Arabidopsis and Saccharomyces have ∼38, 55 and 25 such entities, respectively (4). Differing from DNA helicases and DExH proteins, DEAD helicases are poor in unwinding long nucleic acid duplexes and are best suited for separating short RNA hybrids. DEAD-box proteins bind with high-affinity RNA–protein complexes while exhibiting little RNA sequence preference. This suggests that the specificity determinants for DEAD helicases may be through the recognition of protein factors. In this regard, a better understanding of the roles for DEAD proteins depends on the clear characterization of their respective interacting proteins.', 'Although the precise substrate for most helicases awaits definition, DEAD helicases are generally thought to participate pleiotropically in many aspects of RNA metabolism including transcription, mRNA splicing, mRNA export, translation, RNA stability and mitochondrial gene expression (5–8). Some examples of helicases and their attributed functions include the following. UAP56, Brr2, Prp16, Prp22 and Prp43 play roles in RNA-splicing (4,9), while Dbp5 (10,11) and DDX3 (12) chaperone RNAs from the nucleus into the cytoplasm. eIF4a and Ded1 serve for translation of mRNAs while Rh1B, Ski2, Dob1, Dhh1 helicases contribute to mRNA stability (4). Other DEAD helicases act in ribosome biogenesis through regulation of small nucleolar RNAs and ribosomal RNAs (rRNAs) interactions (13,14). Finally, Neurospora and Trypanosoma DEAD proteins contribute to mitochondrial gene expression (15,16); a Cryptococcus DEAD helicase is required for cryptococcosis pathogenesis (17), and the dipteran Chironomus tentans uses a hrp84 DEAD helicase to regulate mRNA transport from the nucleus into the cytoplasm onto polyribosomes (18).', 'Given that helicases significantly contribute to normal cellular metabolism, are they similarly essential to viruses? The operational answer appears to be a qualified ‘yes’. Indeed, when DEAD/DEAH-box helicase motif (InterPro IPR001410) was used to search the EMBL-EBI database, 1561 matches to individual viral sequence entries were found (), suggesting that many viruses have evolved to encode directly helicase or helicase-like proteins. The strongest biological evidence which supports the importance of a helicase in the virus life cycle comes from those viruses with an RNA genome. Hence, all positive-strand RNA viruses encode one or more helicase/helicase-like open reading frame (ORF) which, aside from the RNA-dependant RNA polymerase, is the most highly conserved viral sequence. Although less ubiquitous, helicases are also found in other types of viruses (see some examples listed in Table 1). Direct mutagenesis studies have confirmed that a helicase function is biologically required for the replication of many viruses including vaccinia virus (19), poliovirus (20), alphaviruses (21), brome mosaic virus (22), nidoviruses (23,24) and flaviviruses (25–27).', 'In 1981, the first cases of acquired immunodeficiency syndrome (AIDS) were described in American homosexual men. Thereafter, within three short years, French and American scientists confirmed that the human immunodeficiency virus (HIV) is the causative agent for AIDS. In the ensuing 20 years, >20 million individuals have died from AIDS; and currently, in 2006, ∼50 million people worldwide are infected by HIV-1 with 3 million incremental AIDS deaths and 4–5 million new infections occurring annually. The magnitude of this burden casts urgency to medical research on HIV/AIDS.', \"HIV-1 is a retrovirus of the lentivirus genus with an RNA genome of ∼9 kilobases which encodes nine polypeptides. The major HIV-1 structural proteins are encoded by three genes, gag (group-specific antigen), pol (polymerase) and env (envelope), while the accessory proteins, Vif, Vpu, Vpr and Nef, and the regulatory proteins, Tat and Rev, are the primary translation products of multiply-spliced mRNA. HIV-1 infects CD4+ human T-cells and macrophages and integrates as a provirus into the host cell's DNA. Gene expression of HIV-1 is governed transcriptionally by a viral protein, Tat (28,29), via its binding to a nascent viral TAR RNA (30), and post-transcriptionally by a second viral protein Rev (31,32) through its association with the viral RRE RNA. Both Tat and Rev interact with several host cell proteins in their transcriptional and post-transcriptional functions (33). HIV-1 does not encode for any RNA helicase; however, findings suggest that host cell RNA helicases may be involved in the reverse transcription of HIV-1 RNA, in HIV-1 mRNA transcription and in the nucleus-to-cytoplasm transport of viral mRNA.\", 'A recent unexpected finding revealed the possibility that an RNA helicase may potentially contribute roles in HIV-1 particle assembly and reverse transcription (34). Using proteomic analyses, Roy et al. (34) reported that the DEAH protein RNA helicase A (RHA) was found associated with HIV-1 Gag and packaged into HIV-1 virions in an RNA-dependent manner. When RHA was knocked down in cells, HIV-1 particles which were produced from these cells were significantly less infectious. This appears to be compatible with two possible explanations. First, it is conceivable that RHA participates in the formation of infectious virus particles either by shaping Gag–RNA interaction during viral particle assembly or by budding. Failure of RHA to properly restructure viral RNP could explain the observed reduced infectivity. Second, Roy et al. (34) reported evidence that HIV-1 particles that do not contain RHA showed reduced virion-endogenous reverse transcriptase activity. In this respect, it may be that RHA assists HIV-1 reverse transcriptase to more efficiently copy RNA by unwinding RNA secondary structure or by promoting the interaction of viral RNA with the nucleocapsid protein in order to assemble a better reverse transcription complex.', \"Separate from reverse transcription, the unwinding of highly structured RNAs might also be reasoned to be important for transcription (35). However, direct evidence for an RNA helicase role has been somewhat elusive. There are several examples which seemingly support an activity for RNA helicase in transcription. First, in vaccinia virus, it has been postulated that the NPH-II helicase assists transcription by strand-separating duplexed RNA structures to prevent R-loop formation behind the elongating RNA polymerase (36). Second, RHA has been invoked to provide a factor-recruitment role, bridging at the promoter the CREB-binding protein (37) and RNA polymerase II (37). Third, the p68 DEAD-box helicase was shown recently to be a novel transcriptional co-activator for p53's transcriptional function (38). Interestingly, in the latter two instances, neither the ATPase nor the helicase activity of RHA and p68 is apparently required for their attributed transcriptional roles.\", 'For HIV-1, two recent studies provide clues that RNA helicases may also serve co-factor function for transcription from the viral long terminal repeat (LTR). Fujii et al. (39) observed that RHA conserves in its N-terminus two double-stranded RNA-binding (dsRBD) domains characterized previously for the TAR RNA-binding protein, TRBP (40,41). These investigators found in both reporter and virus replication assays that RHA activated, in a TAR RNA-binding dependant manner, HIV-1 LTR-directed transcription (39). Next, Cocude et al. (42) separately described that the expression of a DExH RNA helicase, RH116, was significantly induced after infection of HeLa-CD4 cells by HIV-1. RH116 was found localized in the nucleus of HIV-1 infected cells and was observed to augment the transcription of unspliced HIV-1 transcripts. While both studies offer tantalizing evidence for roles by two different RNA helicases in HIV-1 transcription, critical details on how the helicases function by binding to TAR RNA and how this binding might cooperate with the activities of the viral transcriptional activator, Tat, are lacking. Pending more direct experimental tests, it remains unclear whether RHA and RH116 provide direct roles in transcription or indirectly influence the milieu of polymerase II initiation/elongation at the LTR.', 'Downstream from transcription, the fate of HIV-1 encoded RNA is regulated at the step of export of unspliced/partially spliced moieties from the nucleus into the cytoplasm. Unspliced and partially spliced viral RNAs code for genomic RNAs that are packaged into progeny virions and structural proteins. Hence, the egress of these RNAs from the nucleus into the cytoplasm is critical to the life cycle of the virus. Exit of HIV RNAs from the nucleus is a significant issue because unspliced/partially spliced cellular mRNAs are routinely retained in and not permitted export from the nucleus (43–46). A large body of work has suggested an elegant solution to this conundrum. Thus, it was established that the HIV-1 encoded Rev protein binds a highly secondary structured element (Rev responsive element; RRE) present in all unspliced and partially spliced HIV transcripts (47–57); and this binding specifically distinguishes, for purposes of nuclear export, viral transcripts from cellular RNAs.', \"New evidence now suggests that RNA helicases are also co-factors for Rev-directed export of HIV-1 mRNAs (58). In its role of transporting unspliced and incompletely spliced viral RNAs from the nucleus, Rev directly interacts with nuclear export receptor CRM1 (59,60), and CRM1 is required for Rev-mediated export of HIV RNAs (59,61,62). A recent report provides data that an RNA helicase, DDX3, is an additional player in the Rev–CRM1–RRE complex (12). Thus, it was shown that DDX3 over-expression enhanced Rev-dependent, but not other export, pathway; and that DDX3 is a nucleo-cytoplasmic shuttling protein which binds CRM1 and Rev. Moreover, DDX3's necessity for Rev/RRE/CRM1 function was demonstrated by knock-down of cell endogenous DDX3. Finally, because DDX3 locates to nuclear pore complexes (NPC), Yedavalli et al. (12) further proposed that this human helicase, like the analogous yeast Dbp5p (11), may function with Rev/CRM1 to remodel and ‘thread’ large unspliced HIV-1 RNAs through the nuclear pore, facilitating their final release to the cytoplasmic side of the NPC.\", 'The above DDX3 results are consistent with two additional papers which described similar findings for a related RNA helicase, DDX1. Thus, Pomerantz and co-workers (63) showed that DDX1 binds directly to the N-terminus of Rev and to the RRE-RNA motif and participates in the export of unspliced HIV-1 RNA from the nucleus to the cytoplasm. Additionally, they illustrated that reduced expression of DDX1 in astrocytes explains the previously observed tissue restricted function of HIV-1 Rev (64). Fully spliced viral mRNAs encoding for viral Tat, Rev and Nef, proteins have been shown previously to exit the nucleus using the cellular mRNA export pathway. Export of these mRNA may require the RNA helicase Dbp5 (65,66). As yet, the involvement of Dbp5 in export of spliced HIV-1 viral RNA has not been fully clarified.', 'The story of HIV-1 and RNA helicases is, however, likely to be more complex than and unlikely to conclude simply with RHA, RH116, DDX1 and DDX3 (Figure 1). HIV-1 RNAs are extensively regulated through splicing. Splicing is a multiple-step process requiring the recognition of splice sites by spliceosomes. It is generally believed that remodeling of RNA–RNA and RNA–protein interactions within the spliceosome is catalyzed by a family of DEAD/DExH box RNA helicases. To date, seven mammalian proteins that are RNA helicases have been implicated in mRNA splicing (67,68). Whether there is specific preference by subclasses of RNA helicases for viral mRNA splicing remains to be clarified. Moreover, how cellular RNA helicases might contribute to the translation of viral mRNAs also require further investigation.', \"Recently both Van't Wout et al. (69) and Krishnan and Zeichner (70) have provided evidence that the expression of several cellular RNA helicases including DDX24, DDX21, DDX18, DDX11 and DDX9 is modulated during HIV-1 infection; however, the precise cellular role and significance of these helicases for HIV-1 pathogenesis have not be characterized. Interestingly, Krishnan and Zeichner reported microarray data which examined the transition of HIV-1 infection from latency to productive replication, and found that several cellular RNA helicases were upregulated (71). For future understanding of functions, it will be important to design experiments which can segregate helicases which serve direct, although perhaps overlapping and redundant, roles on HIV-1 from those that might participate indirectly in the viral life cycle. Nevertheless, the convergence of evidence would support that several discrete cellular RNA helicases contribute importantly to the efficient execution of several steps in the HIV-1 replicative cycle.\", \"Given that the HIV-1/AIDS disease burden has reached pandemic global proportions, new antiviral strategies that target molecularly delineated mechanisms used by this virus are urgently needed (72). Is there a possibility that host cell helicases can be therapeutic targets for anti-HIV-1 chemotherapy? Implicit within this question is the concept that one could attack a host cell protein in order to treat an infecting pathogen. Although targeting a cellular protein involved in a viral pathway risks obvious cytotoxicity, this approach avoids the inherent problem posed by rapid HIV-1 mutation to all currently utilized chemotherapeutics targeted to virus-encoded proteins. We note that inhibition of cell-encoded enzymes in medical therapy is not an unprecedented strategy. Suppression of angiotensin-converting enzyme (ACE) is widely used to treat hypertension, congestive heart failure, myocardial infarction, endothelial dysfunction and renal disease (73). Elsewhere, aromatase inhibitors have been used to treat hormone-dependent breast cancer (74), and inhibitors of cellular secretory proteases are contemplated for Alzheimer's disease (75). We recently inhibited the cellular polyprotein convertase, furin, at minimal toxicity to the cell in order to block HIV-1 replication (76). Thus, a priori exclusion of cellular helicase as an antiviral target is not warranted.\", 'Guarded optimism that small molecule helicase inhibitors can be developed against viruses arises from encouraging progress in non-retroviral systems. Unlike HIV-1, human herpesviruses physically encode helicases. The herpes simplex virus UL5 and UL9 genes are helicases in superfamily 1 and 2, respectively (77). HSV UL5 together with UL8 and UL52 form a heterotrimeric helicase–primase complex responsible for unwinding duplex viral DNA at replication forks. Two recent studies provide proof-of-concept that the HSV helicase–primase can be targeted at low host cell toxicity by two new classes of drugs, amino-thiazolyphenyl-molecules (78) and thiazole amide derivatives (79). In addition, other studies suggest that the NS3 protein, a RNA helicase encoded by Hepatitis C virus and related West Nile virus and Japanese Encephalitis virus can be targeted to inhibit viral replication (80–82). This conceptual break through in drug development is important because it indicates that target discrimination between different helicases by small molecule inhibitors is possible. Of relevance to HIV-1, a synthetic immunomodulator Murabutide was shown recently to suppress HIV-1 replication in macrophages and T cells. Murabutide was shown to inhibit the activity of RNA helicase RH116, blocking its positive transcriptional activity for HIV-1 gene expression (42).', 'If one looks beyond the signature motifs conserved amongst helicases, then it becomes clear that the different proteins are widely divergent in their coding sequences. In principle, this suggests that individual helicases can be abrogated with specificity in a knowledge-directed manner. In theory, a helicase can be attacked by (i) inhibition of NTPase activity through direct competition for NTP binding, (ii) inhibition of substrate binding through direct competition at active site, (iii) allosteric mechanisms to affect NTP-binding/NTP hydrolysis and/or polynucleotide binding, and (iv) inhibition of unwinding activity by steric hinderance of helicase translocation along the polynucleotide substrate (83,84). Because the NTP-binding and substrate-binding pockets may be sufficiently similar between various helicases, specificity of inhibition through these sites will likely be extremely difficult, although perhaps not impossible. On the other hand, the tremendous variations in sequence and sizes of helicases, in their oligomerization states, in their discrete domains responsible for protein–protein interactions and/or for targeting to specific nucleic acids (85), and in their differential localizations within cells (86) offer interventional possibilities outside of the NTP- or polynucleotide-binding sites.', 'We are in the preliminary stages of screening ring-expanded nucleoside analogs found previously to be successful NTPase/helicase inhibitors of West Nile virus, Hepatitis C virus and Japanese encephalitis virus (81,82). We have observed that a few of these candidate inhibitors have substantial anti-HIV-1 activity at doses that do not incur cytotoxicity to cells treated in tissue culture for 1 week. Further studies are needed before concluding that these compounds exert specific inhibition of DDX3, one of the other cellular helicases, or some other target altogether.', \"There is another area where a cellular helicase activity and HIV-1 are likely to intersect. An emerging research focus is the role of small interfering RNAs (87) and microRNAs (miRNA) as innate cell defenses against viruses including HIV-1 (87–90). In human cells, the precursor for miRNA (pre-miRNA) is processed by DICER (Figure 2A) which is a ribonuclease with a bona fide RNA helicase domain (91,92). A surprising recent finding revealed that the human TRBP, which has been shown to be a potent binder of the HIV-1 TAR RNA RNA (40,41), is an indispensable dsRNA-binding partner of DICER which allows the latter to associate with pre-miRNA (91,92). Without TRBP, DICER's miRNA processing activity is lost. Thus an intriguing scenario can potentially unfold. Accordingly, whereas the ribonuclease–helicase protein DICER requires TRBP to process duplex-structured miRNAs in order that the cell can use such matured miRNAs for antiviral defense, it could be speculated that HIV-1 has evolved to restrict this defense by the ability to transcribe viral TAR RNA to squelch TRBP away from DICER (Y. Bennasser, M. L. Yeung and K. T. Jeang, manuscript submitted) (Figure 2B). If this thinking is correct, then HIV-1 has developed mechanisms not only to co-opt the active functions of a virus-propitious cellular helicase (i.e. DDX3) but also to inactivate the role of a second virus-pernicious helicase (i.e. DICER) for purposes of selfish gain.\", 'In a separate perspective, virus infection can trigger through double-stranded viral RNAs an innate antiviral immune response. Thus viral dsRNAs can be recognized by cellular proteins [pattern-recognition receptors (PRRs)] which initiate antiviral responses by inducing the production of a variety of cytokines including type I interferons (IFN-α and IFN-β) and initiating additional inflammatory and adaptive immune responses. Recently, DExD/H RNA helicases such as RIG-1 (retinoic acid inducible gene-1) (93) and Mda5 (melanoma differentiation-associated gene 5) (94) have been identified as suppressors of viral replication by binding to virus associated dsRNA and activating type I interferon-dependent antiviral immunity. Over-expression of RIG-1 and Mda5 was found to enhance dsRNA induced type I interferon antiviral response. Currently, it remains speculative whether helicases like RIG-1 and Mda5 may recognize HIV-1 dsRNA and trigger an innate immune response. Intriguingly, several reports exist in the literature that HIV-1 infection does induce activation of type 1 interferons (95,96).', \"In conclusion, by studying helicase proteins one can gain insights into normal cellular metabolic processes, abnormal inherited human diseases (e.g. Bloom syndrome, Werner syndrome, Cockayne's syndrome and xeroderma pigmentosum; all diseases with mutations in cellular helicases), and remarkably also the biology of viruses.\"], 'cord_uid': 'vefs1h6o'}\n{'title': 'Amino acid residues critical for RNA-binding in the N-terminal domain of the nucleocapsid protein are essential determinants for the infectivity of coronavirus in cultured cells', 'text': ['RNA viruses, including coronaviruses, encode a group of structural phosphoproteins with basic patches on their surface. As a pivotal structural component of the virion, this group of proteins plays essential roles in packaging the RNA genome to form a ribonucleoprotein (RNP) complex resulting from assembly of the viral RNA and multiple copies of the (nucleo)capsid protein. A detailed dissection and characterization of the intrinsic RNA-binding properties of these proteins are thus essential for understanding several important processes in the life cycle of RNA viruses, including assembly of the nucleocapsid, the specific encapsidation of viral RNA and morphogenesis of virions. In this study, amino acid residues in the N-terminal domain of the nucleocapsid (N) protein of a coronavirus that are critical for its RNA-binding activity are identified and their roles in viral infectivity are analyzed, using site-directed mutagenesis based on our previous structural studies.', 'Coronaviruses are enveloped viruses containing the largest known single-stranded, positive-sense RNA genome of ∼30 kb (1). During the coronavirus life cycle, the N protein is synthesized in large amounts and is thought to play an important role by specifically packaging the viral genome into a filamentous nucleocapsid of ∼10 to 15 nm in diameter and several 100 nm in length, a macromolecular structure that is visible by using electron microscopy (2). In addition to its structural role, the N protein also participates in viral RNA transcription, replication and in modulating the metabolism of host cells (3–13). Using X-ray crystallography techniques, we recently determined a 3D structure of the N protein from coronavirus infectious bronchitis virus (IBV), a prototype coronavirus, and showed that the basic building block for nucleocapsid assembly was a dimer of the N protein (14). The 409 amino acids long IBV N protein is composed of two globular domains that are resistant to proteolysis. Amino acids 29 to 160 form the N-terminal RNA binding domain and residues 218 to 329 form its C-terminal dimerization domain (5,14). Even though the structure of the N-terminal domain was determined in the absence of a nucleic acid ligand, its overall shape and the distribution of electrostatic charges suggest a plausible model for RNA binding. The outer surface of the N-terminal domain is enriched in aromatic and basic residues, a common feature for RNA-binding proteins (15). The shape of this domain is reminiscent of a hand having basic fingers, a hydrophobic palm and an acidic ‘wrist’. The positively charged ‘fingers-like’ β-hairpin extension could neutralize the phosphate groups emanating from RNA, whilst the base moieties could make contact with exposed aromatic residues from the hydrophobic palm (14).', 'Based on this structural hypothesis, we chose to mutate a number of evolutionarily conserved residues in the N-terminal domain of the IBV N protein in order to assess their roles in viral genomic RNA binding and in viral replication. We used purified recombinant wild type and mutant proteins containing the N-terminal domain expressed in Escherichia coli for RNA binding assays. These studies led to the identification of a number of amino acid residues essential for the RNA-binding activity of the domain. Subsequently, we introduced mutations that either severely or mildly impair the RNA binding activities of the N-terminal domain into an infectious cDNA clone system derived from the genomic RNA of IBV to assess their effects on viral replication and infectivity. We identified several residues exposed to the solvent whose substitutions to alanine yield a decrease of RNA binding and a concomitant reduction in virus replication. Interestingly, residues Arg-76 and Tyr-94, located at the base of the positively charged flexible hairpin loop and on the hydrophobic platform, respectively, were critical for RNA binding and viral infectivity.', 'His-tagged wild type and mutant N-terminal domains of IBV N protein were expressed in E.coli BL-21 by induction with 1 mM isopropyl-β-d-thiogalactopyranoside (IPTG). Cells were lysed by sonication and purified by metal affinity chromatography with Protino-Ni 150 kit (Macherey Nagel).', 'PCR fragments covering the IBV genome from 27 100 to 27 608 nt were cloned into a plasmid in either forward or reverse orientation under the control of a T7 promoter. The Dig-labeled sense (+) and anti-sense (−) RNA probes were made in vitro using the DIG RNA labeling kit according to the instructions of the manufacturer (Roche).', \"Five micrograms of purified proteins were resolved on an SDS-15% polyacrylamide gel and transferred onto nitrocellulose membranes (Hybond C-Extra, Amersham Biosciences) using a semi-dry transfer apparatus. Membranes were washed for 10 min with the probe buffer [1× Denhardt's Reagent, 1 mM EDTA, 10 mM Tris–HCl (pH 7.5) and 50 mM NaCl], blocked for 1 h with 25 µg/ml yeast tRNA (Ambion) and subsequently incubated with 10 µg of DIG-labeled RNA probe in the same probe buffer for 1 h. Membranes were washed three times for 15 min each with the probe buffer, before proceeding to detection with CDP-Star (Roche) according to the manufacturer's instructions.\", 'Construction of an infectious IBV clone was carried out essentially as described (16,17). Briefly, five fragments spanning the entire IBV genome were obtained by RT–PCR from Vero cells infected with the Vero cell-adapted IBV p65. The PCR products were purified from agarose gels and cloned into pCR-XL-TOPO (Invitrogen) or pGEM-T Easy (Promega) vectors. Subsequently, fragment A was removed from pCR-XL-TOPO by digestion with NheI and EcoRI, and subcloned into pKT0 vector. Plasmids were digested with either BsmBI (fragment A) or BsaI (fragments B, C, D and E). The digested plasmids were separated on 0.8% agarose gels containing crystal violet. Bands corresponding to each of the fragments were cut from the gels and purified with QIAquick gel extraction kit (QIAGEN Inc.). Fragments A and B, and fragments C, D and E were first ligated with T4 DNA ligase at 4°C overnight. The two reaction mixtures were then mixed and further ligated at 4°C overnight. The final ligation products were extracted with phenol/chloroform/isoamyl alcohol (25:24:1), precipitated with ethanol and detected by electrophoresis on 0.4% agarose gels.', 'Full-length transcripts were generated in vitro using the mMessage mMachine T7 kit (Ambion, Austin, TX). The N transcripts were generated by using a linearized pKT0-IBVN containing IBV N gene and the 3′-untranslated region (3′-UTR) as templates. The in vitro synthesized full-length and N transcripts were treated with DNase I and purified with phenol/chloroform. Vero cells were grown to 90% confluence, trypsinized, washed twice with cold phosphate-buffered saline (PBS), and resuspended in PBS. RNA transcripts were added to 400 µl of Vero cell suspension in an electroporation cuvette, and electroporated with one pulse at 450 V, 50 µF with a Bio-Rad Gene Pulser II electroporator. The transfected Vero cells were cultured overnight in 1% FBS-containing MEM in a 60 mm dish or a 6-well plate and further cultured in MEM without FBS.', 'Mutations were introduced into the corresponding fragments by using QuickChange site-directed mutagenesis kit (Stratagene), and confirmed by sequencing of the whole fragments.', 'Confluent monolayers of Vero cells on 6-well plates were infected with wild-type and mutant viruses at a multiplicity of ∼1 PFU/cell. After 1 h of incubation at 37°C, cells were washed twice with PBS and cultured in 3 ml of MEM containing 0.5% carboxymethy cellulose for 3 days. The cells were fixed and stained with 0.1% toluidine.', 'Vero cells were infected with wild-type and recombinant IBV, and harvested at different times post-infection. Viral stocks were prepared by freezing/thawing of the cells three times. The plaque-forming units per ml of each sample were determined by infecting Vero cells on 6-well plates in duplicate with 10-fold serial dilution of each viral stock.', \"Vero cells were infected with wild type and Q74A mutant virus at a multiplicity of ∼1 PFU/cell. Total RNA was extracted from cells infected with wild type or mutant viruses. Ten micrograms of RNA were added to a mixture of 1× MOPS, 37% formaldehyde and formamide and incubated at 65°C for 20 min before subjected to gel electrophoresis. The segregated RNA bands were transferred onto a Hybond N+ membrane (Amersham Biosciences) via capillary action overnight and fixed by ultraviolet (UV) crosslinking (Stratalinker). Hybridization of Dig-labeled DNA probes was carried out at 50°C in hybridization oven overnight. Membranes were washed three times for 15 min each with the probe buffer, before proceeding to detection with CDP-Star (Roche) according to the manufacturer's instructions.\", \"Vero cells were infected with wild type and Q74A mutant virus at a multiplicity of ∼1 PFU/cell. Total proteins extracted from Vero cells were lysed with 2× SDS loading buffer in the presence of 200 mM DTT plus 10 mM of iodoacetamide and subjected to SDS–PAGE. Proteins were transferred to PVDF membrane (Stratagene) and blocked overnight at 4°C in blocking buffer (5% fat free milk powder in PBST buffer). The membrane was incubated with 1:2000 diluted primary antibodies in blocking buffer for 2 h at room temperature. After washing three times with PBST, the membrane was incubated with 1:2000 diluted anti-mouse or anti-rabbit IgG antibodies conjugated with horseradish peroxidase (HRP) (DAKO) in blocking buffer for 1 h at room temperature. After washing three times with PBST, the polypeptides were detected with a chemiluminescence detection kit (ECL, Amersham Biosciences) according to the manufacturer's instructions.\", 'PCR products covering the IBV sequence from 25 957–26 352 nt were amplified by using the forward primer 5′-CGGGCATATGTCTTCTGGAAATGCATCTTGG-3′ and the reverse primer 5′-CGGGATCCTTACAGGGGAATGAAGTCCCAAC-3′. The PCR fragments were digested with NdeI and BamHI, and ligated into NdeI- and BamHI-digested pET-16b (Novagen). Each mutation was introduced by two-round PCR and the mutation introduced was confirmed by automated nucleotide sequencing.', 'The N-terminal RNA-binding domain of N protein from both IBV and SARS-CoV contains a flexible and positively charged hairpin loop that extends much beyond the protein core and that could grasp a nucleic acid substrate by neutralizing its phosphate groups (Figure 1) (14,18). Positively charged lysine and arginine residues in the N-terminal domain of SARS-CoV N protein were proposed to bind a 32 nt stem–loop structure located at the 3′ end of the SARS-CoV RNA genome (18). Indeed, the structure of this domain possesses some features reminiscent of RNA-binding proteins sharing the RNP fold (19–22). These RNA-binding proteins, including the U1A spliceosomal protein (23) and the coat protein from MS2 bacteriophage (24), bind the RNA ligand with residues emanating from the surface of a four-stranded anti-parallel β sheet. Presumably, strands β2 and β3 as well as the flexible β-hairpin from the IBV nucleocapsid protein could fulfill a similar role by interacting with phosphate groups from an RNA ligand (Figure 1). The β3 strand is the longest in the N-terminal domain of IBV N protein and several evolutionarily conserved residues project from it (Figure 1). Nucleotide bases in single-stranded RNA have a tendency to stack either on adjacent bases or with aromatic side chains of the protein (20).', 'The above structural model was partially supported by the demonstration that the N-terminal domain of IBV N protein could interact with RNA fragments corresponding to the 3′ end of the viral genome (14). To more vigorously test this model, systematic mutagenesis of residues essential for the RNA-binding activity was carried out. We targeted the exposed hydrophobic residues Tyr-92 and Tyr-94 (strand β3) that could form stacking interactions with the nucleotide bases as well as several basic or polar residues emanating from the β-hairpin (Figure 2a). As summarized in Figure 2a, 10 amino acids were mutated to alanine. These residues are either exposed to the solvent (Gln-74, Arg-76, Lys-78, Lys-81, Tyr-92 and Tyr-94) or have been conserved during evolution (Tyr-70 and Arg-73) and are thus likely to be involved in RNA binding or to play an undefined functional role. In addition, mutation of Asp-111 to an alanine was included as an additional control for the RNA-binding assay (Figure 2a).', 'The wild type and mutant constructs, spanning 29 to 160 amino acids with a hexa-histidine tag at their N-termini, were cloned into a bacterial expression vector, expressed in E.coli and purified to near homogeneity. Approximately equal amounts of the purified proteins were separated on SDS–PAGE (Figure 2b, upper panels). After transfer to a Hybond C extra membrane, the RNA-binding activities of wild type and mutant proteins were analyzed by northwestern blot with two probes corresponding to the positive- and negative-sense 3′-UTR of IBV, respectively. A moderate decrease in binding to both the positive and negative sense RNA probes was consistently observed with most mutant proteins, and representative gels are shown in Figure 2b (Figure 2b, middle and lower panels). Among all mutants, mutation of Tyr-94 to an alanine residue showed the most significant reduction in binding of the domain to both positive and negative probes. The RNA binding activities of this mutant to positive and negative probes were reduced to 27 and 25%, respectively, of the wild type were observed (Figure 2b, lane 9). Relatively less reduction in the RNA-binding activity was observed for the R76A, K78A, K81A and Y92A mutant proteins which retain between 50–74% of the wild-type binding activity (Figure 2b, lanes 5–8). With the exception of the Q74A and D111A mutants, which displayed a slight increase in the RNA binding activity to both the positive- and negative-sense RNAs, (Figure 2b, lanes 4 and 10), all other mutants showed a certain degree of reduction in their binding activity to either RNA probes (Figure 2b). As a control, a circular dichroism analysis of the two corresponding R76A and Y94A single mutants was performed to check whether the mutant proteins were properly folded. The spectra obtained for these two mutants were similar to the wild-type protein, ruling out any drastic conformational change introduced by the mutations (data not shown).', 'As none of the single mutations introduced could totally abrogate the RNA-binding activity of the IBV N-terminal domain, two mutant constructs carrying double mutations were made. As can be seen in Figure 2b (lanes 13 and 14), mutations of both Arg-73 and Lys-78 to alanine residues (R73/K78A) reduced the RNA-binding activities to the positive- and negative-sense RNA probes between 44 and 10% of wild-type, respectively. Mutations of both Arg-76 and Tyr-94 (R76/Y94A) to alanine reduced the RNA-binding to the positive- and negative-sense RNA probes to only 7 and 19% of wild-type, respectively (Figure 2b, lanes 15 and 16). Taken together, these results are consistent with the presence of an extended RNA binding site covering a large part of the accessible surface of the N-terminal domain of the coronavirus N protein. They are also in agreement with the assumption that removal of a single positive charge at the protein surface by mutation of a lysine or arginine residue to an alanine is insufficient to totally disrupt the formation of the protein–RNA complex.', 'The R76A and Y94A mutations were first introduced into an infectious clone of IBV, and in vitro transcribed full-length RNA molecules derived from wild type and mutant constructs were generated by in vitro transcription using the T7 RNA polymerase in the presence of a cap analog. Electroporation of wild-type RNA transcripts together with an RNA fragment covering the N protein region into Vero cells showed the formation of massive CPE at 2 days post-electroporation (Figure 3, panel rIBV+N). In contrast, no CPE formation was observed in cells transfected with transcripts derived from either mutant up to 5 days post-electroporation, suggesting that no infectious virus could be rescued from the two mutant transcripts (Figure 3, panels R76A+N and Y94A+N). These results indicate that R76A and Y94A mutations, which severely reduce the RNA-binding activity of the N-terminal domain of the N protein, may abolish the infectivity of the in vitro synthesized full-length IBV transcripts.', 'Introduction of Q74A and Y92A mutations into the full-length infectious clone of IBV was then carried out. Typical CPEs were observed in cells transfected with Q74A mutant transcripts at 3 days post-electroporation and recombinant viruses were recovered (Figure 3, panel Q74A). In cells electroporated with Y92A transcripts, a typical CPE was observed at 2 days post-electroporation [Figure 3, panel Y92A(1)]. Incubation of the transfected cells for one more day saw the appearance of more CPEs [Figure 3, panel Y92A(2)]. However, prolonged incubation of the transfected cells up to 6 days saw no spread of the infection from the two initiated infected loci to neighboring cells. Re-infection of fresh Vero cells with either medium collected from these cells or total cell lysates prepared by freezing and thawing of the transfected cells showed no detection of the CPE formation, indicating that the minute amount of viruses rescued from the initially transfected cells could not maintain infectivity in subsequent passage.', 'The R76A and Y94A mutant N constructs were then used to test if they could facilitate the rescue of the in vitro synthesized full-length IBV transcripts in trans. As mentioned, efficient recovery of infectious virus from the in vitro transcribed full-length IBV RNA requires co-transfection of these in vitro transcripts together with the in vitro synthesized RNA covering the N gene. Electroporation of wild-type full-length IBV transcripts together with RNAs synthesized in vitro from wild-type and the two mutant N constructs, respectively, showed efficient recovery of the infectious virus with CPE observed on almost the whole monolayer at 3 days post-electroporation (Figure 3, panels rIBV+N-R76A and rIBV+N-Y94A). In contrast, cells transfected with the full-length IBV transcripts only (without co-electroporation with the N transcripts) showed the formation of a single CPE on the whole monolayer at 3 days post-electroporation (Figure 3, panel rIBV-N).', 'As no infectious virus was recovered from cells transfected with R76A and Y94A mutant transcripts, total RNA was extracted from cells electroporated with wild-type and mutant full-length transcripts and RT–PCR amplification of subgenomic mRNAs was carried out to check whether a low level of RNA replication and subgenomic mRNA transcription occurred in cells transfected with the mutant transcripts. The forward primer used in this reaction corresponds to the leader sequence from 26–46 nt in the genomic RNA and the downstream primers covers IBV sequences from 24 784 to 24 803 nt. If transcription of subgenomic mRNAs did occur, a 415 bp PCR product corresponding to the 5′-terminal region of the subgenomic mRNA4 and a 1010 bp fragment corresponding to the 5′-terminal region of the subgenomic mRNA3 would be expected. As shown in Figure 4a, a dominant 415 bp band and a weak 1010 bp band were observed in cells electroporated with wild-type full-length transcripts at 2 days post-electroporation (lane 2). Sequencing of the PCR fragments confirmed that they represent the correct sequences of the corresponding regions of the subgenomic mRNAs 3 and 4, respectively. The same PCR products were not detected in cells electroporated with the R76A and Y94A mutant transcripts, respectively, at 24, 48 and 72 h post-electroporation (Figure 4a, lanes 3–8). In cells electroporated with Q74A mutants at 3 days post-electroporation, RT–PCR amplification of the subgenomic mRNA 3 and 4 showed the detection of the 415 and 1010 bp bands (Figure 4a, lane 9). The same PCR products were not evident in cells electroporated with the Y92A mutant transcripts (Figure 4a, lane 10).\\n', 'RT–PCR amplification of the negative strand RNA was performed to check if RNA replication occurred in these transfected cells. The primer pair was chosen so that the IBV sequence from 14 931 to 15 600 nt would be amplified by the RT–PCR. If replication of viral RNA occurred, a 670 bp PCR fragment would be expected. As shown in Figure 4b, RT–PCR fragments amplified from both positive (lanes 4, 7 and 9) and negative (lanes 3, 6 and 10) strand RNA templates were obtained from cells transfected with the mutant transcripts. The amount of the negative strand RNA was approximately half of that of the positive strand RNA (Figure 4b). As positive control, the same RT–PCR fragment for the negative strand RNA was observed in cells transfected with wild-type transcripts (Figure 4b, lanes 8 and 11). In the negative controls, total RNA extracted from cells transfected with Y94A (Figure 4b, lane 2) and R76A (Figure 4b, lane 5) was analyzed by PCR directly using the primer set for the negative strand RNA. No corresponding PCR fragment was detected from these samples, demonstrating that the detection of the negative strand RNA in cells transfected with wild-type and mutant constructs by RT–PCR is due to the replication of viral RNA. These results confirm that transcription of the negative strand RNA has taken place in cells transfected with the mutant transcripts.', 'The transcription of negative strand RNA in cells electroporated with wild-type, R76A, Y92A and Y94A transcripts was then quantitatively determined and compared by real time RT–PCR at 24 and 48 h post-electroporation, respectively. Compared to cells transfected with wild-type transcripts, the relative amounts of negative strand RNA in cells transfected with R76A, Y92A and Y94A were 48, 46 and 50%, respectively, at 24 h post-electroporation (Figure 4c, lanes 1–4). These figures were dropped to 41, 45 and 23%, respectively, at 48 h post-electroporation (Figure 4c, lanes 5–6). Taken together, these results demonstrate that transcription of the negative strand RNA has taken place in cells transfected with the mutant transcripts. As the level of negative strand RNA was not increased in cells transfected with the mutant transcripts over a time-course experiment, it suggests that transcription of these RNA species may occur only in the initially transfected cells using the in vitro transcripts as templates.', 'The growth properties of the Q74A mutant virus on Vero cells were tested by analysis of plaque sizes and growth curves of passage 3 mutant virus. Compared to cells infected with wild-type recombinant virus (rIBV), slightly smaller sized plaques were observed in cells infected with the Q74A mutant virus (Figure 5a). Analysis of the growth curves of wild-type and the mutant virus demonstrated that the mutant virus grew slightly more slowly than the wild type recombinant virus. When 10-fold more mutant virus was used, 2- to 10-fold more mutant virus was produced at 8–24 h post-infection (Figure 5a). The titers of the mutant virus dropped more rapidly than wild-type virus at 36 and 48 h post-infection (Figure 5a).\\n', 'Further characterization of the mutant virus was subsequently carried out by analysis of the genomic and subgenomic RNA, and the expression of viral structural proteins S and N. Northern blot analysis of cells infected with the 5th passage of rIBV and Q74A mutant viruses showed that similar amounts of genomic and subgenomic RNAs were detected (Figure 5b). Quantitative comparison of the subgenomic RNA synthesis by densitometry analysis of the bands shown in Figure 5b demonstrated that in cells infected with Q74A mutant virus (p5), the relative amounts of mRNA 2, 3, 4, 5 and 6 are ∼98, 101, 121, 104, 106 and 101%, respectively, of those in cells infected with rIBV. However, Western blot analysis of cells infected with rIBV and Q74A mutant virus showed that more S and N proteins were detected in cells infected with wild-type virus (Figure 5c, lanes 1–14).', 'To confirm if the Q74A mutation is genetically stable during passage of the mutant virus in cells, viral RNA was isolated from cells infected with passage 2 and 5 virus and amplified by RT–PCR. Sequencing analysis showed that the two mutated nucleotides (CA-GC) remained stable (Figure 5d). Furthermore, no compensatory mutation was found in other region of the N protein, confirming that Q74A mutant virus is stable in cell culture system.', 'The intrinsic RNA-binding activity of the nucleocapsid protein of RNA viruses is essential for packaging the RNA genome into a RNP complex. In previous structural studies of the N protein from two coronaviruses, the N-terminal domain was shown to be resistant to proteolysis and to fold independently (14,15). It serves as a functional unit critical for the RNA-binding activity of the protein (14). Based on these structural results, the importance of individual residues in RNA-binding was studied. Here we showed that Tyr-94 in strand β3 binds to the RNA ligand. The positively charged Arg-76 is located in the immediate vicinity of this aromatic residue at the base of the extended flexible hairpin loop, within a cluster of positively charged residues. Thus, the interaction between Arg-76 and Tyr-94 with RNA appears to play a crucial role during virus assembly. Since no single mutation could totally disrupt nucleic acid binding, other aromatic or basic residues located close to these two residues at the surface of the N-terminal domain are likely to contribute to RNA binding by creating a broad surface that makes contact with the genomic RNA. Using a simple docking experiment, the RNA binding site seems capable of accommodating up to 8–9 nt bases per N-terminal domain monomer. The Arg-76 and Tyr-94 residues identified in this paper thus provide more precise mapping of the location of the RNA-binding site on the IBV N protein.', 'It is interesting to note that the main effect of these mutations on viral life cycles was on subgenomic RNA transcription. However, the replication of negative strand RNA is nearly normal in cells transfected with these mutant constructs. Is it possible that these mutations will directly impair an undefined function of the N protein that is essential for subgenomic RNA transcription? Two possibilities were considered. First, the coronavirus N protein is directly involved in subgenomic RNA replication. The MHV N protein was shown to be involved in coronaviral RNA transcription (4), but the underlying mechanism was unrevealed. Alternatively, the observed phenotypic changes in subgenomic RNA replication caused by these mutations may be due to a generally reduced viral replication rate and infectivity. In fact, several other mutations that affect viral replication cycles were found to have defects in subgenomic RNA transcription (25–27). In this study, we have repeatedly observed that detection of subgenomic RNA replication in cells infected with low multiplicity of infectivity of IBV by RT–PCR is technically challenging. Positive results can be obtained only if transcription of subgenomic RNA reaches a certainly high level. When in vitro transcripts containing Y92A mutation were introduced into cells, one or two CPEs were always observable in a monolayer, suggesting the presence of low level viral RNA replication. However, detection of subgenomic RNA was consistently unsuccessful in cells electroporated with this mutant transcript.', 'Coronavirus N protein is a multi-functional protein. The protein was shown to be able to facilitate rescue of infectious virus from several coronaviruses by co-electroporation of the in vitro synthesized N transcripts with the in vitro synthesized full-length transcripts (17,28–30). The mechanism for this trans-effect is currently unknown. In this study, the R76A and Y94A mutant N constructs could promote rescue of the infectivity of the in vitro synthesized IBV RNA as efficiently as did the wild type N transcripts, suggesting that the reduced RNA-binding activity in the N-terminal domain does not affect this function of the N protein. It also suggests that RNA-binding per se is not a major factor that contributes to this important function of the N protein.', 'Are the reduced replication rate and loss of the infectivity of the mutant transcripts caused by the effects of these mutations on the functionality of the N protein or on the integrity of the genomic RNA itself? The nice correlation between the reduced RNA-binding activity of the N-terminal domain from the mutant proteins and the loss of infectivity of the corresponding mutant transcripts would support that the failure to rescue the viral infectivity is due to the specific mutation that impairs the functionality of the N protein. In addition, in the cases of Q74A and R76A mutants, the two mutations are physically adjacent in the genome, but yet totally different phenotypes were observed from the two mutants. It would support that the failure to recover infectious viruses from the mutant transcripts is due to the fact that these mutations reduce/impair the RNA-binding activity of the N protein. If this were the case, however, supplement of wild type N protein in trans would rescue the infectivity of the mutant transcripts. Two experiments were conducted to test this possibility, but none of them were successful. First, co-transfection of the mutant full-length transcripts with wild-type N transcripts into cells failed to recover infectious virus from the mutant transcripts, suggesting that wild-type N RNA cannot rescue the infectivity of the mutant transcripts in trans. The second experiment was to introduce the mutant full-length transcripts into cells stably expressing a six-His-tagged IBV N protein. Once again, it failed to rescue the infectivity of the mutant transcripts. At present, it is unclear if this was due to the low-level expression of N protein in these cells or because the N protein expressed in these cells contains the six-His tag at its C-terminus.', 'Residues Arg-76 and Tyr-94 in the IBV N protein are well conserved across the whole family of coronaviruses. Structurally, these two residues may correspond to the Arg-94 and Tyr-112 residues in the SARS-CoV N protein (see Figure 1a). It is possible that introduction of similar alanine mutations into the SARS-CoV genome may yield a similar phenotype observed here for IBV. The fact that substitutions of Gln-74, Arg-76, Tyr-72 and Tyr-94 in the IBV genome by using an infectious clone of IBV showed nice correlation between the RNA-binding activity and viral infectivity would suggest potential strategies for intervention of viral replication. A compound which could specifically bind to this region of the N protein might interfere with RNA binding and thus inhibit virus replication. In addition, this study points to a number of residues on the surface of the N protein that contribute less to the RNA-binding activity and viral infectivity for the design of attenuated coronavirus strains as potential vaccine candidates.'], 'cord_uid': 'at9jg710'}\n{'title': 'Automated identification of multiple micro-organisms from resequencing DNA microarrays', 'text': ['For both surveillance and diagnostic applications, fine-scale pathogen identification and near-neighbor discrimination is important; therefore, an assay that monitors at this very specific level is desirable for many types of samples such as clinical and environmental (1–3). To successfully use any method based on DNA or RNA detection, these assays must be coupled with large databases of nucleic acid sequence information for assay design to ensure that the desired information is provided and for the interpretation of raw data. Several well-established techniques use PCR to amplify individual target pieces of sequenced genomes to provide detection of organisms (4). These methods can roughly be divided into approaches that target individual short sequence lengths or probes (<40 bp) and methods that examine longer probes. The advantage of using short probes is that when the uniqueness of the probe has been assured and unique primers are also selected, this method gives good specificity. This approach is capable of providing fine-scale identification of several genetically close organisms by selecting a sufficient number of probes. However, this can rapidly lead to a very large number of total probes being required to detect all organisms of interest. In addition these selected probes, which in the initial selection process were determined to be unique, are often later found to be less specific as more organisms are sequenced or are less specific under conditions that differ from the original conditions. This is particularly a problem for organisms belonging to a family with a high mutation rate and also for pathogens that have relatively few neighboring pathogens sequenced. In addition, PCR approaches focused on short unique probes are not capable of detecting the presence of new significant mutations nor can they easily resolve base sequence details. Approaches that use longer individual probes avoid many of these issues at the cost of being less specific. This issue means most of these approaches are not suitable for providing the information desired, providing impetus to this work.', 'High-density resequencing microarrays produce variable length segments, 102–105 bp, of direct sequence information. This target sequence falls in the longer target regime of PCR approaches but rather than being hybridized to a longer less-specific probe on the microarray, many shorter specific probes are placed on the microarray to allow more detailed determinations from the entire PCR amplicon. This also means that the specificity of the primers used can be relaxed. They have been successfully used to detect single nucleotide polymorphism (SNP) and genetic variants from viral, bacterial and eukaryotic genomes (5–12). Their use for SNP detection has clearly established their ability to provide reliable quality sequence information. In most cases, the microarrays were designed to study a limited number of genetically similar target pathogens and for many cases, the detection methods relied only on recognizing hybridization patterns for identification (6,9,10,13,14). Taking advantage of the sequential base resolution capability of resequencing microarrays that is required for SNP detection, resequencing has recently been successfully adapted recently using a different approach for organism identification of multiple bacterial and viral pathogens while allowing for fine detailed discrimination of closely related organisms and tracking mutations within the targeted pathogen (15–16). The new methodology differed from earlier work by using the resolved bases as the query of a similarity search of DNA databases to identify the most likely species and variants that match the base calls from the hybridization observed. The system was capable of testing for 26 pathogens simultaneously and could detect the presence of multiple pathogens. A software program, resequencing pathogen identifier (REPI), was used to simplify data analysis by performing similarity searches of a genetic database using basic local alignment search tool (BLAST) (17). The REPI program used BLAST default settings and would only return sequences that might represent the hybridization if the expect value, a quantity calculated by the BLAST program that indicates the likelihood that the sequence match found would have occurred by random chance in the database, was <10−9. This screened out all cases that had insufficient signal; however, the final determination of what pathogen(s) was detected and to what degree discrimination was possible required manual examination of the returned results. This method successfully allowed fine discrimination of various adenoviruses and strain identifications of Flu A and B samples in agreement with conventional sampling results (15,16). Two important advantages of this approach were that the information was always recovered at the most detailed level possible and that it was capable of still recognizing organisms with recent mutations. This approach also maintained specificity well, as it was not dependent on the uniqueness of a few individual short probes.', 'Although this analysis method has utility, there are several shortcomings: it is time consuming, not optimized to maximize sensitivity, has complicated results, is suitable only for an expert, and contains redundant or duplicate information. The process was time consuming because only the initial screening was handled automatically while the remaining steps required manual interpretation before the detection analysis was complete. Because a simple criterion (expect value cutoff of 10−9) and non-optimized BLAST parameters were used to consider a pathogen detected, the REPI algorithm provided a list of candidate organisms but did not make a final simple conclusion or relate the results of one prototype sequence to another. Instead a manual process was used to make the final determination, but because the REPI program provided all similar results and the use of public nucleic acid databases containing redundant entries, a large amount of data was presented to a user that was not useful. In addition, with a manual process it was not possible to establish that the algorithm developed was generally applicable for any organism where nucleic acid base resolved sequence information has been provided.', 'In this paper, we describe a new software expert system, Computer-Implemented Biological Sequence Identifier system 2.0 (CIBSI 2.0), that successfully uses resolved base sequence information from custom designed Affymetrix resequencing microarrays to provide a simple list of organisms that are detected. This algorithm addresses the most important shortcoming of previous methods by incorporating new features to completely automate pathogen identification. We have demonstrated the effectiveness of this algorithm for identification via several examples. The single program is capable of making correct decisions for all 26 pathogens contained on the Respiratory Pathogen Microarray v.1 (RPM v.1), whether detected alone or in combinations, with improved sensitivity. Although the program is currently applied to resequencing microarrays, the methodologies developed remain generally applicable. Only the first portion of the algorithm handles issues specific to microarrays while the remainder deals with sequences that are suitable for use as a query by the BLAST algorithm. In developing the general identification algorithm, we have identified and resolved issues specific to resequencing microarrays that complicate their use. Because the entire decision process for what is detected has been automated, it is straightforward to test whether the rules used to make identifications are rigorous and applicable to any pathogen. With this efficient program, resequencing based assays can provide a competitive method to test simultaneously for many possible pathogens, providing output that can be interpreted by a non-expert.', 'The details of the RPM v.1 design and the experimental methods have been discussed in previous work (15,16,18) (Lin et al., submitted for publication). Briefly, the RPM v.1 chip design includes 57 tiled regions allowing resequencing of 29.7 kb of sequences from 27 respiratory pathogens and biothreat agents. These were selected based upon clinical relevance for the population of immediate interest (United States military recruit in training) (19–21). Partial sequences from the genes containing diagnostic regions were tiled for the detection of these pathogens. The experimental microarray data used in the present analysis were obtained using a variety of purified nucleic acid templates and clinical samples culture (throat swabs and nasal washes) using random and multiplexed RT–PCR amplification schemes (for more detail description of amplification methods see Supplementary Data). Resequencing microarrays provide base call resolution by comparing the intensities between a set of four 25mer probes that differ from each other at the same position (13th base). An amplicon or target sequence is represented by numerous overlapping probe sets. GCOS™ software v1.3 (Affymetrix Inc., Santa Clara, CA) was used to align and scan hybridized microarrays to determine the intensity of each probe in every probe set. Base calls were made based on the intensity data of each probe set using GDAS v3.0.2.8 software (Affymetrix Inc.) which used an implementation of the ABACUS algorithm (5). The sequences were represented in FASTA format for later analysis steps.', 'In this paper, target pathogens are the organisms the assay was specifically designed to detect. The sets of probes that represent reference sequence selected from target pathogen genomes are referred to as a Prototype Sequence or ‘ProSeq’ for brevity. The set of resolved bases that result from hybridization of genomic material to a ProSeq is referred to as the hybridized sequence or ‘HybSeq’. The HybSeq is split into possible subsequences or ‘SubSeqs’.', 'The CIBSI 2.0 program implemented in Perl described in this study handled a hierarchy of three tasks (Figure 1): (I) ProSeq identification; (II) ProSeq grouping; and (III) pathogen determination. The most developed and important portion of the algorithm deals with ProSeq identification Task(I) and is handled in three important subtasks: initial filtering of individual HybSeqs into SubSeqs suitable for sequence similarity comparisons (Figure 2), database querying of individual SubSeqs (Figure 3) and taxonomic comparison of BLAST returns for each SubSeq (Figure 4). The NCBI BLAST and taxonomy databases were used for the queries and images were obtained on February 7, 2006. For the ProSeq grouping Task(II), ProSeqs were compared to determine if they supported the same identified organism. In the pathogen determination Task(III), detected organisms were compared to the list of target pathogens the assay was designed for in order to determine if any were positively detected or were possibly related close genetic near neighbors. The level of discrimination that a particular sample supported was automatically determined.', 'An initial filtering algorithm, REPI, was developed previously (16) and the general concepts with revisions were incorporated into the current (automated detection) algorithm used in the CIBSI 2.0 program. Filtering and subsequence selection were used to remove potential biasing caused by reference sequence choice and by other sources (i.e. primers). When PCR amplification was used, microarrays were hybridized in the presence of only primers to determine locations where they resulted in hybridization. Any portions of the ProSeqs that hybridized with the primers were masked as N calls so that the HybSeq did not contain biased information. Normally the primers are designed to be outside the ProSeq region to minimize the interference caused by primers, and so minimize the bases to be masked. There is still the chance that some bases require masking because with the large number of primers used in the multiplex, short stretches of a ProSeq not corresponding to primer locations may still hybridize with the primers. Such regions could be removed from the reference sequences and so not appear on the microarray. However, determining such locations are a difficult and time-consuming task that for most cases is not worth the effort. The first subtask of ProSeq identification Task(I) is noted in Figure 1 and shown schematically in detail in Figure 2. This subtask uses a procedure to examine a HybSeq to find the longest possible subsequence of base calls (SubSeq) that can be submitted as a query to BLAST. It produces a group of SubSeq that contain all portions of a HybSeq that have a chance of producing a limited list of returns from a BLAST query. When a HybSeq has two regions separated by a long stretch of continuous N calls, the relational positioning of the two regions cannot be trusted and so must be sent as separate queries. In addition for shorter subsequences, the number of base calls that must be made is dependent on the length. It was also recognized that for very long sequences a longer WORD size in BLAST may be used. A detailed description of the criteria and process used for each step is contained in Supplementary Data. Upon completion, the algorithm returned to the Task(I) loop and performed the BLAST subtask.', 'The database query subtask performed a batch similarity search of a database using SubSeq as the queries. The BLAST program used was the NCBI Blastall –p blastn version 2.12 with a defined set of parameters. The masking of low complex regions was performed for the seeding phase to speed up the query; however, low complexity repeats were included in the actual scoring. The entire nucleotide database from NCBI acquired on February 7, 2006 was used as the reference database. (Note that earlier images of the database were used during development but all experiments were rerun with the algorithm as described with the image of the database obtained on this date.) The default gap penalty and nucleotide match score were used. The nucleotide mismatch penalty, –q, parameter was set to −1 rather than the default. The results of any BLAST query with an expect value <0.0001 were returned in tabular format from the blastall program. The information about each return (bit score, expect value, mismatches, length of match) was placed in the Return{hash key}{info} hash using the SubSeq identity as the hash key for further analysis.', 'The next subtask of ProSeq identification Task(I) carried out was the determination of SubSeq() states and is shown in Figure 3. The BLAST algorithm gives a ranking score which can be reported as accounting for the size of the database (expect value) or not (bit score). The full taxonomic classifications of every return for a SubSeq were retrieved from the NCBI taxonomy database obtained on February 7, 2006. Using the scores and taxonomy relationships it was possible to find a reduced number of returns that had the best match with the HybSeq. These results were summarized by identifying the taxonomic class to which all the returns belonged to, ‘identified organism’, and a parameter that indicated how they are related to each other, ‘organism uniqueness’. A detailed description of the steps is contained in Supplementary Data.', 'After each SubSeq was examined, the algorithm moved to the next subtask, which was to determine the identified organism of the ProSeq from the SubSeq (Figure 4). The subsequences from the same ProSeq were only allowed to support a single ‘identified organism’ determination. The procedure shown in Figure 4 demonstrates the decision method used to arrive at this determination (detailed description in Supplementary Data). After the subtask covered in Figure 4 was completed, the ProSeq identification Task(I) loop continued until all ProSeqs were examined. A list of ProSeqs that had detected organisms was built up in the Result1 array.', 'After the ProSeq identification Task(I) was completed, ProSeq grouping Task(II) (Figure 1) was used to examine the identified organism values listed in Result1 and grouped them together if they identified the same taxonomic class. Each entry in Result1 was examined and a new entry was created in Result2 if the identified organism did not appear in this list. The entries of Result2 represented the distinct individual organisms identified, but might still contain redundant information. When the ProSeqs were designed to detect the same organism and they all hybridized well, this grouping led to a reduction in redundant information being reported. But, when one ProSeq did not hybridize as well for a variety of possible reasons, multiple entries would appear in Result2 that actually represent hybridization from the same pathogen. This is because there is an alternative cause for the ProSeq hybridizing in this manner. This hybridization could be caused by two different but closely related organisms both being present in a sample and hybridizing to the microarray. Because we have not yet developed methods to distinguish these cases, no further reduction of the list of organisms is made for in ProSeq grouping Task(II) in cases where the level of identification varied on different ProSeq targeted for the same organism.', 'Although it was difficult to relate results from separate ProSeqs to each other, it was desired to have a simple final detection decision be made in pathogen determination Task(III). The first task was specifically implemented so that information about what a ProSeq was intended to detect was not considered and the second task only minimal consideration of this was taken into account. This allowed these initial tasks to be capable of recognizing not only just positive and negative identifications of target pathogens but also cases that were indeterminate. In the final task, the algorithm considered whether the identified organisms belonged to the list of organisms the ProSeqs were designed to detect. The task would group organisms from ProSeq grouping Task(II) together that belonged to or were child classes of the taxonomic class of a target organism. The taxonomic class reported was the common taxonomic group of all the organisms. When all the ProSeqs for a pathogen hybridized well, a fine level discrimination was reported. But if one or more ProSeqs hybridized less well, the reported positive target pathogen was only identified at the level of the less detailed level. This is conservative because methods have not yet been developed to clearly discriminate mixtures of very closely related organisms causing different ProSeqs to hybridize from variable hybridization of a single organism on several ProSeqs. The results of all three tasks were reported and a more experienced user can view ProSeq grouping Task(II) results to clarify some cases. Note that organisms identified in ProSeq grouping Task(II) that only belonged to target pathogens were reported as positives. Clear negative ProSeqs were not mentioned in the output. ProSeqs that were indeterminate or that detected close genetic organisms were never reported as positives. These organisms were instead reported as being detected.', \"Purified Chlamydia pneumoniae nucleic acid samples with 10–1000 genome copies (via method in Lin et al., submitted for publication) were chosen to illustrate how pathogen detection and identification were done when multiple ProSeqs were targeted for the same pathogen. RPM v.1 has three highly conserved ProSeqs selected from the genes encoding for the major outer membrane proteins VD2 and VD4, and the DNA-directed RNA polymerase (rpoB) gene. The HybSeqs from the different samples differed only in the number of unique base calls as shown in Table 1. The percentage of the ProSeq called varied from 80 to 100% except for one case at a concentration of 10 that had only 11% of the rpoB ProSeq producing unique base calls. Because the samples at this concentration are not reproducibly generating the same percentage of base calls, this is probably the detection limit of this ProSeq of the assay. Table 1 listed the determinations made for the SubSeq and at the end of each task for the various samples. The ProSeq from the different cases produced the same number of SubSeqs. These SubSeqs from different samples reported different bit scores for the same top ranked returns from BLAST. In fact VD2 and VD4 produced exactly the same results. The NCBI taxonomy database classified the returns into four distinct groups, which represented the C.pneumoniae taxonomic group and three child strain groups. AE001652, AE002167, AE017159 and BA000008 appeared in the returns of all the ProSeqs for each sample, since they represented database entries of completely sequenced genomes. One rpoB SubSeq produced for its organism uniqueness, SeqUniqu. All other SubSeqs were TaxAmbig as multiple returns from different taxonomic classes were returned. Since the VD2 and VD4 ProSeq each have a single SubSeq, Task(I) assigned the ProSeq the state of the SubSeq. For the rpoB ProSeq, the bit score of one SubSeq was large enough that the algorithm assigned that SubSeq's identification to the ProSeq. Task II of the algorithm grouped all three ProSeqs together since they all had the same identified organism and TaxAmbig was assigned. The result of Task(III) was positive for target pathogen C.pneumoniae and this decision was straightforward as all the ProSeqs agreed with each other and belonged to the same target pathogen taxonomic class. Although the rpoB ProSeq was SeqUniqu, this was not the final conclusion for Task(II) as the ProSeq that was SeqUniqu was not the child taxonomic group and other ProSeq were TaxAmbig. The three recognized strains scored the same, which indicated that the sequence selected for the ProSeqs was very conserved and would not allow discrimination between the strains.\", 'Influenza and Human Adenovirus (HAdV) were the only pathogens that had ProSeq selected that would permit detailed strain level discrimination as discussed in previous work (15,16). This previous work using manual analysis found that the microarray results were in excellent agreement with the conventional sequencing results for clinical samples. A few of the results of running the CIBSI 2.0 program using the updated NCBI database on the raw microarray results are presented in Table 2 (the results for all samples used in the previous work are presented in Supplementary Table A). The identified organisms were not identical to the original findings due to the difference in database used and because all ProSeqs were considered rather than only the Flu A and B hemagglutinin. In fact, the conventional sequencing results that were submitted to NCBI from that work were found for every sample to be among the returns with the best score for the hemagglutinin ProSeq (Supplementary Table B). It should be noted that the previous work based its analysis upon only the results of the hemagglutinin ProSeq. For 8 of 13 Influenza A and 3 of 12 Influenza B cases, the results of ProSeq identification Task(I) and ProSeq grouping Task(II) found that the conventional sequencing was the single best return for the hemagglutinin ProSeq. Owing to the large number of isolate sequences in the database for the hemagglutinin gene it was not surprising that in some cases a single unique entry was not found. In each of the remaining five Influenza A samples, the other sequences returned differed by <0.2% from the conventional sequence. The fewer samples with unique isolate identifications for Influenza B were due to an older reference sequence used for the ProSeq, which allowed less hybridization to occur (18). This also meant that when multiple sequences were returned for a sample they represented greater genetic variation, up to 2%. As a result of the current method of making pathogen determination Task(III) level identification, the final organism reported was less specific (H3N2 or Flu B) for every sample than what was reported as possible in ProSeq grouping Task(II). For HAdV samples, the algorithm also reproduced the finer scale discriminations that had been made previously by manual methods (data not shown).', 'The next example of detection for the Mycoplasma pneumoniae pathogen demonstrated a case where there was only a single ProSeq for the target pathogen. A total of 48 test samples were performed using multiplex PCR (via method in Lin et al., submitted for publication) where for 46 of the samples M.pneumoniae organism was spiked into nasal wash with several other pathogens from 100 to 100 000 colony forming units per ml, the remaining 2 samples were purified with nucleic acid from culture stock at a concentration of 1000 genome copies per reaction volume. This ProSeq was also not optimal for fine discrimination because it was selected from a highly conserved region (345 bp) of the cytadhesin P1 gene. In every case taxonomic database entries for M.pneumoniae or its one recognized distinct strain tied for MaxScore (Supplementary Table 3). To better understand these returns, the database sequences were examined and subdivided into three groups of sequences, A, B and C, based on how well they matched the reference sequence used to make the ProSeq. The placement of the database entries into the three groups was determined from a CLUSTAL alignment of the sequences of this gene. This alignment confirmed that the database entries differed significantly more from each other in regions not represented by the ProSeq and contained sufficient variability that would have allowed finer discrimination. Members of Group A exactly matched the ProSeq and could not be distinguished between on the microarray. Similarly, members of group B matched the ProSeq except at the 199th position where the base called was C rather than T. Group C sequences contained a few database entries that were more variable and might be distinguished from other entries within the ProSeq. For the 48 experimental tests of M.pneumoniae, as much as 80% of the ProSeq hybridized for 19 samples, yet only 5 of these samples had an unambiguous base call at the 199th position. When it was unambiguous, it always matched group B sequences. In the cases where an N base call was made at the 199th location, both groups A and B sequences were returned with the same score. Regardless of this, the target pathogen positively identified was M.pneumoniae for every sample tested.', 'These examples showed how decisions were made independent of whether single or multiple ProSeqs were dedicated to a target pathogen. They also illustrated that the level of discrimination possible was strongly determined by the quality of the selected ProSeq. It is possible that for some pathogens fine level discrimination is not required and the currently tested selections on RPM v.1 would provide satisfactory information. The CIBSI 2.0 algorithm demonstrated its capability to automatically report the maximum level of discrimination that could be supported by the HybSeq information.', 'To demonstrate how the algorithm handled closely related genetic species, a sample of a non-targeted pathogen was considered using multiplex PCR (via method in Lin et al., submitted for publication). For Variola major virus, one of the biothreat pathogens on the RPMv.1, the validation runs demonstrated that Variola major virus purified DNA templates of plasmids were always positively identified when detected (Table 3). Table 4 shows the results when purified Vaccinia genomic DNA was spiked into nasal washes and processed at various concentrations using multiplex PCR. The array has two ProSeqs from hemagglutinin (VMVHA, ∼500 bp) and cytokine response modifier B (VMVcrmB, ∼300 bp) genes for Variola major virus detection. The percentage of the ProSeq that hybridizes is sufficient that if hybridization patterns were only considered one might assume that this tile is identifying the presence of its target. This would indicate that reference sequence selected was not the best choice. However, when our algorithm was applied none of the samples is in fact identified as Variola major or minor virus. Vaccinia was always one of the Orthopoxvirus species listed with the highest scores for VMVcrmB ProSeq, but in only seven cases was it uniquely identified as the probable species detected. In only one sample at the lowest concentration and fraction of VMVcrmB hybridizing, did this ProSeq even identify Variola major and minor virus as one among the Orthopoxvirus species that could be the cause of the hybridization. The lower limit of detection for the amplification method used was between this concentration and the one above it for Variola major itself. The VMVHA ProSeq exhibited much lower sensitivity and made identifications of Orthopoxvirus species in only two experiments and Variola major virus was listed as one of the tied best scoring returns. In both cases, VMVcrmB ProSeq specifically identified Vaccinia virus as the best match. The percentage of the hybridized ProSeq correlated with concentration of the sample.', 'This example demonstrated the importance of the filtering portion of the algorithm by considering the HybSeqs of the ProSeqs for the H1N1 neuraminidase (NA1) and matrix genes from human Influenza A/Puerto Rico/8/34 (H1N1) strain. Filtering was necessary because sending the HybSeq of a ProSeq to BLAST in a single query can bias the scores against strains that have insertions or deletions relative to the ProSeq, especially when using BLAST parameters that maximized the use of base calls. The sliding window test was the portion of the algorithm that controlled filtering. If filtering were turned off, the entire HybSeq would be used in a single subsequence for two influenza ProSeqs that showed significant hybridization. A/Weiss/43 (H1N1) strain was identified as the most likely strain from the HybSeq of the NA1 ProSeq while the HybSeq of the matrix ProSeq correctly identified A/Puerto Rico/8/34. To better understand the source of biasing, CLUSTAL alignment of the NA1 gene of the two strains and the reference sequence used to make the ProSeq are shown in Figure 5. The two strains showed 95% identity (67 mismatches in 1362 aligned bases); however, there was a stretch of 45 bases inserted in both A/Weiss/43 and the NA1 ProSeq compared to A/Puerto Rico/8/34. With the default filtering on, the NA1 ProSeq was split into five SubSeqs as the algorithm encountered large stretches of no calls. In ProSeq identification Task(I), the algorithm determined that three of the SubSeqs, which were shorter, had an identified organism of H1N1 as several isolates including A/Puerto Rico/8/34 tied with the best score, while the two other SubSeqs had an identified organism of only the A/Puerto Rico/8/34 strain as being the closest match. The organism identified by the NA1 ProSeq was A/Puerto Rico/8/34 because one of the SubSeqs had a much higher score. This ProSeq supported the same strain identification made in the matrix ProSeq. The organism identified was A/Puerto Rico/8/34 because two ProSeqs detected only that organism. The correct target pathogen was detected with filtering while without filtering the target pathogen level of identification would have been Influenza A (H1N1 subtype) because two organisms were detected, A/Puerto Rico/8/34 and A/Weiss/43. Splitting the HybSeq into SubSeqs to remove biasing may reduce the level of identification as occurred for the three of the five SubSeqs in this case. The previous example for Vaccinia was another instance where the wrong species (Camel Pox or Callithrix jacchus) identification would occur if the filtering was not used. Clinical samples in Table 2 showed that HybSeqs split into multiple SubSeqs were capable of very specific identification.', 'To illustrate the other filtering performed, when multiplex strategies rather than generic were used for amplification Figure 5 also contains the raw and mask filtered results of this region for the A/Puerto Rico/8/34 sample. It was necessary to perform additional filtering to remove potential biasing caused by the specific primers as described in the methods. In the case shown in Figure 5, a sequence of 18 bases present in the raw result are made N after filtering since they are in a location that interacts with the primers. If these base calls were included in the subsequences constructed, even though the HybSeq would still be split into the same number of SubSeq, the query for the ProSeq would favor an incorrect strain.', 'The algorithm we have developed successfully provided pathogen identification to the maximum level of detail possible (species or strain) depending on the quality of each ProSeq. This identification capability requires minimal input on the identity of the pathogens, making non-expert use feasible. The crucial feature incorporated that allowed complete automation was the use of taxonomic databases, which classify organisms into ordered groups and provide relationships between organism entries, allowing removal of redundancies, comparison of different related prototype sequences and simplification of data presentation. This allows databases, i.e. NCBI, that are redundant and subject to minimal curation but which constantly receive updated and new sequence information to be used with great success. Although we have demonstrated this using only the NCBI databases, other databases or custom made ones could have easily been used, which might improve performance. The algorithm is capable of providing accurate identifications at all analysis levels for pathogens that are less variable or are represented by highly conserved ProSeqs. For more variable or rapidly mutating pathogens, e.g. Influenza A virus, ProSeq identification Task(I) and ProSeq grouping Task(II) still provided accurate detailed identifications, but the pathogen determination Task(III) was unable to report fine scale discrimination. The comparison of the conventionally sequenced Influenza virus gene sequences illustrated that the algorithm is capable of automatically adjusting for updates in databases. The algorithm demonstrated its capability to properly distinguish hybridizations on a ProSeq caused by the specified pathogen from those caused by genetically close (near neighbor) strains and did not make incorrect identifications, eliminating one potential cause of false positives. Filtering the raw hybridization results served to reduce the computation time, accounted for potential primer interference and more importantly reduced potential biasing. This simple integrated algorithm provided sufficient and accurate identification, so that immediate use of the RPM v.1 or similar resequencing arrays and assay is possible. Although not discussed in this paper, the algorithm has successfully detected the presence of simulated multi-infections (Lin et al., submitted for publication). The algorithm as currently developed will detect mixtures when the organisms have sufficient variation; however, detection of a mixture of an organism and its mutation strain in a sample is uncertain in its present phase. In principle it may be possible to detect such mixtures as the resequencing microarray can detect and sequence diploid organisms.', 'Besides demonstrating the success of the CIBSI 2.0 program, the work involved in developing the algorithm allowed insight into the importance of proper ProSeq selection. The RPM v.1 was the first resequencing array designed specifically for multiple pathogen detection using database similarity searching and served as a prototype for this application. We have demonstrated that a single ProSeq with as few as 100 bp, when designed correctly, can be sufficient to unambiguously identify an organism. However, it is clearly indicated that several longer ProSeqs provide better confirmation and more detailed information of a pathogen. Although the algorithm provides accuracy equivalent to manual analysis for determinations of individual ProSeqs, the current algorithm is only partially successful in integrating information from multiple ProSeqs. The emphasis of the design to this point has been on capabilities that are generally applicable to any pathogen. We are incorporating these insights in our newer more comprehensive resequencing array designs. Improving on level of detail reported in pathogen determination Task(III) will require more information about an individual pathogen and may have to be developed for each specific pathogen or class of pathogens. This information is also required for the algorithm to identify which differences between a sample and database entries represent significant mutations. Future work will involve improving the use of the current taxonomic database or potentially developing a new relational database that is specific to our needs and then incorporating more specific information of target pathogens. The hierarchal design of the data analysis makes it easy to incorporate analysis that build upon the analysis already performed.', 'We have met with some success in the current version but want to have increased automated discrimination. We have a well-defined path to completing this aim. The use of properly designed resequencing microarrays and this automated detection algorithm provides a way forward to developing assays that can test for multiple organisms simultaneously while providing fine strain level discrimination giving access to information about detailed strain recognition, antibiotic resistance markers and pathogenicity. This is a capability that other approaches cannot currently provide. In addition, since the design of the original 30 kb RPM microarray, the possible sequence content of the current array has increased 10-fold to 300 kb and further increases in array density are still attainable. This, coupled with our identification algorithms, will allow the analysis of partial sequence information from even more organisms for applications such as differential diagnostics for illnesses with multiple potential causes (i.e. febrile respiratory illness), tracking of emergent pathogens, distinction of biological threats from harmless near genetic neighbors in surveillance applications and for tracking the impact of co-infections or super infections. The concept of categorizing and reporting different degrees of identification depending on the quality of samples and set of target sequences is not limited to resequencing microarrays but is more generally applicable to any platform that is capable of returning sequence level calls that can be used to query a reference DNA database. As the trend for assays that test for multiple pathogens increases, automated analysis tools, such as this one, become more crucial for rapid identification in simple formats useful to the non-expert on a day to day basis. The remaining hurdle to using resequencing microarrays as a routine assay method is now clearly the sample processing methods. Further automating these steps is an important area of future research and development.', 'The program can be obtained free of charge for research purposes by contacting the authors.', 'Supplementary Data are available at NAR Online.'], 'cord_uid': 'cxzlmfst'}\n{'title': 'A DNA biochip for on-the-spot multiplexed pathogen identification', 'text': [\"Decentralized medical testing plays a vital role in today's health care system. The blood glucose meter, which was the first commercial handheld device for medical diagnostics and developed three decades ago, is by far one of the most successful examples in point-of-care testing (POCT). In the years to come, driven by the ever-growing threats from emerging infectious diseases (e.g. avian flu and severe acute respiratory syndrome), the development of small-size instruments for on-the-spot pathogen detection is expected to be an important segment of the POCT market. This trend has already commenced with a few companies having launched self-test products for hepatitis and human immunodeficiency virus detection based on antibody–antigen interactions (e.g. OraSure Technologies and ACON Laboratories). These systems give a visual readout indicating the presence or absence of the target virus in ∼15 min. One of the main shortcomings of these immunological techniques is their limited sensitivity. To address this issue, there have been significant efforts to develop nucleic acid (NA)-based analyzers (1–3). The miniaturization of NA analytical platforms has many advantages over the conventional bench-top counterparts. These include low sample/reagent consumption (volume of micro- down to picoliter) as well as short assay time (minutes rather than days). Most importantly, they permit the integration of a number of functions including sample preparation, target amplification, and product detection, thus enabling a fully automated operation that can be used by untrained individuals.\\nTo date, several integrated NA-based analytical systems have been commercialized [ (GeneXpert from Cepheid),  (Direct Tube Sampling Systems from Gen-Probe),  (RAZOR from Idaho Technology) and  (Liat Analyzer from IQuum)]. Despite their wide use in clinical/central laboratories, their application for on-site pathogen monitoring on a routine basis is still limited due to the large footprint and high instrument cost (mainly the complex optics). A promising alternative would be the inherently simple and lost-cost electrochemical method. Over the past decade, despite a great deal of work having been carried out on electrochemical sequence-specific NA sensing (4–6), little work has been undertaken on the integration of these with upstream functionalities. In 2004, Liu et al. (7) successfully demonstrated a fully integrated biochip for cell isolation and lysis, target amplification, as well as electrochemical amplicon detection. One relevant feature of their approach is the incorporation of on-chip mixers, valves, and pumps in a self-contained device. However, the design and fabrication of the chip involves many complicated steps, which limits its practical application.\", 'In a previous study, our group demonstrated a proof-of-concept experiment that both DNA amplification by the PCR and sequence-specific electrochemical amplicon detection could be done in a single microchamber (8), in contrast to the commonly used device which involved multi-chambers with complex microfluidic control elements (9). This microdevice had an 8 μL reaction chamber etched in a silicon substrate with a thin-film heater and temperature sensor patterned on top for rapid thermal cycling. An oligonucleotide capture probe-modified detection electrode was placed on a glass substrate used to seal the microchamber. To develop this prototype device into practical use, sample preparation functionality as well as the ability to perform multiplexed analysis would need to be addressed.', 'In this work, we present a complete DNA-based assay in a single silicon–glass microchamber for multiple pathogen detection. A model system of E.coli and B.subtilis was used. The assay involves the following steps: (i) sample preparation using thermal cell lysis and magnetic particle-based target genome isolation; (ii) target DNA amplification by the PCR; (iii) hybridization of the amplicons to their complementary oligonucleotide capture probes immobilized onto individual detection electrode surfaces and (iv) electrochemical transduction of the recognition event via gold nanoparticles with signal amplification using electrocatalytic silver deposition (10). An issue of importance much needed to be addressed was the compatibility of all the materials and processing steps. In particular, the chemistry used for the site-specific probe immobilization together with that of the magnetic particles used for genome isolation should be PCR-compatible.', 'Oligonucleotides and PCR reagents were obtained from Invitrogen (Carlsbad, CA, USA), unless otherwise stated. Other chemicals were purchased from Sigma–Aldrich (St. Louis, MO, USA). Electrochemical measurements were performed using a VMP multichannel potentiostat (Princeton Applied Research, Oak Ridge, TN, USA) controlled by EC-Lab software (version 6.70, Bio-Logic Science Instruments, Claix, France). The thermal control system for the PCR consisted of a data acquisition card (PCI-MIO-16E-1, National Instruments, Austin, TX, USA) along with a signal conditioning board (SC-2042-RTD, National Instruments) connected to the temperature sensors. A digital feedback proportional-integral-derivative (PID) control algorithm was implemented in LabVIEW software (National Instruments) to control voltage supply to the heater from a power source (HP6629A, Hewlett-Packard, Rockville, MD, USA).', \"The silicon chip (thickness of 400 μm) had two fluid injection holes (top side, diameter of 500 μm, depth of 100 μm) and a chamber (bottom side, length and width of 5 mm, depth of 325 μm) etched by the inductively coupled plasma/deep reactive ion etching (ICP/DRIE) process, see Figure 1A (left panels). Thin-film platinum (100 nm) was patterned on top of the silicon substrate as heater and temperature sensors (Figure 1A, upper left). The glass chip had platinum pseudo-reference and counter electrodes (thickness of 100 nm) as well as four working electrodes made of indium tin oxide (ITO) (thickness of 100 nm), see right panel of Figure 1A. Ultra-violet curing optical cement (Type UV-69, Summers Optical, Hatfield, PA, USA) was used to bond the silicon and glass chips, the curing procedure was in accordance with the manufacturer's instruction. Pipet tips were glued to the fluid injection holes with epoxy (Figure 1B).\", 'An oligonucleotide capture probe specific for E.coli amplicon [pyrrole-5′-ACAACACGTTTAGCCTGACC-3′ (pyrrole-EC), Apibio, France] was first electrochemically polymerized onto two of the four ITO working electrodes. A mixture of 60 mM pyrrole, 20 μM pyrrole-EC, and 0.1 M LiClO4 was introduced into the microchamber, followed by a cyclic voltammetric scan of the two electrodes between −0.5 and +0.65 V at a scan rate of 50 mV/s for three times. The microchamber was then washed with deionized water and dried with nitrogen gas. The same procedure was repeated for the other two ITO working electrodes, except an oligonucleotide capture probe specific for B.subtilis amplicon [pyrrole-5′-CCTACGGGAGGCAGCAG-3′ (pyrrole-BS), Apibio, France] was used.', 'Ten microliters of avidin-coated magnetic particles (3.0 μm, VMS-30-10, Spherotech, Libertyville, IL, USA) were washed with an equal volume of saline/sodium citrate buffer (SSC, 150 mM NaCl/15 mM sodium citrate, pH 7.0). After centrifugation and pipetting, the supernatant was removed and the magnetic particles were incubated with 10 μL of 10 nM biotinylated genome capture probe overnight at room temperature. Oligonucleotide sequence of the genome capture probe for E.coli was 5′-biotin-GACAAGAAAATCTCCAACATCC-3′ while that for B.subtilis was 5′-biotin-CCAGTTTCCAATGACCCTCCCC-3′. The capture probe functionalized magnetic particles were finally washed with 10 μL of the SSC buffer, resuspended in 10 μL of the SSC buffer, and stored at 4°C.', \"The sample containing E.coli or B.subtilis or both (1 μL), which was cultured in Luria–Bertaini broth overnight at 37°C, was mixed with 1 μL each of the biotinylated genome capture probe (1 nM) for E.coli and B.subtilis along with 7 μL of the SSC buffer. The mixture was injected into the reaction chamber and sealed with Bostik's Blu-Tack. Then, the silicon–glass device was placed in a plexiglass holder with contact pins for electrical connections to the heater and temperature sensors (Figure 1C). The chamber was maintained at 90°C for 5 min to lyse the cell and at the same time denature the genomic DNAs. After that, the temperature was cooled to 50°C and held for 10 min to allow specific hybridization between the denatured genomic DNAs and capture probes. Subsequently, 1 μL of the functionalized magnetic particles were added into the chamber and incubated for 10 min to capture the specific genomes onto the magnetic particles. Finally, SSC buffer was used to remove any unwanted materials with an external magnet to keep the particles within the microchamber.\", 'The PCR master mix consisted of 1× buffer (20 mM Tris−HCl, pH 8.4, 50 mM KCl), 2 mM MgCl2, 0.2 mM dNTPs, 200 nM of biotinylated E.coli forward primer (5′-biotin-GACAAGAAAATCTCCAACATCC-3′), 2 nM of E.coli reverse primer (5′-ACAACACGTTTAGCCTGACC-3′), 2 nM of B.subtilis forward primer (5′-CCTACGGGAGGCAGCAG-3′), 200 nM of B.subtilis biotinylated reverse primer (5′-biotin-CCAGTTTCCAATGACCCTCCCC-3′), 0.5 μg/μL BSA, and 0.4 U/μL Taq polymerase. The mixture was pipetted into the microchamber and subjected to the following thermal cycling profile: initial denaturation at 94°C for 5 min; 40 cycles at 94°C for 30 s, at 60°C for 30 s, 72°C for 30 s; and final extension at 72°C for 5 min.', 'After the asymmetric PCR, the solution was allowed to stand at room temperature for 1 h. Unhybridized amplicons were washed away with the SSC buffer. Gold nanoparticle label was bound to the hybridized amplicons by exposing the electrode to a streptavidin−gold nanoparticle (5 nm) solution (the stock was diluted 10 times with 0.05 M 4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid/0.2 M NaCl) for 30 min at room temperature. The unbound gold nanoparticles were removed by flushing the microchamber with phosphate-buffered nitrate solution (0.3 M NaNO3/10 mM sodium phosphate, pH 7.0). Electrocatalytic silver deposition onto the hybrid-bound gold nanoparticles was then achieved by applying a potential of −0.18 V in a silver nitrate solution (1 mM AgNO3/1 M KNO3) for 20 s. Finally, the amount of deposited silver was determined by measuring the oxidative silver dissolution response with an applied anodic current of 10 μA in the same silver nitrate solution, and the time to reach a potential of +0.65 V was taken as the signal.', 'The single microchamber design poses particular challenges to the electrochemical platform used for the sequence-specific PCR amplicons detection. Addressability and compatibility are two important considerations regarding immobilization chemistry for the oligonucleotide detection capture probes. For a multiplexed assay, it is necessary to individually modify the detection platform so that each individual electrode in an electrode array has a specific capture probe. When using either high temperature or ultra-violet glue to seal the microchamber, it is recommended that immobilization should be carried out after the silicon−glass bonding process so as to prevent damage to the capture probes. In doing so, the more common chemical attachment (spotting) method cannot be used because all the active electrode surfaces are embedded within the same microchamber and they would receive identical modifications. One simple way to achieve site-specific probe immobilization onto individual electrode surfaces can be achieved by electrochemical copolymerization of pyrrole and pyrrole−oligonucleotide (11). Figure 2 illustrates the strategy to immobilize different capture probes onto each individual electrode. A solution of pyrrole and oligonucleotide 1 bearing a pyrrole group is introduced into the microchamber. When a cyclic voltammetric scan is applied to electrode 1, with other electrodes disconnected or grounded, oligonucleotide 1 is selectively deposited on this particular electrode. Then, the microchamber is washed with water to ensure there is no pyrrole−oligonucleotide 1 monomer is left. This procedure is repeated for the other electrodes with different pyrrole−oligonucleotide polymerization solutions. In our model system with two target analytes and four working electrodes, the capture probes specific to E.coli and B.subtilis are immobilized in duplicate. Before proceeding to the complete analytical protocol, the ability of these immobilized capture probes to recognize their complementary targets should be tested. Figure 3 shows the fluorescence images of the four functionalized electrodes (A and D: B.subtilis probe; B and C: E.coli probe) exposed to a sample containing a fluorescently-labeled sequence complementary to the E.coli probe. It is clear that electrodes B and C exhibit much higher fluorescence intensity than electrodes A and D, indicating the highly specific probe immobilization as well as hybridization offered by the electrochemical pyrrole-based attachment chemistry. Another criterion for the selection of immobilization method is the compatibility with other processes, in particular the PCR. Due to the fact that the detection electrodes are within the reaction chamber, the linkage between the immobilized capture probe and electrode surface must be strong enough to survive through the thermal cycling process (especially the high denaturation temperature). Moreover, the detector surface should interact only with the specific amplicon but not with other components employed in the assay protocol.', 'The assay procedure used in this work is schematically represented in Figure 4. It involves three main steps: sample preparation, target DNA amplification, and product detection, all performed within the same microchamber. Intact cells are first broken down by applying a high temperature (90°C, controlled by the on-chip heater and temperature sensor) to free the genomic DNA. To remove all the interfering substances (e.g. cell debris and protein) that may affect the subsequent DNA amplification process, magnetic particles are used to isolate the specific genomes. Biotinylated genome capture probes for the two model species are mixed with the intact cells before injecting into the microchamber. When the temperature is lowered to 50°C after the thermal lysis step, these probes hybridize to their complementary target genomes. These probe−genome hybrids are then isolated by the addition of the avidin-coated magnetic particles, followed by thorough washing. It is worth noting that the magnetic particles are pretreated with a small amount of the genome capture probes to minimize nonspecific adsorption of the interfering substances and other genomic DNAs. Subsequently, with the genomes captured on the magnetic particles serving as the template, asymmetric PCR is conducted to generate single-stranded rich target amplicons. After the amplification step, these amplicons hybridize to their corresponding detection electrodes. Next, the hybridized amplicons are labeled with gold nanoparticles via biotin–avidin interaction. Finally, silver metal is electrocatalytically deposited onto the gold nanoparticles and the amount is determined by the electrochemical oxidative dissolution technique (10,12). The detailed procedure for the sample preparation, target DNA amplification, and product detection steps is given in the Materials and Methods section.', 'The ability of this single microchamber to detect specific cell type using the above protocol is demonstrated by running a series of experiments with E.coli cells of different concentrations, taking the signal from the B.subtilis detection capture probe-modified electrode as the background. Figure 5 gives a semi-log plot of the sample to background ratio against the number of cells in the sample. A linear relationship is obtained in the concentration range investigated (102–105 cells/sample). This result confirms the successful isolation of the genome with the magnetic particle-based approach, compatibility of the magnetic particle with the PCR, thermal stability of the immobilized detection capture probe through the temperature cycling process, as well as negligible nonspecific adsorption on the electrode surface.', 'Another attractive feature of this microdevice is multiplexing. By constructing an electrode array, it is possible to identify several species in a single run. The results for the detection of the two model species are presented in Figure 6. If the sample contains E.coli cells only, there is a significant increase in the analytical signal (silver metal stripping time) for the E.coli detection capture probe-functionalized electrode while that for the B.subtilis remains the same as the background signal. When the sample contains B.subtilis cells only, opposite results from the two different electrodes are obtained. Another case is the inclusion of both cell types; not surprisingly, both electrodes have much higher signals than the background.', 'We have demonstrated the utilization of a silicon−glass-based microchamber for DNA-based detection of E.coli and B.subtilis cells. Thermal cell lysis, magnetic particle-based target genome isolation, DNA amplification, and electrochemical sequence-specific amplicons detection have been successfully implemented in this microdevice platform. The portable electrochemical instrumentation as well as a simple microchip design is conducive to the realization of on-site pathogen detection. The selective immobilization of capture probes using the pyrrole-based electropolymerization process provides good thermal stability and PCR process compatibility, which are crucial for the multiplexed analysis. Future work will be directed towards interfacing the microchip with macro-world so as to achieve a fully automatic device that can be used by untrained individuals.'], 'cord_uid': '0sg0hv9w'}\n{'title': 'Molecular dynamics simulations of human tRNAUUULys,3: the role of modified bases in mRNA recognition', 'text': ['Nonstandard bases are most prevalently found in tRNA molecules and may involve a variety of modifications from simple methylation to addition of a peptide moiety (1,2). Some tRNA molecules require posttranscriptionally modified nucleic acid bases for proper translation (3,4), recognition by proteins (5,6) and initiation of human immunodeficiency virus reverse transcription (7). Specifically, ribosomal binding (8,9), proper translocation from the aminoacyl site (A) to peptidyl site (P site) (10,11) and translational speed (12) may be dependent upon nonstandard bases, while the lack of modified bases may result in frameshifting errors (13). Most tRNAs contain at least one modified base (14) and modifications often occur at positions 34 and 37 (4). Modified bases are generally thought to increase stability of tRNA structure by providing additional hydrogen-bonding contacts, better base–base stacking interactions (4,15–19) and metal ion binding sites (20–22). Specifically, base modifications at the 34th and 37th positions of the tRNA are often important for facilitating cognate codon recognition (4). Still, the role specific modifications play in determining RNA structure, stability and function are not well understood.', 'Functional groups in modified nucleosides can dramatically affect the ability of tRNA anticodons to recognize their cognate mRNA codon and therefore participate in accurate translation of the genetic code (15,16). The third human tRNA coding for lysine (tRNALys,3) contains three posttranscriptionally modified bases in the anticodon stem–loop (ASL, Figure 1). Biochemical assays have established that 5-methoxycarbonylmethyl-2-thiouridine at position 34 (mcm3s2U34) and 2-methylthio-N6-threonylcarbamoyl-adenosine at position 37 (ms2t6A37) are required to mimic wild-type human tRNALys,3 binding to programmed ribosomes (9). Pseudouridine (ψ) presence at position 39 is not critical to anticodon–codon recognition but has been shown to contribute to helical stability (17). Escherichia coli tRNALys,, analogous to its human counterpart, contains a 5-methylaminomethyluridine-2-thiouridine at position 34 (mnm5s2U34) and N6-threonylcarbamoyl-adenosine at position 37 (t6A37) that are both important for ribosomal A site binding (8,9,23), translocation from the A site (11) and discrimination between AAA and AAG codons (24,25).', 'Several structures of human tRNALys,3 and E.coli tRNALys, variants with variously modified bases in a variety of contexts have been determined by both NMR spectroscopy (18,26–29) and X-ray crystallography (19,30). The expensive and often complex synthesis of nonstandard bases further complicates incorporation of all fully modified bases. Structural studies indicate that preservation of a U-turn motif in E.coli anticodon loop is important for recognition of cognate mRNA (28). Systematic, detailed NMR studies of fully modified E.coli and partially modified human tRNALys,3 indicate flexibility in the U36 position (29). In the context of the 30S ribosomal subunit, an anticodon loop containing t6A37 has been found to recognize the cognate mRNA codon (19) in a canonical Fuller–Hodgson tRNA form (31).', 'The contributions of each modified base to free tRNA and ribosomally bound tRNA structures remains elusive (4,16,32). Molecular dynamics (MD) studies considering specific contributions of modified bases mcm5s2U34, ms2t6A37, ψ39 to the structure and function of the human ASLLys,3 are reported. Specifically, the role of modified bases in preserving canonical tRNA anticodon structure is examined.', 'Fully refined NMR structures are currently not available for fully modified tRNALys,3; therefore all simulations were begun from X-ray crystal structures. MD simulations of the tRNAUUULys,3 anticodon stem–loop with bases 27–43 (ASLLys,3 wild type) derived from the fully modified tRNALys,3 X-ray crystal structure solved to 3.3 Å resolution (PDB code 1FIR) (30) were performed with different combinations of mcm5s2U34, ms2t6A37, ψ39 and unmodified bases (Table 1). All variations of ASLLys,3 were derived from the Bénas et al. (30) structure. While this study was being conducted, the crystal structure of E.coli ASLLys,, which contains t6A37, with the 30S ribosomal subunit and mRNA was solved to 3.0 Å resolution (PDB code 1XMQ) (19). In E.coli ASLLys,, the ureido ring of t6A37 stacks with A38 stabilizing anticodon–codon interactions. To ensure adequate sampling of relevant structures, an additional system, wild-type 2, was derived by grafting the E.coli ASLLys, anticodon loop containing an ureido ring in t6A onto the Bénas ASLLys,3 stem and applying the appropriate full-human ASLLys,3 modifications.', 'All MD simulations were performed according to the same protocol. Trajectories were propagated with the Cornell et al. force field (33), as implemented in the SANDER module of the program AMBER 6.0 (34). ASLs were solvated by a 10 Å rectangular box of TIP3P water (35) and Na+ ions were added to neutralize the system. Simulations were performed under periodic boundary conditions, employing the particle mesh Ewald (36) formalism to account for long range electrostatics.', 'MD trajectories were propagated with a 2.0 fs time step, employing the SHAKE algorithm (37) on all hydrogen atoms, with a nonbonded cutoff of 9.0 Å. The pairlist was updated every 25 steps. Constant pressure (1 atm) and temperature (300 K) were maintained according to the Berendsen coupling algorithm (38). The equilibration protocol of Cheatham was followed (39) similar to previous simulations of nucleic acids (40–45). Simulations were allowed to equilibrate under production-run conditions for at least 2.0 ns before collection of statistics over 4.0 ns.', 'Parameters for nonstandard bases were derived to be consistent with the Cornell et al. force field (33) and specific parameters are available in Supplementary Data. Atom types, force constants and equilibrium values were assigned by analogy to the Cornell et al. nucleic acid, threonine side chain (t6) or methionine (ms2) force-field parameters (33), or the General Amber Force Field (46) for sulfur-related parameters. Partial atomic charges were derived from Hartree–Fock (47)/6-31G* (48–50) electrostatic potential charges fitted with a two-step restrained electrostatic potential procedure (51). Electronic structure calculations were performed with Gaussian 03M, revision B.04 (52).', 'Quantum mechanical calculations employing the modified Perdew-Wang 1-parameter model for kinetics (MPW1K) (53,54) and the 6-31+G (d,p) basis set (55,56) were used to determine the rotational barrier around the C10-N11 amide bond involved in ureido ring formation.', 'Trajectories were analyzed using the CARNAL and PTRAJ modules of AMBER 6.0 (34). All statistics were collected on 1 ps snapshots over 4.0 ns of data. Helical parameters were measured using the 3DNA program (57). Trajectories and structures were visualized using VMD (58) and UCSF chimera (59). Criteria for hydrogen bonding was a 3.5 Å distance between heavy atoms and a heavy atom-H-heavy atom angle of 180° ± 60°. Solvent accessible surface area (SASA) was calculated as a Connolly surface (60), as implemented in the MM-PBSA module of AMBER 8.0 (61). When determining if bases were parallel, the angle between normal vectors to the plane of the respective bases was used. Planes were defined using three points; N1, N3 and C5 for pyrimidines and N3, C6 and C8 for purines and the normal vectors were defined to point in the 3′ direction.', 'Trajectory stability was based upon steady fluctuations around constant values for root mean square deviation (r.m.s.d.) values from the average (Figure 2) and starting structures (Supplementary Data) as well as monitoring of helical parameters of the double-stranded region such as local twist, x-displacement from the helical axis and base pair inclination (Supplementary Data).', 'Overall, the helical portion of all ASL structures exhibited classic A-form geometry (62) with x-displacement values ranging from −4.5 ± 0.6 to −5.3 ± 0.9 Å and base pair rise parameters of 2.5 ± 0.3 to 2.8 ± 0.3 Å. The majority of structural analysis occurred in the loop region of the ASLs.', 'The wild-type simulation was unusually stable with an average r.m.s.d. from the starting structure of only 1.8 ± 0.3 Å. To ensure that the wild-type simulation was indeed naturally stable and that periodic effects or lack of sampling time did not influence the results, data for two additional simulations were collected (N. E. McCrate, unpublished data). In the first simulation, the water box surrounding the RNA was increased to 15 Å beyond the solute. In the other simulation, with 10 Å of water added beyond the solute, data were collected for 40 ns. Both additional simulations retained the structure observed in the reported wild-type simulation.', 'All simulations contained a hydrogen bond between C32 O2 or N3 and the A38 exocyclic amine to varying extents of the simulation but averaging 74% occupancy. Simulations containing no modified bases or only ψ39, exhibited an additional hydrogen bond between the exocyclic amine of A37 and the O2 of U33 56 and 62% of the simulations time, respectively. Addition of either mcm5s2U34 and/or ms2t6A37 decreased the A37–U33 interaction to <4% of the simulation time.', 'The side chain of mcm5s2U34 did not form hydrogen bonds often with other parts of the ASL. The only detectable hydrogen bond observed occurred between the mcm5 carbonyl oxygen and the hydroxyl of ms2t6A37 in the wild-type simulation and the simulation containing both mcm5s2U34 and ms2t6A37 for <20% of the simulation time.', 'In the models presented herein there seems to be no correlation between presence of a U33 NH3 hydrogen bond to U36 O2P and any one conformation. Simulations containing mcm5s2U34 seem to lose a U33 NH3 hydrogen bond with U36 O2P (<3%) while simulations lacking a modification at position 34 close to form this hydrogen bond >90% of the simulation time. The exceptions to this trend are both wild-type simulations, which contain this hydrogen bond 100 and 91% for wild-type and wild-type 2 simulations, respectively.', 'In the fully modified ASL structure including mcm5s2U34, ms2t6A37 and ψ39 (wild-type ASLLys,3), the structure retained a stair-stepped conformation, in which the uridines (Figure 3) are offset and nearly parallel to each other, observed previously in the X-ray crystal structures of free human tRNALys,3 (30) and when E.coli t6A ASLLys, pairs with its cognate mRNA codon in the context of the ribosome (19).', \"To characterize the stair-stepped conformation, the relative positions of the anticodon bases to each other were determined. To analyze the positions of the anticodon bases relative to each other, the distances between the center of mass (COM) of anticodon bases' aromatic rings were measured. The distance vectors were transformed into a relative coordinate system utilizing a rotation matrix and decomposed into x (dx), y (dy) and z (dz) components, to describe the relative positions of the bases. By analogy, the dx, dy and dz components describe the height width and length of ‘the stair’. From visual inspection, the wild-type simulation was determined to exhibit the prototypical stair-stepped conformation. To determine whether the geometry of the anticodon for each model matched that of the wild-type, dx, dy and dz between each set of rings (U34–U35 and U35–U36) were classified as ‘stair stepped’ if the values for both pairs of rings fell within 4 SDs of the average values calculated for the wild-type model and the angle between the normal vectors of the planes of the bases was 0 ± 45° (63) (Table 1).\", 'To aid in data analysis, snapshots of each simulation were assessed for presence of a stair-stepped conformation. Based upon the criteria outlined above, the percent occupancies of stair-stepped interactions for positions 34–36 were determined (Table 2). Simulations that contained an ms2t6 modification at position 37 exhibited the stair-stepped anticodon configuration 75–100% of the simulation time, while those ASLs lacking base modifications at position 37 were not stair-stepped for most, if not all, of the simulation. In the wild-type 2 model, in which the anticodon appears to be stair-stepped, the U36 base is positioned closer to the N9 of ms2t6A37 than is observed in the other models. As a result of this difference in position, the wild-type 2 simulation is classified as having a less stair-stepped codon (46%) than the other models because it is stair-stepped in a slightly different conformation.', 'When ms2t6A37 is substituted with A37, U36 rotates around the glycosidic bond and is exposed to solvent. This was observed in all simulations lacking ms2t6 modifications at position 37. Quantification of U36 solvent exposure through the SASA of the U36 base indicates that the presence of the ms2t6 modification at position 37 reduces the SASA of U36 (Table 2). Relatively smaller U36 SASA values correspond to the stair-stepped conformation, while larger SASA values correspond to U36 unstacking. In addition, mcm5s2U at position 34 appears to slightly lower the SASA of U36 as all models lacking mcm5s2 modifications have slightly higher U36 SASA than their modified counterparts (10–30 Å2, Table 1).', 'From visual inspection of the wild-type simulations, it seemed that bases 36–38 were interacting, perhaps by base–base stacking interactions. All simulations were examined to determine the extent of base–base stacking in the anticodon loop. A stacking interaction was defined to occur when the distance measured from the COM between two aromatic rings was <4 Å and the angle between the normal vectors of planes formed by the bases was ±20°, as determined appropriate by previous studies (63,64).', 'Based on the stacking criteria outlined above, the percent occupancy of stacking interactions for positions 36–38 was determined (Table 2). In all simulations containing ms2t6A37, for a portion of the simulation (3–34%), U36 was in close proximity to the A37 5-membered ring. The stacking data indicate that the presence of ms2t6A37 may slightly enhance base–base stacking between U36 and position 37.', 'Modification of A37 encourages the base to remain intercalated between A38 and U36. In general, A37, modified or unmodified, hydrogen bonds with functional groups in the anticodon loop. Simulations containing ms2t6A37 tended to interact across the loop with base and sugar atoms in residues 32–35 for relatively short lifetimes. For instance, in the wild-type trajectory, the hydroxyl group from t6 was hydrogen bonded for most of the simulation time to some group in the tRNA. It preferred to interact with sugar oxygens in residue 33 but also hydrogen bonded with base oxygens in U34 and U35 carbonyls. When ms2t6 modifications were removed, the A37 disrupted the U36–U35 interaction and formed new h-bonds through N6 with bases in closer proximity, such as residues U35, U36 and A38. In the absence of ms2t6, the ASL rearranges to provide new hydrogen-bonding partners for the truncated N6 position of A37.', 'In the context of the ribosome, E.coli t6A37 contains an ureido ring which stacks with the 6-membered ring of A38 (19). The crystal structure of t6A alone contains an ureido ring (65). In the fully refined tRNALys3 X-ray structure (PDB code 1FIR), ms2t6A37 seems to adopt a conformation without an ureido ring although initial attempts were to try to have a clearer electron density showing an ureido ring-like conformation [P. Bénas, personal communication and figures in Ref. (30)]. Quantum mechanical calculations on N9-methylated ms2t6A at the MPW1K/6-31+G (d,p) level of theory indicate that the barrier to rotation around the C10-N11 bond to form an ureido ring is ∼9.0 kcal/mol at room temperature.', 'When ψ39 is present, a water bridge forms between A38 O2P, ψ39 NH3 and ψ39 O1P. Although the full water bridge is only present for ∼30% of the production run (detailed tables in Supplementary Data), the component hydrogen bonds between ψ39 NH3-water-A38 O2P and ψ39 NH3-water-ψ39 O1P are present for larger percentages of the production run, typically 50–80%. The waters involved in the water bridge are short lived with the bulk of the bridging interaction being attributable to water molecules having lifetimes at or below 40 ps and nearly all residence lifetimes being attributable to waters with lifetimes ≤120 ps.', 'MD simulations with the Cornell et al. force field (33) and TIP3P (35) water have been shown to identify long-lived, structural water molecules in other systems (41,44). The water molecule that coordinates with ψ39 is present most of the time but is not long lived. Pseudouridine has been shown to stabilize tRNA structure through a water-mediated hydrogen-bonding network (67). The crystal structure of yeast tRNAPhe, which contains a ψ55, has a water molecule bridging the N1-H position to the neighboring phosphate oxygens of T54 and ψ55 (68). Additionally, a long-lived water molecule coordinated to ψ has been found in tRNAAsp through MDs simulations (41). This water-mediated hydrogen bond with ψ is believed to restrict ψ motion and stabilize A–ψ base pairs (41,69,70), which are most prevalently found at positions 31–39 (14).', 'Overall the wild-type simulation retained the stair-stepped conformation with dx, dy and dz values all similar to that found in the X-ray crystal structure of human tRNALys,3 (30) and E.coli t6A-ASL bound in the 30S ribosome (19). Modifications at position 37 stabilize the anticodon conformation. From this data it appears that the presence of ms2t6A37 encourages retention of a highly ordered stair-stepped conformation.', 'Base modifications at position 37 seem to stabilize U36 participation in the stair-stepped conformation. U36 is the first base of the anticodon and is required for proper reading of the cognate codon base. The ms2t6A37 base is required for proper positioning of U36 to recognize its cognate codon base. Uridine with its hydrophilic carbonyls and N3, in general, is predisposed to an extra-helical position resulting in an increase in solvent exposure as noted in a systematic study of RNAs containing a single-base bulge (71). The highly ordered stair-stepped conformation of the anticodon bases seems to be important to codon recognition as originally hypothesized (31) and observed in crystal structures of tRNAs alone (72), interacting with other tRNAs through crystal packing effects (30,73–75), and in the context of the ribosome (19,76).', 'Base–base stacking data indicate that modifications on A37 may position the base to better interact with U36. However, this stacking interaction is not long lived in simulations. In order to have favorable base–base stacking interactions, the interaction of two hydrophobic bases must be enthalpically favorable (77,78). In this case, the negative electron densities of the aromatic bases may repel each other and electrostatic interactions, in which positive portions of the base above are interacting with the negative electron density of the base below, may predominate. This favorable electrostatic interaction has been observed in simple systems, such as benzene and water. In these benzene–water pairs, the partially positive hydrogens of the water molecules are interacting with the negative π-cloud of the benzene ring (79). It may be more favorable for a canonical anticodon to be stair-stepped because the positive portions of bases are aligned to interact with the negative π-cloud of the neighboring, lower base. When A37 is modified, it can partake in hydrogen bonds across the anticodon loop, effectively holding it up above U36. When modifications are removed, the base no longer has enthalpically favorable hydrogen-bonding interactions unless U36 rotates into solution, leaving unmodified A37 to favorably interact with U35 through electrostatic interactions and hydrogen bond with closer bases through its truncated 6-amino group.', \"The bulky ms2t6 modification containing hydrogen bond donors and acceptors may act sterically to restrict A37 movement while discouraging the displacement of U36 through weak stacking interactions and stabilizing the position of A37 through hydrogen bond contacts across the loop. In the wild-type 2 simulation, where the bulk of the t6A substituent's hydrogen bond donors and acceptors are solvent exposed, U36 exhibits markedly increased conformational freedom. This behavior suggests a delicate balance of forces, of which cross-strand hydrogen bonding is a portion, preventing the displacement of U36 (Figure 4).\", 'NMR studies indicate that the SCH3 of ms2t6A weakens the ureido hydrogen-bonding interaction as compared to t6A (29). Taken together with quantum calculations indicating facile interconversion at room temperature between an ureido ring conformation and a non-ureido ring structure, the conformation around C10-N11 may change based upon the context of the ribosome. In the E.coli 30S ribosomal subunit, primarily responsible for decoding (80), A1493, A1492 and G530 have been found to recognize the first two base pairs of the anticodon–codon complex in the minor groove (76), triggering a closure of the small subunit around the tRNA (81). In this system, the ureido ring allows greater stacking interactions with A38 (19), possibly giving U36 more flexibility to adjust to the codon. The prokaryotic and eukaryotic ribosomes contain conserved regions of rRNA around the decoding site (82) and mRNA crosslinks to A1493 and A1492 corresponding residues in human 18S subunits (83). However, human tRNALys,3 may rotate the t6 moiety around the C10-N11 bond allowing the t6 side chain to hydrogen bond across the loop, thereby stabilizing U36 for correct positioning in the ribosome in a canonical fashion. Accuracy in tRNA–mRNA recognition is both dependent upon a kinetic proofreading mechanism and the stability of the cognate codon–anticodon complex (84). The role of modified bases in stablizing codon–anticodon complexes as opposed to destabilizing the loop conformation for an induced-fit in the ribosome seems to be base and system dependent (4).', 'Supplementary Data are available at NAR Online.'], 'cord_uid': 'br2p09pg'}\n{'title': 'A novel endonuclease IV post-PCR genotyping system', 'text': ['Nucleic acid assays utilizing cleavage enzymes to generate a fluorescent signal from dual-dye-labeled oligonucleotide probes have been extensively used in diagnostics (1–3). In these assays, the ability of fluorescent dyes to transfer energy absorbed from light to nearby molecules forms the basis of homogeneous nucleic acid based assays (1,4,5). In the unhybridized state the fluorophore and quencher are within close proximity, which allows the quencher to absorb energy from the fluorophore to affect quenching. The use of fluorogenic probes in a 5′-nuclease polymerase assay, where the probe is enzymatically cleaved to release the fluorophore (2), is well known. The signal amplification reaction utilizes invasive cleavage with structure-specific 5′-nucleases (6). The method requires annealing of two oligonucleotides, called the upstream oligonucleotide and the probe, to a target sequence, which results in the formation of a unique substrate for the 5′-nuclease. In the DNAzyme-PCR strategy the primer contains a target-specific sequence and harbors the antisense sequence of a 10–23 DNAzyme. During amplification, amplicons are produced containing active sense copies of DNAzymes that cleave a reporter substrate included in the reaction mixture (7). A target nucleic acid sequence can be amplified exponentially in vitro under isothermal conditions by using three enzymatic activities essential to retroviral replication: reverse transcriptase, RNase H (RNA cleavage) and a DNA-dependent RNA polymerase (8). Cycling probe technology represents a simple method for the detection of DNA target sequences, utilizing a chimeric DNA–RNA–DNA probe which is cleaved by RNase H when hybridized with its complementary target (9).', 'Apurinic/apyrimidinic (AP) sites in DNA arise via spontaneous or mutagen-induced hydrolysis of the N-glycosylic bond, or through the repair activity of DNA glycosylases (10). If left unrepaired, AP sites are potentially lethal or mutagenic (11). To cope with the deleterious consequences of AP sites, organisms possess AP endonucleases that initiate the repair of these DNA lesions (10). Class II AP endonucleases initiate repair by catalyzing the hydrolysis of the 5′-phosphodiester of an abasic site to generate a 3′-OH group and a 5′-abasic phosphate residue (10). The substrate specificity of human apurinic endonuclease on synthetic substrate analogs has been reported previously (12).', 'Here we describe a novel endonuclease IV (Endo IV) assay substrate (Figure 1C) that mimics the abasic lesions that normally occur in double-stranded DNA (Figure 1A and B). The first lesion (Figure 1A) is a typical abasic (apurinic or apyrimidinic site generated by spontaneous or enzymatic loss of a nucleic acid base. The second lesion (Figure 1B) is an atypical abasic site appearing as a result of inherent instability of the 3′-phosphodiester bond in abasic deoxyribose in lesion 1 or its cleavage by a Class I AP endonuclease. As shown by the arrow, AP endonucleases cleave the phosphodiester linkage at the abasic sites. In the novel Endo IV assay a short probe with a fluorophore linked through a phosphate at the 3′ end and a short enhancer oligonucleotide generate an artificial, lesion 2 type Endo IV substrate. This arrangement allows the specific and efficient cleavage of the phosphodiester bond of the probe by Endo IV and release of fluorescent dye (Figure 1C). The addition of a quencher at the 5′ end of the probe allows quenching of the fluorescence in the uncleaved probe. The specific cleavage of the phosphate bond and the generation of fluorescence is the basis of the new assay.', 'In this study we used endonuclease IV from Escherichia coli (Endo IV). The enzyme functions as a Class II AP endonuclease and as a 3′ terminal phosphodiesterase (13,14). Endo IV is enzymatically similar to exonuclease III, the major AP endonuclease of E.coli, but it differs from the latter enzyme mainly in lacking the associated exonuclease activity (14). It was also found that E.coli Endo IV is fairly heat stable to ∼70°C (15,16).', 'Full characterization for all the compounds described below can be found in the Supplementary Data.', 'A total of 102 unrelated Centre Etude Polymorphism Humaine DNA samples were obtained from the Coriell Institute of Medical Research () after specifying that the DNA samples should be used for research purpose only. The list of templates used is available at .', 'For the synthesis of the fluorophore-modified controlled pore glass(CPG) supports, please see the Supplementary Data.', 'Fluorogenic probes were synthesized on fluorophore-linker-based CPG supports (Scheme 1) using standard 3′-DNA-phosphoramidites. An Eclipse Dark Quencher was introduced at the 5′ end using the corresponding phosphoramidite (). Probes were purified by reverse-phase high-performance liquid chromatography (HPLC), dried and re-dissolved in 1× TE buffer. A nearest-neighbor model was applied to calculate extinction coefficient (ɛ260) of oligonucleotides (17). A260 measurements were made in PBS (pH 7.2) at ambient temperature and assumed to be a random coil DNA structure in solution. For each Eclipse Quencher, fluorophore 1 (FL1), fluorophore 2 (FL2) or fluorophore 3 (FL3) substitution an ɛ260 correction of +6600, +18 500, or +25 100 or 28 600 M−1 cm−1 was used, respectively.', '3′-Hydroxyprolinol modified probe precursor was synthesized using a modified CPG support (18), and 3′-DNA-phosphoramidites and Eclipse Quencher phosphoramidite on a 0.2 μmol scale. The probe was purified by reverse-phase HPLC using a gradient of acetonitrile in 0.1 M triethylammonium bicarbonate buffer (pH 8–9) followed by drying in a SpeedVac evaporator. The probe precursor was re-dissolved in 50 μl of dry DMSO and treated with 10 μmol of pentafluorophenyl N-FMOC-6-aminohexanoate (19) for 10 h. The FMOC-aminohexanoyl modified probe was isolated by reverse-phase HPLC and deprotected by treatment with concentrated ammonium hydroxide for 1 h at room temperature. The deprotected oligonucleotide was further purified by reverse-phase HPLC using the triethylammonium bicarbonate buffer as described above, dried and re-dissolved in 50 μl of dry DMSO. To this solution, 10 μmol compound 8c (Scheme 2) and 1 μl triethylamine were added. After being kept at room temperature for 10 h the reaction was combined with 1 ml of 2% solution of NaClO4 in acetone. The precipitated crude conjugate was further purified by reverse-phase HPLC, dried and re-constituted in 1× TE buffer. An additional molar absorbance (ɛ260) of 28 500 M−1 cm−1 was used in the probe concentration calculation to correct for the presence of the dye.', 'The synthesis of special CPG supports for the preparation of required oligonucleotide probes are shown in Schemes 1–3.', 'Methyl 3-(3-chloro-2,4-dihydroxyphenyl) propanoate (4a) (Scheme 1) was synthesized starting from 3-chloro-2,4-dimethoxybenzaldehyde (1) (20), which was converted to substituted cinnamic acid 2 by Knoevenagel condensation. Intermediate 2 was hydrogenated in the presence of Pd/C to yield substituted phenyl propionic acid 3, which was first demethylated with HBr/HOAc and then converted to the methyl ester 4a.', 'Synthesis of the pentafluorophenyl ester dye intermediates 11 required to synthesize the dye-modified CPG supports 12 is shown in Scheme 2. Compounds 4a and b were reacted with substituted phthalic anhydrides in the presence of AlCl3 to give the benzophenones 5, which were reacted with the resorcinol analogs 6 (a–c) in trifluoroacetic acid/methanesulfonic acid to yield the carboxyethyl substituted dyes 7. Treatment of compounds 7 with trifluoroacetic anhydride generated lactones 8, which were first reacted with the DMT-protected hydroxyprolinol 9 (21) and then with trimethylacetic anhydride to afford the protected dyes 10. Reaction of intermediates 10 first with succinic anhydride and then with pentafluorophenyl trifluoroacetate, yielded the PFP esters 11.', 'The reaction of the PFP ester 11 with the long chain aminoalkyl CPG (LCAA-CPG) yielded the desired CPG supports 12 as shown in Scheme 1.', 'Endo IV cleavage activity was measured in real-time as shown in Figure 2. The model system in Figure 3A was used to determine optimum E.coli Endo IV (Trevigen, Gaithersburg, MD) concentration, magnesium concentration and pH; the latter two were determined to be at 20 mM Tris buffer, pH 8.6. Results are shown in the Supplementary Data.', 'PCR was conducted in 384-well plates on MJ Research PTC-200 Peltier Thermal Cycler (Waltman, MA). Fifty cycles of three-step PCR profile (95°C for 5 s, 58°C for 30 s, 72°C for 30 s) were run after an initial 2 min denaturation step at 95°C. Each 5 μl PCR contained 2.5 μl JumpStart Taq ReadyMix (Sigma, MI) and 5–10 ng lyophilized human genomic DNA from Coriell Biorepository (Camdem, NJ). Primer concentrations were 0.1 and 1.0 μM for forward and reverse primers, respectively. A Biomek® 2000 Laboratory Automation Station (Beckman Coulter, USA) was used to set up PCRs.', 'The Endo IV Master Mixture was prepared by mixing appropriate volumes of 5× Endo IV reaction buffer [100 mM Tris–HCl (pH 8.5), 25 mM MgCl2, 5× E.coli Endo IV enzyme solution, 50% glycerol, 50 mM Tris–HCl (pH 7.5), 0.1 M NaCl, 1 mM DTT, 0.1% Triton X-100, 0.02 U E.coli Endo IV enzyme and 20× of Endo IV Probes/Enhancer Master Mixture], containing each probe and enhancer at 20 μM concentration.', 'The endo IV reaction was prepared by adding 5 μl Endo IV Master Mixture to 5 μl PCR mixture. Reaction was incubated at 50°C for 50–60 min. Fluorescent detection was performed on ABI 7700 or 7900 in the two channels typically used to detect FAM (for FL1) and VIC (for FL2, FL3 and FL4) utilizing the ABI software for Allelic Discrimination. Alternatively, the fluorescence could also be detected in a LightCycler® LC24 Real-Time PCR System. All assays unless otherwise indicated were performed at least in triplicate.', \"Thermodynamic parameters of duplex formation were derived by the van't Hoff analysis methods. The shape of each melting curve was fitted to the two-state model with linear base lines (22) using a nonlinear least-square program (23). Unless otherwise stated, Tm of DNA duplexes were measured at a concentration of 5 × 10−7 M in 1× PCR buffer containing 10 mM Tris–HCl (pH 8.5), 5 mM MgCl2 and 40 mM NaCl, as described earlier (23).\", 'The MGB Eclipse® Design Software 3.0 () was used to design the primers for PCR. It was also used to calculate the Tm of the enhancer and probe.', 'An example of the new Endo IV-based assay is shown in Figure 2. A probe and an enhancer are hybridized to a complementary synthetic target with a one base gap between them.', 'The signal is substantially diminished in the absence of the enhancer. A rigid hydroxyprolinol linker (Figure 3B) between the oligonucleotide and fluorophore FL1, critical for sensitivity and specificity, was used in this experiment. The results in Figure 2 illustrate the importance of all components used in the assay. The cleavage of the fluorophore is highly target dependent and its rate is amplified in the presence of the enhancer oligonucleotide. Very low non-target-dependent cleavage is an important feature of the Endo IV assay.', 'Effects of pH and cations on E.coli Endo IV activity was determined using the probe, enhancer and target (Figure 3A). It was shown that Endo IV cleavage of the probe has a relatively flat pH optimum in the range between pH 8.5 and 9.5. For compatibility with post-PCR Endo IV amplification the assay buffer with pH 8.6 was chosen, similar to that of the PCR master mixture. Inclusion of the divalent magnesium cation in the buffer ranging from 0 to 20 mM, resulted in more than 2-fold increase in cleavage rate, the optimum range was ∼5–8 mM. A concentration of 5 mM was chosen for the Endo IV assay. It was determined that the monovalent cations, such as Li, Na, K and Rb, inhibit Endo IV cleavage activity for the concentrations studied in the range from 0 to 100 mM. An inhibition of ∼30% was observed at 20 mM monovalent cation concentration, which increased to ∼60–70% at 100 mM concentration. The optimum probe concentration was determined to be ∼600 nM, please see Supplementary Figures 1–3.', 'Figure 3A shows a model Endo IV assay that requires a target complementary to an enhancer and a probe labeled at the 5′ end with an Eclipse Dark Quencher and a fluorescent dye at the 3′ end. As shown, the probe (calculated Tm = 46.7°C) and enhancer (calculated Tm = 50.2°C) are separated by one base to mimic a natural abasic site. Four probes with different fluorescent dyes and linkers (Figure 3B) were evaluated for their performance in the new assay. Fluorophores FL1, FL2 and FL3 of the probes I, II and III (with similar quantum yields) contained a hydroxyprolinol linker between the 3′-phosphate of the probe sequence and the core of the dyes. Fluorophore FL4 of probe IV was analogous to FL3 of probe III, with an additional aminocaproic spacer introduced between the hydroxylprolinol moiety and the dye; the purpose of this spacer was to distance the fluorophore from the cleavage site.', 'The probe specificity was evaluated by comparing the cleavage rates in the presence and absence of target. Target-dependent (specific) and target-independent (non-specific) cleavage rates of the probe were measured as the percent target cleaved per minute. The results are summarized in Table 1. Probes I–III demonstrated similar specific cleavage. A 2- to 3-fold decrease in the rate of non-specific cleavage for probes II and III versus probe I was observed. These low, non-specific cleavage rates translate into high-specific to non-specific cleavage ratios. It appears that chloro- or chloro/methyl substitutions in the fluorophores of probes II and III, respectively, are beneficial for the reduction of non-specific cleavage. One possible explanation for this reduction may be an effect of increased hydrophobicity of FL2 and FL3 on enzyme activity. In the case of probe IV, with the extended hydroxyprolinol linker, the rate of specific cleavage almost doubled. However, the corresponding increase in the rate of non-specific cleavage was ∼20 times higher than that observed in probe III, substantially compromising the ratio of specific to non-specific cleavage rates. Once again, this result implies the existence of inhibiting interactions between Endo IV enzymatic activity and fluorophore which can be adjusted by moving the fluorophore away from the active center.', 'The rigid hydroxyprolinol linker with low non-specific cleavage rates utilized in probes I–III was identified as the linker of choice in the Endo IV assays.', 'The use of a flexible, straight chain linker (C6) instead of hydroxyprolinol-based linker increased the rates of non-specific cleavage to even higher degree than the extended hydroxyprolinol linker and reduced the specific to non-specific ratios to an unacceptable level (data not shown).', 'It appears that both the structure of the sterically constrained hydroxyprolinol linker and proximity of the bulky dye moiety to the cleavage site moderate cleavability of the 3′-terminal phosphodiester bond and practically eliminate non-template-dependent cleavage.', 'Figure 4A shows the target, probe and the enhancer aligned with the target to indicate different gaps. The Tm of the enhancer is typically chosen to be at least 5°C higher than that of the probe. The cleavage rates in the presence of enhancer are shown with 1–5 base gaps (Figure 4B). As expected, based on the natural abasic substrate requirement shown in Figure 1B, a one base gap gave the highest cleavage rate. The probe in the absence of enhancer showed a cleavage rate nearly equivalent to that of a 4 base gap in the presence of enhancer. As shown in Figure 4C, the maximum cleavage rate is achieved at a concentration of enhancer greater than ∼80 nM. In practice, the concentration of the enhancer is typically at least equivalent or larger than that of the probe. Some cleavage is observed with no enhancer present.', 'Figure 5 shows how the cleavage rate of probes with different Tms changes with temperature. Probe lengths ranging from a 6mer to a 14mer were investigated and are shown in Figure 5 with calculated Tms of between 14 and 60°C. All the probes showed a bell-shaped relationship for cleavage rate with temperature. As expected, the cleavage rate for all probes increased with increased temperature. A relatively sharp cleavage rate optimum was observed for the probes a, b and c with calculated Tms of 60, 48 and 42, respectively. It appears that the best probe cleavage occurs close to the calculated Tm or slightly above it. Performing the assay at a temperature slightly higher than the Tm (Figure 5) showed not only optimum activity but also appears to allow probe cleavage cycling, an important feature of this assay.', 'The ability of the Endo IV assay to detect different target concentrations is shown in Figure 6. It was observed that in the case of 125 nM target concentration, the probe-based fluorescence plateaued in ∼1 h, while in the presence of 25 nM target a fluorescence plateau was reached in ∼3 h. The limit of detection of this assay is 0.04 nM, clearly distinguishable from the stable background (Figure 6). This stable background, a characteristic feature of the assay, is maintained over a period of 15 h. This property is also observed in the assay with other nucleic acid targets.', 'The ability of the Endo IV assay to discriminate mismatches at different positions of a 14mer probe is illustrated in Figure 7. Mismatches were introduced in the probe one at a time, from positions 1 to 8. The Endo IV assay shows excellent specificity for the different mismatches from base 1 to base 6 with large match/mismatch signal ratios. As shown in Figure 7, the mismatch signal is often about the same as that of the NTC or slightly higher. The exceptions are the difficult 4-G/T- and 8-T/G-mismatches (24) where match/mismatch ratios of ∼16 and 7 were observed, respectively. These ratios are quite adequate to differentiate matched and mismatched sequences. It appears (data not shown) that the discrimination in positions 1 and 2 is largely determined by substrate requirements while discrimination in other positions is determined by thermodynamic considerations. Experiments with a large number of assays have also indicated that, with 10mer to 12mer probes, satisfactory discrimination is observed with the mismatch position in all except the last three bases at the 5′ end.', 'The exquisite specificity of the Endo IV assay appears to be the result of at least two factors. The first is the strict intrinsic substrate specificity requirement of the Endo IV enzyme which appears to require the rigid prolinol linker (Figure 3B, n = 0) used in the probes I–III. Second, the use of short probes (10mer to 12mer) gives the Endo IV a distinct advantage over most other assays where probes are typically much longer (1,2,6,25).', 'The application of the Endo IV cleavage as a post-PCR detection system is shown in Figures 8 and 9. Following an asymmetric PCR using primers to produce predominantly single-stranded amplicon, two allele-specific probes, an enhancer and Endo IV enzyme are added to the amplification mixture and incubated isothermally for about an hour. The cleaved fluorescence from the two allele-specific probes, each labeled with a different fluorescent dye, is then plotted in a scatter diagram as shown in Figures 8 and 9 for the genotyping analysis of two single nucleotide polymorphisms (SNPs) which are of immediate importance to in cancer (). The scatter diagrams of two polymorphisms, namely agouti signal molecular epidemiology studies protein (ASIP-01) and adenomatous polyposis coli (APC-03) () are shown in Figures 8 and 9, respectively. In both cases, a probe specific to the wild-type allele is labeled with FL1 and a probe specific for the mutant allele labeled with FL3 is used.', 'The probe sets, designed to analyze ASIP-01 and APC-03 polymorphisms, were used to assay 102 unrelated human DNA samples obtained from the Coriell Institute. In the case of the ASIP-01 target the mismatch was situated two bases from the 3′ end of the probe, while in the case of APC-03, the mismatch was six bases from the 3′ end of the probe.', 'From our experience, the spacing of the allele clusters in Figures 8 and 9 are comparable or better than those scatter plots observed in other methods (2,26,27). The exquisite specificity achieved with the Endo IV enzyme, combined with the shorter probes used and the high signal accumulation, gives this assay an advantage over other SNP assays. In comparison with other signal accumulating assays (2), the Endo IV assay has similar levels of high signal accumulation, but uses shorter probes with higher specificity than those assays. In the case of hybridization-based assays (5,25), the Endo IV assay has not only a signal accumulation advantage, but also uses shorter probes with a specificity advantage. In addition, the shorter probes allow much more flexibility in assay design compared to other methods with longer probes.', 'We disclosed a novel nucleic acid signal detection system based on the cleavage of a phosphodiester linkage in a dual-dye-labeled oligonucleotide probe by the Endo IV enzyme from E.coli. The parameters influencing the activity of the Endo IV enzyme were optimized. This optimized Endo IV detection assay requires, in addition to an oligonucleotide probe substrate, a fluorophore attached through a relatively rigid linker to the oligonucleotide and a helper oligonucleotide. The rigid linker conveys the exquisite specificity of Endo IV cleavage of the phosphodiester linkage in the probe and shows high-specific/non-specific rate ratios. The synthesis of fluorophore analogs coupled to a rigid linker and solid support, used in the synthesis of labeled oligonucleotides, was also disclosed. Excellent mismatch discrimination is observed from positions 1 to 6 in a 12mer probe. The ability to put the mismatch in any one of positions 1–6, allows for flexibility in probe design for genotyping assays where sequence and secondary structural constraints are considered. This method allows the use of probes shorter than those typically used in other methods (28) contributing to the improved mismatch discrimination.', 'The combination of the Endo IV assay with a PCR as a post-amplification detection system has been demonstrated. Post-PCR Endo IV genotyping shows excellent spacing of scatter plot endpoint signals of no template controls, wild-type, mutant and heterozygous samples allowing clear allele differentiation. In contrast, other endpoint genotyping methods may suffer from overlapping endpoint signals due to PCR efficiency issues (25,29). This Endo IV post-PCR genotyping has been successfully used in an industry setting for >5000 genotype assays.', 'Supplementary Data are available at NAR online.'], 'cord_uid': 'kt9pmfdr'}\n{'title': 'PATRIC: The VBI PathoSystems Resource Integration Center', 'text': [\"Bioterrorism became an important national security issue (1) following the deliberate release of anthrax spores into the US postal system in October 2001 (2). Meanwhile, emerging and reemerging infectious diseases (3) have had profound effects on public health in many parts of the world. Recognizing the pathogens responsible for these diseases as threats to homeland security, the National Institute of Allergy and Infectious Diseases (NIAID) of the US National Institutes of Health has embarked upon a series of initiatives aimed at developing a comprehensive understanding of the organisms identified as NIAID category A, B and C priority pathogens (for a complete list, see ). The Virginia Bioinformatics Institute's PathoSystems Resource Integration Center (PATRIC) is one of eight Bioinformatics Resource Centers (BRCs) established to study the NIAID priority pathogens and develop these information resources for the research community. While database resources for bacterial ((4) and those cited in (5)) and viral (6,7) genomics have been available for number of years, this project seeks to integrate genomics with comparative genomics and pathway analysis and ultimately proteomics, transcriptomics, immune epitope mapping, host-response and other downstream technologies. The goal is to help researchers and clinicians better detect and respond to biothreat agents (and infectious diseases in general) by facilitating the development of diagnostics, vaccines and therapeutics. This requires access to comprehensive information on the molecular biology, physiology and pathogenicity of these organisms.\", 'PATRIC is responsible for the eight organism categories listed in Table 1. The three genera of proteobacteria are all intracellular pathogens that are known or potential biowarfare agents. In the 1950s, Brucella suis was the first infectious agent developed for use as a biowarfare agent by the United States. Brucellosis, caused by Brucella sp., is an important agricultural disease infecting cattle, sheep, goats and swine as well as humans. It is highly contagious and readily dispersed as an aerosol (8). Coxiella burnetii, the causative agent of Q fever, is a highly infectious agent of relatively low lethality. Its interest as a biowarfare agent stems from its high infectivity, stability to heat and desiccation and potential for aerosol dispersal. The genus Rickettsia contains the organisms responsible for numerous types of typhus and arthropod-borne spotted fevers (9,10). Rickettsia prowazekii was developed as a bioweapon by the USSR in the 1930s and was used by the Japanese in Manchuria during World War II (11).', 'The five categories of viruses studied by PATRIC are all positive-strand ssRNA viruses, with the exception of Lyssaviruses, which have negative-strand ssRNA genomes. While there are no reports of any of these viruses being weaponized, they represent the causative agents for a number of emerging and reemerging diseases including Severe Acute Respiratory Syndrome (SARS), rabies and transmissible gastroenteritis. Recombinant vaccines for these viruses are either still in development or unavailable in areas where these infections are endemic or epidemic, compounding the public health risk.', 'The pace of research on these organisms has increased significantly since the turn of the millennium, with outbreaks, such as that of SARS in 2003 (12,13), spawning a flurry of scientific activity. The widespread use of automated DNA sequencing, microarray gene expression analysis and other high-throughput laboratory technologies has increased the volume of data produced, but not necessarily its accessibility. Currently, significant genomics and bioinformatics expertise is required to extract, process and interpret this wealth of data.', 'To address these problems, PATRIC has created an interdisciplinary team of bioinformaticians, software engineers, computational biologists and organism experts to build a publicly accessible resource aimed at providing high quality, analyzed and curated data to the infectious disease community working on these pathogens. To date, we have achieved the following objectives:\\ncollection and organization of existing genomic data for the eight pathosystems under a single, unified frameworkgenome annotation and curation following standardized proceduresvisualization of raw data from analytical programs, as well as curated datacreation of orthologous gene groups within each organism category allowing comparative analysis of gene contentprediction and visualization of bacterial metabolic pathways to complement functional analysis of proteinsintegration of online literature reviews from PathInfo (14) for selected organisms.', 'Longer-term goals include integration of data from gene expression and proteomics experiments (including host-response), predicted protein and RNA secondary and tertiary structures, and well-cataloged literature compilations. Ultimately, we hope our website will become an essential tool for researchers working on these pathogens and provide networking opportunities within the pathogen research communities.', 'PATRIC is implemented on Oracle 9i RDBMS using the Genomics Unified Schema (GUS) version 3.5, developed at the Computational Biology and Informatics Laboratory at the University of Pennsylvania (see ). GUS is used to store all sequence data and associated annotation with the exception of metabolic pathway data, which is stored in a separate schema developed for use with the Pathway Tools software (15) (see section 4.3). To minimize modifications to GUS and ensure compatibility with future releases, the Curation/Annotation Schema was developed to support PATRIC-specific curation activities.', 'The database is populated with all known full-length or nearly full-length genomic sequences for the eight organism categories listed in Table 1. Automated scripts query GenBank (16) daily to identify new or updated records. The corresponding sequences, annotation and associated literature are retrieved from NCBI and loaded following curatorial review to remove redundancies and assign unique names to each genome. RefSeq (17) records are used when available to take advantage of their more thorough and consistent annotation. Draft genome sequences from Joint Genome Institute (JGI)/Los Alamos National Labs (LANL) and the NIAID-funded Microbial Sequencing Centers will also be part of the PATRIC dataset. In addition to genome sequences and primary annotation from the original GenBank or RefSeq entry, the database stores the results of all automated and manual analyses described in the following section.', 'Due to the large number of closely related genomes in each organism category, we have adopted an annotation strategy in which automated methods are applied to all genomes while detailed manual curation is applied to a limited number of reference genomes. The species B.suis 1330, C.burnetii RSA 493 and R.prowazekii str. Madrid E were chosen as reference genomes for their respective categories. Each viral category has (or will have) multiple reference genomes, representing phylogenetically diverse strains.', 'Automated nucleic acid and protein sequence annotation is accomplished using a Java-based genome annotation pipeline (unpublished), which reads an XML script containing the names and parameters of the analytical applications. The bacterial pipeline executes the gene prediction programs Glimmer (20) and GeneMark (21,22) followed by start site correction programs RBSfinder (23) and TICO (24). BLASTX (25) searches the non-redundant protein database, complementing the ab initio gene prediction methods. RNA genes are identified by tRNAscan-SE (26) and BLASTN searching against a ribosomal RNA database (27,28). The annotation protocol containing the full list of applications and parameters is available online at  under ‘standard operating procedures’.', 'Results of the genome analysis pipeline are merged with original GenBank or RefSeq features for automated interpretation. A decision tree is used to classify genes into categories based on the level of agreement between the various prediction methods. Genes that are unambiguously predicted by multiple methods are automatically ‘finalized’, creating new ‘gene’, ‘CDS’ and/or ‘[t/r]RNA’ features. The remaining genes are marked for manual curation. For viral genomes, an abbreviated pipeline is executed that emphasizes sequence alignment for gene identification and employs GeneMarkHMM optimized for mammalian (host) genomes.', 'After curatorial review, finalized protein-coding (CDS) features are translated and subjected to another pipeline executing InterProScan and structure prediction methods such as MEMSAT 2 (29). Currently, each protein is associated with GO terms (30), TIGRroles, Enzyme Commission numbers (31) based on Pfam (32) and TIGRfam alignments (for a description of TIGRfam and TIGRroles, see: ). The protocol for automated proteome annotation is also available online. Manually curated protein sequences will be available in early 2007.', 'Once protein sequences are inferred from each genome in an organism category, putative ortholog groups are generated using BLASTP for all pairwise genome combinations and applying the conventional bidirectional-best-hit (BBH) criterion (33). While putative ortholog groups within the bacterial categories are generally well defined, many viral proteins cannot be readily clustered using the stringent BBH criterion. This is an active area of curation. Using the ortholog groups as a starting point, a reference protein list is created for each bacterial category consisting of the proteins of the reference genome (each representing one ortholog group) plus a representative protein from each ortholog group identified in the associated genomes. A gene occurring in only a single genome constitutes a ‘group’ of one and would be included in the reference list. The reference protein lists will be manually curated and include, whenever possible, detailed functional descriptions, gene symbols, GO terms and EC numbers. Thus, every protein in the database will either be manually curated or be linked to an ortholog group member that has been manually curated.', 'The ortholog groups are further processed to create multiple sequence alignments (MSAs) using MUSCLE (34) with default parameters. Phylogenetic estimations using the neighbor-joining method (35) were created based on trimmed alignments using PHYLIP (36). Trees were validated by bootstrapping (37) using a minimum of 100 replicates.', \"To help users understand the function of the bacterial proteins in context, we have adopted the Pathway Tools system (15) to derive pathways from genome annotation and to fill potential gaps in annotation known as pathway holes. The system takes a list of protein names, descriptions and EC numbers as input. Proteins with EC numbers can be assigned roles directly; the roles of other proteins are suggested by lexicographic analysis of descriptive information and/or analysis of gene order from homologous regions of related genomes and confirmed or rejected by the curation staff. The output is a database with integrated web server that allows users to browse and query the organism's metabolic pathways. This system has been integrated with the PATRIC web site, allowing users to access pathway information for all bacterial reference genomes. The current analysis was based on preexisting RefSeq or GenBank annotations; later releases will incorporate data curated in house, unifying the genomic and pathway versions of the data. The analysis of pathways can facilitate the identification of metabolic choke points, critical enzymes that could be targeted by drugs that may have valuable antimicrobial properties. Pathway analysis can also yield clues to pathogenesis by comparing virulent and avirulent strains and examining the roles of genes not present in both strains.\", 'The PATRIC website is hosted on a Sun Microsystems v20z server running SuSE Linux using the Apache web server. Applications are written in PHP and Perl, accessing data from an Oracle 9i server hosted on a Sun Microsystems E15000 running Sun OS.', \"The conceptual organization of the website is described in Figure 1. The website's home page contains news, a navigation bar and the list of PATRIC organisms. Users can select their organism of interest from the list to access the corresponding organism category page. This page contains a table of genomes currently in our database with links to the three principal representations of individual genomes: the genome summary, genome browser and gene table. These pages allow users to view a summary of genome sequencing information and to identify specific genes and link to their corresponding gene, protein and pathway information pages. The gene information page displays the output of sequence analysis software run by the annotation pipeline, as well as curated data. Similarly, the protein information page displays InterProScan and TIGRfam alignments and associated information such as GO terms and EC numbers. For bacterial genomes, the pathway information page illustrates the protein's position in the organism's metabolic network and links to a wealth of information provided by PathwayTools.\", 'The organism category page also contains links to a pathogen summary, ortholog group table and a phylogenetic tree based on 16S rRNAs for bacteria or a selected protein family for viruses. For bacterial genomes, detailed pathosystem information is available, provided by the VBI PathInfo documents (14). The ortholog group table shows the presence or absence of reference gene list proteins for each organism in the organism category and provides links to an MSA and tree viewer and the Base-By-Base MSA editor (38) for every ortholog group. Base-By-Base allows users to add sequences to the MSA, recalculate it using Clustal (39), T-Coffee (40) or MUSCLE and generate the corresponding tree using neighbor-joining or a number of clustering algorithms.', 'The PATRIC website also supports analytical and query tools. A database search page allows user-supplied sequences to be BLASTed against reference and curated sequences from PATRIC organisms. The page also supports MUMMER (41) comparisons between genomes in the database or with a user-supplied sequence. A query tool is available throughout the site by which users can retrieve genes by name, ID, description, as well as GO and EC identifiers and descriptions. Questions, comments and suggestions concerning the website and its contents may be submitted via the ‘feedback’ page, accessible from the menu bar.', 'The PATRIC database is hosted at the Virginia Bioinformatics Institute at Virginia Tech and can be accessed via web browser at . Sequences and annotation in GFF3 format (see ) can be downloaded by following the ‘downloads’ link on the main menu bar. GFF3 files are also available through BRC-Central at: .', \"This paper presents the first detailed description of the PATRIC website. Future development will advance on several fronts. Genome and proteome curation will continue, complemented by improved tools for query, analysis and visualization. For viruses, we will transition to the more widely accepted ICTV taxonomy (42). The website's user interface is being enhanced to integrate organism-, tool/task- and data-centric approaches to data access, allowing users more efficient and effective access to PATRIC resources. This will be followed up by prioritized curation targeted at potential drug and vaccine targets, virulence factors and genes with differential representation or polymorphisms associated with clinically significant phenotypes. Leveraging another NIAID-funded VBI project, the Administrative Resource for Biodefense Proteomics Research (), we plan to integrate expression profiling and proteomics data from pathogen and host to better understand the pathosystem's biology and help the community identify targets for counter-measures. The integration of these disparate data types into a single, easy-to-use system is a goal that we anticipate will enable pathogen researchers to make full use of available data to develop diagnostics, vaccines and therapeutics.\"], 'cord_uid': '3ulketgy'}\n{'title': \"MMDB: annotating protein sequences with Entrez's 3D-structure database\", 'text': [\"The molecular modeling database (MMDB) is Entrez's ‘Structure’ database (1). Querying MMDB with text terms, e.g. one may identify structures of interest based on a protein name. Links between databases provide other search mechanisms. A query of Entrez PubMed database, e.g. will identify articles citing a particular protein name. Links from this set of articles to ‘Structure’ may identify structures not found by direct query, since PubMed abstracts contain additional descriptive terms. Currently, MMDB and its visualization services handle ∼25 000 user queries per day.\", 'Experimental three-dimensional (3D) structure data are obtained from the Protein Data Bank (PDB) (2). Author-annotated features provided by PDB are recorded in MMDB. The agreement between atomic coordinate and sequence data is verified, and sequence data are obtained from PDB coordinate records, if necessary, to resolve ambiguities(3). Data are mapped into a computer friendly format and transferred between applications using Abstract Syntax Notation 1 (ASN.1). This validation and encoding supports the interoperable display of sequence, structure and alignment. Uniformly defined secondary-structure and 3D-domain features are added to support structure neighbor calculations. MMDB currently contains ∼39 000 structure entries, corresponding to ∼90 000 chains and 170 000 3D domains.', \"The MMDB web server generates structure summary pages, which provide a concise description of an MMDB entry's content and the available annotation (4). Sequences derived from MMDB are entered into Entrez's protein or nucleic acid sequence database, preserving links to the corresponding 3D structures. Links to PubMed are generated by matching citations. Links to Entrez's organism taxonomy database are generated by semi-automatic processing of ‘source records' and other descriptive text provided by PDB. Ligands and other small molecules are identified and added to the PubChem resource, accessible at , also preserving reciprocal links to 3D structure. Sequence neighbors are identified by BLAST (5), and links to the Conserved Domain Database (CDD) (6) by the RPS-BLAST algorithm (5). Structure neighbors are identified by VAST (7). The 3D structure viewer supported by Entrez, Cn3D (8), provides molecular-graphics visualization.\", 'In the Entrez database system, protein sequences are neighbored to each other by comparing each newly entered sequence to all other database entries. These database scans are run with the BLAST (5) engine, which identifies sequence neighbors with significant similarity, and the resulting sequence identifiers and taxonomy indices are stored, so that Entrez can provide ‘Related Sequences’ links for all protein records in the collection. The ‘Related Structure’ service is built on top of this system. Sequence neighbors directly linked to MMDB are identified and alignments are re-computed by employing the ‘BlastTwoSequences’ tool (9) to restore alignment footprints. The ‘Related Structure’ web interface provides direct access to this information. Initially this service had been restricted to sequences from microbial genomes (10), but it has now been expanded to cover all proteins in Entrez and is updated daily to provide a comprehensive 3D-structure annotation service. Identification of structure-linked neighbors and the visualization of sequence-structure alignment is also possible using Entrez and the Cn3D alignment viewer/editor, but ‘Related Structures’ provides a convenient new summary and ‘one click’ shortcuts to 3D visualization. These 3D views may be used to identify conserved residues and map site-specific features derived from the 3D structure. Currently ∼48% of non-identical protein sequences in Entrez have been linked to at least one related structure, employing a conservative threshold for alignment length (50 aligned residues or more) and similarity (30% or more identical residues in the aligned footprint); see Figure 1 for details.', \"A search with the term ‘Angiotensin converting enzyme’ in Entrez's protein database retrieves >400 hits. One may configure the Entrez browser to filter search results by various criteria, and one pre-configured filter selects those protein sequences with ‘Related Structures’ (configuration of Entrez can be achieved by following links to ‘My NCBI’, or by clicking on the ‘toolbox’ icon shown at the top of Entrez document summaries.). In this example, the ‘Related Structures’ filter shows that >240 of the identified sequence records have links to related structures.\", 'One such protein sequence is the ACE protein from Rattus norvegicus (accession no. ‘NP_036676’). On the ‘Links’ menu for this record, ‘Related structures’ generates a request to the Related Structure service (). The resulting page indicates with a horizontal bar, the sequence region annotated by each related structure (Figure 2). The display also supports sorting by a variety of alignment parameters such as score or length and selection of sequence-dissimilar ‘non redundant’ subsets. A ‘Table’ option switches to a text view, listing descriptions of each structure as well as alignment scores.', \"Using the table view with this example, one may notice that several related structures are complexes of the same protein with different drugs/inhibitors, e.g. structures with PDB codes 1O86 (11), 1UZF (12) and 1UZE (12). Clicking on the graphical alignment footprint of 1O86, a human ACE enzyme in complex with lisinopril, one can see a text representation of the corresponding BLAST alignment, and a Cn3D view of the alignment can be launched by clicking on ‘Get 3D Structure data’ (Figure 3). One may see that the query protein is highly similar in sequence to the human ACE enzyme, as identical residue pairs are colored red by default. The sequence identity across the aligned region is 82%, and it appears that the core of the structure is mostly formed by residues conserved between the two aligned rows, while non-conserved residues are mainly located on the structure's surface.\", \"One may further identify the catalytic center by identifying residues that contact the catalytic Zinc ion. Those sites can then be mapped from the structure to aligned regions in the sequence window using Cn3D's highlighting functionality. One may also examine the sequence-structure alignments with related structures 1UZE and 1UZF, human ACE binding to enalaprilat and captopril, respectively, drugs with chemical structures similar to that of lisinopril. This allows one to identify conserved interactions between the ACE enzyme and this series of antihypertensive drugs. Similarly, by examining the related structure 2AJF (13), one may be able to identify residues critical for cross-species infection by studying the protein–protein interactions between the receptor binding domain from SARS Coronavirus Spike and human versus rat angiotensin-converting enzyme 2.\", \"The ‘Related Structure’ service is also integrated with NCBI's protein BLAST service. A ‘Related Structures’ link is provided when one or more similar proteins with known 3D structures have been identified by BLAST. The NCBI single-nucleotide polymorphism resource (SNP) also links to the ‘Related Structure’ service, which in this context provides a mapping of both synonymous and non-synonymous coding SNPs onto experimentally determined 3D structures. ‘Related Structure’ may be expanded further in the future, to provide visualization for other NCBI resources and to support additional filtering and selection among related structures, e.g. to highlight those annotated with conserved domain footprints by the CDD resource or those linked to small molecules in the PubChem database.\"], 'cord_uid': '6qpsxmgi'}\n{'title': 'VIRUS ON T H E CEREBRAL ACTIVITY OF P L E U R O P N E U M O N I A -L I K E ORGANISMS I N M I C E', 'text': ['Shortly after the isolation of MHV(C) from Balb C mice, in 1955, it was found that Swiss weanlings were equally susceptible and because of their availability they were subsequently used in activity tests (1) . Princeton mice were consistently resistant. Intraperitoneal injection in Swiss mice was generally followed by focal or diffuse necrosis of the liver but was not attended by nervous manifestations. On intracerebral injection the virus produced significant changes in the brain, with some destruction of nerve cells. Lymphocytic infiltration was commonly observed in brain sections but polymorphonuclear leukocytes were absent. Lesions produced in the liver, following carriage of the virus from the brain, were generally focal in character. The death rate was variable. In one series of passages, 12 of 25 mice died between the 3rd and the 5th day after injection. Survival after the 3rd day was irregularly attended by paralysis of the hind legs. The most common sign was a rigid backward extension of one or both limbs. The affected leg was functionless and was dragged by the mouse in moving. This type of paralysis was observed in 9 of the 25 mice. At autopsy on the 5th or 6th day, 5 of them were erect and active while 4 were prone and unable to move. A tendency to roll sideways, if lightly prodded, was observed in one of the latter mice. All of the cranially injected animals showed weight losses of 1 to 3 gin.', 'Unusually acute signs of illness appeared in Swiss mice during the first cranial passage series. By the 3rd transfer, leukocytes were demonstrable in direct films 179 and later in sections from the brain. A cerebral suspension prepared from the mice of the 5th transfer yielded a pure growth of PPLO. The passage series was continued with a Coors filtrate of the contaminated suspension and subsequently maintained in the absence of PPLO. In 1938, Sabin (2) described a nervous disorder of mice that appeared during the cerebral passage of toxoplasma. The characteristic sign was a rolling motion on the long axis of the body. It was also observed after intraabdominal and intrathoracic injection. The incidence was 20 to 40 per cent. Two kinds of brain lesions were found in sections. The first one was indicated by destruction of the periventricular tissue and extensive leukocytic infiltration. It was present in all mice. The second lesion, consisting of almost complete necrosis and lysis of the posterior pole of the cerebellum, occurred only in mice with rolling signs.', 'Sabin (3) subsequently found that a pleuropneumonia-like organism, later termed type A (5) , was the etiologic factor. The disease was fully reproduced by intracerebral injection of the agent in pure culture. If grown in glucose-serum broth the organism produced a true exotoxin which resulted in the same nervous signs and cerebellar lesion on intravenous injection. Findlay et al. (4) , also in 1938, noted that mice injected intracerebrally, in series, with a neurotropic strain of yellow fever virus developed nervous signs within 2 days, instead of 4 to 5 days. Rolling with a sideways motion was observed in some of the mice. An agent that was not cultivable in ordinary media was separated from the brain but was subsequently lost on passage.', 'A similar condition was later observed during brain passage of the S strain of lymphocytic choriomeningitis virus. Rolling occurred in about 10 per cent of the injected mice. Seventy-five per cent died in 2 to 7 days. Some of the survivors developed hydrocephalus between the 8th and the 21st day.', 'Brain sections showed neurolysis and an intense infiltration of leukocytes. The lateral ventricles, meninges, and choroid plexus were invaded. In hydrocephalus the leukocytes were largely replaced by lymphocytes and mononuclears.', 'PPLO designated L5 were invariably isolated from the brains of mice with rolling signs. This organism by itself was incapable of reproducing the disease but mixed with a neurotropic virus or agar, or after rapid intracerebral passage, its virulence was raised and the rolling signs resulted. Several other types of PPLO (L,, Ls, and L4) were inactive in the presence of virus.', \"Cross-protection tests indicated that Sabin's type A organism was closely related to Ls but his was considerably more virulent.\", 'The first experiments on the behavior of PPLO in the brain were made in normal mice with 2 strains of the catarrhal type, designated S and J. Swiss and Princeton weanlings were injected intracerebrally with pure cultures of the organisms in the absence of any other microbial agent. Subsequent passages were made with brain suspensions.', 'The mice used in all of the following experiments were Swiss and Princeton weanlings, 10 to 12 gin. in weight, from colonies maintained at The Rockefeller Institute. They were injected in groups of 4 or 6, using mice of either sex but the same sex in any one group. The intracerebrai injections were made with approximately 0.02 ml. of inoculum, in mice lightly anesthetized with ether. The brain suspensions were prepared in saline from minced tissue, using a TenBroeck grinder, and were about 10 per cent in concentration. Freshly prepared suspensions were generally held for a short period, until sedimentation had occurred, and the supernatant used. In some instances penicillin was added in the amount of 1000 units per ml.', 'The PPLO strain designated S was the one originally isolated from the brains of Swiss mice during the cranial passage of MHV(C). The results of nasal and intraperitoneal injection in Princeton weanlings had indicated that it was a catarrhal type. The intracerebral passage in Swiss mice was begun with the 2nd transfer of the organism grown for 48 hours in 20 per cent horse serum--heart infusion bouillon (pH 8).', \"Most of the injected mice were killed on the 7th day and their brains removed aseptically. At least one brain from each passage group was placed in Zenker's solution, for histologic study, and sections cut from 3 levels including the cerebellum. They were stained with eosinmethylene blue. A small amount of tissue was aspirated with a capillary pipette from 2 or more of the remaining brains, the pipette being inserted between the hemispheres. Gramstained films of the tissue were prepared for microscopic examination. Cultures were also made on 20 per cent horse serum-nutrient agar plates containing 2500 units of penicillin. Finally, a suspension was prepared from the pooled brains for passaging. In some of the later tests cultures were also made from the suspensions. Several of the mice in this experiment were held under observation for 14 days and killed without a complete examination. Five subsequent passages were made, in each case with a brain suspension from the preceding transfer. Essentially the same methods were used in all of the following tests.\", 'The PPLO strain designated J was originally isolated during an outbreak of infectious catarrh in a small colony of Japanese mice. It has been maintained for many years in this laboratory by nasal passage or contact in Princeton mice. Intracerebral passage was begun in Princeton weanlings with a freshly isolated 72 hour old culture, the 3rd transfer on isolation from middle ear exudate. Four subsequent cerebral passages were made in Princeton mice and one in Swiss mice.', 'Twenty-eight of the 32 Swiss mice used in the 6 cerebral passages of the S strain of PPLO were killed on the 7th day after injection, and 4 on the 14th day. All of the animals gained weight during the observation period and were normal in appearance. At autopsy they showed no consistent pathologic findings. Leukocytes were not observed in either direct films or sections and there was no indication of encephalitis or meningitis. Thirteen of 16 brain cultures gave a pure growth of PPLO and 3 were negative. The amount of growth was regularly scanty, 8 of the plates having 10 colonies or less.', 'Similar results were obtained with the cerebral passage of the J strain in Princeton mice. The 25 animals used in the 5 transfers were normal in appearance during the observation period of 7 days. At autopsy, leukocytes were not observed in either brain films or sections. Cultures from the suspensions yielded only a scanty growth of PPLO colonies. In the single experiment with Swiss mice the injected animals were held for 4 weeks. They were likewise normal during life and at autopsy. The terminal culture from the brain suspension yielded 8 PPLO colonies.', 'The Reaction with Catarrhal Strains of PPLO.--This series was begun with the S strain of PPLO. Eight successive cranial transfers were made in Swiss weanlings, starting with a brain suspension which contained both the organisms and the virus. Prior to use this suspension had been maintained in a frozen state at --20°C. for 21 weeks.', 'Five mice were used in each of the 8 brain passages. Mter injection they were held under observation until one or more deaths had occurred. At this time, usually the 3rd or 4th day, the survivors were killed with ether and autopsied. Direct brain films and cultures were made from at least one mouse in each passage. Six brains were sectioned for later study.', 'Twenty of the 40 mice in this series of passages died, 14 on the 3rd day and 6 on the 4th. The 20 survivors were acutely ill when killed for autopsy. They regularly showed a marked loss in weight, up to 4 grn. per mouse. Roughening of the hair was a variable finding. Many of the mice were inactive and some were prone. Convulsive movements were irregularly observed and a tendency to roll was seen in 3. At autopsy the livers commonly showed a normal red color but were regularly spotted with small white loci. These areas were often few in number, 10 or less, and in some instances were visible only on low power magnification. The Gram-stained films regularly showed leukocytes in considerable numbers. In some films extra-and intracellular groupings of PPLO were clearly demonstrable. Areas of inflammation with innumerable leukocytes were observed in the brain sections. A pure growth of PPLO colonies, varying from several thousand to innumerable, was obtained on the plate cultures from 7 of the passages. The growth from the brains of the first passage, which had been made with a previously frozen suspension, was relatively meagre. A culture made from the pooled livers of the 2 survivors in the second passage was bacteriologically sterile.', 'Supplementary experiments in this group were carried out with 3 additional PPLO strains of the catarrhal type. One strain (J) was recovered from the middle ear exudate of experimentally infected mice and 2 strains (RZ and RH) from naturally infected albino rats. All 3 strains were of recent isolation. Two cerebral passages were made in Swiss weanlings with each of the strains. In preparing the inoculum for the first passage a brain suspension containing only MHV(C) was mixed with an equal volume of a fluid culture of PPLO. The second passage was made with a brain suspension which contained both agents.', 'With the RZ strain of PPLO the outward signs of illness were delayed until the 5th day. In other respects the outcome of these tests essentially duplicated the results with the S strain. Signs of acute illness were regularly observed in the injected mice. Thirteen of the 30 animals used in the combined tests died. A tendency to roll was observed in only 2 of the 17 survivors. At autopsy a sparse distribution of small surface foci was observed in all of the livers. Leukocytes were regularly present in brain films and numerous colonies of PPLO were obtained on plate cultures. Brain sections from 2 mice in each group showed extensive leukocytic infiltration.', \"In the experiment with the J strain of PPLO, cultures were also made from the heart's blood, liver, and spleen. All of these suspensions were bacteriologically sterile. Aside from the liver, the abdominal and thoracic organs of these mice, and indeed of all the mice in the entire series of experiments, were normal in the gross.\", 'The effect of long continued cultivation on the cerebral activity of PPLO was determined with a culture of the J strain that had been transferred 100 times in serum-bouillon at intervals of 2 to 3 days. In the presence of MHV(C) its pathogenicity was fully as great as that of the recently isolated strains.', 'One titration experiment was carried out with brains removed on the 3rd day from Swiss mice injected intracerebrally with MHV(C) and the J strain of PPLO. The highest dilution of the suspension that gave a leukocytic response with positive cultures was 10 -4 . The brains from mice which received the 10 -5 dilution showed no lesions and were bacteriologically sterile.', 'The Reaction with Conjunctival Strains of PPLO.--The observations on the activity of MHV(C) and PPLO in the brain were continued with 2 strains of the conjunctival type, designated P and B. These organisms are much less pathogenic for mice than those of the catarrhal type and are commonly inactive on nasal or intraperitoneal injection in susceptible weanlings. Under natural conditions they are associated with a low grade inflammation of the conjunctivae (6).', 'The P strain was obtained from Princeton mice infected by direct contact. A passage series with this strain was begun in 1950 and has been maintained to date. A culture of recent isolation from conjunctival washings (the 3rd subculture) and one that had been recultured after 79 transfers and then stored for 8 months at -20°C. were used. The B strain was obtained from a naturally infected mouse, one of a group sent to us for examination from the Brookhaven National Laboratory. The 2nd culture transfer was employed.', 'In the presence of active MHV(C) 2 cerebral passages were made in Swiss weanlings with each of the 2 P strain cultures and 10 passages with the B strain. A single passage was also made with the latter culture in the absence of virus.', 'On combination with MHV(C) the activity of the P strain in the brain was essentially the same as that of the catarrhal types. The subculture of the transplant that had been transferred 79 times and then stored in a frozen state was fully as active as the recently isolated one. The first experiments with the B strain showed evidence of an accelerated lethal outcome which was substantiated by subsequent passages. Outward signs of illness were also more con-spicuous with this strain. In other respects its behavior resembled that of the P strain. In the absence of virus the organisms survived in the brain for 2 weeks but produced no demonstrable reaction.', 'Twelve of the 20 mice used in the 2 experiments with the P strain died on the 3rd to the 5th day after injection. Prior to death they showed external signs of illness. One animal tended to roll. Eight mice were killed and autopsied on the 3rd or 4th day. Their livers were spotted with surface areas of focal necrosis. Leukocytes were regularly present in the brains and numerous colonies of PPLO were obtained on culture.', 'Deaths in the passage series with the B strain began a day earlier, 15 of the 50 mice dying on the 2nd day after injection. Signs of acute illness were observed in many of the survivors at this time. Fourteen additional deaths occurred between the 3rd and the 5th day. Leukocytes were invariably seen in films or sections from the 21 mice that were killed for autopsy. All of the brain cultures yielded many colonies of PPLO. Two of these mice were hydrocephalic, one showed circling movements, and 5 displayed a tendency to roll.', 'Earlier observations on the behavior of MHV(C) in different strains of mice had indicated that the virus alone was commonly inactive on intraperitoneal or intracerebral injection in Princeton weanlings (1) . As an extension of the present findings on the behavior of MHV(C) and PPLO in the brains of Swiss mice a group of experiments was carried out in Princeton mice to determine whether or not either agent was influenced by the presence of the other.', 'In a preliminary experiment 5 Princeton weanlings were injected intracerebrally with a brain suspension containing only the virus. These mice showed no outward signs of illness and when killed on the 7th day were strictly normal. Leukocytes were not demonstrable in brain films or sections and cultures were bacteriologically sterile.', 'Ten mice were then injected intracerebrally with a mixture of the virus and the S strain of catarrhal PPLO. A similar number received the virus and the J strain of the organisms. The 20 mice, in the 2 groups, were unaffected by the inoculum. When killed on the 7th day the livers were free from focal lesions and the brains showed no indication of leukocytic infiltration. On culture, brain suspensions prepared from each of the 2 groups yielded a sparse growth of PPLO colonies.', 'The highly virulent Princeton type of mouse hepatitis virus, MHV(Pr), is pathogenic for Princeton weanlings on intracerebral injection (1). Cerebral multiplication of the virus is not attended, however, by outward signs of nervous disorder and at autopsy there is little evidence of cellular injury in the brain. Carriage of the virus from the brain to the liver commonly occurs and is followed by the customary necrosis. The hepatic reaction is generally fatal.', 'Swiss weanlings are naturally resistant to MHV(Pr). The virus fails to multiply in the brain and is usually not recoverable by passage to susceptible Princeton mice. At autopsy the brains and livers are regularly lesion-free.', 'Two additional groups of experiments were carried out to determine the cerebral response of Princeton and Swiss weanlings to MHV(Pr) in combination with PPLO.', 'Five cerebral passages were made in a total of 25 Princeton mice with a mixture of MHV(Pr) and the J strain of PPLO. Fifteen deaths were recorded as follows: 1 on the 2nd day, 10 on the 3rd, 3 on the 4th, and 1 on the 5th. Ten mice were killed and autopsied, most of them on the 3rd day. Prior to death they appeared acutely ill but were free from outward signs of nervous disorder. The livers of these mice showed focal or semidiffuse necrosis. Leukocytes were observed in brain films and massive areas of leukocytic infiltration in sections. Innumerable colonies of PPLO were obtained on culture.', 'Twenty Swiss weanlings were injected intracerebrally with a mixture of MHV(Pr) and PPLO, 10 receiving the J strain and 10 the S strain. There were no deaths. All of the mice gained weight normally and showed no outward signs of illness when killed on the 7th or the 10th day. At autopsy the livers were normal. Leukocytes were not observed in either films or sections from the brains. Eight of 10 brain cultures yielded a few discrete colonies of PPLO and 2 were without growth.', 'A group of experiments was carried out in Princeton and Swiss weaniings to determine the effect of agar on the growth and activity of PPLO in the brain. Findlay et al. (4) had reported that cultures of PPLO which failed to produce rolling signs on intracerebral injection did so in some mice on the addition of agar.', 'The J strain of PPLO was used in all of these tests. Heart infusion agar (pH 8) was liquefied, cooled to 40°C., and diluted 1 to 5 with saline, The undiluted medium contained 1.5 per cent agar and was used as routine in the isolation of PPLO. The diluted semisolid agar was mixed with an equal volume of young PPLO cultures in 20 per cent horse serum-bouillon. Twenty-four weanlings of each mouse strain were injected intracerebrally with this inoculum. One series of 3 cerebral passages was made in Princeton mice, starting with a brain suspension from 2 animals injected with the PPLO-agar mixture. Finally, as a control on the activity of the medium, mice of both strains were injected with a sterile mixture of agar and horse serum-bouillon.', 'Six of the 48 mice used in the first group of experiments died on the 7th to the 19th day after injection. The survivors were killed and autopsied at in-tervals of 1 and 3 weeks. All of the mice, including those that died, gained weight normally and were free from any outward signs of nervous disorder. Thirteen of the 21 Princeton mice that were killed and 6 of the Swiss showed leukocytes in either films or sections of the brain and a good growth of PPLO on culture. In the absence of an inflammatory reaction the growth was meagre.', 'As Findlay et al. (4) had previously noted in their experiments, a significant number of the mice were frankly hydrocephalic. Twelve Princeton weanlings and 10 Swiss showed free fluid in the cranial cavity, accompanied in some instances by a marked elevation of the skull and circumscribed sunken areas in the anterior portion of one or both hemispheres. The brain usually appeared paler than normal.', 'All of the 18 mice in the passage series survived and were normal in appearance through the 14th day when they were killed. At autopsy leukocytes were not observed in brain films or sections and only a meagre growth of PPLO was obtained on culture. In the first passage, however, 3 of the mice were hydrocephalic. Their skulls were not elevated but the cranial cavities contained free fluid and areas of collapse were present in one or both hemispheres.', 'Two of the 6 Princeton mice injected with the sterile mixture of agar and horse serum-bouillon died. The 10 survivors in both groups showed no outward signs of nervous disorder and were killed on the 21st day. The skulls of 2 Princeton mice were elevated and free fluid was present in the cranial cavities. Four of the Swiss mice were also hydrocephalic, though their skulls were not elevated. Films and sections from the brains of these mice showed no leukocytes and cultures were bacteriologically sterile.', 'Brain sections from the mice injected with the fluid cultures of PPLO alone were essentially normal. A few lymphocytes were occasionally observed but leukocytes were regularly absent. In some sections the ventricles were somewhat dilated.', 'Sections of the cerebrum from the mice injected with a mixture of virus with neurotropic activity and PPLO invariably showed an inflammatory reaction with a preponderance of leukocytes. The most common lesion was a roughly semicircular band composed largely of leukocytes, a few lymphocytes, much necrotic debris, red blood cells, and fibrin (Figs. 1 and 2). It partially enclosed the ventricular spaces and the intermeshing horns of the bilateral ganglia. Large mononuclear cells packed with fat vacuoles (gitter cells) and droplets of free fat were generally observed at its margin. The nerve cells of the adjacent ganglia often showed necrosis with deep blue pyknotic nuclei and pink staining debris. In a few sections the reaction was limited to one or more sharply demarcated abscesses. Dilation of the ventricles sometimes occurred but was not a characteristic finding. Leukocytie involvement of the meninges was observed but was rarely conspicuous. Leukocytes also invaded the cerebellum but much less frequently. A few sections showed degeneration of nerve cells particularly those in the granular layer (Fig. 5) . It was not consistent, however~ in respect to the appearance of rolling signs prior to autopsy.', 'Brain sections from the mice injected with the agar-PPLO mixture showed 2 types of cellular response. In the presence of actively growing PPLO there was a marked inflammatory reaction which was commonly circumscribed (Fig. 4) . Leukocytes predominated but lymphocytes and large mononuclear cells, with and without fat droplets, were also present. There was much necrotic debris. In some mice there was definite evidence of encapsulation by connective tissue. Necrosis of nerve cells in the nearby ganglia was generally not observed. Portions of the meninges, particularly in the cerebellum, were distended by an infiltration of leukocytes and lymphocytes. Some of the sections also showed widely dilated ventricles.', 'In the absence of actively growing PPLO there was commonly a lymphocytic infiltration together with hydrocephalus. With no or low magnification the ground substance of the brain was separated symmetrically by unstained open spaces (Fig. 3) . Microscopically the central and lateral ventricles were greatly dilated and often extended beyond the limits of the ependymal cell layer. In some instances this layer had broken away from the margin of the cavity. Rupture of the distended ventricles, possibly at autopsy, with release of the fluid may account for the areas of collapse observed in the intact brain. Lymphocytes and large mononuclear cells were usually present within and around the margin of the ventricles and also in the meninges. Leukocytes were absent. The nerve ceils of the ganglia were intact and appeared normal. Sections from some of the mice injected with the sterile mixture of agar and horse serum-bouillon showed a similar distension of the ventricles but little or no cellular reaction.', 'In naturally infected mice the sites most often frequented by PPLO are the nasal passages and the middle ears. Localization in the lungs, conjunctivae, and joints also occurs but is more sporadic. The present findings together with the earlier ones of Sabin (2, 3) and of Findlay et al. (4) indicate that PPLO may occasionally invade the central nervous system.', 'It is not surprising that PPLO have been recovered in a few instances from the brains of supposedly normal mice. The first isolate that we obtained was an organism of the catarrhal type which is widely distributed in nature. This type is enzootic in the Swiss colony, from which our mice were obtained, but is not present in the Princeton colony. In diseased mice the organisms occasionally migrate inwardly from the tympanic cavity to the labyrinth. A small local abscess that impinges on the brain may be produced and provide access to the meninges. Carriage via the circulating blood is also a possibility and would account for the transport of the conjunctival type which ordinarily does not invade the middle ear.', \"The normal mouse brain evidently fails to provide optimal conditions for the growth of most PPLO. Sabin's type A organism (5) which produces an extotoxin is an outstanding exception. On cranial injection the organisms that we studied survived in the brain for a period of weeks. The level of growth was barely sufficient, however, to maintain them by serial passage and too low to provoke a counter-reaction. Growth was markedly increased by the simultaneous injection of mouse hepatitis virus and was attended by a vigorous leukocytic response. There was no indication of specificity in relation to the type of PPLO, as earlier reported by Findlay et al. (4) . Multiplication of the virus in the brain was a prerequisite but encephalitis was not essential. AI-though direct proof is lacking, it is tempting to attribute the growth enhancement to fatty substances liberated in the brain by the virus. Edward (7) and Smith, Lecce, and Lynn (8) have presented evidence that lipoids, probably lipoproteins, are generally needed for the growth of PPLO in vitro.\", 'Rolling occurred so infrequently in mice injected with the mixture of MHV and PPLO that it could hardly be called a characteristic sign. It was also observed in an occasional mouse injected with MHV(C) alone. This virus was productive of considerable injury to the brain but rarely affected the nerve cells of the cerebral ganglia. In the absence of the virus or agar PPLO showed little or no evidence of neurotropism. The intracerebral injection of both microbic agents commonly resulted, however, in extensive destruction of the nerve cells in the anterior horns of the cerebrum. This outcome was indicative of an interaction between the 2 agents and suggestive of synergism.', 'Our findings in regard to the effect of agar on the cerebral behavior of PPLO in mice agreed with the earlier observations of Findlay et al. (4) as to its enhancing properties but differed from theirs in detail. In our experiments none of the mice injected with the agar-PPLO mixture showed any outward signs of nervous disorder. In Swiss mice there was a marked difference between agar and MHV(C) in respect to the number of individuals with an increased growth of PPLO and a leukocytic response at autopsy. The actual rates were about 30 per cent with agar and practically 100 per cent with the virus. Princeton mice were more responsive, the rate with agar being 60 per cent. The present observations suggest that hydrocephalus which frequently appeared in the injected mice was in part a response to the agar as the same condition also followed the injection of a sterile agar inoculum. SUMMARY Pleuropneumonia-like organisms (PPLO) of the catarrhal type were isolated from the brain of a Swiss mouse during the cranial passage of mouse hepatitis virus-MHV(C). Cranial injection of the PPLO alone in Swiss and Princeton weanlings was attended by a meagre growth of the organisms in the brain, with no pathologic change.', 'The growth of both catarrhal and conjunctival strains of PPLO in the brains of Swiss mice was greatly enhanced by the simultaneous injection of MHV(C). Rolling was not a characteristic sign prior to autopsy. Brain sections regularly showed a vigorous leukocytic response, commonly accompanied by the destruction of nerve cells in the anterior horns of the cerebrum. Injected in Princeton mice together with the virus, the organisms barely survived and were inactive. MHV(Pr) enhanced the growth and pathogenicity of PPLO in the brains of Princeton mice but failed to do so in Swiss.', 'The behavior of PPLO in the brain was likewise affected by the presence of agar, as earlier observed by Findlay et al. In comparison with the effect of MHV, the enhancement was reduced in rate in both strains of mice and was not accompanied by outward signs of nervous disorder. Hydrocephalus which often followed injection of the PPLO-agar mixture was also produced by agarbouillon alone.', 'All of the following cross-sections of the brain were stained with eosin-rnethyl ene blue. They were photographed by Mr. J. A. Carlile. '], 'cord_uid': 'vhfticne'}\n"
     ]
    }
   ],
   "source": [
    "for k, v in cov_dm.paper_dict.items():\n",
    "    print(v)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 93,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'title': 'The gene of an archaeal α-l-fucosidase is expressed by translational frameshifting',\n 'text': ['Translation is optimally accurate and the correspondence between the nucleotide and the protein sequences are often considered as an immutable dogma. However, the genetic code is not quite universal: in certain organelles and in a small number of organisms the meaning of different codons has been reassigned and all the mRNAs are decoded accordingly. More surprisingly, the standard rules of genetic decoding are altered in specific genes by different events that are globally termed recoding (1). In all cases, translational recoding occurs in competition with normal decoding, with a proportion of the ribosomes not obeying to the ‘universal’ rules. Translational recoding has been identified in both prokaryotes and eukaryotes. It has crucial roles in the regulation of gene expression and includes stop codon readthrough, ribosome hopping and ±1 programmed frameshifting [for reviews see (2–4)].',\n  'In stop codon readthrough a stop codon is decoded by a tRNA carrying an unusual amino acid rather than a translational release factor. Specific stimulatory elements downstream to the stop codon regulate this process (5). Hopping, in which the ribosome stops translation in a particular site of the mRNA and re-start few nucleotides downstream, is a rare event and it has been studied in detail only in the bacteriophage T4 (6). In programmed frameshifting, ribosomes are induced to shift to an alternative, overlapping reading frame 1 nt 3′-wards (+1 frameshifting) or 5′-wards (−1 frameshifting) of the mRNA. This process is regulated and its frequency varies in different genes. The ±1 programmed frameshifting has been studied extensively in viruses, retrotransposons and insertion elements for which many cases are documented (7–9). Instead, this phenomenon is by far less common in cellular genes. A single case of programmed +1 frameshifting is known in prokaryotes (10,11) while in eukaryotes, including humans, several genes regulated by this recoding event have been described previously [(4) and references therein]. Compared to +1 frameshifting, −1 frameshifting is less widespread with only two examples in prokaryotes (12–14) and few others in eukaryotes (15–17).',\n  'The programmed −1 frameshifting is triggered by several elements in the mRNA. The slippery sequence, showing the X-XXY-YYZ motif, in which X can be any base, Y is usually A or U, and Z is any base but G, has the function of favouring the tRNA misalignment and it is the site where the shift takes place (3,18). Frameshifting could be further stimulated by other elements flanking the slippery sequence: a codon for a low-abundance tRNA, a stop codon, a Shine–Dalgarno sequence and an mRNA secondary structure. It has been reported that these elements, alone or in combination, enhance frameshifting by pausing the translating ribosome on the slippery sequence (4,18).',\n  'Noticeably, known cases of recoding in Archaea [recently reviewed in (19)] are limited to termination codon readthrough events that regulate the incorporation of the 21st and 22nd amino acids selenocysteine and pyrrolysine, respectively (20–23).',\n  'No archaeal genes regulated by translational programmed frameshifting and ribosome hopping have been identified experimentally so far; therefore, if compared with the others domains of life, the study of translational recoding in Archaea is still at its dawn.',\n  'We showed that the α-l-fucosidase gene from the crenarchaeon Sulfolobus solfataricus is putatively expressed by programmed −1 frameshifting (24). This gene, named fucA1, is organized in the open reading frames (ORFs) SSO11867 and SSO3060 of 81 and 426 amino acids, respectively, which are separated by a −1 frameshifting in a 40 base overlap (Figure 1A). We have reported previously that the region of overlap between the two ORFs had the characteristic features of the genes expressed by programmed −1 frameshifting including a slippery heptanucleotide A-AAA-AAT (codons are shown in the zero frame) flanked by a putative stem–loop and the rare codons CAC (Figure 1A) resembling the prokaryotic stem–loops/hairpins and the Shine–Dalgarno-like sites (24). We showed that the frameshifting, obtained by mutating by site-directed mutagenesis the fucA1 gene exactly in the position predicted from the slippery site, produced a full-length gene, named fucA1A, encoding for a polypeptide of 495 amino acids (Figure 1B). This mutant gene expressed in Escherichia coli a fully functional α-l-fucosidase, named Ssα-fuc, which was thermophilic, thermostable and had an unusual nonameric structure (24,25). More recently, we determined the reaction mechanism and the function of the residues of the active site of the mutant enzyme (26,27).',\n  'The functionality of the product of the mutant gene fucA1A does not provide direct experimental evidence that programmed −1 frameshifting occurs in vivo and in S.solfataricus. To address these issues, we report here the study of the expression of the wild-type split gene fucA1 and of its mutants in the slippery sequence. We demonstrate here that fucA1 is expressed by programmed −1 frameshifting in both E.coli and S.solfataricus. This is the first experimental demonstration that this kind of recoding is present in the Archaea domain of life. The relevance of programmed −1 frameshifting in Archaea is also discussed.',\n  'S.solfataricus cells were grown, and cell extracts obtained, as described previously (24,28).',\n  'The expression in the E.coli strain BL21(RB791) of the wild-type gene fucA1 and of the mutant genes fucA1A [previously named FrameFuc in (24)], fucA1B, fucA1sm and fucA1tm as fusions of glutathione S-transferase (GST) and the purification of the recombinant proteins were performed as reported previously (23). The nomenclature used in this paper for the different α-fucosidase genes is listed in Table 1.',\n  \"For the western blot studies, equal amounts of E.coli cultures expressing the wild-type and mutant fucA1 genes, normalized for the OD600, were resuspended in SDS–PAGE loading buffer containing 0.03 M Tris–HCl buffer, pH 6.8, 3% SDS (w/v), 6.7% glycerol (w/v), 6.7% 2-mercaptoethanol (w/v) and 0.002% blue bromophenol (w/v). The samples were incubated at 100°C for 5 min (unless otherwise indicated) and were directly loaded on to the gel. Western blot analyses were performed by blotting SDS–PAGEs of the concentrations indicated on Hybond-P polyvinylidenfluorid filters (Amersham Biosciences, Uppsala, Sweden); polyclonal anti-Ssα-fuc antibodies from rabbit (PRIMM, Milan, Italy) and anti-GST antibodies (Amersham Biosciences) were diluted 1:5000 and 1:40 000, respectively. The filters were washed and incubated with the ImmunoPure anti-rabbit IgG antibody conjugated with the horseradish peroxidase (HRP) from Pierce Biotechnology (Rockford, IL, USA). Filters were developed with the ECL-plus Western Blotting Detection system (Amersham Biosciences) by following the manufacturer's indications. The molecular weight markers used in the western blot analyses were the ECL streptavidin–HRP conjugate (Amersham Biosciences).\",\n  'The protein concentration of the samples was measured with the method of Bradford (29) and the amounts of sample loaded on to the SDS–PAGEs are those indicated. The quantification of the bands identified by western blot was performed by using the program Quantity One 4.4.0 in a ChemiDoc EQ System (Bio-Rad, Hercules, CA, USA) with the volume analysis tool. The frameshifting efficiency was calculated as the ratio of the intensity of the bands of the frameshifted product/frameshifted product + termination product.',\n  'The mutants in the slippery sequence of the wild-type gene fucA1 were prepared by site-directed mutagenesis from the vector pGEX-11867/3060, described previously (24,27). The synthetic oligonucleotides used (PRIMM) were the following: FucA1sm-rev, 5′-TTTAGGTGATATTGGTGTTCTGGTCTATCT-3′; FucA1sm-fwd, 5′-GAACACCAATATCACCTAAAGAATTCGGCCCA-3′; FucA1tm-rev, 5′-AGGTGATATTGGTGTTCTGGTCTATCTGGC-3′; FucA1tm-fwd, 5′-CCAGAACACCAATATCACCTCAAGAACTCGGCCCAGT-3′, where the mismatched nucleotides in the mutagenic primers are underlined. Direct sequencing identified the plasmids containing the desired mutations and the mutant genes, named fucA1sm and fucA1tm, were completely re-sequenced.',\n  'The mutant Ssα-fucB was prepared by site-directed mutagenesis from the vector pGEX-11867/3060, by using the same site-directed mutagenesis kit described above. The synthetic oligonucleotides used were FucA1sm-rev (described above) and the following mutagenic oligonucleotide: Fuc-B, 5′-GAACACCAATATCACCTAAAGAAGTTCGGCCCAGT-3′, where the mismatched nucleotides are underlined. Direct sequencing identified the plasmid containing the desired mutations and the mutant gene, named fucA1B, was completely re-sequenced. The enzymatic characterization of Ssα-fucB was performed as described previously (24,27).',\n  'Samples of the proteins expressed in E.coli from the wild-type gene fucA1 and the mutants fucA1A and fucA1sm, purified as described, were fractionated on an SDS–PAGE. Protein bands were excised from the gel, washed in 50 mM ammonium bicarbonate, pH 8.0, in 50% acetonitrile, reduced with 10 mM DTT at 56°C for 45 min and alkylated with 55 mM iodoacetamide for 30 min at room temperature in the dark. The gel pieces were washed several times with the buffer, resuspended in 50 mM ammonium bicarbonate and incubated with 100 ng of trypsin for 2 h at 4°C and overnight at 37°C. The supernatant containing peptides was analysed by MALDIMS on an Applied Biosystem Voyager DE-PRO mass spectrometer using α-cyano-4-hydroxycynnamic acid as matrix. Mass calibration was performed by using the standard mixture provided by manufacturer.',\n  'Liquid chromatography online tandem mass spectrometry (LCMSMS) analyses were performed on a Q-TOF hybrid mass spectrometer (Micromass, Waters, Milford, MA, USA) coupled with a CapLC capillary chromatographic system (Waters). Peptide ions were selected in the collision cell and fragmented. Analysis of the daughter ion spectra led to the reconstruction of peptide sequences.',\n  'Genomic DNA from S.solfataricus P2 strain was prepared as described previously (24). A DNA fragment of 1538 nt containing the complete fucA1 gene, was prepared by PCR, by using the following synthetic oligonucleotides (Genenco, Florence, Italy): FucA1-fwd, 5′-CTGGAGGCGCGCTAATACGACTCACTATAGGTCAGTTAAATGTCACAAAATTCT-3′; FucA1-rev, 5′-GACTTGGCGCGCCTATCTATAATCTAGGATAACCCTTAT-3′, in which the sequence corresponding to the genome of S.solfataricus is underlined. In the FucA1-fwd primer, the sequence of the promoter of the T7 RNA polymerase is in boldface and the sequence of the BssHII site is shown in italics. The PCR amplification was performed as described previously (24) and the amplification products were cloned in the BssHII site of the plasmid pBluescript II KS+. The fucA1 gene was completely re-sequenced to check if undesired mutations were introduced by PCR and the recombinant vector obtained, named pBlu-FucA1, was used for translation in vitro experiments.',\n  'The plasmids expressing the mutant genes fucA1A, fucA1sm and fucA1tm for experiments of translation in vitro were prepared by substituting the KpnI–NcoI wild-type fragment, containing the slippery site, with those isolated from the mutants. To check that the resulting plasmids had the correct sequence, the mutant genes were completely re-sequenced.',\n  'The mRNAs encoding wild-type fucA1 and its various mutants were obtained by in vitro run-off transcription. About 2 μg of each plasmid was linearized with BssHII and incubated with 50 U of T7 RNA polymerase for 1 h 30 min at 37°C. The transcription mixtures were then treated with 10 U of DNAseI (RNAse free) for 30 min. The transcribed RNAs were recovered by extracting the samples twice with phenol (pH 4.7) and once with phenol/chloroform 1:1 followed by precipitation with ethanol. The mRNAs were resuspended in DEPC-treated H2O at the approximate concentration of 0.6 pmol/μl.',\n  'In vitro translation assays were performed essentially as described by Condò et al. (28). The samples (25 μl final volume) contained 5 μl of S.solfataricus cell extract, 10 mM KCl, 20 mM Tris–HCl, pH 7.0, 20 mM Mg acetate, 3 mM ATP, 1 mM GTP, 5 μg of bulk S.solfataricus tRNA, 2 μl of [35S]methionine (1200 Ci/mmol at 10 mCi/ml) and ∼10 pmol of each mRNA. The mixtures were incubated at 70°C for 45 min. After this time, the synthesized proteins were resolved by electrophoresis 12.5% acrylamide–SDS gels and revealed by autoradiography of the dried gels on an Instant Imager apparatus.',\n  'Cells of S.solfataricus, strain P2, were grown in minimal salts culture media supplemented with yeast extract (0.1%), casamino acids (0.1%), plus glucose (0.1%) (YGM) or sucrose (0.1%) (YSM). The extraction of total RNA was performed as reported previously (24). Total RNA was extensively digested with DNAse (Ambion, Austin, TX, USA) and the absence of DNA was assessed by the lack of PCR amplification with each sets of primers described below. The RT–PCR experiments were performed as reported previously (24) by using the primers described previously that allowed the amplification of a region of 833 nt (positions 1–833, in which the A of the first ATG codon is numbered as one) overlapping the ORFs SSO11867 and SSO3060 (24).',\n  'For real-time PCR experiments total cDNA was obtained using the kit Quantitect RT (Qiagen GmbH, Hilden, Germany) from 500 ng of the same preparation of RNA described above. cDNA was then amplified in a Bio-Rad LightCycler using the DyNAmo HS Syber Green qPCR Kit (Finnzymes Oy, Espoo, Finland). Synthetic oligonucleotides (PRIMM) used for the amplification of a region at the 3′ of the ORF SSO3060 were as follows: 5′-Real: 5′-TAAATGGCGAAGCGATTTTC-3′; 3′-Real: 5′-ATATGCCTTTGTCGCGGATA-3′ for the gene fucA1. 5′-GAATGGGGGTGATACTGTCG-3′ and 5′-TTTACAGCCGGGACTACAGG-3′ for the 16S rRNA gene.',\n  'For each amplification of the fucA1 gene was used ∼2500-fold more cDNA than that used for the amplification of the 16S rRNA. Controls with no template cDNA were always included. PCR conditions were 15 min at 95°C for initial denaturation, followed by 40 cycles of 10 s at 95°C, 25 s at 56°C and 35 s at 72°C, and a final step of 10 min at 72°C. Product purity was controlled by melting point analysis of setpoints with 0.5°C temperature increase from 72 to 95°C. PCR products were analysed on 2% agarose gels and visualized by ethidium bromide staining.',\n  'The expression values of fucA1 gene were normalized to the values determined for the 16S rRNA gene. Absolute expression levels were calculated as fucA1/16S ratio in YSM and YGM cells, respectively. Relative mRNA expression levels (YSM/YGM ratio) were calculated as (fucA1/16S ratio in YGM cells)/(fucA1/16S ratio in YSM). Each cDNA was used in triplicate for each amplification.',\n  'The wild-type fucA1 gene, expressed in E.coli as a GST-fused protein, produced trace amounts of α-fucosidase activity (2.3 × 10−2 units mg−1 after removal of GST), suggesting that a programmed −1 frameshifting may occur in E.coli (24). The enzyme was then purified by using the GST purification system and analysed by SDS–PAGE revealing a major protein band (Figure 2A). The sample and control bands were excised from the gel, digested in situ with trypsin and directly analysed by matrix-assisted laser desorption/ionization mass spectrometry (MALDIMS). As shown in Figure 2B and C, both spectra revealed the occurrence of an identical mass signal at m/z 1244.6 corresponding to a peptide (Peptide A) encompassing the overlapping region of the two ORFs. This result was confirmed by liquid chromatography online tandem mass spectrometry (LCMSMS) analysis of the peptide mixtures. The fragmentation spectra of the two signals showed the common sequence Asn-Phe-Gly-Pro-Val-Thr-Asp-Phe-Gly-Tyr-Lys in which the amino acid from the ORF SSO11867 is underlined. These results unequivocally demonstrate that the protein containing the Peptide A is produced in E.coli by a frameshifting event that occurred exactly within the slippery heptamer predicted from the analysis of the DNA sequence in the region of overlap between the ORFs SSO11867 and SSO3060 (Figure 1A).',\n  'Remarkably, the MALDIMS analysis of the products of the wild-type fucA1 gene revealed the presence of a second Peptide B at m/z 1258.6 that is absent in the spectra of the Ssα-fuc control protein (Figure 2B and C). The sequence of Peptide B obtained by LCMSMS (Figure 2D) was Lys-Phe-Gly-Pro-Val-Thr-Asp-Phe-Gly-Tyr-Lys. This sequence differs only by one amino acid from Peptide A demonstrating that the interrupted gene fucA1 expresses in E.coli two full-length proteins originated by different −1 frameshifting events. Polypeptide A results from a shift in a site A and it is identical to Ssα-fuc prepared by site-directed mutagenesis (24), suggesting that the expression occurred with the simultaneous P- and A-site slippage. Instead, polypeptide B, named Ssα-fucB, is generated by frameshifting in a second site B as the result of a single P-site slippage (Figure 2E).',\n  'To measure the global efficiency of frameshifting in the two sites of the wild-type gene fucA1 we analysed the total extracts of E.coli by western blot using anti-GST antibodies (Figure 2F). Two bands with marked different electrophoretic mobility were observed: the polypeptide of 78.7 ± 1.1 kDa migrated like GST-Ssα-fuc fusion and was identified as originated from frameshifting in either site A or B of fucA1. The protein of 38.1 ± 1.2 kDa, which is not expressed by the mutant gene fucA1A (not shown), had an electrophoretic mobility compatible with GST fused to the polypeptide encoded by the ORF SSO11867 solely (27 and 9.6 kDa, respectively). This polypeptide originated from the translational termination of the ribosome at the OCH codon of the fucA1 N-terminal ORF (Figure 1A). The calculated ratio of frameshifting to the termination products was 5%.',\n  'To test if the full-length α-fucosidase produced by the −1 frameshifting event in site B (Ssα-fucB), resulting from the single P-site slippage has different properties from Ssα-fuc, whose sequence arises from the simultaneous P- and A- site slippage, we prepared the enzyme by site-directed mutagenesis. The slippery sequence in fucA1 A-AAA-AAT was mutated in A-AAG-AAG-T where mutations are underlined. The new mutant gene was named fucA1B. The first G, producing the conservative mutation AAA→AAG, was made to disrupt the slippery sequence and hence reducing the shifting efficiency. The second G was inserted to produce the frameshifting that results in the amino acid sequence of Peptide B. Therefore, the sequence of the two full-length mutant genes fucA1A and fucA1B differs only in the region of the slippery sequence: A-AAG-AAT-TTC-GGC and A-AAG-AAG-TTC-GGC, respectively (the mutations are underlined, the nucleotides in boldface were originally in the −1 frame) (Table 1).',\n  'The recombinant Ssα-fucB was purified up to ∼95% (Materials and Methods). Gel filtration chromatography demonstrated that in native conditions Ssα-fucB had the same nonameric structure of Ssα-fuc with an identical molecular weight of 508 kDa (data not shown). In addition, Ssα-fucB had the same high substrate selectivity of Ssα-fuc. The two enzymes have high affinity for 4-nitrophenyl-α-l-fucoside (4NP-Fuc) substrate at 65°C; the KM is identical within the experimental error (0.0287 ± 0.005 mM) while the kcat of Ssα-fucB (137 ± 5.7 s−1) is ∼48% of that of Ssα-fuc (287 ± 11 s−1). In addition, 4-nitrophenyl-α-l-arabinoside, -rhamnoside, 4-nitrophenyl-α-d-glucoside, -xyloside, -galactoside and -mannoside were not substrates of Ssα-fucB as shown previously for Ssα-fuc (24). This suggests that the different amino acid sequence did not significantly affect the active site. Both enzymes showed an identical profile of specific activity versus temperature with an optimal temperature higher than 95°C (data not shown). The heat stability and the pH dependence of Ssα-fuc and Ssα-fucB are reported in Figure 3. At 80°C, the optimal growth temperature of S.solfataricus, the half-life of Ssα-fucB is 45 min, almost 4-fold lower than that of Ssα-fuc (Figure 3A). The two enzymes showed different behaviour at pH <6.0 at which Ssα-fucB is only barely active and stable (Figure 3B); however, the two enzymes showed similar values of specific activity at pHs above 6.0, which is close to the intracellular pH of S.solfataricus (30).',\n  'The experimental data reported above indicate that the predicted slippery heptanucleotide in the region of overlap between the ORFs SSO11867 and SSO3060 of the wild-type gene fucA1 could regulate in cis the frameshifting events observed in E.coli. To test this hypothesis, we mutated the sequence A-AAA-AAT into A-AAG-AAT and C-AAG-AAC (mutations are underlined) obtaining the fucA1 single mutant (fucA1sm) and triple mutant (fucA1tm) genes, respectively. It is worth noting that the mutations disrupt the slippery sequence, but they maintain the −1 frameshift between the two ORFs (Table 1).',\n  'Surprisingly, the expression of fucA1sm in E.coli produced a full-length polypeptide that, after purification by affinity chromatography and removal of the GST protein, showed the same electrophoretic migration of Ssα-fuc and Ssα-fucB (Figure 4A). This protein was then characterized by mass spectrometry analyses following in situ tryptic digestion. Interestingly, the MALDI spectra revealed the presence of a single peptide encompassing the overlapping region between the two ORFs with a mass value of 1259.7 Da (peptide C; Figure 4B). The sequence of peptide C, determined from the fragmentation spectra obtained by LCMSMS analysis, was Glu-Phe-Gly-Pro-Val-Thr-Asp-Phe-Gly-Tyr-Lys (Figure 4C). Remarkably, apart from the Glu residue, this sequence is identical to that of peptide B produced from fucA1, indicating that in the mutant gene fucA1sm only one of the two frameshifting events observed in the wild-type fucA1 gene had occurred. The presence of a Glu instead of Lys was not unexpected. The mutation A-AAA-AAT→A-AAG-AAT in fucA1sm was conservative in the zero frame of the ORF SSO11867 (AAA→AAG, both encoding Lys), but it produced the mutation AAA→GAA (Lys→Glu) in the −1 frame of the ORF SSO3060.',\n  'It is worth noting that the frameshifting efficiency of the gene fucA1sm, calculated by western blot as described above, was 2-folds higher (10%) if compared to fucA1 (5%) (Figure 4D). This indicates that the mutation cancelled the frameshifting site A and, in the same time, enhanced the frameshifting efficiency of site B.',\n  'In contrast, the triple mutant fucA1tm produced in E.coli only the low molecular weight band resulting from translational termination (Figure 4D). No full-length protein could be detected in western blots probed with either anti-GST (Figure 4D) or anti-Ssα-fuc antibodies (Figure 4E). These data show that the disruption of the heptameric slippery sequence completely abolished the frameshifting in E.coli confirming that this sequence has a direct role in controlling the frameshifting in vivo.',\n  'To test whether fucA1 is expressed in S.solfataricus we analysed the extracts of cells grown on yeast extract, sucrose and casaminoacids medium (YSM). Accurate assays showed that S.solfataricus extracts contained 3.4 × 10−4 units mg−1 of α-fucosidase activity. These very low amounts hampered the purification of the enzyme. The extracts of S.solfataricus cells grown on YSM revealed by western blot a band of a molecular mass >97 kDa and no signals were detected with the pre-immune serum confirming the specificity of the anti-Ssα-fuc antibodies (Figure 5A). The different molecular mass may result from post-translational modifications occurred in the archaeon or from the incomplete denaturation of a protein complex. In particular, the latter event is not unusual among enzymes from hyperthermophilic archaea (31,32). To test which hypotheses were appropriate, cellular extracts of S.solfataricus were analysed by western blot extending the incubation at 100°C to 2 h. Interestingly, this treatment shifted the high-molecular mass band to 67.6 ± 1.2 kDa (Figure 5B and C), which still differs from that of the recombinant Ssα-fuc, 58.9 ± 1.2 kDa, leaving the question on the origin of this difference unsolved. To try to shed some light we immunoprecipitated extracts of S.solfataricus with anti-Ssα-fuc antibodies and we analysed the major protein band by MALDIMS. Unfortunately, we could not observe any peptide compatible with the fucosidase because the heavy IgG chain co-migrated with the band of the expected molecular weight (data not shown).',\n  'To test if the scarce amounts of the α-fucosidase in S.solfataricus extracts was the result of reduced expression at transcriptional level, we performed a northern blot analysis of total RNA extracted from cells grown either on YSM or YGM media. We could not observe any signal by using probes matching the 3′ of the ORF SSO3060 (data not shown). These results suggest that fucA1 produced a rare transcript; therefore, we analysed the level of mRNA by RT–PCR and by real-time PCR. A band corresponding to the region of overlap between the ORFs SSO11867 and SSO3060 was observed in the RNA extracted from cells grown on YSM and YGM media, demonstrating that under these conditions the two ORFs were co-transcribed (Figure 6A).',\n  'The experiments of real-time PCR shown in Figure 6B demonstrated that rRNA16S was amplified after ∼17 cycles while the amplification of fucA1 mRNA was observed after 38 cycles, despite the fact that we used ∼2500-fold more cDNA for the amplification of fucA1. This indicates that the gene fucA1 is transcribed at very low level. No significant differences in the fucA1 mRNA level were observed in cells grown in YSM or YGM media. This is further confirmed by the analysis by western blot of the extracts of the same cells of S.solfataricus used to prepare the total RNAs, which revealed equal amounts of α-fucosidase in the two extracts (Figure 6C). Therefore, the low α-fucosidase activity observed under the conditions tested is the result of the poor transcription of the fucA1 gene.',\n  'To determine whether, and with what efficiency, the −1 frameshifting could be performed by S.solfataricus ribosomes, mRNAs obtained by in vitro transcription of the cloned wild-type fucA1 gene and the mutants thereof were used to program an in vitro translation system prepared as described by Condò et al. (28). To this aim, a promoter of T7 polymerase was inserted ahead of the gene of interest to obtain RNA transcripts endowed with the short 5′-untranslated region of 9 nt observed for the natural fucA1 mRNA (24). Autoradiography of an SDS–PAGE of the translation products (Figure 7) revealed that the wild-type fucA1 transcript produced a tiny but clear band whose molecular weight corresponded to that of the full-length Ssα-fuc obtained by site-directed mutagenesis (24); the latter was translated quite efficiently in the cell-free system in spite of being encoded by a quasi-leaderless mRNA. Judging from the relative intensity of the signals given by the translation products of the wild-type fucA1 and the full-length mutant fucA1A, the efficiency of the −1 frameshifting in the homologous system was ∼10%. No signals corresponding to the polypeptides expected from the separated ORFs SSO11867 and SSO3060 (9.6 and 46.5 kDa, respectively) were observed. However, it should be noted that the product of SSO11867, even if synthesized, is too small to be detected in the gel system employed for this experiment. The larger product of ORF SSO3060, on the other hand, is certainly absent. These data unequivocally demonstrate that the ribosomes of S.solfataricus can decode the split fucA1 gene by programmed −1 frameshifting with considerable efficiency producing a full-length polypeptide from the two ORFs SSO11867 and SSO3060.',\n  'Remarkably, under the same conditions at which fucA1 drives the expression of the full-length protein, we could not observe any product from the fucA1sm and fucA1tm constructs. These data demonstrate that the integrity of the heptanucleotide is essential for the expression of the fucA1 gene in S.solfataricus, thus further confirming that the gene is decoded by programmed −1 frameshifting in this organism. In addition, the lack of expression of fucA1sm by translation in vitro in S.solfataricus contrasts with the efficient expression of this mutant in E.coli, indicating that the two organisms recognize different sequences regulating the translational frameshifting.',\n  'The identification of genes whose expression is regulated by recoding events is often serendipitous. In the framework of our studies on glycosidases from hyperthermophiles, we identified in the genome of the archaeon S.solfataricus a split gene encoding a putative α-fucosidase, which could be expressed through programmed −1 frameshifting (24). We tackled this issue by studying the expression of fucA1 in S.solfataricus and in E.coli to overcome the problems connected to the scarcity of expression of the α-fucosidase gene and to the manipulation of hyperthermophiles. As already reported by others, in fact, it is a common strategy to study recoding events from different organisms in E.coli (23,33).',\n  'The expression in E.coli of the wild-type split gene fucA1 led to the production by frameshifting of two full-length polypeptides with an efficiency of 5%. This is a value higher than that observed in other genes expressed by translational frameshifting in a heterologous system such as the proteins gpG and gpGT (0.3–3.5%) (33).',\n  'The gene fucA1 is expressed in S.solfataricus at very low level under the conditions tested. In particular, the transcriptional analysis of the gene revealed that it is expressed at very low level in both YSM and YGM media. Similarly, no differences in the two media could be found by western blot probed with anti-Ssα-fuc antibodies, indicating that the low expression of the enzyme in S.solfataricus is the result of scarce transcription rather than suppressed translation.',\n  'Western blots allowed us to identify a specific band ∼8.7 kDa heavier than that of the recombinant Ssα-fuc and experiments of translation in vitro showed that the wild-type gene expresses a full-length polypeptide exhibiting the same molecular mass of the recombinant protein. This demonstrates that the translational machinery of S.solfataricus is fully competent to perform programmed frameshifting. It seems likely that the observed discrepancy in molecular mass might arise from post-translational modifications that cannot be produced by the translation in vitro. Further experiments are required to characterize the α-l-fucosidase identified in S.solfataricus.',\n  'MALDIMS and LCMSMS analyses of the products in E.coli of the wild-type split gene fucA1 demonstrated that two independent frameshifting events occurred in vivo in the proposed slippery site. In particular, the sequences obtained by LCMSMS demonstrate that peptide A results from a simultaneous backward slippage of both the P- and the A-site tRNAs (Figure 8A). Instead, the sequence of peptide B is the result of the re-positioning on the −1 frame of only the P-site tRNA; in fact, the next incorporated amino acid is specified by the codon in the new frame (Figure 8B). Therefore, the expression by −1 frameshifting of the wild-type gene fucA1 in E.coli follows the models proposed for ribosomal frameshifting (34). We confirmed the significance of the slippery heptanucleotide in promoting the programmed frameshifting in vivo by mutating the putative regulatory sequence. The triple mutant fucA1tm gave no full-length products; presumably, the mutations in both the P- and in the A-site of the slippery sequence dramatically reduced the efficiency of the −1 frameshifting as observed previously in metazoans (35). This result confirms that the intact slippery sequence in the wild-type gene fucA1 is absolutely necessary for its expression in E.coli. In contrast, surprisingly, the single mutant fucA1sm showed an even increased frequency of frameshifting (10%) if compared to the wild-type and produced only one polypeptide by shifting specifically in site B. We explained this result observing that the mutation in the P-site of the slippery sequence A-AAA-AAT→A-AAG-AAT created a novel slippery sequence A-AAG identical to that controlling the expression by programmed −1 frameshifting of a transposase gene in E.coli (36). Therefore, apparently, the single mutation inactivated the simultaneous P- and A-site tRNA re-positioning and, in the same time, fostered the shifting efficiency of the tRNA in the P-site. It is worth noting that, instead, in S.solfataricus, only the simultaneous slippage is effective (Figure 8B) and even the single mutation in the slippery sequence of fucA1sm completely annulled the expression of the gene. This indicates that this sequence is essential in the archaeon and that programmed frameshifting in S.solfataricus and E.coli exploits different mechanisms. Furthermore, since the only difference between the enzymes produced by the frameshifting sites A and B, Ssα-fuc and Ssα-fucB, respectively, is the stability at 80°C, which is the S.solfataricus physiological temperature, the functionality of Ssα-fucB in the archaeon appears questionable.',\n  'The reason why fucA1 is regulated by programmed −1 frameshifting is not known. However, the physiological significance of programmed frameshifting has been assigned to a minority of the cellular genes while for most of them it is still uncertain [see (4) and reference therein; (16)]. This mechanism of recoding is exploited to set the ratio of two polypeptides such as the τ and γ subunits of the DNA polymerase III holoenzyme in E.coli (12). Alternatively, programmed frameshifting balances the expression of a protein, as the bacterial translational release factor 2 and the eukaryotic ornithine decarboxylase antizyme [see (4) and (18) and references therein]. In the case of fucA1, the polypeptide encoded by the smaller ORF SSO11867 could never be detected by western blots analyses. In addition, the modelling of Ssα-fuc on the high-resolution crystal structure of the α-l-fucosidase from Thermotoga maritima (25,37) showed that the fucA1 N-terminal polypeptide is not an independent domain. Moreover, we have shown recently that SSO11867 includes essential catalytic residues (27), excluding the possibility that a functional α-fucosidase can be obtained from the ORF SSO3060 alone. Therefore, several lines of evidence allow us to exclude that programmed −1 frameshifting is used to set the ratio of two polypeptides of the α-fucosidase from S.solfataricus. More probably, this translational mechanism might be required to control the expression level of fucA1.',\n  'Noticeably, this is the only fucosidase gene expressed by programmed −1 frameshifting. Among carbohydrate active enzymes, the only example of expression through this recoding mechanism is that reported for a gene encoding for a α(1,2)-fucosyltransferase from Helicobacter pylori that is interrupted by a −1 frameshifting (38). In this case, the expression by programmed frameshifting would lead to a functional enzyme synthesizing components of the surface lipopolysaccharides to evade the human immune defensive system. It is hard to parallel this model to fucA1. Nevertheless, the monosaccharide fucose is involved in a variety of biological functions (39). Therefore, the α-l-fucosidase might play a role in the metabolism of fucosylated oligosaccharides; experiments are currently in progress to knockout the wild-type fucA1 gene and to insert constitutive functional mutants of this gene in S.solfataricus.',\n  'FucA1 is the only archaeal α-l-fucosidase gene identified so far; hence, it is probably the result of a horizontal gene transfer event in S.solfataricus. However, since there are no α-fucosidases genes regulated by programmed frameshifting in Bacteria and Eukarya, it is tempting to speculate that this sophisticated mechanism of translational regulation pre-existed in S.solfataricus and it was applied to the fucosidase gene for physiological reasons. The identification of other genes interrupted by −1 frameshifts in S.solfataricus would open the possibility that they are regulated by programmed −1 frameshifting. Recently, the computational analysis of prokaryotic genomes revealed that seven Archaea harbour interrupted coding sequences, but S.solfataricus is not included in this study (40). A computational analysis on several archaeal genomes revealed that 34 interrupted genes are present in the genome of S.solfataricus, 11 of these genes are composed by two ORFs separated by −1 frameshifting and could be expressed by recoding (B. Cobucci-Ponzano, M. Rossi and M. Moracci, manuscript in preparation).',\n  'We have experimentally shown here, for the first time, that programmed −1 frameshifting is present in the Archaea domain. This finding is the missing piece in the puzzle of the phylogenetic distribution of programmed frameshifting demonstrating that this mechanism is universally conserved.'],\n 'cord_uid': '69gftii4'}"
      ]
     },
     "execution_count": 93,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "cov_dm.get_document_from_dict('69gftii4')\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 2",
   "language": "python",
   "name": "python2"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.6"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 0
}
